{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Biopython"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "BioPython is a very popular third party Python package for handling biological data."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "In order to install it, type in a terminal window:"
   ]
  },
  {
   "cell_type": "raw",
   "metadata": {},
   "source": [
    "bash\n",
    "conda install -c anaconda biopython"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "and follow the instructions"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "You can check the basic installation and inspect the version by doing:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1.72\n"
     ]
    }
   ],
   "source": [
    "import Bio\n",
    "print(Bio.__version__)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Biopython has three major functionalities\n",
    "-  Sequence handling\n",
    "-  3D structure\n",
    "-  Population Genetics"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Some useful references: __[Website](https://biopython.org/)__, __[Tutorial](http://biopython.org/DIST/docs/tutorial/Tutorial.html)__    (most of the examples are coming from these sources)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Working with Sequences in BioPython\n",
    "\n",
    " -  Reading sequence files\n",
    " -  Writing sequence files\n",
    " -  Working with sequence features\n",
    " -  Sequence alignments"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Sequences are just strings of letters. In principle, we could deal with them defining normal python strings.\n",
    "However, BioPython has some nice features that helps us distinguishing those strings as belonging to DNA, RNA or proteins and working with them."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## BioPython.Seq"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Sequences are handled by the __[Seq](https://biopython.org/wiki/Seq)__ class. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#let's make a generic sequence\n",
    "from Bio.Seq import Seq\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Help on class Seq in module Bio.Seq:\n",
      "\n",
      "class Seq(builtins.object)\n",
      " |  Read-only sequence object (essentially a string with an alphabet).\n",
      " |  \n",
      " |  Like normal python strings, our basic sequence object is immutable.\n",
      " |  This prevents you from doing my_seq[5] = \"A\" for example, but does allow\n",
      " |  Seq objects to be used as dictionary keys.\n",
      " |  \n",
      " |  The Seq object provides a number of string like methods (such as count,\n",
      " |  find, split and strip), which are alphabet aware where appropriate.\n",
      " |  \n",
      " |  In addition to the string like sequence, the Seq object has an alphabet\n",
      " |  property. This is an instance of an Alphabet class from Bio.Alphabet,\n",
      " |  for example generic DNA, or IUPAC DNA. This describes the type of molecule\n",
      " |  (e.g. RNA, DNA, protein) and may also indicate the expected symbols\n",
      " |  (letters).\n",
      " |  \n",
      " |  The Seq object also provides some biological methods, such as complement,\n",
      " |  reverse_complement, transcribe, back_transcribe and translate (which are\n",
      " |  not applicable to sequences with a protein alphabet).\n",
      " |  \n",
      " |  Methods defined here:\n",
      " |  \n",
      " |  __add__(self, other)\n",
      " |      Add another sequence or string to this sequence.\n",
      " |      \n",
      " |      If adding a string to a Seq, the alphabet is preserved:\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet import generic_protein\n",
      " |      >>> Seq(\"MELKI\", generic_protein) + \"LV\"\n",
      " |      Seq('MELKILV', ProteinAlphabet())\n",
      " |      \n",
      " |      When adding two Seq (like) objects, the alphabets are important.\n",
      " |      Consider this example:\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet.IUPAC import unambiguous_dna, ambiguous_dna\n",
      " |      >>> unamb_dna_seq = Seq(\"ACGT\", unambiguous_dna)\n",
      " |      >>> ambig_dna_seq = Seq(\"ACRGT\", ambiguous_dna)\n",
      " |      >>> unamb_dna_seq\n",
      " |      Seq('ACGT', IUPACUnambiguousDNA())\n",
      " |      >>> ambig_dna_seq\n",
      " |      Seq('ACRGT', IUPACAmbiguousDNA())\n",
      " |      \n",
      " |      If we add the ambiguous and unambiguous IUPAC DNA alphabets, we get\n",
      " |      the more general ambiguous IUPAC DNA alphabet:\n",
      " |      \n",
      " |      >>> unamb_dna_seq + ambig_dna_seq\n",
      " |      Seq('ACGTACRGT', IUPACAmbiguousDNA())\n",
      " |      \n",
      " |      However, if the default generic alphabet is included, the result is\n",
      " |      a generic alphabet:\n",
      " |      \n",
      " |      >>> Seq(\"\") + ambig_dna_seq\n",
      " |      Seq('ACRGT')\n",
      " |      \n",
      " |      You can't add RNA and DNA sequences:\n",
      " |      \n",
      " |      >>> from Bio.Alphabet import generic_dna, generic_rna\n",
      " |      >>> Seq(\"ACGT\", generic_dna) + Seq(\"ACGU\", generic_rna)\n",
      " |      Traceback (most recent call last):\n",
      " |         ...\n",
      " |      TypeError: Incompatible alphabets DNAAlphabet() and RNAAlphabet()\n",
      " |      \n",
      " |      You can't add nucleotide and protein sequences:\n",
      " |      \n",
      " |      >>> from Bio.Alphabet import generic_dna, generic_protein\n",
      " |      >>> Seq(\"ACGT\", generic_dna) + Seq(\"MELKI\", generic_protein)\n",
      " |      Traceback (most recent call last):\n",
      " |         ...\n",
      " |      TypeError: Incompatible alphabets DNAAlphabet() and ProteinAlphabet()\n",
      " |  \n",
      " |  __contains__(self, char)\n",
      " |      Implement the 'in' keyword, like a python string.\n",
      " |      \n",
      " |      e.g.\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet import generic_dna, generic_rna, generic_protein\n",
      " |      >>> my_dna = Seq(\"ATATGAAATTTGAAAA\", generic_dna)\n",
      " |      >>> \"AAA\" in my_dna\n",
      " |      True\n",
      " |      >>> Seq(\"AAA\") in my_dna\n",
      " |      True\n",
      " |      >>> Seq(\"AAA\", generic_dna) in my_dna\n",
      " |      True\n",
      " |      \n",
      " |      Like other Seq methods, this will raise a type error if another Seq\n",
      " |      (or Seq like) object with an incompatible alphabet is used:\n",
      " |      \n",
      " |      >>> Seq(\"AAA\", generic_rna) in my_dna\n",
      " |      Traceback (most recent call last):\n",
      " |         ...\n",
      " |      TypeError: Incompatible alphabets DNAAlphabet() and RNAAlphabet()\n",
      " |      >>> Seq(\"AAA\", generic_protein) in my_dna\n",
      " |      Traceback (most recent call last):\n",
      " |         ...\n",
      " |      TypeError: Incompatible alphabets DNAAlphabet() and ProteinAlphabet()\n",
      " |  \n",
      " |  __eq__(self, other)\n",
      " |      Compare the sequence to another sequence or a string (README).\n",
      " |      \n",
      " |      Historically comparing Seq objects has done Python object comparison.\n",
      " |      After considerable discussion (keeping in mind constraints of the\n",
      " |      Python language, hashes and dictionary support), Biopython now uses\n",
      " |      simple string comparison (with a warning about the change).\n",
      " |      \n",
      " |      Note that incompatible alphabets (e.g. DNA to RNA) will trigger a\n",
      " |      warning.\n",
      " |      \n",
      " |      During this transition period, please just do explicit comparisons:\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet import generic_dna\n",
      " |      >>> seq1 = Seq(\"ACGT\")\n",
      " |      >>> seq2 = Seq(\"ACGT\")\n",
      " |      >>> id(seq1) == id(seq2)\n",
      " |      False\n",
      " |      >>> str(seq1) == str(seq2)\n",
      " |      True\n",
      " |      \n",
      " |      The new behaviour is to use string-like equality:\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet import generic_dna\n",
      " |      >>> seq1 == seq2\n",
      " |      True\n",
      " |      >>> seq1 == \"ACGT\"\n",
      " |      True\n",
      " |      >>> seq1 == Seq(\"ACGT\", generic_dna)\n",
      " |      True\n",
      " |  \n",
      " |  __getitem__(self, index)\n",
      " |      Return a subsequence of single letter, use my_seq[index].\n",
      " |      \n",
      " |      >>> my_seq = Seq('ACTCGACGTCG')\n",
      " |      >>> my_seq[5]\n",
      " |      'A'\n",
      " |  \n",
      " |  __hash__(self)\n",
      " |      Hash for comparison.\n",
      " |      \n",
      " |      See the __cmp__ documentation - this has changed from past\n",
      " |      versions of Biopython!\n",
      " |  \n",
      " |  __imul__(self, other)\n",
      " |      Multiply Seq in-place.\n",
      " |      \n",
      " |      Note although Seq is immutable, the in-place method is\n",
      " |      included to match the behaviour for regular Python strings.\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet import generic_dna\n",
      " |      >>> seq = Seq('ATG', generic_dna)\n",
      " |      >>> seq *= 2\n",
      " |      >>> seq\n",
      " |      Seq('ATGATG', DNAAlphabet())\n",
      " |  \n",
      " |  __init__(self, data, alphabet=Alphabet())\n",
      " |      Create a Seq object.\n",
      " |      \n",
      " |      Arguments:\n",
      " |       - seq - Sequence, required (string)\n",
      " |       - alphabet - Optional argument, an Alphabet object from\n",
      " |         Bio.Alphabet\n",
      " |      \n",
      " |      You will typically use Bio.SeqIO to read in sequences from files as\n",
      " |      SeqRecord objects, whose sequence will be exposed as a Seq object via\n",
      " |      the seq property.\n",
      " |      \n",
      " |      However, will often want to create your own Seq objects directly:\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet import IUPAC\n",
      " |      >>> my_seq = Seq(\"MKQHKAMIVALIVICITAVVAALVTRKDLCEVHIRTGQTEVAVF\",\n",
      " |      ...              IUPAC.protein)\n",
      " |      >>> my_seq\n",
      " |      Seq('MKQHKAMIVALIVICITAVVAALVTRKDLCEVHIRTGQTEVAVF', IUPACProtein())\n",
      " |      >>> print(my_seq)\n",
      " |      MKQHKAMIVALIVICITAVVAALVTRKDLCEVHIRTGQTEVAVF\n",
      " |      >>> my_seq.alphabet\n",
      " |      IUPACProtein()\n",
      " |  \n",
      " |  __le__(self, other)\n",
      " |      Implement the less-than or equal operand.\n",
      " |  \n",
      " |  __len__(self)\n",
      " |      Return the length of the sequence, use len(my_seq).\n",
      " |  \n",
      " |  __lt__(self, other)\n",
      " |      Implement the less-than operand.\n",
      " |  \n",
      " |  __mul__(self, other)\n",
      " |      Multiply Seq by integer.\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet import generic_dna\n",
      " |      >>> Seq('ATG') * 2\n",
      " |      Seq('ATGATG')\n",
      " |      >>> Seq('ATG', generic_dna) * 2\n",
      " |      Seq('ATGATG', DNAAlphabet())\n",
      " |  \n",
      " |  __ne__(self, other)\n",
      " |      Implement the not-equal operand.\n",
      " |  \n",
      " |  __radd__(self, other)\n",
      " |      Add a sequence on the left.\n",
      " |      \n",
      " |      If adding a string to a Seq, the alphabet is preserved:\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet import generic_protein\n",
      " |      >>> \"LV\" + Seq(\"MELKI\", generic_protein)\n",
      " |      Seq('LVMELKI', ProteinAlphabet())\n",
      " |      \n",
      " |      Adding two Seq (like) objects is handled via the __add__ method.\n",
      " |  \n",
      " |  __repr__(self)\n",
      " |      Return (truncated) representation of the sequence for debugging.\n",
      " |  \n",
      " |  __rmul__(self, other)\n",
      " |      Multiply integer by Seq.\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet import generic_dna\n",
      " |      >>> 2 * Seq('ATG')\n",
      " |      Seq('ATGATG')\n",
      " |      >>> 2 * Seq('ATG', generic_dna)\n",
      " |      Seq('ATGATG', DNAAlphabet())\n",
      " |  \n",
      " |  __str__(self)\n",
      " |      Return the full sequence as a python string, use str(my_seq).\n",
      " |      \n",
      " |      Note that Biopython 1.44 and earlier would give a truncated\n",
      " |      version of repr(my_seq) for str(my_seq).  If you are writing code\n",
      " |      which need to be backwards compatible with old Biopython, you\n",
      " |      should continue to use my_seq.tostring() rather than str(my_seq).\n",
      " |  \n",
      " |  back_transcribe(self)\n",
      " |      Return the DNA sequence from an RNA sequence by creating a new Seq object.\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet import IUPAC\n",
      " |      >>> messenger_rna = Seq(\"AUGGCCAUUGUAAUGGGCCGCUGAAAGGGUGCCCGAUAG\",\n",
      " |      ...                     IUPAC.unambiguous_rna)\n",
      " |      >>> messenger_rna\n",
      " |      Seq('AUGGCCAUUGUAAUGGGCCGCUGAAAGGGUGCCCGAUAG', IUPACUnambiguousRNA())\n",
      " |      >>> messenger_rna.back_transcribe()\n",
      " |      Seq('ATGGCCATTGTAATGGGCCGCTGAAAGGGTGCCCGATAG', IUPACUnambiguousDNA())\n",
      " |      \n",
      " |      Trying to back-transcribe a protein or DNA sequence raises an\n",
      " |      exception:\n",
      " |      \n",
      " |      >>> my_protein = Seq(\"MAIVMGR\", IUPAC.protein)\n",
      " |      >>> my_protein.back_transcribe()\n",
      " |      Traceback (most recent call last):\n",
      " |         ...\n",
      " |      ValueError: Proteins cannot be back transcribed!\n",
      " |  \n",
      " |  complement(self)\n",
      " |      Return the complement sequence by creating a new Seq object.\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet import IUPAC\n",
      " |      >>> my_dna = Seq(\"CCCCCGATAG\", IUPAC.unambiguous_dna)\n",
      " |      >>> my_dna\n",
      " |      Seq('CCCCCGATAG', IUPACUnambiguousDNA())\n",
      " |      >>> my_dna.complement()\n",
      " |      Seq('GGGGGCTATC', IUPACUnambiguousDNA())\n",
      " |      \n",
      " |      You can of course used mixed case sequences,\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet import generic_dna\n",
      " |      >>> my_dna = Seq(\"CCCCCgatA-GD\", generic_dna)\n",
      " |      >>> my_dna\n",
      " |      Seq('CCCCCgatA-GD', DNAAlphabet())\n",
      " |      >>> my_dna.complement()\n",
      " |      Seq('GGGGGctaT-CH', DNAAlphabet())\n",
      " |      \n",
      " |      Note in the above example, ambiguous character D denotes\n",
      " |      G, A or T so its complement is H (for C, T or A).\n",
      " |      \n",
      " |      Trying to complement a protein sequence raises an exception.\n",
      " |      \n",
      " |      >>> my_protein = Seq(\"MAIVMGR\", IUPAC.protein)\n",
      " |      >>> my_protein.complement()\n",
      " |      Traceback (most recent call last):\n",
      " |         ...\n",
      " |      ValueError: Proteins do not have complements!\n",
      " |  \n",
      " |  count(self, sub, start=0, end=9223372036854775807)\n",
      " |      Return a non-overlapping count, like that of a python string.\n",
      " |      \n",
      " |      This behaves like the python string method of the same name,\n",
      " |      which does a non-overlapping count!\n",
      " |      \n",
      " |      For an overlapping search use the newer count_overlap() method.\n",
      " |      \n",
      " |      Returns an integer, the number of occurrences of substring\n",
      " |      argument sub in the (sub)sequence given by [start:end].\n",
      " |      Optional arguments start and end are interpreted as in slice\n",
      " |      notation.\n",
      " |      \n",
      " |      Arguments:\n",
      " |       - sub - a string or another Seq object to look for\n",
      " |       - start - optional integer, slice start\n",
      " |       - end - optional integer, slice end\n",
      " |      \n",
      " |      e.g.\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> my_seq = Seq(\"AAAATGA\")\n",
      " |      >>> print(my_seq.count(\"A\"))\n",
      " |      5\n",
      " |      >>> print(my_seq.count(\"ATG\"))\n",
      " |      1\n",
      " |      >>> print(my_seq.count(Seq(\"AT\")))\n",
      " |      1\n",
      " |      >>> print(my_seq.count(\"AT\", 2, -1))\n",
      " |      1\n",
      " |      \n",
      " |      HOWEVER, please note because python strings and Seq objects (and\n",
      " |      MutableSeq objects) do a non-overlapping search, this may not give\n",
      " |      the answer you expect:\n",
      " |      \n",
      " |      >>> \"AAAA\".count(\"AA\")\n",
      " |      2\n",
      " |      >>> print(Seq(\"AAAA\").count(\"AA\"))\n",
      " |      2\n",
      " |      \n",
      " |      An overlapping search, as implemented in .count_overlap(),\n",
      " |      would give the answer as three!\n",
      " |  \n",
      " |  count_overlap(self, sub, start=0, end=9223372036854775807)\n",
      " |      Return an overlapping count.\n",
      " |      \n",
      " |      For a non-overlapping search use the count() method.\n",
      " |      \n",
      " |      Returns an integer, the number of occurrences of substring\n",
      " |      argument sub in the (sub)sequence given by [start:end].\n",
      " |      Optional arguments start and end are interpreted as in slice\n",
      " |      notation.\n",
      " |      \n",
      " |      Arguments:\n",
      " |       - sub - a string or another Seq object to look for\n",
      " |       - start - optional integer, slice start\n",
      " |       - end - optional integer, slice end\n",
      " |      \n",
      " |      e.g.\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> print(Seq(\"AAAA\").count_overlap(\"AA\"))\n",
      " |      3\n",
      " |      >>> print(Seq(\"ATATATATA\").count_overlap(\"ATA\"))\n",
      " |      4\n",
      " |      >>> print(Seq(\"ATATATATA\").count_overlap(\"ATA\", 3, -1))\n",
      " |      1\n",
      " |      \n",
      " |      Where substrings do not overlap, should behave the same as\n",
      " |      the count() method:\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> my_seq = Seq(\"AAAATGA\")\n",
      " |      >>> print(my_seq.count_overlap(\"A\"))\n",
      " |      5\n",
      " |      >>> my_seq.count_overlap(\"A\") == my_seq.count(\"A\")\n",
      " |      True\n",
      " |      >>> print(my_seq.count_overlap(\"ATG\"))\n",
      " |      1\n",
      " |      >>> my_seq.count_overlap(\"ATG\") == my_seq.count(\"ATG\")\n",
      " |      True\n",
      " |      >>> print(my_seq.count_overlap(Seq(\"AT\")))\n",
      " |      1\n",
      " |      >>> my_seq.count_overlap(Seq(\"AT\")) == my_seq.count(Seq(\"AT\"))\n",
      " |      True\n",
      " |      >>> print(my_seq.count_overlap(\"AT\", 2, -1))\n",
      " |      1\n",
      " |      >>> my_seq.count_overlap(\"AT\", 2, -1) == my_seq.count(\"AT\", 2, -1)\n",
      " |      True\n",
      " |      \n",
      " |      HOWEVER, do not use this method for such cases because the\n",
      " |      count() method is much for efficient.\n",
      " |  \n",
      " |  endswith(self, suffix, start=0, end=9223372036854775807)\n",
      " |      Return True if the Seq ends with the given suffix, False otherwise.\n",
      " |      \n",
      " |      This behaves like the python string method of the same name.\n",
      " |      \n",
      " |      Return True if the sequence ends with the specified suffix\n",
      " |      (a string or another Seq object), False otherwise.\n",
      " |      With optional start, test sequence beginning at that position.\n",
      " |      With optional end, stop comparing sequence at that position.\n",
      " |      suffix can also be a tuple of strings to try.  e.g.\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> my_rna = Seq(\"GUCAUGGCCAUUGUAAUGGGCCGCUGAAAGGGUGCCCGAUAGUUG\")\n",
      " |      >>> my_rna.endswith(\"UUG\")\n",
      " |      True\n",
      " |      >>> my_rna.endswith(\"AUG\")\n",
      " |      False\n",
      " |      >>> my_rna.endswith(\"AUG\", 0, 18)\n",
      " |      True\n",
      " |      >>> my_rna.endswith((\"UCC\", \"UCA\", \"UUG\"))\n",
      " |      True\n",
      " |  \n",
      " |  find(self, sub, start=0, end=9223372036854775807)\n",
      " |      Find method, like that of a python string.\n",
      " |      \n",
      " |      This behaves like the python string method of the same name.\n",
      " |      \n",
      " |      Returns an integer, the index of the first occurrence of substring\n",
      " |      argument sub in the (sub)sequence given by [start:end].\n",
      " |      \n",
      " |      Arguments:\n",
      " |       - sub - a string or another Seq object to look for\n",
      " |       - start - optional integer, slice start\n",
      " |       - end - optional integer, slice end\n",
      " |      \n",
      " |      Returns -1 if the subsequence is NOT found.\n",
      " |      \n",
      " |      e.g. Locating the first typical start codon, AUG, in an RNA sequence:\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> my_rna = Seq(\"GUCAUGGCCAUUGUAAUGGGCCGCUGAAAGGGUGCCCGAUAGUUG\")\n",
      " |      >>> my_rna.find(\"AUG\")\n",
      " |      3\n",
      " |  \n",
      " |  lower(self)\n",
      " |      Return a lower case copy of the sequence.\n",
      " |      \n",
      " |      This will adjust the alphabet if required. Note that the IUPAC\n",
      " |      alphabets are upper case only, and thus a generic alphabet must be\n",
      " |      substituted.\n",
      " |      \n",
      " |      >>> from Bio.Alphabet import Gapped, generic_dna\n",
      " |      >>> from Bio.Alphabet import IUPAC\n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> my_seq = Seq(\"CGGTACGCTTATGTCACGTAG*AAAAAA\",\n",
      " |      ...          Gapped(IUPAC.unambiguous_dna, \"*\"))\n",
      " |      >>> my_seq\n",
      " |      Seq('CGGTACGCTTATGTCACGTAG*AAAAAA', Gapped(IUPACUnambiguousDNA(), '*'))\n",
      " |      >>> my_seq.lower()\n",
      " |      Seq('cggtacgcttatgtcacgtag*aaaaaa', Gapped(DNAAlphabet(), '*'))\n",
      " |      \n",
      " |      See also the upper method.\n",
      " |  \n",
      " |  lstrip(self, chars=None)\n",
      " |      Return a new Seq object with leading (left) end stripped.\n",
      " |      \n",
      " |      This behaves like the python string method of the same name.\n",
      " |      \n",
      " |      Optional argument chars defines which characters to remove.  If\n",
      " |      omitted or None (default) then as for the python string method,\n",
      " |      this defaults to removing any white space.\n",
      " |      \n",
      " |      e.g. print(my_seq.lstrip(\"-\"))\n",
      " |      \n",
      " |      See also the strip and rstrip methods.\n",
      " |  \n",
      " |  reverse_complement(self)\n",
      " |      Return the reverse complement sequence by creating a new Seq object.\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet import IUPAC\n",
      " |      >>> my_dna = Seq(\"CCCCCGATAGNR\", IUPAC.ambiguous_dna)\n",
      " |      >>> my_dna\n",
      " |      Seq('CCCCCGATAGNR', IUPACAmbiguousDNA())\n",
      " |      >>> my_dna.reverse_complement()\n",
      " |      Seq('YNCTATCGGGGG', IUPACAmbiguousDNA())\n",
      " |      \n",
      " |      Note in the above example, since R = G or A, its complement\n",
      " |      is Y (which denotes C or T).\n",
      " |      \n",
      " |      You can of course used mixed case sequences,\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet import generic_dna\n",
      " |      >>> my_dna = Seq(\"CCCCCgatA-G\", generic_dna)\n",
      " |      >>> my_dna\n",
      " |      Seq('CCCCCgatA-G', DNAAlphabet())\n",
      " |      >>> my_dna.reverse_complement()\n",
      " |      Seq('C-TatcGGGGG', DNAAlphabet())\n",
      " |      \n",
      " |      Trying to complement a protein sequence raises an exception:\n",
      " |      \n",
      " |      >>> my_protein = Seq(\"MAIVMGR\", IUPAC.protein)\n",
      " |      >>> my_protein.reverse_complement()\n",
      " |      Traceback (most recent call last):\n",
      " |         ...\n",
      " |      ValueError: Proteins do not have complements!\n",
      " |  \n",
      " |  rfind(self, sub, start=0, end=9223372036854775807)\n",
      " |      Find from right method, like that of a python string.\n",
      " |      \n",
      " |      This behaves like the python string method of the same name.\n",
      " |      \n",
      " |      Returns an integer, the index of the last (right most) occurrence of\n",
      " |      substring argument sub in the (sub)sequence given by [start:end].\n",
      " |      \n",
      " |      Arguments:\n",
      " |       - sub - a string or another Seq object to look for\n",
      " |       - start - optional integer, slice start\n",
      " |       - end - optional integer, slice end\n",
      " |      \n",
      " |      Returns -1 if the subsequence is NOT found.\n",
      " |      \n",
      " |      e.g. Locating the last typical start codon, AUG, in an RNA sequence:\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> my_rna = Seq(\"GUCAUGGCCAUUGUAAUGGGCCGCUGAAAGGGUGCCCGAUAGUUG\")\n",
      " |      >>> my_rna.rfind(\"AUG\")\n",
      " |      15\n",
      " |  \n",
      " |  rsplit(self, sep=None, maxsplit=-1)\n",
      " |      Do a right split method, like that of a python string.\n",
      " |      \n",
      " |      This behaves like the python string method of the same name.\n",
      " |      \n",
      " |      Return a list of the 'words' in the string (as Seq objects),\n",
      " |      using sep as the delimiter string.  If maxsplit is given, at\n",
      " |      most maxsplit splits are done COUNTING FROM THE RIGHT.\n",
      " |      If maxsplit is omitted, all splits are made.\n",
      " |      \n",
      " |      Following the python string method, sep will by default be any\n",
      " |      white space (tabs, spaces, newlines) but this is unlikely to\n",
      " |      apply to biological sequences.\n",
      " |      \n",
      " |      e.g. print(my_seq.rsplit(\"*\",1))\n",
      " |      \n",
      " |      See also the split method.\n",
      " |  \n",
      " |  rstrip(self, chars=None)\n",
      " |      Return a new Seq object with trailing (right) end stripped.\n",
      " |      \n",
      " |      This behaves like the python string method of the same name.\n",
      " |      \n",
      " |      Optional argument chars defines which characters to remove.  If\n",
      " |      omitted or None (default) then as for the python string method,\n",
      " |      this defaults to removing any white space.\n",
      " |      \n",
      " |      e.g. Removing a nucleotide sequence's polyadenylation (poly-A tail):\n",
      " |      \n",
      " |      >>> from Bio.Alphabet import IUPAC\n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> my_seq = Seq(\"CGGTACGCTTATGTCACGTAGAAAAAA\", IUPAC.unambiguous_dna)\n",
      " |      >>> my_seq\n",
      " |      Seq('CGGTACGCTTATGTCACGTAGAAAAAA', IUPACUnambiguousDNA())\n",
      " |      >>> my_seq.rstrip(\"A\")\n",
      " |      Seq('CGGTACGCTTATGTCACGTAG', IUPACUnambiguousDNA())\n",
      " |      \n",
      " |      See also the strip and lstrip methods.\n",
      " |  \n",
      " |  split(self, sep=None, maxsplit=-1)\n",
      " |      Split method, like that of a python string.\n",
      " |      \n",
      " |      This behaves like the python string method of the same name.\n",
      " |      \n",
      " |      Return a list of the 'words' in the string (as Seq objects),\n",
      " |      using sep as the delimiter string.  If maxsplit is given, at\n",
      " |      most maxsplit splits are done.  If maxsplit is omitted, all\n",
      " |      splits are made.\n",
      " |      \n",
      " |      Following the python string method, sep will by default be any\n",
      " |      white space (tabs, spaces, newlines) but this is unlikely to\n",
      " |      apply to biological sequences.\n",
      " |      \n",
      " |      e.g.\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> my_rna = Seq(\"GUCAUGGCCAUUGUAAUGGGCCGCUGAAAGGGUGCCCGAUAGUUG\")\n",
      " |      >>> my_aa = my_rna.translate()\n",
      " |      >>> my_aa\n",
      " |      Seq('VMAIVMGR*KGAR*L', HasStopCodon(ExtendedIUPACProtein(), '*'))\n",
      " |      >>> for pep in my_aa.split(\"*\"):\n",
      " |      ...     pep\n",
      " |      Seq('VMAIVMGR', HasStopCodon(ExtendedIUPACProtein(), '*'))\n",
      " |      Seq('KGAR', HasStopCodon(ExtendedIUPACProtein(), '*'))\n",
      " |      Seq('L', HasStopCodon(ExtendedIUPACProtein(), '*'))\n",
      " |      >>> for pep in my_aa.split(\"*\", 1):\n",
      " |      ...     pep\n",
      " |      Seq('VMAIVMGR', HasStopCodon(ExtendedIUPACProtein(), '*'))\n",
      " |      Seq('KGAR*L', HasStopCodon(ExtendedIUPACProtein(), '*'))\n",
      " |      \n",
      " |      See also the rsplit method:\n",
      " |      \n",
      " |      >>> for pep in my_aa.rsplit(\"*\", 1):\n",
      " |      ...     pep\n",
      " |      Seq('VMAIVMGR*KGAR', HasStopCodon(ExtendedIUPACProtein(), '*'))\n",
      " |      Seq('L', HasStopCodon(ExtendedIUPACProtein(), '*'))\n",
      " |  \n",
      " |  startswith(self, prefix, start=0, end=9223372036854775807)\n",
      " |      Return True if the Seq starts with the given prefix, False otherwise.\n",
      " |      \n",
      " |      This behaves like the python string method of the same name.\n",
      " |      \n",
      " |      Return True if the sequence starts with the specified prefix\n",
      " |      (a string or another Seq object), False otherwise.\n",
      " |      With optional start, test sequence beginning at that position.\n",
      " |      With optional end, stop comparing sequence at that position.\n",
      " |      prefix can also be a tuple of strings to try.  e.g.\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> my_rna = Seq(\"GUCAUGGCCAUUGUAAUGGGCCGCUGAAAGGGUGCCCGAUAGUUG\")\n",
      " |      >>> my_rna.startswith(\"GUC\")\n",
      " |      True\n",
      " |      >>> my_rna.startswith(\"AUG\")\n",
      " |      False\n",
      " |      >>> my_rna.startswith(\"AUG\", 3)\n",
      " |      True\n",
      " |      >>> my_rna.startswith((\"UCC\", \"UCA\", \"UCG\"), 1)\n",
      " |      True\n",
      " |  \n",
      " |  strip(self, chars=None)\n",
      " |      Return a new Seq object with leading and trailing ends stripped.\n",
      " |      \n",
      " |      This behaves like the python string method of the same name.\n",
      " |      \n",
      " |      Optional argument chars defines which characters to remove.  If\n",
      " |      omitted or None (default) then as for the python string method,\n",
      " |      this defaults to removing any white space.\n",
      " |      \n",
      " |      e.g. print(my_seq.strip(\"-\"))\n",
      " |      \n",
      " |      See also the lstrip and rstrip methods.\n",
      " |  \n",
      " |  tomutable(self)\n",
      " |      Return the full sequence as a MutableSeq object.\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet import IUPAC\n",
      " |      >>> my_seq = Seq(\"MKQHKAMIVALIVICITAVVAAL\",\n",
      " |      ...              IUPAC.protein)\n",
      " |      >>> my_seq\n",
      " |      Seq('MKQHKAMIVALIVICITAVVAAL', IUPACProtein())\n",
      " |      >>> my_seq.tomutable()\n",
      " |      MutableSeq('MKQHKAMIVALIVICITAVVAAL', IUPACProtein())\n",
      " |      \n",
      " |      Note that the alphabet is preserved.\n",
      " |  \n",
      " |  tostring(self)\n",
      " |      Return the full sequence as a python string (DEPRECATED).\n",
      " |      \n",
      " |      You are now encouraged to use str(my_seq) instead of\n",
      " |      my_seq.tostring().\n",
      " |  \n",
      " |  transcribe(self)\n",
      " |      Return the RNA sequence from a DNA sequence by creating a new Seq object.\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet import IUPAC\n",
      " |      >>> coding_dna = Seq(\"ATGGCCATTGTAATGGGCCGCTGAAAGGGTGCCCGATAG\",\n",
      " |      ...                  IUPAC.unambiguous_dna)\n",
      " |      >>> coding_dna\n",
      " |      Seq('ATGGCCATTGTAATGGGCCGCTGAAAGGGTGCCCGATAG', IUPACUnambiguousDNA())\n",
      " |      >>> coding_dna.transcribe()\n",
      " |      Seq('AUGGCCAUUGUAAUGGGCCGCUGAAAGGGUGCCCGAUAG', IUPACUnambiguousRNA())\n",
      " |      \n",
      " |      Trying to transcribe a protein or RNA sequence raises an exception:\n",
      " |      \n",
      " |      >>> my_protein = Seq(\"MAIVMGR\", IUPAC.protein)\n",
      " |      >>> my_protein.transcribe()\n",
      " |      Traceback (most recent call last):\n",
      " |         ...\n",
      " |      ValueError: Proteins cannot be transcribed!\n",
      " |  \n",
      " |  translate(self, table='Standard', stop_symbol='*', to_stop=False, cds=False, gap=None)\n",
      " |      Turn a nucleotide sequence into a protein sequence by creating a new Seq object.\n",
      " |      \n",
      " |      This method will translate DNA or RNA sequences, and those with a\n",
      " |      nucleotide or generic alphabet.  Trying to translate a protein\n",
      " |      sequence raises an exception.\n",
      " |      \n",
      " |      Arguments:\n",
      " |       - table - Which codon table to use?  This can be either a name\n",
      " |         (string), an NCBI identifier (integer), or a CodonTable\n",
      " |         object (useful for non-standard genetic codes).  This\n",
      " |         defaults to the \"Standard\" table.\n",
      " |       - stop_symbol - Single character string, what to use for\n",
      " |         terminators.  This defaults to the asterisk, \"*\".\n",
      " |       - to_stop - Boolean, defaults to False meaning do a full\n",
      " |         translation continuing on past any stop codons (translated as the\n",
      " |         specified stop_symbol).  If True, translation is terminated at\n",
      " |         the first in frame stop codon (and the stop_symbol is not\n",
      " |         appended to the returned protein sequence).\n",
      " |       - cds - Boolean, indicates this is a complete CDS.  If True,\n",
      " |         this checks the sequence starts with a valid alternative start\n",
      " |         codon (which will be translated as methionine, M), that the\n",
      " |         sequence length is a multiple of three, and that there is a\n",
      " |         single in frame stop codon at the end (this will be excluded\n",
      " |         from the protein sequence, regardless of the to_stop option).\n",
      " |         If these tests fail, an exception is raised.\n",
      " |       - gap - Single character string to denote symbol used for gaps.\n",
      " |         It will try to guess the gap character from the alphabet.\n",
      " |      \n",
      " |      e.g. Using the standard table:\n",
      " |      \n",
      " |      >>> coding_dna = Seq(\"GTGGCCATTGTAATGGGCCGCTGAAAGGGTGCCCGATAG\")\n",
      " |      >>> coding_dna.translate()\n",
      " |      Seq('VAIVMGR*KGAR*', HasStopCodon(ExtendedIUPACProtein(), '*'))\n",
      " |      >>> coding_dna.translate(stop_symbol=\"@\")\n",
      " |      Seq('VAIVMGR@KGAR@', HasStopCodon(ExtendedIUPACProtein(), '@'))\n",
      " |      >>> coding_dna.translate(to_stop=True)\n",
      " |      Seq('VAIVMGR', ExtendedIUPACProtein())\n",
      " |      \n",
      " |      Now using NCBI table 2, where TGA is not a stop codon:\n",
      " |      \n",
      " |      >>> coding_dna.translate(table=2)\n",
      " |      Seq('VAIVMGRWKGAR*', HasStopCodon(ExtendedIUPACProtein(), '*'))\n",
      " |      >>> coding_dna.translate(table=2, to_stop=True)\n",
      " |      Seq('VAIVMGRWKGAR', ExtendedIUPACProtein())\n",
      " |      \n",
      " |      In fact, GTG is an alternative start codon under NCBI table 2, meaning\n",
      " |      this sequence could be a complete CDS:\n",
      " |      \n",
      " |      >>> coding_dna.translate(table=2, cds=True)\n",
      " |      Seq('MAIVMGRWKGAR', ExtendedIUPACProtein())\n",
      " |      \n",
      " |      It isn't a valid CDS under NCBI table 1, due to both the start codon\n",
      " |      and also the in frame stop codons:\n",
      " |      \n",
      " |      >>> coding_dna.translate(table=1, cds=True)\n",
      " |      Traceback (most recent call last):\n",
      " |          ...\n",
      " |      TranslationError: First codon 'GTG' is not a start codon\n",
      " |      \n",
      " |      If the sequence has no in-frame stop codon, then the to_stop argument\n",
      " |      has no effect:\n",
      " |      \n",
      " |      >>> coding_dna2 = Seq(\"TTGGCCATTGTAATGGGCCGC\")\n",
      " |      >>> coding_dna2.translate()\n",
      " |      Seq('LAIVMGR', ExtendedIUPACProtein())\n",
      " |      >>> coding_dna2.translate(to_stop=True)\n",
      " |      Seq('LAIVMGR', ExtendedIUPACProtein())\n",
      " |      \n",
      " |      When translating gapped sequences, the gap character is inferred from\n",
      " |      the alphabet:\n",
      " |      \n",
      " |      >>> from Bio.Alphabet import Gapped\n",
      " |      >>> coding_dna3 = Seq(\"GTG---GCCATT\", Gapped(IUPAC.unambiguous_dna))\n",
      " |      >>> coding_dna3.translate()\n",
      " |      Seq('V-AI', Gapped(ExtendedIUPACProtein(), '-'))\n",
      " |      \n",
      " |      It is possible to pass the gap character when the alphabet is missing:\n",
      " |      \n",
      " |      >>> coding_dna4 = Seq(\"GTG---GCCATT\")\n",
      " |      >>> coding_dna4.translate(gap='-')\n",
      " |      Seq('V-AI', Gapped(ExtendedIUPACProtein(), '-'))\n",
      " |      \n",
      " |      NOTE - Ambiguous codons like \"TAN\" or \"NNN\" could be an amino acid\n",
      " |      or a stop codon.  These are translated as \"X\".  Any invalid codon\n",
      " |      (e.g. \"TA?\" or \"T-A\") will throw a TranslationError.\n",
      " |      \n",
      " |      NOTE - This does NOT behave like the python string's translate\n",
      " |      method.  For that use str(my_seq).translate(...) instead.\n",
      " |  \n",
      " |  ungap(self, gap=None)\n",
      " |      Return a copy of the sequence without the gap character(s).\n",
      " |      \n",
      " |      The gap character can be specified in two ways - either as an explicit\n",
      " |      argument, or via the sequence's alphabet. For example:\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet import generic_dna\n",
      " |      >>> my_dna = Seq(\"-ATA--TGAAAT-TTGAAAA\", generic_dna)\n",
      " |      >>> my_dna\n",
      " |      Seq('-ATA--TGAAAT-TTGAAAA', DNAAlphabet())\n",
      " |      >>> my_dna.ungap(\"-\")\n",
      " |      Seq('ATATGAAATTTGAAAA', DNAAlphabet())\n",
      " |      \n",
      " |      If the gap character is not given as an argument, it will be taken from\n",
      " |      the sequence's alphabet (if defined). Notice that the returned\n",
      " |      sequence's alphabet is adjusted since it no longer requires a gapped\n",
      " |      alphabet:\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet import IUPAC, Gapped, HasStopCodon\n",
      " |      >>> my_pro = Seq(\"MVVLE=AD*\", HasStopCodon(Gapped(IUPAC.protein, \"=\")))\n",
      " |      >>> my_pro\n",
      " |      Seq('MVVLE=AD*', HasStopCodon(Gapped(IUPACProtein(), '='), '*'))\n",
      " |      >>> my_pro.ungap()\n",
      " |      Seq('MVVLEAD*', HasStopCodon(IUPACProtein(), '*'))\n",
      " |      \n",
      " |      Or, with a simpler gapped DNA example:\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet import IUPAC, Gapped\n",
      " |      >>> my_seq = Seq(\"CGGGTAG=AAAAAA\", Gapped(IUPAC.unambiguous_dna, \"=\"))\n",
      " |      >>> my_seq\n",
      " |      Seq('CGGGTAG=AAAAAA', Gapped(IUPACUnambiguousDNA(), '='))\n",
      " |      >>> my_seq.ungap()\n",
      " |      Seq('CGGGTAGAAAAAA', IUPACUnambiguousDNA())\n",
      " |      \n",
      " |      As long as it is consistent with the alphabet, although it is\n",
      " |      redundant, you can still supply the gap character as an argument to\n",
      " |      this method:\n",
      " |      \n",
      " |      >>> my_seq\n",
      " |      Seq('CGGGTAG=AAAAAA', Gapped(IUPACUnambiguousDNA(), '='))\n",
      " |      >>> my_seq.ungap(\"=\")\n",
      " |      Seq('CGGGTAGAAAAAA', IUPACUnambiguousDNA())\n",
      " |      \n",
      " |      However, if the gap character given as the argument disagrees with that\n",
      " |      declared in the alphabet, an exception is raised:\n",
      " |      \n",
      " |      >>> my_seq\n",
      " |      Seq('CGGGTAG=AAAAAA', Gapped(IUPACUnambiguousDNA(), '='))\n",
      " |      >>> my_seq.ungap(\"-\")\n",
      " |      Traceback (most recent call last):\n",
      " |         ...\n",
      " |      ValueError: Gap '-' does not match '=' from alphabet\n",
      " |      \n",
      " |      Finally, if a gap character is not supplied, and the alphabet does not\n",
      " |      define one, an exception is raised:\n",
      " |      \n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> from Bio.Alphabet import generic_dna\n",
      " |      >>> my_dna = Seq(\"ATA--TGAAAT-TTGAAAA\", generic_dna)\n",
      " |      >>> my_dna\n",
      " |      Seq('ATA--TGAAAT-TTGAAAA', DNAAlphabet())\n",
      " |      >>> my_dna.ungap()\n",
      " |      Traceback (most recent call last):\n",
      " |         ...\n",
      " |      ValueError: Gap character not given and not defined in alphabet\n",
      " |  \n",
      " |  upper(self)\n",
      " |      Return an upper case copy of the sequence.\n",
      " |      \n",
      " |      >>> from Bio.Alphabet import HasStopCodon, generic_protein\n",
      " |      >>> from Bio.Seq import Seq\n",
      " |      >>> my_seq = Seq(\"VHLTPeeK*\", HasStopCodon(generic_protein))\n",
      " |      >>> my_seq\n",
      " |      Seq('VHLTPeeK*', HasStopCodon(ProteinAlphabet(), '*'))\n",
      " |      >>> my_seq.lower()\n",
      " |      Seq('vhltpeek*', HasStopCodon(ProteinAlphabet(), '*'))\n",
      " |      >>> my_seq.upper()\n",
      " |      Seq('VHLTPEEK*', HasStopCodon(ProteinAlphabet(), '*'))\n",
      " |      \n",
      " |      This will adjust the alphabet if required. See also the lower method.\n",
      " |  \n",
      " |  ----------------------------------------------------------------------\n",
      " |  Data descriptors defined here:\n",
      " |  \n",
      " |  __dict__\n",
      " |      dictionary for instance variables (if defined)\n",
      " |  \n",
      " |  __weakref__\n",
      " |      list of weak references to the object (if defined)\n",
      "\n"
     ]
    }
   ],
   "source": [
    "help(Seq)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "my_seq = Seq(\"CCCAGGGAG\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'Bio.Seq.Seq'>\n",
      "['alphabet', 'back_transcribe', 'complement', 'count', 'count_overlap', 'endswith', 'find', 'lower', 'lstrip', 'reverse_complement', 'rfind', 'rsplit', 'rstrip', 'split', 'startswith', 'strip', 'tomutable', 'tostring', 'transcribe', 'translate', 'ungap', 'upper']\n"
     ]
    }
   ],
   "source": [
    "print(type(my_seq)) #print the type of the object\n",
    "\n",
    "#let's see what attributes this object has\n",
    "dir(my_seq) #prints all the attribute of an object\n",
    "\n",
    "\n",
    "attributes = [a for a in dir(my_seq) if not a.startswith(\"_\")] #\n",
    "print(attributes)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "The alphabet object is perhaps the important thing that makes the Seq object more than just a string."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Alphabet()"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "my_seq.alphabet"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The currently available alphabets for Biopython are defined in the Bio.Alphabet module"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "from Bio.Alphabet import generic_dna, generic_protein, generic_rna"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "my_dna = Seq(\"CCCGGAGAG\", generic_dna)\n",
    "my_rna = Seq(\"ACCCGUUGU\", generic_rna)\n",
    "my_protein = Seq(\"AKKKGGGUUULL\", generic_protein)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Biopython will now know the difference for example between a <font color=blue>DNA</font> base <font color=blue>A</font> for adenine and the <font color=red>protein</font> residue <font color=red>A</font> for alanine."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "DNAAlphabet()\n"
     ]
    }
   ],
   "source": [
    "print(my_dna.alphabet)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Let's look at how to utilize this class to do interesting work as, for example, transcribe and translate a sequence "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ACUAGCAGCGGA\n",
      "RNAAlphabet()\n",
      "TSSG\n",
      "ExtendedIUPACProtein()\n"
     ]
    }
   ],
   "source": [
    "my_gene = Seq(\"ACTAGCAGCGGA\", generic_dna)\n",
    "\n",
    "#get the mRNA\n",
    "\n",
    "my_transcript = my_gene.transcribe()\n",
    "print(my_transcript)\n",
    "print(my_transcript.alphabet)\n",
    "\n",
    "\n",
    "#get the protein from the mRNA\n",
    "my_protein = my_transcript.translate()\n",
    "print(my_protein)\n",
    "print(my_protein.alphabet)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Sequences act like strings"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "In many ways, we can deal with Seq objects as if they were normal Python strings, for example getting the length, or iterating over the elements:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "my_seq = Seq(\"GATCG\", generic_dna)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "You can access elements of the sequence in the same way as for strings (but remember, Python counts from zero!):"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "G\n",
      "T\n",
      "G\n"
     ]
    }
   ],
   "source": [
    "print(my_seq[0]) #first letter\n",
    "print(my_seq[2]) #third letter\n",
    "print(my_seq[-1]) #last letter"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Slicing a sequence:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Seq('ATGGGC', DNAAlphabet())"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "my_seq = Seq(\"GATCGATGGGCCTATATAGGATCGAAAATCGC\", generic_dna)\n",
    "my_seq[5:11]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Two things are interesting to note:\n",
    "- First, this follows the normal conventions for Python strings. So the first element of the sequence is 0. When you do a slice the first item is included (i.e. 5 in this case) and the last is excluded (11 in this case)\n",
    "- The new object produced is another Seq object which retains the alphabet information from the original Seq object."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The Seq object has a .count() method, just like a string. Note that this means that like a Python string, this gives a non-overlapping count:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2\n",
      "2\n"
     ]
    }
   ],
   "source": [
    "print(\"AAAA\".count(\"AA\"))\n",
    "print(Seq(\"AAAA\").count(\"AA\"))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Naturally, you can in principle add any two Seq objects together - just like you can with Python strings to concatenate them. However, you **can't add sequences with incompatible alphabets**, such as a protein sequence and a DNA sequence:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "ename": "TypeError",
     "evalue": "Incompatible alphabets ProteinAlphabet() and DNAAlphabet()",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-9-d9b00ad83065>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0mdna_seq\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mSeq\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"ACGT\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mgeneric_dna\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m \u001b[0mprotein_seq\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mdna_seq\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/Bio/Seq.py\u001b[0m in \u001b[0;36m__add__\u001b[0;34m(self, other)\u001b[0m\n\u001b[1;32m    309\u001b[0m                 raise TypeError(\n\u001b[1;32m    310\u001b[0m                     \"Incompatible alphabets {0!r} and {1!r}\".format(\n\u001b[0;32m--> 311\u001b[0;31m                         self.alphabet, other.alphabet))\n\u001b[0m\u001b[1;32m    312\u001b[0m             \u001b[0;31m# They should be the same sequence type (or one of them is generic)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    313\u001b[0m             \u001b[0ma\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mAlphabet\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_consensus_alphabet\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0malphabet\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mother\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0malphabet\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mTypeError\u001b[0m: Incompatible alphabets ProteinAlphabet() and DNAAlphabet()"
     ]
    }
   ],
   "source": [
    "protein_seq = Seq(\"EVRNAK\", generic_protein)\n",
    "dna_seq = Seq(\"ACGT\", generic_dna)\n",
    "\n",
    "protein_seq + dna_seq"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Seq('EVRNAKRAV', ProteinAlphabet())"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "protein_seq = Seq(\"EVRNAK\", generic_protein)\n",
    "protein2_seq = Seq(\"RAV\", generic_protein)\n",
    "\n",
    "\n",
    "protein_seq + protein2_seq \n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "You can also go straight from the DNA to the protein."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<a id=\"section1\"></a>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MAIVMGR*KGAR*\n"
     ]
    }
   ],
   "source": [
    "coding_dna = Seq(\"ATGGCCATTGTAATGGGCCGCTGAAAGGGTGCCCGATAG\", generic_dna)\n",
    "myprot = coding_dna.translate()\n",
    "print(myprot)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "As you can see, we got some STOP codons represented as * and translation continued."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We can get translation to actually stop when it encounters a STOP codon."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MAIVMGR\n"
     ]
    }
   ],
   "source": [
    "myprot = coding_dna.translate(to_stop=True)\n",
    "print(myprot)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We can be even more realistic and only allow translation of valid genes (i.e. with a valid start and stop codon and proper number of bases).\n",
    "\n",
    "This is done by setting the ** cds=True ** keyword argument, which stands for \"coding sequence\".\n",
    "\n",
    "If we don't have a valid coding sequence, we get an exception."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "ename": "TranslationError",
     "evalue": "Extra in frame stop codon found.",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyError\u001b[0m                                  Traceback (most recent call last)",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/Bio/Seq.py\u001b[0m in \u001b[0;36m_translate_str\u001b[0;34m(sequence, table, stop_symbol, to_stop, cds, pos_stop, gap)\u001b[0m\n\u001b[1;32m   2587\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 2588\u001b[0;31m             \u001b[0mamino_acids\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mforward_table\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mcodon\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   2589\u001b[0m         \u001b[0;32mexcept\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0mKeyError\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mCodonTable\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mTranslationError\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/Bio/Data/CodonTable.py\u001b[0m in \u001b[0;36m__getitem__\u001b[0;34m(self, codon)\u001b[0m\n\u001b[1;32m    384\u001b[0m             \u001b[0;32mif\u001b[0m \u001b[0mx\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0mKeyError\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 385\u001b[0;31m                 \u001b[0;32mraise\u001b[0m \u001b[0mKeyError\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcodon\u001b[0m\u001b[0;34m)\u001b[0m  \u001b[0;31m# it's a stop codon\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    386\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mx\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mKeyError\u001b[0m: 'TGA'",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[0;31mTranslationError\u001b[0m                          Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-15-c32b2502db4f>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mmyprot\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mcoding_dna\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mtranslate\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcds\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/Bio/Seq.py\u001b[0m in \u001b[0;36mtranslate\u001b[0;34m(self, table, stop_symbol, to_stop, cds, gap)\u001b[0m\n\u001b[1;32m   1161\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1162\u001b[0m         protein = _translate_str(str(self), codon_table, stop_symbol, to_stop,\n\u001b[0;32m-> 1163\u001b[0;31m                                  cds, gap=gap)\n\u001b[0m\u001b[1;32m   1164\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1165\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mgap\u001b[0m \u001b[0;32mand\u001b[0m \u001b[0mgap\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mprotein\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.6/site-packages/Bio/Seq.py\u001b[0m in \u001b[0;36m_translate_str\u001b[0;34m(sequence, table, stop_symbol, to_stop, cds, pos_stop, gap)\u001b[0m\n\u001b[1;32m   2591\u001b[0m                 \u001b[0;32mif\u001b[0m \u001b[0mcds\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   2592\u001b[0m                     raise CodonTable.TranslationError(\n\u001b[0;32m-> 2593\u001b[0;31m                         \"Extra in frame stop codon found.\")\n\u001b[0m\u001b[1;32m   2594\u001b[0m                 \u001b[0;32mif\u001b[0m \u001b[0mto_stop\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   2595\u001b[0m                     \u001b[0;32mbreak\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mTranslationError\u001b[0m: Extra in frame stop codon found."
     ]
    }
   ],
   "source": [
    "myprot = coding_dna.translate(cds=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Seq('MAIVM', ExtendedIUPACProtein())"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gene = Seq(\"ATGGCCATTGTAATGTAG\", generic_dna)\n",
    "gene.translate(cds=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "\n",
    "# Reading sequence files "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Dealing with assorted sequence file formats is one of the strengths of Biopython.The primary module we'll be using is __[Seq.IO](https://biopython.org/wiki/SeqIO)__, which is short for sequence input/output."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "For these examples we're going to use files for the famous bacteria Esherichia coli K12, and some potato genes.\n",
    "Tfrom Bio import SeqIO\n",
    "o download the sample data needed for this section use the  fetch_sample_data.sh script."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 350,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Downloading files using curl\n",
      "==============================================\n",
      "Fetching Escherichia coli K-12 files from NCBI\n",
      "==============================================\n",
      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
      "                                 Dload  Upload   Total   Spent    Left  Speed\n",
      "100 17.4M  100 17.4M    0     0  1367k      0  0:00:13  0:00:13 --:--:-- 2025k\n",
      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
      "                                 Dload  Upload   Total   Spent    Left  Speed\n",
      "100 4597k  100 4597k    0     0   833k      0  0:00:05  0:00:05 --:--:--  916kk    0     0  31575      0  0:02:29  0:00:03  0:02:26 31566\n",
      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
      "                                 Dload  Upload   Total   Spent    Left  Speed\n",
      "100 4501k  100 4501k    0     0   621k      0  0:00:07  0:00:07 --:--:--  976kk   25 1170k    0     0   249k      0  0:00:18  0:00:04  0:00:14  249k\n",
      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
      "                                 Dload  Upload   Total   Spent    Left  Speed\n",
      "100 1779k  100 1779k    0     0   356k      0  0:00:04  0:00:04 --:--:--  372k\n",
      "==========================================================\n",
      "Fetching proteins from Potato Genome Sequencing Consortium\n",
      "==========================================================\n",
      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
      "                                 Dload  Upload   Total   Spent    Left  Speed\n",
      "100 7629k  100 7629k    0     0   885k      0  0:00:08  0:00:08 --:--:-- 1383k\n",
      "Archive:  PGSC_DM_v3.4_pep_representative.fasta.zip\n",
      "  inflating: PGSC_DM_v3.4_pep_representative.fasta  \n",
      "====================================\n",
      "Fetching PF08792 alignment from PFAM\n",
      "====================================\n",
      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
      "                                 Dload  Upload   Total   Spent    Left  Speed\n",
      "100  2056  100  2056    0     0  11053      0 --:--:-- --:--:-- --:--:-- 10994\n"
     ]
    }
   ],
   "source": [
    "! ./fetch_sample_data.sh\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We'll start by looking at the protein sequence in the FASTA amino acid file, NC_000913.faa. "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The annotation for a FASTA  sequence is tipically held in the header:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">gi|16127995|ref|NP_414542.1| thr operon leader peptide [Escherichia coli str. K-12 substr. MG1655]\r\n",
      "MKRISTTITTTITITTGNGAG\r\n",
      ">gi|16127996|ref|NP_414543.1| Bifunctional aspartokinase/homoserine dehydrogenase 1 [Escherichia coli str. K-12 substr. MG1655]\r\n",
      "MRVLKFGGTSVANAERFLRVADILESNARQGQVATVLSAPAKITNHLVAMIEKTISGQDALPNISDAERI\r\n",
      "FAELLTGLAAAQPGFPLAQLKTFVDQEFAQIKHVLHGISLLGQCPDSINAALICRGEKMSIAIMAGVLEA\r\n",
      "RGHNVTVIDPVEKLLAVGHYLESTVDIAESTRRIAASRIPADHMVLMAGFTAGNEKGELVVLGRNGSDYS\r\n",
      "AAVLAACLRADCCEIWTDVDGVYTCDPRQVPDARLLKSMSYQEAMELSYFGAKVLHPRTITPIAQFQIPC\r\n",
      "LIKNTGNPQAPGTLIGASRDEDELPVKGISNLNNMAMFSVSGPGMKGMVGMAARVFAAMSRARISVVLIT\r\n",
      "QSSSEYSISFCVPQSDCVRAERAMQEEFYLELKEGLLEPLAVTERLAIISVVGDGMRTLRGISAKFFAAL\r\n",
      "ARANINIVAIAQGSSERSISVVVNNDDATTGVRVTHQMLFNTDQVIEVFVIGVGGVGGALLEQLKRQQSW\r\n"
     ]
    }
   ],
   "source": [
    "! head NC_000913.faa"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The SeqIO.parse takes a path to a file and a format, in this case \"fasta\" and produces an iterator over each entry in the file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "from Bio import SeqIO"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n",
      "<class 'Bio.SeqRecord.SeqRecord'>\n"
     ]
    }
   ],
   "source": [
    "filename = \"NC_000913.faa\"\n",
    "for record in SeqIO.parse(filename, \"fasta\"):\n",
    "    print(type(record))\n",
    "    "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Exercise:**  count the records with Biopython using the SeqIO.parse function and print how many records you have in NC_000913.faa"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "There were 4140 records in file \n"
     ]
    }
   ],
   "source": [
    "# Solution:\n",
    "count = 0\n",
    "for record in SeqIO.parse(filename, \"fasta\"):\n",
    "    count = count + 1\n",
    "print(\"There were \" + str(count) + \" records in file \") "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Looking at the Records"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "In the above example, we used a for loop to count the records in a FASTA file, but didn't actually look at the information in the records. \n",
    "The **SeqIO.parse** function was creating **SeqRecord** objects. \n",
    "Biopython's SeqRecord objects are a container holding the sequence, and any annotation about it - most importantly the identifier."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gi|16132220|ref|NP_418820.1|\n"
     ]
    }
   ],
   "source": [
    "print(record.id)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MRITIILVAPARAENIGAAARAMKTMGFSDLRIVDSQAHLEPATRWVAHGSGDIIDNIKVFPTLAESLHDVDFTVATTARSRAKYHYYATPVELVPLLEEKSSWMSHAALVFGREDSGLTNEELALADVLTGVPMVADYPSLNLGQAVMVYCYQLATLIQQPAKSDATADQHQLQALRERAMTLLTTLAVADDIKLVDWLQQRLGLLEQRDTAMLHRLLHDIEKNITK\n"
     ]
    }
   ],
   "source": [
    "print (record.seq)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    " This simple example prints out the record identifers and their lengths:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Record gi|16127995|ref|NP_414542.1|, length 21\n",
      "Record gi|16127996|ref|NP_414543.1|, length 820\n",
      "Record gi|16127997|ref|NP_414544.1|, length 310\n",
      "Record gi|16127998|ref|NP_414545.1|, length 428\n",
      "Record gi|16127999|ref|NP_414546.1|, length 98\n",
      "Record gi|16128000|ref|NP_414547.1|, length 258\n",
      "Record gi|16128001|ref|NP_414548.1|, length 476\n",
      "Record gi|16128002|ref|NP_414549.1|, length 317\n",
      "Record gi|16128003|ref|NP_414550.1|, length 195\n",
      "Record gi|16128004|ref|NP_414551.1|, length 188\n",
      "Record gi|16128005|ref|NP_414552.1|, length 237\n",
      "Record gi|16128007|ref|NP_414554.1|, length 134\n",
      "Record gi|16128008|ref|NP_414555.1|, length 638\n",
      "Record gi|16128009|ref|NP_414556.1|, length 376\n",
      "Record gi|16128010|ref|NP_414557.1|, length 370\n",
      "Record gi|16128012|ref|NP_414559.1|, length 69\n",
      "Record gi|49175991|ref|YP_025292.1|, length 50\n",
      "Record gi|16128013|ref|NP_414560.1|, length 388\n",
      "Record gi|16128014|ref|NP_414561.1|, length 301\n",
      "Record gi|16128015|ref|NP_414562.1|, length 167\n",
      "Record gi|16128016|ref|NP_414563.1|, length 91\n",
      "Record gi|16128017|ref|NP_414564.1|, length 87\n",
      "Record gi|16128018|ref|NP_414565.1|, length 72\n",
      "Record gi|16128019|ref|NP_414566.1|, length 313\n",
      "Record gi|16128020|ref|NP_414567.1|, length 938\n",
      "Record gi|16128021|ref|NP_414568.1|, length 164\n",
      "Record gi|16128022|ref|NP_414569.1|, length 149\n",
      "Record gi|16128023|ref|NP_414570.1|, length 316\n",
      "Record gi|16128024|ref|NP_414571.1|, length 304\n",
      "Record gi|16128025|ref|NP_414572.1|, length 273\n",
      "Record gi|16128026|ref|NP_414573.1|, length 382\n",
      "Record gi|16128027|ref|NP_414574.1|, length 1073\n",
      "Record gi|90111079|ref|NP_414576.4|, length 131\n",
      "Record gi|90111080|ref|NP_414577.2|, length 196\n",
      "Record gi|221800779|ref|NP_414578.2|, length 261\n",
      "Record gi|221800780|ref|NP_414579.4|, length 517\n",
      "Record gi|16128032|ref|NP_414580.1|, length 405\n",
      "Record gi|16128033|ref|NP_414581.1|, length 380\n",
      "Record gi|16128034|ref|NP_414582.1|, length 504\n",
      "Record gi|90111081|ref|NP_414583.2|, length 256\n",
      "Record gi|16128036|ref|NP_414584.1|, length 313\n",
      "Record gi|16128037|ref|NP_414585.1|, length 428\n",
      "Record gi|16128038|ref|NP_414586.1|, length 95\n",
      "Record gi|16128039|ref|NP_414587.1|, length 443\n",
      "Record gi|16128040|ref|NP_414588.1|, length 176\n",
      "Record gi|16128041|ref|NP_414589.1|, length 620\n",
      "Record gi|16128042|ref|NP_414590.1|, length 159\n",
      "Record gi|16128043|ref|NP_414591.1|, length 280\n",
      "Record gi|16128044|ref|NP_414592.1|, length 125\n",
      "Record gi|16128045|ref|NP_414593.1|, length 273\n",
      "Record gi|16128046|ref|NP_414594.1|, length 329\n",
      "Record gi|16128047|ref|NP_414595.1|, length 428\n",
      "Record gi|16128048|ref|NP_414596.1|, length 784\n",
      "Record gi|16128049|ref|NP_414597.1|, length 271\n",
      "Record gi|16128052|ref|NP_414600.1|, length 219\n",
      "Record gi|16128053|ref|NP_414601.1|, length 968\n",
      "Record gi|16128054|ref|NP_414602.1|, length 783\n",
      "Record gi|16128055|ref|NP_414603.1|, length 231\n",
      "Record gi|16128056|ref|NP_414604.1|, length 500\n",
      "Record gi|16128057|ref|NP_414605.1|, length 566\n",
      "Record gi|16128058|ref|NP_414606.1|, length 292\n",
      "Record gi|16128059|ref|NP_414607.1|, length 254\n",
      "Record gi|16128060|ref|NP_414608.1|, length 232\n",
      "Record gi|16128061|ref|NP_414609.1|, length 536\n",
      "Record gi|16128062|ref|NP_414610.1|, length 327\n",
      "Record gi|16128063|ref|NP_414611.1|, length 551\n",
      "Record gi|226524697|ref|YP_002791237.1|, length 43\n",
      "Record gi|49175994|ref|YP_025293.1|, length 392\n",
      "Record gi|16128065|ref|NP_414613.1|, length 201\n",
      "Record gi|16128066|ref|NP_414614.1|, length 466\n",
      "Record gi|90111082|ref|NP_414615.4|, length 363\n",
      "Record gi|16128068|ref|NP_414616.1|, length 523\n",
      "Record gi|16128069|ref|NP_414617.1|, length 28\n",
      "Record gi|90111083|ref|NP_414618.4|, length 314\n",
      "Record gi|90111084|ref|YP_025294.2|, length 574\n",
      "Record gi|16128071|ref|NP_414620.1|, length 163\n",
      "Record gi|16128073|ref|NP_414622.1|, length 334\n",
      "Record gi|16128074|ref|NP_414623.1|, length 152\n",
      "Record gi|16128075|ref|NP_414624.1|, length 313\n",
      "Record gi|16128076|ref|NP_414625.1|, length 121\n",
      "Record gi|16128077|ref|NP_414626.1|, length 588\n",
      "Record gi|16128078|ref|NP_414627.1|, length 495\n",
      "Record gi|16128079|ref|NP_414628.1|, length 452\n",
      "Record gi|16128080|ref|NP_414629.1|, length 360\n",
      "Record gi|16128081|ref|NP_414630.1|, length 438\n",
      "Record gi|16128082|ref|NP_414631.1|, length 414\n",
      "Record gi|16128083|ref|NP_414632.1|, length 355\n",
      "Record gi|16128084|ref|NP_414633.1|, length 491\n",
      "Record gi|16128085|ref|NP_414634.1|, length 306\n",
      "Record gi|16128086|ref|NP_414635.1|, length 276\n",
      "Record gi|16128087|ref|NP_414636.1|, length 420\n",
      "Record gi|16128088|ref|NP_414637.1|, length 383\n",
      "Record gi|16128089|ref|NP_414638.1|, length 305\n",
      "Record gi|90111085|ref|NP_414639.2|, length 170\n",
      "Record gi|16128091|ref|NP_414640.1|, length 901\n",
      "Record gi|16128092|ref|NP_414641.1|, length 129\n",
      "Record gi|16128094|ref|NP_414643.1|, length 65\n",
      "Record gi|16128095|ref|NP_414644.1|, length 247\n",
      "Record gi|16128096|ref|NP_414645.1|, length 206\n",
      "Record gi|16128097|ref|NP_414646.1|, length 347\n",
      "Record gi|16128099|ref|NP_414648.1|, length 400\n",
      "Record gi|16128100|ref|NP_414649.1|, length 461\n",
      "Record gi|16128101|ref|NP_414650.1|, length 146\n",
      "Record gi|16128102|ref|NP_414651.1|, length 297\n",
      "Record gi|16128103|ref|NP_414652.1|, length 183\n",
      "Record gi|16128104|ref|NP_414653.1|, length 284\n",
      "Record gi|16128105|ref|NP_414654.1|, length 457\n",
      "Record gi|16128106|ref|NP_414655.1|, length 254\n",
      "Record gi|16128107|ref|NP_414656.1|, length 887\n",
      "Record gi|16128108|ref|NP_414657.1|, length 630\n",
      "Record gi|16128109|ref|NP_414658.1|, length 474\n",
      "Record gi|16128110|ref|NP_414659.1|, length 617\n",
      "Record gi|16128111|ref|NP_414660.1|, length 865\n",
      "Record gi|90111086|ref|NP_414661.2|, length 120\n",
      "Record gi|16128113|ref|NP_414662.1|, length 264\n",
      "Record gi|16128114|ref|NP_414663.1|, length 288\n",
      "Record gi|90111087|ref|NP_414664.4|, length 115\n",
      "Record gi|16128116|ref|NP_414665.1|, length 516\n",
      "Record gi|16128117|ref|NP_414666.1|, length 796\n",
      "Record gi|90111088|ref|NP_414667.4|, length 178\n",
      "Record gi|16128119|ref|NP_414668.1|, length 220\n",
      "Record gi|16128120|ref|NP_414669.1|, length 308\n",
      "Record gi|16128121|ref|NP_414670.1|, length 256\n",
      "Record gi|16128122|ref|NP_414671.1|, length 146\n",
      "Record gi|16128123|ref|NP_414672.1|, length 409\n",
      "Record gi|16128124|ref|NP_414673.1|, length 126\n",
      "Record gi|16128125|ref|NP_414674.1|, length 300\n",
      "Record gi|16128126|ref|NP_414675.1|, length 283\n",
      "Record gi|16128127|ref|NP_414676.1|, length 264\n",
      "Record gi|16128128|ref|NP_414677.1|, length 412\n",
      "Record gi|16128129|ref|NP_414678.1|, length 198\n",
      "Record gi|16128130|ref|NP_414679.1|, length 201\n",
      "Record gi|90111089|ref|NP_414680.4|, length 189\n",
      "Record gi|16128132|ref|NP_414681.1|, length 865\n",
      "Record gi|16128133|ref|NP_414682.1|, length 246\n",
      "Record gi|16128134|ref|NP_414683.1|, length 194\n",
      "Record gi|16128135|ref|NP_414684.1|, length 159\n",
      "Record gi|162135892|ref|NP_414685.4|, length 465\n",
      "Record gi|226524698|ref|NP_414686.3|, length 308\n",
      "Record gi|16128138|ref|NP_414687.1|, length 151\n",
      "Record gi|16128139|ref|NP_414688.1|, length 234\n",
      "Record gi|90111091|ref|NP_414689.4|, length 176\n",
      "Record gi|162135893|ref|NP_414690.4|, length 809\n",
      "Record gi|16128142|ref|NP_414691.1|, length 844\n",
      "Record gi|16128143|ref|NP_414692.1|, length 747\n",
      "Record gi|16128144|ref|NP_414693.1|, length 265\n",
      "Record gi|16128145|ref|NP_414694.1|, length 296\n",
      "Record gi|16128146|ref|NP_414695.1|, length 660\n",
      "Record gi|16128147|ref|NP_414696.1|, length 426\n",
      "Record gi|16128148|ref|NP_414697.1|, length 473\n",
      "Record gi|16128149|ref|NP_414698.1|, length 114\n",
      "Record gi|16128150|ref|NP_414699.1|, length 207\n",
      "Record gi|16128151|ref|NP_414700.1|, length 266\n",
      "Record gi|16128152|ref|NP_414701.1|, length 232\n",
      "Record gi|16128153|ref|NP_414702.1|, length 505\n",
      "Record gi|16128154|ref|NP_414703.1|, length 474\n",
      "Record gi|90111093|ref|NP_414704.4|, length 385\n",
      "Record gi|16128156|ref|NP_414705.1|, length 128\n",
      "Record gi|90111094|ref|NP_414706.2|, length 270\n",
      "Record gi|16128159|ref|NP_414708.1|, length 274\n",
      "Record gi|16128160|ref|NP_414709.1|, length 890\n",
      "Record gi|16128161|ref|NP_414710.1|, length 264\n",
      "Record gi|16128162|ref|NP_414711.1|, length 241\n",
      "Record gi|16128163|ref|NP_414712.1|, length 283\n",
      "Record gi|16128164|ref|NP_414713.1|, length 241\n",
      "Record gi|16128165|ref|NP_414714.1|, length 185\n",
      "Record gi|16128166|ref|NP_414715.1|, length 398\n",
      "Record gi|16128167|ref|NP_414716.1|, length 253\n",
      "Record gi|90111095|ref|NP_414717.2|, length 285\n",
      "Record gi|16128169|ref|NP_414718.1|, length 450\n",
      "Record gi|16128170|ref|NP_414719.1|, length 810\n",
      "Record gi|16128171|ref|NP_414720.1|, length 161\n",
      "Record gi|16128172|ref|NP_414721.1|, length 341\n",
      "Record gi|16128173|ref|NP_414722.1|, length 151\n",
      "Record gi|16128174|ref|NP_414723.1|, length 262\n",
      "Record gi|16128175|ref|NP_414724.1|, length 382\n",
      "Record gi|16128176|ref|NP_414725.1|, length 198\n",
      "Record gi|16128177|ref|NP_414726.1|, length 1160\n",
      "Record gi|16128178|ref|NP_414727.1|, length 319\n",
      "Record gi|16128179|ref|NP_414728.1|, length 713\n",
      "Record gi|90111096|ref|NP_414729.4|, length 129\n",
      "Record gi|16128181|ref|NP_414730.1|, length 432\n",
      "Record gi|90111097|ref|NP_414731.2|, length 84\n",
      "Record gi|49176004|ref|YP_026160.1|, length 66\n",
      "Record gi|16128183|ref|NP_414732.1|, length 181\n",
      "Record gi|16128184|ref|NP_414733.1|, length 140\n",
      "Record gi|16128185|ref|NP_414734.1|, length 236\n",
      "Record gi|90111098|ref|NP_414735.4|, length 274\n",
      "Record gi|16128187|ref|NP_414736.1|, length 572\n",
      "Record gi|16128188|ref|NP_414737.1|, length 235\n",
      "Record gi|16128189|ref|NP_414738.1|, length 134\n",
      "Record gi|16128190|ref|NP_414739.1|, length 271\n",
      "Record gi|16128191|ref|NP_414740.1|, length 217\n",
      "Record gi|16128192|ref|NP_414741.1|, length 343\n",
      "Record gi|16128193|ref|NP_414742.1|, length 191\n",
      "Record gi|16128194|ref|NP_414743.1|, length 267\n",
      "Record gi|16128195|ref|NP_414744.1|, length 304\n",
      "Record gi|16128196|ref|NP_414745.1|, length 266\n",
      "Record gi|16128197|ref|NP_414746.1|, length 207\n",
      "Record gi|16128198|ref|NP_414747.1|, length 452\n",
      "Record gi|16128199|ref|NP_414748.1|, length 251\n",
      "Record gi|90111099|ref|NP_414749.2|, length 240\n",
      "Record gi|16128201|ref|NP_414750.1|, length 155\n",
      "Record gi|16128202|ref|NP_414751.1|, length 243\n",
      "Record gi|16128203|ref|NP_414752.1|, length 261\n",
      "Record gi|145698220|ref|YP_001165307.1|, length 159\n",
      "Record gi|16128205|ref|NP_414754.1|, length 256\n",
      "Record gi|16128206|ref|NP_414755.1|, length 157\n",
      "Record gi|90111100|ref|NP_414756.2|, length 814\n",
      "Record gi|16128208|ref|NP_414757.1|, length 192\n",
      "Record gi|16128209|ref|NP_414758.1|, length 255\n",
      "Record gi|16128210|ref|NP_414759.1|, length 246\n",
      "Record gi|16128211|ref|NP_414760.1|, length 92\n",
      "Record gi|16128212|ref|NP_414761.1|, length 86\n",
      "Record gi|16128213|ref|NP_414762.1|, length 249\n",
      "Record gi|16128214|ref|NP_414763.1|, length 165\n",
      "Record gi|16128217|ref|NP_414766.1|, length 351\n",
      "Record gi|16128218|ref|NP_414767.1|, length 97\n",
      "Record gi|16128219|ref|NP_414768.1|, length 132\n",
      "Record gi|16128220|ref|NP_414769.1|, length 150\n",
      "Record gi|16128223|ref|NP_414772.1|, length 485\n",
      "Record gi|16128224|ref|NP_414773.1|, length 152\n",
      "Record gi|16128225|ref|NP_414774.1|, length 414\n",
      "Record gi|16128227|ref|NP_414776.1|, length 351\n",
      "Record gi|16128228|ref|NP_414777.1|, length 367\n",
      "Record gi|16128229|ref|NP_414778.1|, length 417\n",
      "Record gi|16128230|ref|NP_414779.1|, length 113\n",
      "Record gi|16128231|ref|NP_414780.1|, length 105\n",
      "Record gi|94541094|ref|YP_588435.1|, length 73\n",
      "Record gi|16128232|ref|NP_414781.1|, length 158\n",
      "Record gi|16128233|ref|NP_414782.1|, length 152\n",
      "Record gi|16128234|ref|NP_414783.1|, length 79\n",
      "Record gi|16128235|ref|NP_414784.1|, length 155\n",
      "Record gi|145698221|ref|NP_414785.4|, length 147\n",
      "Record gi|90111102|ref|NP_414786.4|, length 273\n",
      "Record gi|90111103|ref|NP_414787.4|, length 287\n",
      "Record gi|16128239|ref|NP_414788.1|, length 297\n",
      "Record gi|16128241|ref|NP_414790.1|, length 383\n",
      "Record gi|16128244|ref|NP_414793.1|, length 338\n",
      "Record gi|90111105|ref|NP_414794.4|, length 467\n",
      "Record gi|16128246|ref|NP_414795.1|, length 310\n",
      "Record gi|90111106|ref|NP_414796.2|, length 348\n",
      "Record gi|16128249|ref|NP_414798.1|, length 167\n",
      "Record gi|16128250|ref|NP_414799.1|, length 91\n",
      "Record gi|16128252|ref|NP_414801.1|, length 384\n",
      "Record gi|345452715|ref|NP_414802.2|, length 302\n",
      "Record gi|16128254|ref|NP_414803.1|, length 655\n",
      "Record gi|16128255|ref|NP_414804.1|, length 460\n",
      "Record gi|16128256|ref|NP_414805.1|, length 536\n",
      "Record gi|16128257|ref|NP_414806.1|, length 252\n",
      "Record gi|16128258|ref|NP_414807.1|, length 334\n",
      "Record gi|16128259|ref|NP_414808.1|, length 167\n",
      "Record gi|16128260|ref|NP_414809.1|, length 91\n",
      "Record gi|16128262|ref|NP_414811.1|, length 208\n",
      "Record gi|16128263|ref|NP_414812.1|, length 232\n",
      "Record gi|16128264|ref|NP_414813.1|, length 284\n",
      "Record gi|16128265|ref|NP_414814.1|, length 146\n",
      "Record gi|16128266|ref|NP_414815.1|, length 466\n",
      "Record gi|16128268|ref|NP_414817.1|, length 318\n",
      "Record gi|16128269|ref|NP_414818.1|, length 732\n",
      "Record gi|16128270|ref|NP_414819.1|, length 318\n",
      "Record gi|16128271|ref|NP_414820.1|, length 229\n",
      "Record gi|16128272|ref|NP_414821.1|, length 204\n",
      "Record gi|16128273|ref|NP_414822.1|, length 109\n",
      "Record gi|90111108|ref|NP_414823.2|, length 236\n",
      "Record gi|16128275|ref|NP_414824.1|, length 547\n",
      "Record gi|16128276|ref|NP_414825.1|, length 841\n",
      "Record gi|16128277|ref|NP_414826.1|, length 222\n",
      "Record gi|16128278|ref|NP_414827.1|, length 195\n",
      "Record gi|16128279|ref|NP_414828.1|, length 196\n",
      "Record gi|16128280|ref|NP_414829.1|, length 75\n",
      "Record gi|94541137|ref|YP_588437.1|, length 46\n",
      "Record gi|16128281|ref|NP_414830.1|, length 87\n",
      "Record gi|226524699|ref|YP_002791238.1|, length 33\n",
      "Record gi|226524700|ref|NP_414832.2|, length 99\n",
      "Record gi|16128284|ref|NP_414833.1|, length 288\n",
      "Record gi|90111110|ref|NP_414835.2|, length 197\n",
      "Record gi|90111111|ref|NP_414837.2|, length 78\n",
      "Record gi|90111112|ref|NP_414838.2|, length 441\n",
      "Record gi|16128290|ref|NP_414839.1|, length 284\n",
      "Record gi|16128291|ref|NP_414840.1|, length 239\n",
      "Record gi|16128292|ref|NP_414841.1|, length 475\n",
      "Record gi|90111113|ref|NP_414842.4|, length 231\n",
      "Record gi|16128295|ref|NP_414844.1|, length 222\n",
      "Record gi|16128296|ref|NP_414845.1|, length 556\n",
      "Record gi|16128297|ref|NP_414846.1|, length 490\n",
      "Record gi|49176011|ref|NP_414847.3|, length 195\n",
      "Record gi|16128299|ref|NP_414848.1|, length 677\n",
      "Record gi|16128300|ref|NP_414849.1|, length 362\n",
      "Record gi|16128301|ref|NP_414850.1|, length 310\n",
      "Record gi|16128302|ref|NP_414851.1|, length 165\n",
      "Record gi|16128303|ref|NP_414852.1|, length 201\n",
      "Record gi|16128304|ref|NP_414853.1|, length 287\n",
      "Record gi|16128305|ref|NP_414854.1|, length 515\n",
      "Record gi|16128306|ref|NP_414855.1|, length 472\n",
      "Record gi|16128308|ref|NP_414857.1|, length 316\n",
      "Record gi|16128309|ref|NP_414858.1|, length 460\n",
      "Record gi|16128310|ref|NP_414859.1|, length 349\n",
      "Record gi|16128311|ref|NP_414860.1|, length 271\n",
      "Record gi|90111114|ref|NP_414861.2|, length 81\n",
      "Record gi|16128313|ref|NP_414862.1|, length 223\n",
      "Record gi|16128314|ref|NP_414863.1|, length 91\n",
      "Record gi|16128315|ref|NP_414864.1|, length 528\n",
      "Record gi|16128316|ref|NP_414865.1|, length 296\n",
      "Record gi|16128318|ref|NP_414867.1|, length 389\n",
      "Record gi|16128319|ref|NP_414868.1|, length 483\n",
      "Record gi|16128320|ref|NP_414869.1|, length 628\n",
      "Record gi|16128321|ref|NP_414870.1|, length 419\n",
      "Record gi|16128322|ref|NP_414871.1|, length 427\n",
      "Record gi|157783149|ref|NP_414872.3|, length 299\n",
      "Record gi|16128324|ref|NP_414873.1|, length 219\n",
      "Record gi|16128325|ref|NP_414874.1|, length 156\n",
      "Record gi|16128326|ref|NP_414875.1|, length 384\n",
      "Record gi|16128327|ref|NP_414876.1|, length 203\n",
      "Record gi|16128328|ref|NP_414877.1|, length 417\n",
      "Record gi|16128329|ref|NP_414878.1|, length 1024\n",
      "Record gi|49176012|ref|NP_414879.3|, length 360\n",
      "Record gi|145698223|ref|NP_414880.2|, length 277\n",
      "Record gi|16128332|ref|NP_414881.1|, length 554\n",
      "Record gi|16128333|ref|NP_414882.1|, length 314\n",
      "Record gi|459201369|ref|NP_414883.5|, length 288\n",
      "Record gi|90111116|ref|NP_414884.2|, length 269\n",
      "Record gi|16128336|ref|NP_414885.1|, length 316\n",
      "Record gi|16128337|ref|NP_414886.1|, length 337\n",
      "Record gi|90111117|ref|NP_414887.2|, length 403\n",
      "Record gi|90111118|ref|NP_414888.4|, length 179\n",
      "Record gi|16128340|ref|NP_414889.1|, length 277\n",
      "Record gi|16128341|ref|NP_414890.1|, length 369\n",
      "Record gi|162135894|ref|NP_414891.4|, length 91\n",
      "Record gi|16128343|ref|NP_414892.1|, length 257\n",
      "Record gi|145698224|ref|NP_414894.2|, length 121\n",
      "Record gi|16128346|ref|NP_414895.1|, length 301\n",
      "Record gi|16128348|ref|NP_414897.1|, length 398\n",
      "Record gi|90111121|ref|NP_414898.4|, length 185\n",
      "Record gi|90111122|ref|NP_414899.2|, length 320\n",
      "Record gi|16128351|ref|NP_414900.1|, length 255\n",
      "Record gi|16128352|ref|NP_414901.1|, length 275\n",
      "Record gi|16128353|ref|NP_414902.1|, length 283\n",
      "Record gi|90111123|ref|NP_414903.4|, length 324\n",
      "Record gi|16128357|ref|NP_414906.1|, length 288\n",
      "Record gi|226524701|ref|NP_414907.2|, length 99\n",
      "Record gi|226524702|ref|NP_414909.2|, length 207\n",
      "Record gi|16128361|ref|NP_414910.1|, length 385\n",
      "Record gi|16128362|ref|NP_414911.1|, length 406\n",
      "Record gi|16128363|ref|NP_414912.1|, length 364\n",
      "Record gi|16128364|ref|NP_414913.1|, length 102\n",
      "Record gi|90111125|ref|NP_414914.2|, length 70\n",
      "Record gi|16128366|ref|NP_414915.1|, length 364\n",
      "Record gi|16128367|ref|NP_414916.1|, length 86\n",
      "Record gi|49176017|ref|NP_414917.2|, length 471\n",
      "Record gi|162135895|ref|NP_414918.4|, length 106\n",
      "Record gi|16128370|ref|NP_414919.1|, length 371\n",
      "Record gi|16128371|ref|NP_414920.1|, length 269\n",
      "Record gi|90111127|ref|NP_414921.2|, length 152\n",
      "Record gi|16128373|ref|NP_414922.1|, length 174\n",
      "Record gi|16128374|ref|NP_414923.1|, length 63\n",
      "Record gi|16128375|ref|NP_414924.1|, length 225\n",
      "Record gi|16128376|ref|NP_414925.1|, length 94\n",
      "Record gi|16128378|ref|NP_414927.1|, length 303\n",
      "Record gi|90111128|ref|NP_414928.2|, length 302\n",
      "Record gi|49176018|ref|NP_414930.3|, length 394\n",
      "Record gi|16128382|ref|NP_414931.1|, length 1048\n",
      "Record gi|16128383|ref|NP_414932.1|, length 400\n",
      "Record gi|16128384|ref|NP_414933.1|, length 229\n",
      "Record gi|16128385|ref|NP_414934.1|, length 431\n",
      "Record gi|16128386|ref|NP_414935.1|, length 439\n",
      "Record gi|16128387|ref|NP_414936.1|, length 457\n",
      "Record gi|345452716|ref|NP_414937.2|, length 604\n",
      "Record gi|16128389|ref|NP_414938.1|, length 193\n",
      "Record gi|16128390|ref|NP_414939.1|, length 356\n",
      "Record gi|16128391|ref|NP_414940.1|, length 375\n",
      "Record gi|16128392|ref|NP_414941.1|, length 110\n",
      "Record gi|16128393|ref|NP_414942.1|, length 615\n",
      "Record gi|16128394|ref|NP_414943.1|, length 323\n",
      "Record gi|16128395|ref|NP_414944.1|, length 115\n",
      "Record gi|16128396|ref|NP_414945.1|, length 294\n",
      "Record gi|90111129|ref|NP_414946.4|, length 179\n",
      "Record gi|16128398|ref|NP_414947.1|, length 149\n",
      "Record gi|16128399|ref|NP_414948.1|, length 367\n",
      "Record gi|16128400|ref|NP_414949.1|, length 156\n",
      "Record gi|16128401|ref|NP_414950.1|, length 139\n",
      "Record gi|16128402|ref|NP_414951.1|, length 325\n",
      "Record gi|16128403|ref|NP_414952.1|, length 172\n",
      "Record gi|90111130|ref|NP_414953.2|, length 324\n",
      "Record gi|16128405|ref|NP_414954.1|, length 620\n",
      "Record gi|16128406|ref|NP_414955.1|, length 299\n",
      "Record gi|16128407|ref|NP_414956.1|, length 80\n",
      "Record gi|16128408|ref|NP_414957.1|, length 482\n",
      "Record gi|90111131|ref|NP_414958.4|, length 196\n",
      "Record gi|16128410|ref|NP_414959.1|, length 303\n",
      "Record gi|90111132|ref|NP_414960.2|, length 163\n",
      "Record gi|90111133|ref|NP_414961.4|, length 454\n",
      "Record gi|16128413|ref|NP_414962.1|, length 296\n",
      "Record gi|16128414|ref|NP_414963.1|, length 109\n",
      "Record gi|16128415|ref|NP_414964.1|, length 204\n",
      "Record gi|16128416|ref|NP_414965.1|, length 663\n",
      "Record gi|16128417|ref|NP_414966.1|, length 315\n",
      "Record gi|16128418|ref|NP_414967.1|, length 491\n",
      "Record gi|90111134|ref|NP_414968.4|, length 192\n",
      "Record gi|90111135|ref|NP_414969.4|, length 105\n",
      "Record gi|16128421|ref|NP_414970.1|, length 432\n",
      "Record gi|16128422|ref|NP_414971.1|, length 207\n",
      "Record gi|16128423|ref|NP_414972.1|, length 424\n",
      "Record gi|16128424|ref|NP_414973.1|, length 784\n",
      "Record gi|16128425|ref|NP_414974.1|, length 90\n",
      "Record gi|16128426|ref|NP_414975.1|, length 623\n",
      "Record gi|16128427|ref|NP_414976.1|, length 123\n",
      "Record gi|16128428|ref|NP_414977.1|, length 132\n",
      "Record gi|16128429|ref|NP_414978.1|, length 231\n",
      "Record gi|16128430|ref|NP_414979.1|, length 566\n",
      "Record gi|90111136|ref|NP_414980.2|, length 272\n",
      "Record gi|90111137|ref|NP_414981.4|, length 152\n",
      "Record gi|16128433|ref|NP_414982.1|, length 590\n",
      "Record gi|16128434|ref|NP_414983.1|, length 593\n",
      "Record gi|16128435|ref|NP_414984.1|, length 112\n",
      "Record gi|16128436|ref|NP_414985.1|, length 428\n",
      "Record gi|16128437|ref|NP_414986.1|, length 286\n",
      "Record gi|49176025|ref|NP_414987.3|, length 190\n",
      "Record gi|16128439|ref|NP_414988.1|, length 129\n",
      "Record gi|16128440|ref|NP_414989.1|, length 117\n",
      "Record gi|90111138|ref|NP_414990.2|, length 516\n",
      "Record gi|90111139|ref|NP_414991.4|, length 156\n",
      "Record gi|16128443|ref|NP_414992.1|, length 183\n",
      "Record gi|16128444|ref|NP_414993.1|, length 72\n",
      "Record gi|16128445|ref|NP_414994.1|, length 124\n",
      "Record gi|16128446|ref|NP_414995.1|, length 1049\n",
      "Record gi|16128447|ref|NP_414996.1|, length 397\n",
      "Record gi|16128448|ref|NP_414997.1|, length 215\n",
      "Record gi|16128449|ref|NP_414998.1|, length 1120\n",
      "Record gi|16128450|ref|NP_414999.1|, length 53\n",
      "Record gi|16128451|ref|NP_415000.1|, length 175\n",
      "Record gi|16128452|ref|NP_415001.1|, length 125\n",
      "Record gi|16128453|ref|NP_415002.1|, length 183\n",
      "Record gi|16128454|ref|NP_415003.1|, length 643\n",
      "Record gi|16128455|ref|NP_415004.1|, length 109\n",
      "Record gi|16128456|ref|NP_415005.1|, length 201\n",
      "Record gi|16128457|ref|NP_415006.1|, length 624\n",
      "Record gi|16128458|ref|NP_415007.1|, length 214\n",
      "Record gi|16128459|ref|NP_415008.1|, length 320\n",
      "Record gi|16128460|ref|NP_415009.1|, length 319\n",
      "Record gi|16128461|ref|NP_415010.1|, length 434\n",
      "Record gi|16128462|ref|NP_415011.1|, length 558\n",
      "Record gi|16128463|ref|NP_415012.1|, length 406\n",
      "Record gi|16128464|ref|NP_415013.1|, length 550\n",
      "Record gi|16128465|ref|NP_415014.1|, length 159\n",
      "Record gi|16128466|ref|NP_415015.1|, length 264\n",
      "Record gi|90111140|ref|NP_415016.4|, length 113\n",
      "Record gi|16128468|ref|NP_415017.1|, length 834\n",
      "Record gi|16128469|ref|NP_415018.1|, length 310\n",
      "Record gi|16128470|ref|NP_415019.1|, length 430\n",
      "Record gi|16128471|ref|NP_415020.1|, length 135\n",
      "Record gi|145698226|ref|NP_415021.2|, length 152\n",
      "Record gi|16128473|ref|NP_415022.1|, length 305\n",
      "Record gi|16128474|ref|NP_415023.1|, length 225\n",
      "Record gi|90111141|ref|NP_415024.4|, length 259\n",
      "Record gi|90111142|ref|NP_415025.4|, length 284\n",
      "Record gi|16128477|ref|NP_415026.1|, length 269\n",
      "Record gi|16128478|ref|NP_415027.1|, length 208\n",
      "Record gi|16128479|ref|NP_415028.1|, length 228\n",
      "Record gi|16128480|ref|NP_415029.1|, length 804\n",
      "Record gi|16128481|ref|NP_415030.1|, length 1426\n",
      "Record gi|16128482|ref|NP_415031.1|, length 122\n",
      "Record gi|16128487|ref|NP_415036.1|, length 364\n",
      "Record gi|16128488|ref|NP_415037.1|, length 308\n",
      "Record gi|16128489|ref|NP_415038.1|, length 160\n",
      "Record gi|16128490|ref|NP_415039.1|, length 271\n",
      "Record gi|16128491|ref|NP_415040.1|, length 593\n",
      "Record gi|16128492|ref|NP_415041.1|, length 258\n",
      "Record gi|16128493|ref|NP_415042.1|, length 292\n",
      "Record gi|90111144|ref|NP_415044.4|, length 484\n",
      "Record gi|16128496|ref|NP_415045.1|, length 453\n",
      "Record gi|90111145|ref|NP_415046.4|, length 433\n",
      "Record gi|16128498|ref|NP_415047.1|, length 381\n",
      "Record gi|16128499|ref|NP_415048.1|, length 261\n",
      "Record gi|16128500|ref|NP_415049.1|, length 411\n",
      "Record gi|16128501|ref|NP_415050.1|, length 349\n",
      "Record gi|16128502|ref|NP_415051.1|, length 555\n",
      "Record gi|635341461|ref|YP_009029994.1|, length 419\n",
      "Record gi|16128504|ref|NP_415053.1|, length 271\n",
      "Record gi|16128505|ref|NP_415054.1|, length 297\n",
      "Record gi|16128506|ref|NP_415055.1|, length 355\n",
      "Record gi|16128507|ref|NP_415056.1|, length 169\n",
      "Record gi|16128508|ref|NP_415057.1|, length 240\n",
      "Record gi|16128509|ref|NP_415058.1|, length 164\n",
      "Record gi|16128510|ref|NP_415059.1|, length 461\n",
      "Record gi|16128511|ref|NP_415060.1|, length 173\n",
      "Record gi|90111146|ref|NP_415061.2|, length 70\n",
      "Record gi|16128513|ref|NP_415062.1|, length 288\n",
      "Record gi|90111147|ref|NP_415063.4|, length 180\n",
      "Record gi|16128515|ref|NP_415064.1|, length 230\n",
      "Record gi|16128516|ref|NP_415065.1|, length 867\n",
      "Record gi|145698227|ref|NP_415066.2|, length 327\n",
      "Record gi|16128518|ref|NP_415067.1|, length 171\n",
      "Record gi|16128519|ref|NP_415068.1|, length 210\n",
      "Record gi|16128520|ref|NP_415069.1|, length 387\n",
      "Record gi|226524703|ref|NP_415072.2|, length 99\n",
      "Record gi|16128524|ref|NP_415073.1|, length 288\n",
      "Record gi|16128526|ref|NP_415075.1|, length 110\n",
      "Record gi|16128527|ref|NP_415076.1|, length 508\n",
      "Record gi|16128528|ref|NP_415077.1|, length 183\n",
      "Record gi|16128529|ref|NP_415078.1|, length 265\n",
      "Record gi|145698228|ref|YP_001165308.1|, length 33\n",
      "Record gi|16128530|ref|NP_415079.1|, length 151\n",
      "Record gi|16128531|ref|NP_415080.1|, length 56\n",
      "Record gi|16128532|ref|NP_415081.1|, length 96\n",
      "Record gi|16128533|ref|NP_415082.1|, length 120\n",
      "Record gi|94541097|ref|YP_588439.1|, length 46\n",
      "Record gi|16128534|ref|NP_415083.1|, length 127\n",
      "Record gi|16128535|ref|NP_415084.1|, length 338\n",
      "Record gi|16128537|ref|NP_415086.1|, length 71\n",
      "Record gi|16128538|ref|NP_415087.1|, length 165\n",
      "Record gi|16128539|ref|NP_415088.1|, length 153\n",
      "Record gi|94541098|ref|YP_588440.1|, length 60\n",
      "Record gi|16128540|ref|NP_415089.1|, length 97\n",
      "Record gi|90111149|ref|NP_415090.2|, length 136\n",
      "Record gi|16128542|ref|NP_415091.1|, length 68\n",
      "Record gi|145698229|ref|YP_001165309.1|, length 64\n",
      "Record gi|16128543|ref|NP_415092.1|, length 181\n",
      "Record gi|16128547|ref|NP_415096.1|, length 249\n",
      "Record gi|16128548|ref|NP_415097.1|, length 317\n",
      "Record gi|16128549|ref|NP_415098.1|, length 253\n",
      "Record gi|16128550|ref|NP_415099.1|, length 296\n",
      "Record gi|16128551|ref|NP_415100.1|, length 990\n",
      "Record gi|16128552|ref|NP_415101.1|, length 745\n",
      "Record gi|16128553|ref|NP_415102.1|, length 480\n",
      "Record gi|16128554|ref|NP_415103.1|, length 227\n",
      "Record gi|16128555|ref|NP_415104.1|, length 457\n",
      "Record gi|16128556|ref|NP_415105.1|, length 110\n",
      "Record gi|16128557|ref|NP_415106.1|, length 407\n",
      "Record gi|16128558|ref|NP_415107.1|, length 1047\n",
      "Record gi|16128559|ref|NP_415108.1|, length 458\n",
      "Record gi|16128560|ref|NP_415109.1|, length 415\n",
      "Record gi|16128561|ref|NP_415110.1|, length 217\n",
      "Record gi|16128562|ref|NP_415111.1|, length 122\n",
      "Record gi|16128563|ref|NP_415112.1|, length 82\n",
      "Record gi|16128564|ref|NP_415113.1|, length 372\n",
      "Record gi|49176035|ref|YP_025295.1|, length 50\n",
      "Record gi|16128565|ref|NP_415114.1|, length 370\n",
      "Record gi|345452717|ref|NP_415115.2|, length 206\n",
      "Record gi|16128567|ref|NP_415116.1|, length 746\n",
      "Record gi|635341462|ref|NP_415117.2|, length 400\n",
      "Record gi|94541099|ref|YP_588441.1|, length 72\n",
      "Record gi|16128569|ref|NP_415118.1|, length 1293\n",
      "Record gi|16128570|ref|NP_415119.1|, length 377\n",
      "Record gi|16128571|ref|NP_415120.1|, length 271\n",
      "Record gi|16128572|ref|NP_415121.1|, length 330\n",
      "Record gi|16128573|ref|NP_415122.1|, length 334\n",
      "Record gi|16128574|ref|NP_415123.1|, length 416\n",
      "Record gi|16128575|ref|NP_415124.1|, length 318\n",
      "Record gi|16128576|ref|NP_415125.1|, length 391\n",
      "Record gi|16128577|ref|NP_415126.1|, length 536\n",
      "Record gi|16128578|ref|NP_415127.1|, length 285\n",
      "Record gi|16128579|ref|NP_415128.1|, length 248\n",
      "Record gi|16128580|ref|NP_415129.1|, length 137\n",
      "Record gi|16128581|ref|NP_415130.1|, length 701\n",
      "Record gi|94541100|ref|YP_588442.1|, length 65\n",
      "Record gi|16128582|ref|NP_415132.1|, length 362\n",
      "Record gi|16128583|ref|NP_415133.1|, length 386\n",
      "Record gi|16128584|ref|NP_415134.1|, length 209\n",
      "Record gi|16128585|ref|NP_415135.1|, length 406\n",
      "Record gi|16128586|ref|NP_415136.1|, length 300\n",
      "Record gi|90111151|ref|NP_415137.2|, length 248\n",
      "Record gi|16128588|ref|NP_415138.1|, length 187\n",
      "Record gi|90111152|ref|NP_415139.2|, length 521\n",
      "Record gi|16128590|ref|NP_415140.1|, length 142\n",
      "Record gi|16128591|ref|NP_415141.1|, length 412\n",
      "Record gi|16128593|ref|NP_415143.1|, length 136\n",
      "Record gi|16128594|ref|NP_415144.1|, length 268\n",
      "Record gi|16128595|ref|NP_415145.1|, length 487\n",
      "Record gi|16128596|ref|NP_415146.1|, length 292\n",
      "Record gi|16128597|ref|NP_415147.1|, length 183\n",
      "Record gi|16128598|ref|NP_415148.1|, length 510\n",
      "Record gi|90111153|ref|NP_415149.4|, length 302\n",
      "Record gi|16128600|ref|NP_415150.1|, length 98\n",
      "Record gi|90111154|ref|NP_415151.4|, length 352\n",
      "Record gi|16128602|ref|NP_415152.1|, length 552\n",
      "Record gi|16128603|ref|NP_415153.1|, length 226\n",
      "Record gi|16128604|ref|NP_415154.1|, length 461\n",
      "Record gi|16128605|ref|NP_415155.1|, length 186\n",
      "Record gi|16128606|ref|NP_415156.1|, length 69\n",
      "Record gi|16128607|ref|NP_415157.1|, length 127\n",
      "Record gi|16128610|ref|NP_415160.1|, length 67\n",
      "Record gi|16128611|ref|NP_415161.1|, length 321\n",
      "Record gi|90111155|ref|NP_415162.4|, length 317\n",
      "Record gi|90111156|ref|NP_415163.2|, length 213\n",
      "Record gi|16128614|ref|NP_415164.1|, length 87\n",
      "Record gi|16128615|ref|NP_415165.1|, length 403\n",
      "Record gi|16128616|ref|NP_415166.1|, length 362\n",
      "Record gi|16128617|ref|NP_415167.1|, length 370\n",
      "Record gi|16128618|ref|NP_415168.1|, length 633\n",
      "Record gi|16128619|ref|NP_415169.1|, length 155\n",
      "Record gi|90111157|ref|NP_415170.4|, length 105\n",
      "Record gi|16128621|ref|NP_415171.1|, length 203\n",
      "Record gi|16128622|ref|NP_415172.1|, length 213\n",
      "Record gi|16128623|ref|NP_415173.1|, length 343\n",
      "Record gi|16128624|ref|NP_415174.1|, length 193\n",
      "Record gi|16128625|ref|NP_415175.1|, length 860\n",
      "Record gi|16128626|ref|NP_415176.1|, length 160\n",
      "Record gi|90111158|ref|NP_415177.4|, length 325\n",
      "Record gi|16128628|ref|NP_415178.1|, length 235\n",
      "Record gi|16128629|ref|NP_415179.1|, length 475\n",
      "Record gi|16128630|ref|NP_415180.1|, length 184\n",
      "Record gi|16128631|ref|NP_415181.1|, length 235\n",
      "Record gi|16128632|ref|NP_415182.1|, length 483\n",
      "Record gi|16128633|ref|NP_415183.1|, length 556\n",
      "Record gi|16128634|ref|NP_415184.1|, length 311\n",
      "Record gi|16128635|ref|NP_415185.1|, length 241\n",
      "Record gi|16128636|ref|NP_415186.1|, length 224\n",
      "Record gi|16128637|ref|NP_415187.1|, length 246\n",
      "Record gi|16128638|ref|NP_415188.1|, length 302\n",
      "Record gi|16128639|ref|NP_415189.1|, length 338\n",
      "Record gi|16128640|ref|NP_415190.1|, length 512\n",
      "Record gi|16128641|ref|NP_415191.1|, length 292\n",
      "Record gi|16128642|ref|NP_415192.1|, length 155\n",
      "Record gi|145698230|ref|NP_415193.2|, length 346\n",
      "Record gi|16128644|ref|NP_415194.1|, length 474\n",
      "Record gi|16128645|ref|NP_415195.1|, length 391\n",
      "Record gi|16128650|ref|NP_415200.1|, length 554\n",
      "Record gi|16128651|ref|NP_415201.1|, length 250\n",
      "Record gi|16128652|ref|NP_415202.1|, length 406\n",
      "Record gi|16128653|ref|NP_415203.1|, length 382\n",
      "Record gi|16128654|ref|NP_415204.1|, length 266\n",
      "Record gi|16128655|ref|NP_415205.1|, length 648\n",
      "Record gi|16128656|ref|NP_415206.1|, length 554\n",
      "Record gi|16128657|ref|NP_415207.1|, length 468\n",
      "Record gi|16128658|ref|NP_415208.1|, length 108\n",
      "Record gi|16128659|ref|NP_415209.1|, length 148\n",
      "Record gi|157783150|ref|YP_001491546.1|, length 28\n",
      "Record gi|16128660|ref|NP_415210.1|, length 176\n",
      "Record gi|90111159|ref|NP_415211.2|, length 97\n",
      "Record gi|16128662|ref|NP_415212.1|, length 254\n",
      "Record gi|16128663|ref|NP_415213.1|, length 181\n",
      "Record gi|16128664|ref|NP_415214.1|, length 546\n",
      "Record gi|16128665|ref|NP_415215.1|, length 164\n",
      "Record gi|16128668|ref|NP_415219.1|, length 439\n",
      "Record gi|16128669|ref|NP_415220.1|, length 732\n",
      "Record gi|145698231|ref|YP_001165310.1|, length 85\n",
      "Record gi|16128670|ref|NP_415222.1|, length 225\n",
      "Record gi|16128671|ref|NP_415223.1|, length 894\n",
      "Record gi|16128672|ref|NP_415224.1|, length 190\n",
      "Record gi|16128673|ref|NP_415225.1|, length 682\n",
      "Record gi|16128674|ref|NP_415226.1|, length 557\n",
      "Record gi|94541101|ref|YP_588443.1|, length 29\n",
      "Record gi|16128675|ref|NP_415228.1|, length 68\n",
      "Record gi|16128676|ref|NP_415229.1|, length 1397\n",
      "Record gi|16128677|ref|NP_415230.1|, length 108\n",
      "Record gi|16128679|ref|NP_415232.1|, length 189\n",
      "Record gi|16128681|ref|NP_415234.1|, length 253\n",
      "Record gi|16128682|ref|NP_415235.1|, length 169\n",
      "Record gi|16128683|ref|NP_415236.1|, length 472\n",
      "Record gi|16128684|ref|NP_415237.1|, length 493\n",
      "Record gi|16128685|ref|NP_415238.1|, length 247\n",
      "Record gi|16128686|ref|NP_415239.1|, length 218\n",
      "Record gi|16128687|ref|NP_415240.1|, length 310\n",
      "Record gi|16128688|ref|NP_415241.1|, length 244\n",
      "Record gi|16128689|ref|NP_415242.1|, length 263\n",
      "Record gi|90111163|ref|NP_415243.4|, length 348\n",
      "Record gi|90111164|ref|NP_415244.4|, length 353\n",
      "Record gi|16128692|ref|NP_415245.1|, length 242\n",
      "Record gi|90111165|ref|NP_415246.2|, length 815\n",
      "Record gi|16128694|ref|NP_415247.1|, length 188\n",
      "Record gi|16128695|ref|NP_415248.1|, length 427\n",
      "Record gi|16128696|ref|NP_415249.1|, length 129\n",
      "Record gi|16128697|ref|NP_415250.1|, length 115\n",
      "Record gi|16128698|ref|NP_415251.1|, length 588\n",
      "Record gi|16128699|ref|NP_415252.1|, length 238\n",
      "Record gi|16128701|ref|NP_415254.1|, length 933\n",
      "Record gi|16128702|ref|NP_415255.1|, length 405\n",
      "Record gi|16128703|ref|NP_415256.1|, length 388\n",
      "Record gi|16128704|ref|NP_415257.1|, length 289\n",
      "Record gi|16128705|ref|NP_415258.1|, length 240\n",
      "Record gi|16128706|ref|NP_415259.1|, length 658\n",
      "Record gi|16128707|ref|NP_415260.1|, length 877\n",
      "Record gi|90111166|ref|NP_415261.2|, length 522\n",
      "Record gi|16128709|ref|NP_415262.1|, length 379\n",
      "Record gi|94541102|ref|YP_588444.1|, length 37\n",
      "Record gi|16128710|ref|NP_415263.1|, length 97\n",
      "Record gi|16128711|ref|NP_415264.1|, length 134\n",
      "Record gi|16128712|ref|NP_415265.1|, length 230\n",
      "Record gi|16128713|ref|NP_415266.1|, length 142\n",
      "Record gi|16128714|ref|NP_415267.1|, length 421\n",
      "Record gi|16128715|ref|NP_415268.1|, length 430\n",
      "Record gi|16128716|ref|NP_415269.1|, length 173\n",
      "Record gi|16128717|ref|NP_415270.1|, length 263\n",
      "Record gi|16128718|ref|NP_415271.1|, length 347\n",
      "Record gi|16128719|ref|NP_415272.1|, length 239\n",
      "Record gi|16128720|ref|NP_415273.1|, length 313\n",
      "Record gi|16128721|ref|NP_415274.1|, length 126\n",
      "Record gi|16128722|ref|NP_415275.1|, length 350\n",
      "Record gi|16128723|ref|NP_415276.1|, length 250\n",
      "Record gi|16128724|ref|NP_415277.1|, length 346\n",
      "Record gi|16128725|ref|NP_415278.1|, length 382\n",
      "Record gi|16128726|ref|NP_415279.1|, length 348\n",
      "Record gi|49176045|ref|NP_415280.3|, length 338\n",
      "Record gi|16128728|ref|NP_415281.1|, length 490\n",
      "Record gi|16128729|ref|NP_415282.1|, length 262\n",
      "Record gi|16128730|ref|NP_415283.1|, length 49\n",
      "Record gi|16128731|ref|NP_415284.1|, length 257\n",
      "Record gi|16128732|ref|NP_415285.1|, length 229\n",
      "Record gi|16128733|ref|NP_415286.1|, length 352\n",
      "Record gi|16128734|ref|NP_415287.1|, length 272\n",
      "Record gi|16128735|ref|NP_415288.1|, length 331\n",
      "Record gi|145698233|ref|NP_415289.4|, length 317\n",
      "Record gi|16128737|ref|NP_415290.1|, length 350\n",
      "Record gi|16128738|ref|NP_415291.1|, length 477\n",
      "Record gi|90111167|ref|NP_415292.2|, length 753\n",
      "Record gi|16128740|ref|NP_415293.1|, length 427\n",
      "Record gi|16128741|ref|NP_415294.1|, length 158\n",
      "Record gi|16128742|ref|NP_415295.1|, length 429\n",
      "Record gi|16128743|ref|NP_415296.1|, length 346\n",
      "Record gi|16128744|ref|NP_415297.1|, length 384\n",
      "Record gi|16128745|ref|NP_415298.1|, length 251\n",
      "Record gi|16128746|ref|NP_415299.1|, length 225\n",
      "Record gi|16128747|ref|NP_415300.1|, length 673\n",
      "Record gi|16128748|ref|NP_415301.1|, length 302\n",
      "Record gi|16128749|ref|NP_415302.1|, length 329\n",
      "Record gi|16128750|ref|NP_415303.1|, length 170\n",
      "Record gi|16128751|ref|NP_415304.1|, length 161\n",
      "Record gi|16128752|ref|NP_415305.1|, length 81\n",
      "Record gi|16128753|ref|NP_415306.1|, length 150\n",
      "Record gi|16128754|ref|NP_415307.1|, length 234\n",
      "Record gi|16128755|ref|NP_415308.1|, length 237\n",
      "Record gi|16128756|ref|NP_415309.1|, length 318\n",
      "Record gi|16128757|ref|NP_415310.1|, length 413\n",
      "Record gi|16128758|ref|NP_415311.1|, length 253\n",
      "Record gi|16128759|ref|NP_415312.1|, length 136\n",
      "Record gi|16128760|ref|NP_415313.1|, length 368\n",
      "Record gi|16128761|ref|NP_415314.1|, length 377\n",
      "Record gi|90111168|ref|NP_415315.4|, length 578\n",
      "Record gi|16128763|ref|NP_415316.1|, length 332\n",
      "Record gi|162135896|ref|NP_415317.4|, length 223\n",
      "Record gi|16128765|ref|NP_415318.1|, length 454\n",
      "Record gi|16128766|ref|NP_415319.1|, length 160\n",
      "Record gi|16128767|ref|NP_415320.1|, length 716\n",
      "Record gi|16128768|ref|NP_415321.1|, length 320\n",
      "Record gi|16128769|ref|NP_415322.1|, length 361\n",
      "Record gi|16128770|ref|NP_415323.1|, length 86\n",
      "Record gi|16128771|ref|NP_415324.1|, length 88\n",
      "Record gi|90111170|ref|NP_415325.4|, length 225\n",
      "Record gi|16128773|ref|NP_415326.1|, length 760\n",
      "Record gi|90111171|ref|NP_415327.2|, length 86\n",
      "Record gi|162135897|ref|NP_415328.4|, length 308\n",
      "Record gi|90111173|ref|NP_415329.4|, length 741\n",
      "Record gi|16128777|ref|NP_415330.1|, length 240\n",
      "Record gi|16128778|ref|NP_415331.1|, length 219\n",
      "Record gi|16128779|ref|NP_415332.1|, length 248\n",
      "Record gi|16128780|ref|NP_415333.1|, length 167\n",
      "Record gi|16128781|ref|NP_415334.1|, length 295\n",
      "Record gi|16128782|ref|NP_415335.1|, length 171\n",
      "Record gi|16128783|ref|NP_415336.1|, length 527\n",
      "Record gi|635341463|ref|YP_009029995.1|, length 42\n",
      "Record gi|16128785|ref|NP_415338.1|, length 155\n",
      "Record gi|16128786|ref|NP_415339.1|, length 372\n",
      "Record gi|16128787|ref|NP_415340.1|, length 306\n",
      "Record gi|16128788|ref|NP_415341.1|, length 530\n",
      "Record gi|16128789|ref|NP_415342.1|, length 421\n",
      "Record gi|16128790|ref|NP_415343.1|, length 271\n",
      "Record gi|16128791|ref|NP_415344.1|, length 810\n",
      "Record gi|226524704|ref|NP_415345.2|, length 299\n",
      "Record gi|90111174|ref|NP_415346.4|, length 220\n",
      "Record gi|16128794|ref|NP_415347.1|, length 249\n",
      "Record gi|16128795|ref|NP_415348.1|, length 411\n",
      "Record gi|16128796|ref|NP_415349.1|, length 321\n",
      "Record gi|145698235|ref|NP_415350.2|, length 623\n",
      "Record gi|16128798|ref|NP_415351.1|, length 512\n",
      "Record gi|16128799|ref|NP_415352.1|, length 306\n",
      "Record gi|16128800|ref|NP_415353.1|, length 303\n",
      "Record gi|16128801|ref|NP_415354.1|, length 782\n",
      "Record gi|16128802|ref|NP_415355.1|, length 442\n",
      "Record gi|16128803|ref|NP_415356.1|, length 441\n",
      "Record gi|16128804|ref|NP_415357.1|, length 127\n",
      "Record gi|16128805|ref|NP_415358.1|, length 371\n",
      "Record gi|90111175|ref|NP_415359.4|, length 208\n",
      "Record gi|16128807|ref|NP_415360.1|, length 400\n",
      "Record gi|16128808|ref|NP_415361.1|, length 252\n",
      "Record gi|16128809|ref|NP_415362.1|, length 198\n",
      "Record gi|16128810|ref|NP_415363.1|, length 410\n",
      "Record gi|16128811|ref|NP_415364.1|, length 94\n",
      "Record gi|90111176|ref|NP_415365.4|, length 271\n",
      "Record gi|16128813|ref|NP_415366.1|, length 402\n",
      "Record gi|16128814|ref|NP_415367.1|, length 178\n",
      "Record gi|16128815|ref|NP_415368.1|, length 561\n",
      "Record gi|16128816|ref|NP_415369.1|, length 125\n",
      "Record gi|16128817|ref|NP_415370.1|, length 85\n",
      "Record gi|16128818|ref|NP_415371.1|, length 95\n",
      "Record gi|16128819|ref|NP_415372.1|, length 240\n",
      "Record gi|16128820|ref|NP_415373.1|, length 300\n",
      "Record gi|16128821|ref|NP_415374.1|, length 158\n",
      "Record gi|16128822|ref|NP_415375.1|, length 370\n",
      "Record gi|90111177|ref|NP_415376.4|, length 377\n",
      "Record gi|16128824|ref|NP_415377.1|, length 317\n",
      "Record gi|16128825|ref|NP_415378.1|, length 281\n",
      "Record gi|16128826|ref|NP_415379.1|, length 162\n",
      "Record gi|16128827|ref|NP_415380.1|, length 375\n",
      "Record gi|16128828|ref|NP_415381.1|, length 243\n",
      "Record gi|16128829|ref|NP_415382.1|, length 222\n",
      "Record gi|16128830|ref|NP_415383.1|, length 238\n",
      "Record gi|16128831|ref|NP_415384.1|, length 243\n",
      "Record gi|16128832|ref|NP_415385.1|, length 242\n",
      "Record gi|16128833|ref|NP_415386.1|, length 171\n",
      "Record gi|16128834|ref|NP_415387.1|, length 107\n",
      "Record gi|16128835|ref|NP_415388.1|, length 276\n",
      "Record gi|90111178|ref|NP_415389.2|, length 337\n",
      "Record gi|90111179|ref|NP_415390.4|, length 476\n",
      "Record gi|16128838|ref|NP_415391.1|, length 333\n",
      "Record gi|16128839|ref|NP_415392.1|, length 572\n",
      "Record gi|16128840|ref|NP_415393.1|, length 322\n",
      "Record gi|90111180|ref|NP_415394.4|, length 550\n",
      "Record gi|90111181|ref|NP_415395.2|, length 299\n",
      "Record gi|16128843|ref|NP_415396.1|, length 231\n",
      "Record gi|16128844|ref|NP_415397.1|, length 552\n",
      "Record gi|16128845|ref|NP_415398.1|, length 330\n",
      "Record gi|90111182|ref|NP_415399.4|, length 371\n",
      "Record gi|16128847|ref|NP_415400.1|, length 648\n",
      "Record gi|16128848|ref|NP_415401.1|, length 74\n",
      "Record gi|16128849|ref|NP_415402.1|, length 106\n",
      "Record gi|16128850|ref|NP_415403.1|, length 758\n",
      "Record gi|16128851|ref|NP_415404.1|, length 72\n",
      "Record gi|16128852|ref|NP_415405.1|, length 234\n",
      "Record gi|16128853|ref|NP_415406.1|, length 573\n",
      "Record gi|16128854|ref|NP_415407.1|, length 588\n",
      "Record gi|16128855|ref|NP_415408.1|, length 321\n",
      "Record gi|16128856|ref|NP_415409.1|, length 164\n",
      "Record gi|16128857|ref|NP_415410.1|, length 1329\n",
      "Record gi|145698236|ref|NP_415411.2|, length 203\n",
      "Record gi|16128859|ref|NP_415412.1|, length 447\n",
      "Record gi|16128860|ref|NP_415413.1|, length 430\n",
      "Record gi|90111183|ref|NP_415414.4|, length 814\n",
      "Record gi|16128862|ref|NP_415415.1|, length 205\n",
      "Record gi|16128863|ref|NP_415416.1|, length 287\n",
      "Record gi|16128864|ref|NP_415417.1|, length 208\n",
      "Record gi|16128865|ref|NP_415418.1|, length 382\n",
      "Record gi|90111184|ref|NP_415419.2|, length 476\n",
      "Record gi|16128867|ref|NP_415420.1|, length 302\n",
      "Record gi|16128868|ref|NP_415421.1|, length 196\n",
      "Record gi|16128869|ref|NP_415422.1|, length 246\n",
      "Record gi|16128870|ref|NP_415423.1|, length 760\n",
      "Record gi|16128871|ref|NP_415424.1|, length 285\n",
      "Record gi|90111185|ref|NP_415425.4|, length 586\n",
      "Record gi|16128873|ref|NP_415426.1|, length 230\n",
      "Record gi|16128874|ref|NP_415427.1|, length 362\n",
      "Record gi|16128875|ref|NP_415428.1|, length 427\n",
      "Record gi|90111186|ref|NP_415429.2|, length 254\n",
      "Record gi|16128877|ref|NP_415430.1|, length 227\n",
      "Record gi|16128878|ref|NP_415431.1|, length 557\n",
      "Record gi|16128879|ref|NP_415432.1|, length 94\n",
      "Record gi|90111187|ref|NP_415433.4|, length 754\n",
      "Record gi|16128881|ref|NP_415434.1|, length 582\n",
      "Record gi|16128882|ref|NP_415435.1|, length 328\n",
      "Record gi|16128883|ref|NP_415436.1|, length 410\n",
      "Record gi|16128884|ref|NP_415437.1|, length 60\n",
      "Record gi|16128885|ref|NP_415438.1|, length 248\n",
      "Record gi|16128886|ref|NP_415439.1|, length 297\n",
      "Record gi|16128887|ref|NP_415440.1|, length 259\n",
      "Record gi|16128888|ref|NP_415441.1|, length 261\n",
      "Record gi|16128889|ref|NP_415442.1|, length 440\n",
      "Record gi|226524705|ref|NP_415443.2|, length 234\n",
      "Record gi|16128891|ref|NP_415444.1|, length 1486\n",
      "Record gi|16128892|ref|NP_415445.1|, length 615\n",
      "Record gi|16128893|ref|NP_415446.1|, length 182\n",
      "Record gi|16128894|ref|NP_415447.1|, length 215\n",
      "Record gi|16128895|ref|NP_415448.1|, length 396\n",
      "Record gi|16128896|ref|NP_415449.1|, length 362\n",
      "Record gi|16128897|ref|NP_415450.1|, length 466\n",
      "Record gi|16128898|ref|NP_415451.1|, length 400\n",
      "Record gi|16128899|ref|NP_415452.1|, length 870\n",
      "Record gi|16128900|ref|NP_415453.1|, length 255\n",
      "Record gi|90111188|ref|NP_415454.4|, length 263\n",
      "Record gi|16128902|ref|NP_415455.1|, length 381\n",
      "Record gi|90111189|ref|NP_415456.4|, length 319\n",
      "Record gi|16128904|ref|NP_415457.1|, length 191\n",
      "Record gi|90111190|ref|NP_415458.2|, length 179\n",
      "Record gi|16128906|ref|NP_415459.1|, length 233\n",
      "Record gi|16128907|ref|NP_415460.1|, length 866\n",
      "Record gi|16128908|ref|NP_415461.1|, length 356\n",
      "Record gi|16128909|ref|NP_415462.1|, length 180\n",
      "Record gi|90111191|ref|NP_415463.2|, length 171\n",
      "Record gi|90111192|ref|NP_415464.4|, length 236\n",
      "Record gi|16128912|ref|NP_415465.1|, length 336\n",
      "Record gi|90111193|ref|NP_415466.4|, length 180\n",
      "Record gi|16128914|ref|NP_415467.1|, length 369\n",
      "Record gi|16128915|ref|NP_415468.1|, length 702\n",
      "Record gi|16128916|ref|NP_415469.1|, length 635\n",
      "Record gi|16128917|ref|NP_415470.1|, length 417\n",
      "Record gi|16128918|ref|NP_415471.1|, length 546\n",
      "Record gi|145698237|ref|NP_415472.2|, length 187\n",
      "Record gi|16128920|ref|NP_415473.1|, length 55\n",
      "Record gi|16128921|ref|NP_415474.1|, length 172\n",
      "Record gi|16128922|ref|NP_415475.1|, length 586\n",
      "Record gi|16128923|ref|NP_415476.1|, length 150\n",
      "Record gi|16128924|ref|NP_415477.1|, length 346\n",
      "Record gi|16128925|ref|NP_415478.1|, length 169\n",
      "Record gi|16128926|ref|NP_415479.1|, length 209\n",
      "Record gi|90111194|ref|NP_415480.2|, length 717\n",
      "Record gi|16128928|ref|NP_415481.1|, length 148\n",
      "Record gi|16128929|ref|NP_415482.1|, length 684\n",
      "Record gi|90111195|ref|NP_415483.2|, length 152\n",
      "Record gi|16128931|ref|NP_415484.1|, length 220\n",
      "Record gi|90111196|ref|NP_415485.4|, length 137\n",
      "Record gi|145698238|ref|NP_415486.4|, length 105\n",
      "Record gi|145698239|ref|NP_415487.4|, length 396\n",
      "Record gi|16128935|ref|NP_415488.1|, length 92\n",
      "Record gi|90111197|ref|NP_415489.4|, length 109\n",
      "Record gi|16128937|ref|NP_415490.1|, length 219\n",
      "Record gi|16128938|ref|NP_415491.1|, length 372\n",
      "Record gi|16128939|ref|NP_415492.1|, length 597\n",
      "Record gi|16128940|ref|NP_415493.1|, length 235\n",
      "Record gi|16128941|ref|NP_415494.1|, length 195\n",
      "Record gi|16128942|ref|NP_415495.1|, length 132\n",
      "Record gi|16128943|ref|NP_415496.1|, length 285\n",
      "Record gi|16128944|ref|NP_415497.1|, length 514\n",
      "Record gi|16128945|ref|NP_415498.1|, length 378\n",
      "Record gi|145698240|ref|YP_001165313.1|, length 30\n",
      "Record gi|16128946|ref|NP_415500.1|, length 432\n",
      "Record gi|16128947|ref|NP_415501.1|, length 726\n",
      "Record gi|90111198|ref|NP_415502.4|, length 148\n",
      "Record gi|16128949|ref|NP_415503.1|, length 379\n",
      "Record gi|16128950|ref|NP_415504.1|, length 698\n",
      "Record gi|16128951|ref|NP_415505.1|, length 248\n",
      "Record gi|16128952|ref|NP_415506.1|, length 214\n",
      "Record gi|90111199|ref|NP_415507.2|, length 101\n",
      "Record gi|94541103|ref|YP_588445.1|, length 91\n",
      "Record gi|16128954|ref|NP_415508.1|, length 167\n",
      "Record gi|16128955|ref|NP_415509.1|, length 70\n",
      "Record gi|16128956|ref|NP_415510.1|, length 70\n",
      "Record gi|16128957|ref|NP_415511.1|, length 76\n",
      "Record gi|94541104|ref|YP_588446.1|, length 57\n",
      "Record gi|16128958|ref|NP_415512.1|, length 357\n",
      "Record gi|90111200|ref|NP_415513.2|, length 914\n",
      "Record gi|16128960|ref|NP_415514.1|, length 342\n",
      "Record gi|16128961|ref|NP_415515.1|, length 230\n",
      "Record gi|16128962|ref|NP_415516.1|, length 390\n",
      "Record gi|16128963|ref|NP_415517.1|, length 848\n",
      "Record gi|16128964|ref|NP_415518.1|, length 199\n",
      "Record gi|16128965|ref|NP_415519.1|, length 101\n",
      "Record gi|16128966|ref|NP_415520.1|, length 306\n",
      "Record gi|16128967|ref|NP_415521.1|, length 418\n",
      "Record gi|16128968|ref|NP_415522.1|, length 413\n",
      "Record gi|16128969|ref|NP_415523.1|, length 75\n",
      "Record gi|16128970|ref|NP_415524.1|, length 198\n",
      "Record gi|94541090|ref|NP_415525.2|, length 57\n",
      "Record gi|90111201|ref|NP_415526.4|, length 442\n",
      "Record gi|90111202|ref|NP_415527.4|, length 164\n",
      "Record gi|16128974|ref|NP_415528.1|, length 196\n",
      "Record gi|16128975|ref|NP_415529.1|, length 266\n",
      "Record gi|16128976|ref|NP_415530.1|, length 128\n",
      "Record gi|90111203|ref|NP_415531.2|, length 230\n",
      "Record gi|49176071|ref|NP_415532.3|, length 382\n",
      "Record gi|16128979|ref|NP_415533.1|, length 212\n",
      "Record gi|16128980|ref|NP_415534.1|, length 1320\n",
      "Record gi|16128981|ref|NP_415535.1|, length 502\n",
      "Record gi|16128982|ref|NP_415537.1|, length 375\n",
      "Record gi|16128983|ref|NP_415538.1|, length 423\n",
      "Record gi|16128984|ref|NP_415539.1|, length 354\n",
      "Record gi|16128985|ref|NP_415540.1|, length 137\n",
      "Record gi|16128986|ref|NP_415541.1|, length 441\n",
      "Record gi|16128987|ref|NP_415542.1|, length 672\n",
      "Record gi|16128988|ref|NP_415543.1|, length 807\n",
      "Record gi|16128989|ref|NP_415544.1|, length 452\n",
      "Record gi|16128990|ref|NP_415545.1|, length 288\n",
      "Record gi|226524706|ref|NP_415546.2|, length 99\n",
      "Record gi|16128993|ref|NP_415548.1|, length 328\n",
      "Record gi|90111205|ref|NP_415551.2|, length 312\n",
      "Record gi|16128997|ref|NP_415552.1|, length 245\n",
      "Record gi|16128998|ref|NP_415553.1|, length 184\n",
      "Record gi|90111206|ref|NP_415554.4|, length 163\n",
      "Record gi|16129000|ref|NP_415555.1|, length 277\n",
      "Record gi|16129001|ref|NP_415556.1|, length 138\n",
      "Record gi|16129002|ref|NP_415557.1|, length 129\n",
      "Record gi|16129003|ref|NP_415558.1|, length 216\n",
      "Record gi|16129004|ref|NP_415559.1|, length 151\n",
      "Record gi|16129005|ref|NP_415560.1|, length 151\n",
      "Record gi|16129006|ref|NP_415561.1|, length 110\n",
      "Record gi|16129007|ref|NP_415562.1|, length 103\n",
      "Record gi|16129008|ref|NP_415563.1|, length 177\n",
      "Record gi|90111207|ref|NP_415564.2|, length 473\n",
      "Record gi|16129010|ref|NP_415565.1|, length 385\n",
      "Record gi|16129011|ref|NP_415566.1|, length 511\n",
      "Record gi|16129012|ref|NP_415567.1|, length 847\n",
      "Record gi|16129013|ref|NP_415568.1|, length 75\n",
      "Record gi|90111208|ref|NP_415569.4|, length 124\n",
      "Record gi|16129016|ref|NP_415571.1|, length 408\n",
      "Record gi|16129017|ref|NP_415572.1|, length 306\n",
      "Record gi|16129018|ref|NP_415573.1|, length 350\n",
      "Record gi|16129019|ref|NP_415574.1|, length 191\n",
      "Record gi|16129020|ref|NP_415575.1|, length 188\n",
      "Record gi|16129021|ref|NP_415576.1|, length 46\n",
      "Record gi|16129022|ref|NP_415577.1|, length 372\n",
      "Record gi|49176074|ref|NP_415578.3|, length 84\n",
      "Record gi|16129024|ref|NP_415579.1|, length 81\n",
      "Record gi|16129025|ref|NP_415580.1|, length 348\n",
      "Record gi|16129026|ref|NP_415581.1|, length 186\n",
      "Record gi|16129027|ref|NP_415582.1|, length 215\n",
      "Record gi|90111209|ref|NP_415583.4|, length 402\n",
      "Record gi|16129029|ref|NP_415584.1|, length 194\n",
      "Record gi|16129030|ref|NP_415585.1|, length 215\n",
      "Record gi|16129031|ref|NP_415586.1|, length 307\n",
      "Record gi|16129032|ref|NP_415587.1|, length 511\n",
      "Record gi|16129033|ref|NP_415588.1|, length 138\n",
      "Record gi|16129034|ref|NP_415589.1|, length 97\n",
      "Record gi|16129035|ref|NP_415590.1|, length 219\n",
      "Record gi|16129036|ref|NP_415591.1|, length 138\n",
      "Record gi|16129037|ref|NP_415592.1|, length 134\n",
      "Record gi|16129038|ref|NP_415593.1|, length 231\n",
      "Record gi|16129039|ref|NP_415594.1|, length 402\n",
      "Record gi|16129040|ref|NP_415595.1|, length 251\n",
      "Record gi|16129041|ref|NP_415596.1|, length 260\n",
      "Record gi|16129042|ref|NP_415597.1|, length 232\n",
      "Record gi|49176076|ref|NP_415598.3|, length 365\n",
      "Record gi|16129044|ref|NP_415599.1|, length 313\n",
      "Record gi|16129045|ref|NP_415600.1|, length 547\n",
      "Record gi|16129046|ref|NP_415601.1|, length 317\n",
      "Record gi|16129047|ref|NP_415602.1|, length 1061\n",
      "Record gi|90111210|ref|NP_415603.2|, length 106\n",
      "Record gi|16129049|ref|NP_415604.1|, length 319\n",
      "Record gi|90111211|ref|NP_415605.2|, length 194\n",
      "Record gi|16129051|ref|NP_415606.1|, length 173\n",
      "Record gi|16129052|ref|NP_415607.1|, length 57\n",
      "Record gi|90111212|ref|NP_415608.2|, length 356\n",
      "Record gi|16129054|ref|NP_415609.1|, length 317\n",
      "Record gi|16129055|ref|NP_415610.1|, length 309\n",
      "Record gi|16129056|ref|NP_415611.1|, length 244\n",
      "Record gi|16129057|ref|NP_415612.1|, length 78\n",
      "Record gi|16129058|ref|NP_415613.1|, length 413\n",
      "Record gi|16129059|ref|NP_415614.1|, length 269\n",
      "Record gi|16129060|ref|NP_415615.1|, length 340\n",
      "Record gi|16129061|ref|NP_415616.1|, length 213\n",
      "Record gi|16129062|ref|NP_415617.1|, length 334\n",
      "Record gi|16129063|ref|NP_415618.1|, length 265\n",
      "Record gi|16129064|ref|NP_415619.1|, length 477\n",
      "Record gi|16129065|ref|NP_415620.1|, length 729\n",
      "Record gi|49176077|ref|NP_415621.3|, length 119\n",
      "Record gi|16129067|ref|NP_415622.1|, length 125\n",
      "Record gi|16129068|ref|NP_415623.1|, length 213\n",
      "Record gi|16129069|ref|NP_415624.1|, length 274\n",
      "Record gi|16129070|ref|NP_415625.1|, length 341\n",
      "Record gi|90111213|ref|NP_415626.4|, length 180\n",
      "Record gi|16129072|ref|NP_415627.1|, length 434\n",
      "Record gi|16129073|ref|NP_415628.1|, length 179\n",
      "Record gi|90111214|ref|NP_415629.4|, length 210\n",
      "Record gi|16129075|ref|NP_415630.1|, length 85\n",
      "Record gi|16129076|ref|NP_415631.1|, length 320\n",
      "Record gi|16129077|ref|NP_415632.1|, length 1148\n",
      "Record gi|16129078|ref|NP_415633.1|, length 357\n",
      "Record gi|16129079|ref|NP_415634.1|, length 399\n",
      "Record gi|90111215|ref|NP_415635.4|, length 233\n",
      "Record gi|16129081|ref|NP_415636.1|, length 414\n",
      "Record gi|16129082|ref|NP_415637.1|, length 303\n",
      "Record gi|308209619|ref|NP_415638.2|, length 242\n",
      "Record gi|16129084|ref|NP_415639.1|, length 262\n",
      "Record gi|90111216|ref|NP_415640.2|, length 153\n",
      "Record gi|16129086|ref|NP_415641.1|, length 348\n",
      "Record gi|16129087|ref|NP_415642.1|, length 264\n",
      "Record gi|162135921|ref|NP_415643.3|, length 275\n",
      "Record gi|16129089|ref|NP_415644.1|, length 378\n",
      "Record gi|16129090|ref|NP_415645.1|, length 408\n",
      "Record gi|90111217|ref|NP_415646.4|, length 373\n",
      "Record gi|16129092|ref|NP_415647.1|, length 486\n",
      "Record gi|16129093|ref|NP_415648.1|, length 223\n",
      "Record gi|16129094|ref|NP_415649.1|, length 456\n",
      "Record gi|16129095|ref|NP_415650.1|, length 213\n",
      "Record gi|90111218|ref|NP_415651.4|, length 368\n",
      "Record gi|16129097|ref|NP_415652.1|, length 153\n",
      "Record gi|162135898|ref|NP_415653.4|, length 217\n",
      "Record gi|16129099|ref|NP_415654.1|, length 416\n",
      "Record gi|16129100|ref|NP_415655.1|, length 221\n",
      "Record gi|16129101|ref|NP_415656.1|, length 234\n",
      "Record gi|16129102|ref|NP_415657.1|, length 297\n",
      "Record gi|16129103|ref|NP_415658.1|, length 375\n",
      "Record gi|16129104|ref|NP_415659.1|, length 81\n",
      "Record gi|90111220|ref|NP_415661.2|, length 113\n",
      "Record gi|90111221|ref|NP_415662.2|, length 102\n",
      "Record gi|16129108|ref|NP_415663.1|, length 224\n",
      "Record gi|90111222|ref|NP_415664.2|, length 66\n",
      "Record gi|145698241|ref|NP_415665.2|, length 185\n",
      "Record gi|16129111|ref|NP_415666.1|, length 112\n",
      "Record gi|16129113|ref|NP_415668.1|, length 60\n",
      "Record gi|16129116|ref|NP_415671.1|, length 194\n",
      "Record gi|16129117|ref|NP_415672.1|, length 209\n",
      "Record gi|16129118|ref|NP_415673.1|, length 137\n",
      "Record gi|16129119|ref|NP_415674.1|, length 200\n",
      "Record gi|16129121|ref|NP_415676.1|, length 184\n",
      "Record gi|16129122|ref|NP_415677.1|, length 277\n",
      "Record gi|16129123|ref|NP_415678.1|, length 107\n",
      "Record gi|16129124|ref|NP_415679.1|, length 134\n",
      "Record gi|16129125|ref|NP_415680.1|, length 243\n",
      "Record gi|16129126|ref|NP_415681.1|, length 403\n",
      "Record gi|16129127|ref|NP_415682.1|, length 78\n",
      "Record gi|16129128|ref|NP_415683.1|, length 90\n",
      "Record gi|16129129|ref|NP_415684.1|, length 88\n",
      "Record gi|16129130|ref|NP_415685.1|, length 82\n",
      "Record gi|90111225|ref|NP_415686.2|, length 507\n",
      "Record gi|94541105|ref|YP_588447.1|, length 72\n",
      "Record gi|90111226|ref|NP_415689.2|, length 109\n",
      "Record gi|288551664|ref|NP_415690.5|, length 114\n",
      "Record gi|145698243|ref|YP_001165314.1|, length 57\n",
      "Record gi|145698244|ref|YP_001165315.1|, length 61\n",
      "Record gi|16129137|ref|NP_415692.1|, length 88\n",
      "Record gi|16129138|ref|NP_415693.1|, length 270\n",
      "Record gi|16129139|ref|NP_415694.1|, length 231\n",
      "Record gi|16129140|ref|NP_415695.1|, length 122\n",
      "Record gi|16129141|ref|NP_415696.1|, length 133\n",
      "Record gi|308209620|ref|NP_415697.4|, length 97\n",
      "Record gi|16129143|ref|NP_415698.1|, length 219\n",
      "Record gi|145698245|ref|NP_415699.5|, length 153\n",
      "Record gi|162135899|ref|NP_415700.4|, length 303\n",
      "Record gi|16129146|ref|NP_415701.1|, length 139\n",
      "Record gi|16129147|ref|NP_415702.1|, length 422\n",
      "Record gi|49176085|ref|NP_415703.3|, length 176\n",
      "Record gi|16129149|ref|NP_415704.1|, length 513\n",
      "Record gi|16129150|ref|NP_415705.1|, length 239\n",
      "Record gi|16129151|ref|NP_415706.1|, length 510\n",
      "Record gi|16129152|ref|NP_415707.1|, length 432\n",
      "Record gi|16129153|ref|NP_415708.1|, length 356\n",
      "Record gi|90111230|ref|NP_415709.2|, length 578\n",
      "Record gi|16129155|ref|NP_415710.1|, length 304\n",
      "Record gi|90111231|ref|NP_415711.2|, length 203\n",
      "Record gi|16129157|ref|NP_415712.1|, length 244\n",
      "Record gi|16129158|ref|NP_415713.1|, length 84\n",
      "Record gi|16129159|ref|NP_415714.1|, length 146\n",
      "Record gi|16129160|ref|NP_415715.1|, length 565\n",
      "Record gi|226524708|ref|NP_415716.4|, length 472\n",
      "Record gi|16129162|ref|NP_415717.1|, length 210\n",
      "Record gi|162135900|ref|NP_415718.6|, length 356\n",
      "Record gi|90111234|ref|NP_415719.2|, length 639\n",
      "Record gi|16129165|ref|NP_415720.1|, length 955\n",
      "Record gi|16129166|ref|NP_415721.1|, length 363\n",
      "Record gi|16129167|ref|NP_415722.1|, length 194\n",
      "Record gi|16129168|ref|NP_415723.1|, length 92\n",
      "Record gi|90111235|ref|NP_415724.2|, length 559\n",
      "Record gi|16129170|ref|NP_415725.1|, length 315\n",
      "Record gi|16129171|ref|NP_415726.1|, length 283\n",
      "Record gi|16129172|ref|NP_415727.1|, length 207\n",
      "Record gi|16129173|ref|NP_415728.1|, length 418\n",
      "Record gi|16129174|ref|NP_415729.1|, length 360\n",
      "Record gi|16129175|ref|NP_415730.1|, length 277\n",
      "Record gi|16129176|ref|NP_415731.1|, length 130\n",
      "Record gi|16129177|ref|NP_415732.1|, length 269\n",
      "Record gi|16129178|ref|NP_415733.1|, length 284\n",
      "Record gi|49176087|ref|YP_025297.1|, length 35\n",
      "Record gi|49176088|ref|YP_025298.1|, length 35\n",
      "Record gi|49176089|ref|YP_025299.1|, length 35\n",
      "Record gi|16129179|ref|NP_415734.1|, length 366\n",
      "Record gi|16129180|ref|NP_415735.1|, length 76\n",
      "Record gi|145698247|ref|NP_415736.2|, length 231\n",
      "Record gi|16129182|ref|NP_415737.1|, length 117\n",
      "Record gi|145698248|ref|NP_415738.2|, length 464\n",
      "Record gi|16129184|ref|NP_415739.1|, length 216\n",
      "Record gi|16129185|ref|NP_415740.1|, length 598\n",
      "Record gi|16129186|ref|NP_415741.1|, length 463\n",
      "Record gi|16129187|ref|NP_415742.1|, length 1247\n",
      "Record gi|16129188|ref|NP_415743.1|, length 512\n",
      "Record gi|16129189|ref|NP_415744.1|, length 236\n",
      "Record gi|16129190|ref|NP_415745.1|, length 225\n",
      "Record gi|145698249|ref|NP_415747.2|, length 29\n",
      "Record gi|16129193|ref|NP_415748.1|, length 280\n",
      "Record gi|16129194|ref|NP_415749.1|, length 152\n",
      "Record gi|226524709|ref|NP_415750.2|, length 301\n",
      "Record gi|16129196|ref|NP_415751.1|, length 337\n",
      "Record gi|16129197|ref|NP_415752.1|, length 302\n",
      "Record gi|16129198|ref|NP_415753.1|, length 137\n",
      "Record gi|16129199|ref|NP_415754.1|, length 205\n",
      "Record gi|16129202|ref|NP_415757.1|, length 891\n",
      "Record gi|16129203|ref|NP_415758.1|, length 215\n",
      "Record gi|16129204|ref|NP_415759.1|, length 543\n",
      "Record gi|16129205|ref|NP_415760.1|, length 306\n",
      "Record gi|16129206|ref|NP_415761.1|, length 302\n",
      "Record gi|49176090|ref|YP_025300.1|, length 337\n",
      "Record gi|16129208|ref|NP_415763.1|, length 334\n",
      "Record gi|90111236|ref|NP_415764.4|, length 109\n",
      "Record gi|16129210|ref|NP_415765.1|, length 486\n",
      "Record gi|145698250|ref|YP_001165316.1|, length 57\n",
      "Record gi|16129211|ref|NP_415766.1|, length 417\n",
      "Record gi|90111237|ref|NP_415767.2|, length 98\n",
      "Record gi|16129213|ref|NP_415768.1|, length 239\n",
      "Record gi|16129214|ref|NP_415769.1|, length 132\n",
      "Record gi|16129215|ref|NP_415770.1|, length 179\n",
      "Record gi|16129216|ref|NP_415771.1|, length 247\n",
      "Record gi|16129217|ref|NP_415772.1|, length 212\n",
      "Record gi|16129218|ref|NP_415773.1|, length 168\n",
      "Record gi|16129219|ref|NP_415774.1|, length 166\n",
      "Record gi|90111238|ref|NP_415775.4|, length 59\n",
      "Record gi|16129221|ref|NP_415776.1|, length 268\n",
      "Record gi|16129222|ref|NP_415777.1|, length 397\n",
      "Record gi|635341464|ref|NP_415778.3|, length 453\n",
      "Record gi|16129224|ref|NP_415779.1|, length 531\n",
      "Record gi|16129225|ref|NP_415780.1|, length 520\n",
      "Record gi|16129226|ref|NP_415781.1|, length 14\n",
      "Record gi|16129227|ref|NP_415782.1|, length 293\n",
      "Record gi|162135901|ref|NP_415783.6|, length 206\n",
      "Record gi|16129229|ref|NP_415784.1|, length 631\n",
      "Record gi|16129230|ref|NP_415785.1|, length 291\n",
      "Record gi|16129231|ref|NP_415786.1|, length 196\n",
      "Record gi|16129232|ref|NP_415787.1|, length 252\n",
      "Record gi|16129233|ref|NP_415788.1|, length 349\n",
      "Record gi|16129234|ref|NP_415789.1|, length 83\n",
      "Record gi|16129235|ref|NP_415790.1|, length 865\n",
      "Record gi|16129236|ref|NP_415791.1|, length 324\n",
      "Record gi|635341465|ref|YP_588449.2|, length 54\n",
      "Record gi|145698251|ref|YP_588450.2|, length 55\n",
      "Record gi|16129237|ref|NP_415792.1|, length 891\n",
      "Record gi|16129238|ref|NP_415793.1|, length 196\n",
      "Record gi|16129239|ref|NP_415794.1|, length 254\n",
      "Record gi|16129240|ref|NP_415795.1|, length 102\n",
      "Record gi|16129241|ref|NP_415796.1|, length 389\n",
      "Record gi|16129242|ref|NP_415797.1|, length 245\n",
      "Record gi|90111241|ref|NP_415798.2|, length 108\n",
      "Record gi|16129244|ref|NP_415799.1|, length 72\n",
      "Record gi|16129245|ref|NP_415800.1|, length 249\n",
      "Record gi|145698252|ref|YP_001165317.1|, length 57\n",
      "Record gi|16129246|ref|NP_415801.1|, length 661\n",
      "Record gi|226524710|ref|YP_002791239.1|, length 34\n",
      "Record gi|16129247|ref|NP_415802.1|, length 644\n",
      "Record gi|90111242|ref|NP_415803.2|, length 375\n",
      "Record gi|16129249|ref|NP_415804.1|, length 262\n",
      "Record gi|16129250|ref|NP_415805.1|, length 117\n",
      "Record gi|16129251|ref|NP_415806.1|, length 268\n",
      "Record gi|16129252|ref|NP_415807.1|, length 330\n",
      "Record gi|16129253|ref|NP_415808.1|, length 296\n",
      "Record gi|16129254|ref|NP_415809.1|, length 321\n",
      "Record gi|16129255|ref|NP_415810.1|, length 547\n",
      "Record gi|16129256|ref|NP_415811.1|, length 81\n",
      "Record gi|90111243|ref|NP_415812.4|, length 461\n",
      "Record gi|90111244|ref|NP_415813.4|, length 472\n",
      "Record gi|90111245|ref|NP_415814.4|, length 254\n",
      "Record gi|16129260|ref|NP_415815.1|, length 185\n",
      "Record gi|16129261|ref|NP_415816.1|, length 495\n",
      "Record gi|16129262|ref|NP_415817.1|, length 426\n",
      "Record gi|16129263|ref|NP_415818.1|, length 421\n",
      "Record gi|90111246|ref|NP_415819.4|, length 325\n",
      "Record gi|16129265|ref|NP_415820.1|, length 222\n",
      "Record gi|16129266|ref|NP_415821.1|, length 74\n",
      "Record gi|16129267|ref|NP_415822.1|, length 119\n",
      "Record gi|16129268|ref|NP_415823.1|, length 73\n",
      "Record gi|16129269|ref|NP_415824.1|, length 104\n",
      "Record gi|90111247|ref|NP_415825.4|, length 559\n",
      "Record gi|16129271|ref|NP_415826.1|, length 430\n",
      "Record gi|16129272|ref|NP_415827.1|, length 293\n",
      "Record gi|16129273|ref|NP_415828.1|, length 280\n",
      "Record gi|16129274|ref|NP_415829.1|, length 350\n",
      "Record gi|90111248|ref|NP_415830.2|, length 262\n",
      "Record gi|16129276|ref|NP_415831.1|, length 351\n",
      "Record gi|16129277|ref|NP_415832.1|, length 755\n",
      "Record gi|16129278|ref|NP_415833.1|, length 219\n",
      "Record gi|16129280|ref|NP_415835.1|, length 301\n",
      "Record gi|16129281|ref|NP_415836.1|, length 332\n",
      "Record gi|16129282|ref|NP_415837.1|, length 465\n",
      "Record gi|16129283|ref|NP_415838.1|, length 353\n",
      "Record gi|16129284|ref|NP_415839.1|, length 513\n",
      "Record gi|16129285|ref|NP_415840.1|, length 168\n",
      "Record gi|90111249|ref|NP_415841.4|, length 321\n",
      "Record gi|226524711|ref|NP_415842.2|, length 242\n",
      "Record gi|221800781|ref|NP_415843.2|, length 306\n",
      "Record gi|16129289|ref|NP_415844.1|, length 299\n",
      "Record gi|90111250|ref|NP_415845.2|, length 537\n",
      "Record gi|16129291|ref|NP_415846.1|, length 343\n",
      "Record gi|16129292|ref|NP_415847.1|, length 338\n",
      "Record gi|16129293|ref|NP_415848.1|, length 85\n",
      "Record gi|16129294|ref|NP_415849.1|, length 316\n",
      "Record gi|16129295|ref|NP_415850.1|, length 250\n",
      "Record gi|16129296|ref|NP_415851.1|, length 171\n",
      "Record gi|90111251|ref|NP_415852.2|, length 508\n",
      "Record gi|16129298|ref|NP_415853.1|, length 481\n",
      "Record gi|90111252|ref|NP_415854.4|, length 436\n",
      "Record gi|16129300|ref|NP_415855.1|, length 302\n",
      "Record gi|16129301|ref|NP_415856.1|, length 187\n",
      "Record gi|90111253|ref|NP_415857.2|, length 410\n",
      "Record gi|16129303|ref|NP_415858.1|, length 327\n",
      "Record gi|16129304|ref|NP_415859.1|, length 457\n",
      "Record gi|16129305|ref|NP_415860.1|, length 311\n",
      "Record gi|16129306|ref|NP_415861.1|, length 411\n",
      "Record gi|90111254|ref|NP_415862.4|, length 71\n",
      "Record gi|16129308|ref|NP_415863.1|, length 69\n",
      "Record gi|16129309|ref|NP_415864.1|, length 64\n",
      "Record gi|16129310|ref|NP_415865.1|, length 269\n",
      "Record gi|16129311|ref|NP_415866.1|, length 866\n",
      "Record gi|16129312|ref|NP_415867.1|, length 91\n",
      "Record gi|94541109|ref|YP_588451.1|, length 56\n",
      "Record gi|90111255|ref|NP_415869.2|, length 73\n",
      "Record gi|90111256|ref|NP_415870.4|, length 162\n",
      "Record gi|94541091|ref|NP_415872.2|, length 51\n",
      "Record gi|90111257|ref|NP_415873.2|, length 44\n",
      "Record gi|16129317|ref|NP_415874.1|, length 158\n",
      "Record gi|16129318|ref|NP_415875.1|, length 98\n",
      "Record gi|16129319|ref|NP_415876.1|, length 140\n",
      "Record gi|16129320|ref|NP_415877.1|, length 285\n",
      "Record gi|16129321|ref|NP_415878.1|, length 248\n",
      "Record gi|94541110|ref|YP_588452.1|, length 61\n",
      "Record gi|16129324|ref|NP_415881.1|, length 485\n",
      "Record gi|16129326|ref|NP_415883.1|, length 87\n",
      "Record gi|90111261|ref|NP_415890.2|, length 1120\n",
      "Record gi|16129334|ref|NP_415891.1|, length 191\n",
      "Record gi|16129335|ref|NP_415892.1|, length 196\n",
      "Record gi|345452718|ref|NP_415893.2|, length 77\n",
      "Record gi|90111262|ref|NP_415894.4|, length 144\n",
      "Record gi|16129338|ref|NP_415895.1|, length 377\n",
      "Record gi|16129339|ref|NP_415896.1|, length 1174\n",
      "Record gi|94541111|ref|YP_588453.1|, length 88\n",
      "Record gi|16129340|ref|NP_415897.1|, length 140\n",
      "Record gi|16129341|ref|NP_415898.1|, length 329\n",
      "Record gi|16129342|ref|NP_415899.1|, length 879\n",
      "Record gi|16129343|ref|NP_415900.1|, length 61\n",
      "Record gi|90111263|ref|NP_415901.2|, length 108\n",
      "Record gi|16129345|ref|NP_415902.1|, length 301\n",
      "Record gi|90111264|ref|NP_415903.4|, length 499\n",
      "Record gi|162135920|ref|NP_415904.3|, length 757\n",
      "Record gi|16129348|ref|NP_415905.1|, length 681\n",
      "Record gi|16129349|ref|NP_415906.1|, length 309\n",
      "Record gi|16129350|ref|NP_415907.1|, length 95\n",
      "Record gi|16129351|ref|NP_415908.1|, length 248\n",
      "Record gi|226524712|ref|NP_415909.4|, length 165\n",
      "Record gi|16129353|ref|NP_415910.1|, length 356\n",
      "Record gi|16129354|ref|NP_415911.1|, length 255\n",
      "Record gi|16129355|ref|NP_415912.1|, length 262\n",
      "Record gi|16129356|ref|NP_415913.1|, length 475\n",
      "Record gi|16129357|ref|NP_415914.1|, length 140\n",
      "Record gi|16129358|ref|NP_415915.1|, length 401\n",
      "Record gi|16129359|ref|NP_415916.1|, length 437\n",
      "Record gi|16129360|ref|NP_415917.1|, length 316\n",
      "Record gi|16129361|ref|NP_415918.1|, length 196\n",
      "Record gi|226524713|ref|YP_002791240.1|, length 21\n",
      "Record gi|16129363|ref|NP_415920.1|, length 301\n",
      "Record gi|145698254|ref|NP_415921.2|, length 121\n",
      "Record gi|16129365|ref|NP_415922.1|, length 383\n",
      "Record gi|16129367|ref|NP_415924.1|, length 286\n",
      "Record gi|90111266|ref|NP_415925.2|, length 768\n",
      "Record gi|90111267|ref|NP_415926.4|, length 201\n",
      "Record gi|16129370|ref|NP_415927.1|, length 298\n",
      "Record gi|16129371|ref|NP_415928.1|, length 585\n",
      "Record gi|16129372|ref|NP_415929.1|, length 430\n",
      "Record gi|16129373|ref|NP_415930.1|, length 201\n",
      "Record gi|145698255|ref|NP_415931.4|, length 1300\n",
      "Record gi|16129375|ref|NP_415932.1|, length 266\n",
      "Record gi|16129376|ref|NP_415933.1|, length 479\n",
      "Record gi|90111268|ref|NP_415935.2|, length 176\n",
      "Record gi|16129378|ref|NP_415936.1|, length 57\n",
      "Record gi|49176107|ref|YP_025301.1|, length 49\n",
      "Record gi|16129379|ref|NP_415937.1|, length 55\n",
      "Record gi|16129380|ref|NP_415938.1|, length 546\n",
      "Record gi|90111269|ref|NP_415939.2|, length 307\n",
      "Record gi|16129382|ref|NP_415940.1|, length 447\n",
      "Record gi|145698256|ref|NP_415941.5|, length 551\n",
      "Record gi|90111271|ref|NP_415943.4|, length 74\n",
      "Record gi|16129386|ref|NP_415944.1|, length 179\n",
      "Record gi|16129387|ref|NP_415945.1|, length 326\n",
      "Record gi|16129388|ref|NP_415946.1|, length 330\n",
      "Record gi|16129389|ref|NP_415947.1|, length 197\n",
      "Record gi|16129390|ref|NP_415948.1|, length 222\n",
      "Record gi|288551665|ref|NP_415949.2|, length 382\n",
      "Record gi|90111272|ref|NP_415950.4|, length 391\n",
      "Record gi|16129393|ref|NP_415951.1|, length 178\n",
      "Record gi|90111273|ref|NP_415952.2|, length 653\n",
      "Record gi|16129395|ref|NP_415953.1|, length 76\n",
      "Record gi|94541092|ref|NP_415954.2|, length 58\n",
      "Record gi|145698257|ref|NP_415955.2|, length 138\n",
      "Record gi|16129398|ref|NP_415956.1|, length 468\n",
      "Record gi|16129399|ref|NP_415957.1|, length 381\n",
      "Record gi|16129400|ref|NP_415958.1|, length 337\n",
      "Record gi|16129401|ref|NP_415959.1|, length 313\n",
      "Record gi|16129402|ref|NP_415960.1|, length 264\n",
      "Record gi|16129403|ref|NP_415961.1|, length 474\n",
      "Record gi|145698258|ref|YP_001165318.1|, length 31\n",
      "Record gi|90111274|ref|NP_415962.2|, length 57\n",
      "Record gi|16129405|ref|NP_415963.1|, length 77\n",
      "Record gi|16129406|ref|NP_415964.1|, length 149\n",
      "Record gi|16129407|ref|NP_415965.1|, length 172\n",
      "Record gi|226524714|ref|NP_415966.6|, length 345\n",
      "Record gi|90111276|ref|NP_415967.2|, length 221\n",
      "Record gi|16129410|ref|NP_415968.1|, length 700\n",
      "Record gi|16129411|ref|NP_415969.1|, length 353\n",
      "Record gi|90111277|ref|NP_415970.4|, length 499\n",
      "Record gi|16129413|ref|NP_415971.1|, length 205\n",
      "Record gi|16129414|ref|NP_415972.1|, length 70\n",
      "Record gi|16129416|ref|NP_415974.1|, length 160\n",
      "Record gi|16129419|ref|NP_415977.1|, length 378\n",
      "Record gi|16129420|ref|NP_415978.1|, length 77\n",
      "Record gi|90111278|ref|NP_415979.2|, length 189\n",
      "Record gi|16129422|ref|NP_415980.1|, length 281\n",
      "Record gi|16129423|ref|NP_415981.1|, length 297\n",
      "Record gi|16129424|ref|NP_415982.1|, length 226\n",
      "Record gi|16129425|ref|NP_415983.1|, length 231\n",
      "Record gi|16129426|ref|NP_415984.1|, length 514\n",
      "Record gi|16129427|ref|NP_415985.1|, length 1246\n",
      "Record gi|16129428|ref|NP_415986.1|, length 462\n",
      "Record gi|145698260|ref|NP_415990.5|, length 293\n",
      "Record gi|16129433|ref|NP_415991.1|, length 1015\n",
      "Record gi|16129434|ref|NP_415992.1|, length 294\n",
      "Record gi|16129435|ref|NP_415993.1|, length 217\n",
      "Record gi|145698261|ref|NP_415994.4|, length 94\n",
      "Record gi|90111280|ref|NP_415995.4|, length 336\n",
      "Record gi|90111281|ref|NP_415996.2|, length 565\n",
      "Record gi|16129439|ref|NP_415997.1|, length 45\n",
      "Record gi|90111282|ref|NP_415998.4|, length 71\n",
      "Record gi|16129441|ref|NP_415999.1|, length 143\n",
      "Record gi|16129442|ref|NP_416000.1|, length 308\n",
      "Record gi|16129443|ref|NP_416001.1|, length 328\n",
      "Record gi|16129444|ref|NP_416002.1|, length 298\n",
      "Record gi|16129445|ref|NP_416003.1|, length 340\n",
      "Record gi|16129446|ref|NP_416004.1|, length 516\n",
      "Record gi|16129447|ref|NP_416005.1|, length 193\n",
      "Record gi|162135902|ref|NP_416006.4|, length 799\n",
      "Record gi|145698262|ref|NP_416007.3|, length 460\n",
      "Record gi|16129450|ref|NP_416008.1|, length 439\n",
      "Record gi|16129451|ref|NP_416009.1|, length 511\n",
      "Record gi|16129452|ref|NP_416010.1|, length 466\n",
      "Record gi|16129453|ref|NP_416011.1|, length 931\n",
      "Record gi|16129454|ref|NP_416012.1|, length 790\n",
      "Record gi|16129455|ref|NP_416013.1|, length 561\n",
      "Record gi|90111285|ref|NP_416014.4|, length 385\n",
      "Record gi|90111286|ref|NP_416015.2|, length 560\n",
      "Record gi|16129458|ref|NP_416016.1|, length 253\n",
      "Record gi|16129459|ref|NP_416017.1|, length 65\n",
      "Record gi|16129460|ref|NP_416018.1|, length 759\n",
      "Record gi|16129461|ref|NP_416019.1|, length 304\n",
      "Record gi|16129462|ref|NP_416020.1|, length 167\n",
      "Record gi|16129463|ref|NP_416021.1|, length 176\n",
      "Record gi|16129466|ref|NP_416024.1|, length 440\n",
      "Record gi|16129467|ref|NP_416025.1|, length 88\n",
      "Record gi|49176118|ref|NP_416028.3|, length 530\n",
      "Record gi|16129471|ref|NP_416029.1|, length 317\n",
      "Record gi|16129472|ref|NP_416030.1|, length 511\n",
      "Record gi|16129473|ref|NP_416031.1|, length 342\n",
      "Record gi|16129474|ref|NP_416032.1|, length 330\n",
      "Record gi|16129475|ref|NP_416033.1|, length 340\n",
      "Record gi|16129476|ref|NP_416034.1|, length 291\n",
      "Record gi|16129477|ref|NP_416035.1|, length 96\n",
      "Record gi|16129478|ref|NP_416036.1|, length 252\n",
      "Record gi|90111287|ref|NP_416037.2|, length 304\n",
      "Record gi|49176119|ref|YP_025302.1|, length 483\n",
      "Record gi|16129481|ref|NP_416039.1|, length 315\n",
      "Record gi|16129482|ref|NP_416040.1|, length 119\n",
      "Record gi|16129483|ref|NP_416041.1|, length 308\n",
      "Record gi|90111288|ref|NP_416042.2|, length 462\n",
      "Record gi|16129485|ref|NP_416043.1|, length 293\n",
      "Record gi|16129486|ref|NP_416044.1|, length 371\n",
      "Record gi|16129487|ref|NP_416045.1|, length 396\n",
      "Record gi|16129488|ref|NP_416046.1|, length 221\n",
      "Record gi|90111289|ref|NP_416047.4|, length 144\n",
      "Record gi|90111290|ref|NP_416048.2|, length 127\n",
      "Record gi|16129491|ref|NP_416049.1|, length 72\n",
      "Record gi|90111291|ref|NP_416050.4|, length 299\n",
      "Record gi|16129493|ref|NP_416051.1|, length 395\n",
      "Record gi|145698263|ref|YP_001165319.1|, length 31\n",
      "Record gi|16129494|ref|NP_416052.1|, length 296\n",
      "Record gi|16129495|ref|NP_416054.1|, length 130\n",
      "Record gi|16129496|ref|NP_416055.1|, length 172\n",
      "Record gi|16129497|ref|NP_416056.1|, length 681\n",
      "Record gi|16129498|ref|NP_416057.1|, length 248\n",
      "Record gi|16129499|ref|NP_416058.1|, length 228\n",
      "Record gi|16129500|ref|NP_416059.1|, length 67\n",
      "Record gi|16129501|ref|NP_416060.1|, length 486\n",
      "Record gi|345452719|ref|NP_416062.2|, length 77\n",
      "Record gi|16129504|ref|NP_416063.1|, length 196\n",
      "Record gi|16129505|ref|NP_416064.1|, length 191\n",
      "Record gi|16129506|ref|NP_416065.1|, length 320\n",
      "Record gi|94541112|ref|YP_588454.1|, length 77\n",
      "Record gi|90111292|ref|NP_416067.2|, length 136\n",
      "Record gi|90111293|ref|NP_416068.2|, length 57\n",
      "Record gi|90111294|ref|NP_416069.2|, length 51\n",
      "Record gi|16129511|ref|NP_416070.1|, length 70\n",
      "Record gi|16129512|ref|NP_416071.1|, length 165\n",
      "Record gi|226524715|ref|YP_002791241.1|, length 84\n",
      "Record gi|16129513|ref|NP_416072.1|, length 177\n",
      "Record gi|16129514|ref|NP_416073.1|, length 103\n",
      "Record gi|90111295|ref|NP_416074.4|, length 71\n",
      "Record gi|16129516|ref|NP_416075.1|, length 71\n",
      "Record gi|16129517|ref|NP_416076.1|, length 70\n",
      "Record gi|90111296|ref|NP_416077.4|, length 250\n",
      "Record gi|226524716|ref|NP_416078.4|, length 349\n",
      "Record gi|16129520|ref|NP_416079.1|, length 83\n",
      "Record gi|16129521|ref|NP_416080.1|, length 51\n",
      "Record gi|16129522|ref|NP_416081.1|, length 95\n",
      "Record gi|16129523|ref|NP_416082.1|, length 79\n",
      "Record gi|16129524|ref|NP_416083.1|, length 101\n",
      "Record gi|16129525|ref|NP_416084.1|, length 110\n",
      "Record gi|16129528|ref|NP_416087.1|, length 76\n",
      "Record gi|16129529|ref|NP_416088.1|, length 135\n",
      "Record gi|16129530|ref|NP_416089.1|, length 51\n",
      "Record gi|90111298|ref|NP_416090.2|, length 42\n",
      "Record gi|16129532|ref|NP_416091.1|, length 72\n",
      "Record gi|16129533|ref|NP_416092.1|, length 62\n",
      "Record gi|16129534|ref|NP_416093.1|, length 63\n",
      "Record gi|16129538|ref|NP_416097.1|, length 339\n",
      "Record gi|16129539|ref|NP_416098.1|, length 404\n",
      "Record gi|16129540|ref|NP_416099.1|, length 108\n",
      "Record gi|16129541|ref|NP_416100.1|, length 113\n",
      "Record gi|16129542|ref|NP_416101.1|, length 186\n",
      "Record gi|90111299|ref|NP_416102.2|, length 236\n",
      "Record gi|90111300|ref|NP_416103.2|, length 101\n",
      "Record gi|16129545|ref|NP_416104.1|, length 808\n",
      "Record gi|171701681|ref|NP_416105.4|, length 807\n",
      "Record gi|16129547|ref|NP_416106.1|, length 205\n",
      "Record gi|16129548|ref|NP_416107.1|, length 284\n",
      "Record gi|90111302|ref|NP_416108.2|, length 204\n",
      "Record gi|90111303|ref|NP_416109.2|, length 418\n",
      "Record gi|90111304|ref|NP_416110.4|, length 231\n",
      "Record gi|16129552|ref|NP_416111.1|, length 406\n",
      "Record gi|16129553|ref|NP_416112.1|, length 297\n",
      "Record gi|16129554|ref|NP_416113.1|, length 417\n",
      "Record gi|90111305|ref|NP_416114.2|, length 102\n",
      "Record gi|145698267|ref|YP_001165320.1|, length 27\n",
      "Record gi|16129556|ref|NP_416115.1|, length 273\n",
      "Record gi|16129557|ref|NP_416116.1|, length 109\n",
      "Record gi|16129558|ref|NP_416117.1|, length 121\n",
      "Record gi|16129559|ref|NP_416118.1|, length 344\n",
      "Record gi|16129560|ref|NP_416119.1|, length 462\n",
      "Record gi|16129561|ref|NP_416120.1|, length 510\n",
      "Record gi|16129562|ref|NP_416121.1|, length 314\n",
      "Record gi|16129563|ref|NP_416122.1|, length 460\n",
      "Record gi|16129564|ref|NP_416123.1|, length 240\n",
      "Record gi|16129565|ref|NP_416124.1|, length 111\n",
      "Record gi|145698268|ref|NP_416125.2|, length 239\n",
      "Record gi|16129567|ref|NP_416126.1|, length 433\n",
      "Record gi|16129568|ref|NP_416127.1|, length 309\n",
      "Record gi|16129569|ref|NP_416128.1|, length 467\n",
      "Record gi|16129570|ref|NP_416129.1|, length 548\n",
      "Record gi|49176125|ref|NP_416130.3|, length 391\n",
      "Record gi|16129572|ref|NP_416131.1|, length 502\n",
      "Record gi|90111306|ref|NP_416132.2|, length 421\n",
      "Record gi|16129574|ref|NP_416133.1|, length 457\n",
      "Record gi|16129575|ref|NP_416134.1|, length 603\n",
      "Record gi|16129576|ref|NP_416135.1|, length 196\n",
      "Record gi|16129577|ref|NP_416136.1|, length 255\n",
      "Record gi|16129578|ref|NP_416137.1|, length 342\n",
      "Record gi|16129579|ref|NP_416138.1|, length 530\n",
      "Record gi|16129580|ref|NP_416139.1|, length 390\n",
      "Record gi|16129581|ref|NP_416140.1|, length 333\n",
      "Record gi|90111307|ref|NP_416141.4|, length 346\n",
      "Record gi|49176127|ref|YP_025303.1|, length 41\n",
      "Record gi|16129583|ref|NP_416142.1|, length 71\n",
      "Record gi|90111308|ref|NP_416143.2|, length 146\n",
      "Record gi|16129585|ref|NP_416144.1|, length 193\n",
      "Record gi|16129586|ref|NP_416145.1|, length 192\n",
      "Record gi|16129587|ref|NP_416146.1|, length 740\n",
      "Record gi|16129588|ref|NP_416147.1|, length 352\n",
      "Record gi|16129589|ref|NP_416148.1|, length 206\n",
      "Record gi|16129590|ref|NP_416149.1|, length 231\n",
      "Record gi|16129591|ref|NP_416150.1|, length 211\n",
      "Record gi|16129592|ref|NP_416151.1|, length 500\n",
      "Record gi|16129593|ref|NP_416152.1|, length 201\n",
      "Record gi|16129594|ref|NP_416153.1|, length 287\n",
      "Record gi|16129595|ref|NP_416154.1|, length 424\n",
      "Record gi|16129596|ref|NP_416155.1|, length 218\n",
      "Record gi|90111309|ref|NP_416156.4|, length 109\n",
      "Record gi|16129598|ref|NP_416157.1|, length 369\n",
      "Record gi|49176129|ref|YP_025304.1|, length 155\n",
      "Record gi|90111310|ref|NP_416159.2|, length 144\n",
      "Record gi|16129601|ref|NP_416160.1|, length 78\n",
      "Record gi|90111311|ref|NP_416161.4|, length 285\n",
      "Record gi|16129603|ref|NP_416162.1|, length 670\n",
      "Record gi|16129604|ref|NP_416163.1|, length 173\n",
      "Record gi|49176131|ref|YP_025305.1|, length 298\n",
      "Record gi|90111312|ref|NP_416165.2|, length 79\n",
      "Record gi|145698269|ref|NP_416166.3|, length 199\n",
      "Record gi|16129608|ref|NP_416167.1|, length 365\n",
      "Record gi|16129609|ref|NP_416168.1|, length 135\n",
      "Record gi|16129610|ref|NP_416169.1|, length 215\n",
      "Record gi|16129611|ref|NP_416170.1|, length 1538\n",
      "Record gi|16129612|ref|NP_416171.1|, length 115\n",
      "Record gi|16129613|ref|NP_416172.1|, length 271\n",
      "Record gi|16129614|ref|NP_416173.1|, length 193\n",
      "Record gi|16129615|ref|NP_416174.1|, length 389\n",
      "Record gi|145698270|ref|YP_001165321.1|, length 29\n",
      "Record gi|16129616|ref|NP_416175.1|, length 341\n",
      "Record gi|16129617|ref|NP_416176.1|, length 310\n",
      "Record gi|49176132|ref|YP_025306.1|, length 403\n",
      "Record gi|16129619|ref|NP_416178.1|, length 382\n",
      "Record gi|16129620|ref|NP_416179.1|, length 213\n",
      "Record gi|49176133|ref|YP_025307.1|, length 457\n",
      "Record gi|16129622|ref|NP_416181.1|, length 418\n",
      "Record gi|16129623|ref|NP_416182.1|, length 101\n",
      "Record gi|16129624|ref|NP_416183.1|, length 534\n",
      "Record gi|16129625|ref|NP_416184.1|, length 270\n",
      "Record gi|16129626|ref|NP_416185.1|, length 261\n",
      "Record gi|226524717|ref|NP_416186.4|, length 222\n",
      "Record gi|16129628|ref|NP_416187.1|, length 215\n",
      "Record gi|16129629|ref|NP_416188.1|, length 700\n",
      "Record gi|16129630|ref|NP_416189.1|, length 208\n",
      "Record gi|16129631|ref|NP_416190.1|, length 69\n",
      "Record gi|16129632|ref|NP_416191.1|, length 470\n",
      "Record gi|16129633|ref|NP_416192.1|, length 78\n",
      "Record gi|16129634|ref|NP_416193.1|, length 334\n",
      "Record gi|16129635|ref|NP_416194.1|, length 138\n",
      "Record gi|16129636|ref|NP_416195.1|, length 406\n",
      "Record gi|16129637|ref|NP_416196.1|, length 423\n",
      "Record gi|16129638|ref|NP_416197.1|, length 248\n",
      "Record gi|90111315|ref|NP_416198.2|, length 495\n",
      "Record gi|16129640|ref|NP_416199.1|, length 122\n",
      "Record gi|226524718|ref|NP_416200.2|, length 62\n",
      "Record gi|16129642|ref|NP_416201.1|, length 136\n",
      "Record gi|16129643|ref|NP_416202.1|, length 1018\n",
      "Record gi|16129644|ref|NP_416203.1|, length 370\n",
      "Record gi|90111316|ref|NP_416204.2|, length 118\n",
      "Record gi|16129646|ref|NP_416205.1|, length 404\n",
      "Record gi|226524719|ref|NP_416206.6|, length 421\n",
      "Record gi|16129648|ref|NP_416207.1|, length 288\n",
      "Record gi|16129649|ref|NP_416208.1|, length 252\n",
      "Record gi|16129650|ref|NP_416209.1|, length 531\n",
      "Record gi|90111318|ref|NP_416210.4|, length 383\n",
      "Record gi|16129652|ref|NP_416211.1|, length 303\n",
      "Record gi|16129653|ref|NP_416212.1|, length 254\n",
      "Record gi|16129654|ref|NP_416213.1|, length 312\n",
      "Record gi|16129655|ref|NP_416214.1|, length 429\n",
      "Record gi|16129656|ref|NP_416215.1|, length 97\n",
      "Record gi|145698273|ref|NP_416216.4|, length 548\n",
      "Record gi|16129658|ref|NP_416217.1|, length 792\n",
      "Record gi|16129659|ref|NP_416218.1|, length 277\n",
      "Record gi|16129660|ref|NP_416219.1|, length 348\n",
      "Record gi|16129661|ref|NP_416220.1|, length 63\n",
      "Record gi|16129662|ref|NP_416221.1|, length 478\n",
      "Record gi|16129663|ref|NP_416222.1|, length 237\n",
      "Record gi|16129664|ref|NP_416223.1|, length 154\n",
      "Record gi|16129665|ref|NP_416224.1|, length 249\n",
      "Record gi|16129666|ref|NP_416225.1|, length 183\n",
      "Record gi|16129667|ref|NP_416226.1|, length 326\n",
      "Record gi|16129668|ref|NP_416227.1|, length 99\n",
      "Record gi|16129669|ref|NP_416228.1|, length 795\n",
      "Record gi|16129670|ref|NP_416229.1|, length 327\n",
      "Record gi|16129671|ref|NP_416230.1|, length 14\n",
      "Record gi|16129672|ref|NP_416231.1|, length 118\n",
      "Record gi|49176137|ref|NP_416232.3|, length 65\n",
      "Record gi|16129674|ref|NP_416233.1|, length 180\n",
      "Record gi|16129675|ref|NP_416234.1|, length 642\n",
      "Record gi|145698274|ref|YP_588455.2|, length 35\n",
      "Record gi|16129676|ref|NP_416236.1|, length 252\n",
      "Record gi|49176138|ref|NP_416237.3|, length 309\n",
      "Record gi|16129678|ref|NP_416238.1|, length 96\n",
      "Record gi|16129679|ref|NP_416239.1|, length 286\n",
      "Record gi|16129680|ref|NP_416240.1|, length 178\n",
      "Record gi|16129681|ref|NP_416241.1|, length 222\n",
      "Record gi|226524720|ref|NP_416242.6|, length 196\n",
      "Record gi|16129683|ref|NP_416243.1|, length 463\n",
      "Record gi|226524721|ref|NP_416244.2|, length 267\n",
      "Record gi|145698276|ref|NP_416245.2|, length 80\n",
      "Record gi|49176140|ref|YP_025308.1|, length 753\n",
      "Record gi|16129687|ref|NP_416247.1|, length 249\n",
      "Record gi|16129688|ref|NP_416248.1|, length 450\n",
      "Record gi|16129689|ref|NP_416249.1|, length 280\n",
      "Record gi|16129690|ref|NP_416250.1|, length 116\n",
      "Record gi|16129691|ref|NP_416251.1|, length 452\n",
      "Record gi|16129692|ref|NP_416252.1|, length 106\n",
      "Record gi|16129693|ref|NP_416253.1|, length 112\n",
      "Record gi|16129694|ref|NP_416254.1|, length 275\n",
      "Record gi|16129695|ref|NP_416255.1|, length 295\n",
      "Record gi|90111320|ref|NP_416256.2|, length 191\n",
      "Record gi|16129697|ref|NP_416257.1|, length 161\n",
      "Record gi|16129698|ref|NP_416258.1|, length 322\n",
      "Record gi|16129699|ref|NP_416259.1|, length 447\n",
      "Record gi|16129700|ref|NP_416260.1|, length 492\n",
      "Record gi|16129701|ref|NP_416261.1|, length 344\n",
      "Record gi|16129702|ref|NP_416262.1|, length 406\n",
      "Record gi|16129703|ref|NP_416263.1|, length 268\n",
      "Record gi|90111321|ref|NP_416264.4|, length 236\n",
      "Record gi|90111322|ref|NP_416265.4|, length 225\n",
      "Record gi|16129706|ref|NP_416266.1|, length 235\n",
      "Record gi|16129707|ref|NP_416267.1|, length 182\n",
      "Record gi|90111323|ref|NP_416268.2|, length 388\n",
      "Record gi|145698277|ref|NP_416269.4|, length 511\n",
      "Record gi|16129710|ref|NP_416270.1|, length 217\n",
      "Record gi|90111324|ref|NP_416271.4|, length 435\n",
      "Record gi|90111325|ref|NP_416272.2|, length 206\n",
      "Record gi|16129713|ref|NP_416273.1|, length 135\n",
      "Record gi|16129714|ref|NP_416274.1|, length 90\n",
      "Record gi|16129715|ref|NP_416275.1|, length 447\n",
      "Record gi|90111326|ref|NP_416276.4|, length 346\n",
      "Record gi|16129717|ref|NP_416277.1|, length 653\n",
      "Record gi|16129718|ref|NP_416278.1|, length 347\n",
      "Record gi|16129719|ref|NP_416279.1|, length 183\n",
      "Record gi|16129720|ref|NP_416280.1|, length 618\n",
      "Record gi|16129721|ref|NP_416281.1|, length 338\n",
      "Record gi|90111327|ref|NP_416282.4|, length 213\n",
      "Record gi|16129723|ref|NP_416283.1|, length 452\n",
      "Record gi|16129724|ref|NP_416284.1|, length 252\n",
      "Record gi|16129725|ref|NP_416285.1|, length 326\n",
      "Record gi|90111328|ref|NP_416286.4|, length 315\n",
      "Record gi|16129727|ref|NP_416287.1|, length 278\n",
      "Record gi|16129728|ref|NP_416288.1|, length 347\n",
      "Record gi|16129729|ref|NP_416289.1|, length 459\n",
      "Record gi|16129730|ref|NP_416290.1|, length 358\n",
      "Record gi|90111329|ref|NP_416291.4|, length 90\n",
      "Record gi|16129732|ref|NP_416292.1|, length 137\n",
      "Record gi|16129733|ref|NP_416293.1|, length 331\n",
      "Record gi|90111330|ref|NP_416294.4|, length 294\n",
      "Record gi|16129735|ref|NP_416295.1|, length 284\n",
      "Record gi|16129736|ref|NP_416296.1|, length 248\n",
      "Record gi|16129737|ref|NP_416297.1|, length 644\n",
      "Record gi|16129738|ref|NP_416298.1|, length 427\n",
      "Record gi|16129739|ref|NP_416299.1|, length 491\n",
      "Record gi|90111331|ref|NP_416300.2|, length 496\n",
      "Record gi|16129741|ref|NP_416301.1|, length 167\n",
      "Record gi|90111332|ref|NP_416302.2|, length 34\n",
      "Record gi|16129743|ref|NP_416303.1|, length 148\n",
      "Record gi|16129744|ref|NP_416304.1|, length 273\n",
      "Record gi|16129745|ref|NP_416305.1|, length 393\n",
      "Record gi|90111333|ref|NP_416306.2|, length 115\n",
      "Record gi|16129747|ref|NP_416307.1|, length 84\n",
      "Record gi|90111334|ref|NP_416308.4|, length 341\n",
      "Record gi|226524722|ref|YP_002791242.1|, length 32\n",
      "Record gi|226524723|ref|YP_002791243.1|, length 24\n",
      "Record gi|16129749|ref|NP_416309.1|, length 82\n",
      "Record gi|16129750|ref|NP_416310.1|, length 60\n",
      "Record gi|16129751|ref|NP_416311.1|, length 119\n",
      "Record gi|16129752|ref|NP_416312.1|, length 212\n",
      "Record gi|90111335|ref|NP_416313.4|, length 307\n",
      "Record gi|16129754|ref|NP_416314.1|, length 361\n",
      "Record gi|16129755|ref|NP_416315.1|, length 481\n",
      "Record gi|16129756|ref|NP_416316.1|, length 374\n",
      "Record gi|16129757|ref|NP_416317.1|, length 321\n",
      "Record gi|16129758|ref|NP_416318.1|, length 375\n",
      "Record gi|16129759|ref|NP_416319.1|, length 561\n",
      "Record gi|16129760|ref|NP_416320.1|, length 193\n",
      "Record gi|16129761|ref|NP_416321.1|, length 231\n",
      "Record gi|16129762|ref|NP_416322.1|, length 636\n",
      "Record gi|90111336|ref|NP_416323.4|, length 114\n",
      "Record gi|226524724|ref|NP_416324.2|, length 99\n",
      "Record gi|16129765|ref|NP_416325.1|, length 59\n",
      "Record gi|16129766|ref|NP_416326.1|, length 453\n",
      "Record gi|16129767|ref|NP_416327.1|, length 192\n",
      "Record gi|16129768|ref|NP_416328.1|, length 454\n",
      "Record gi|90111337|ref|NP_416329.4|, length 532\n",
      "Record gi|16129770|ref|NP_416330.1|, length 518\n",
      "Record gi|16129771|ref|NP_416331.1|, length 323\n",
      "Record gi|16129772|ref|NP_416332.1|, length 266\n",
      "Record gi|345452720|ref|NP_416333.4|, length 283\n",
      "Record gi|16129774|ref|NP_416334.1|, length 152\n",
      "Record gi|162135903|ref|NP_416335.4|, length 188\n",
      "Record gi|16129776|ref|NP_416336.1|, length 269\n",
      "Record gi|16129777|ref|NP_416337.1|, length 69\n",
      "Record gi|16129778|ref|NP_416338.1|, length 47\n",
      "Record gi|16129779|ref|NP_416339.1|, length 95\n",
      "Record gi|16129780|ref|NP_416340.1|, length 47\n",
      "Record gi|94541114|ref|YP_588456.1|, length 79\n",
      "Record gi|16129781|ref|NP_416341.1|, length 263\n",
      "Record gi|90111339|ref|NP_416342.4|, length 457\n",
      "Record gi|16129783|ref|NP_416343.1|, length 293\n",
      "Record gi|16129784|ref|NP_416344.1|, length 682\n",
      "Record gi|49176156|ref|YP_026161.1|, length 232\n",
      "Record gi|171701682|ref|NP_416346.4|, length 165\n",
      "Record gi|16129786|ref|NP_416347.1|, length 427\n",
      "Record gi|90111340|ref|NP_416348.2|, length 877\n",
      "Record gi|90111341|ref|NP_416349.2|, length 479\n",
      "Record gi|90111342|ref|NP_416350.4|, length 78\n",
      "Record gi|90111343|ref|NP_416351.2|, length 63\n",
      "Record gi|90111344|ref|NP_416352.4|, length 218\n",
      "Record gi|16129792|ref|NP_416353.1|, length 113\n",
      "Record gi|16129793|ref|NP_416354.1|, length 290\n",
      "Record gi|16129794|ref|NP_416355.1|, length 124\n",
      "Record gi|16129795|ref|NP_416356.1|, length 76\n",
      "Record gi|16129796|ref|NP_416357.1|, length 218\n",
      "Record gi|16129797|ref|NP_416358.1|, length 220\n",
      "Record gi|16129798|ref|NP_416359.1|, length 686\n",
      "Record gi|16129799|ref|NP_416360.1|, length 219\n",
      "Record gi|145698278|ref|NP_416361.2|, length 118\n",
      "Record gi|16129801|ref|NP_416362.1|, length 96\n",
      "Record gi|16129802|ref|NP_416363.1|, length 392\n",
      "Record gi|16129803|ref|NP_416364.1|, length 213\n",
      "Record gi|16129804|ref|NP_416365.1|, length 603\n",
      "Record gi|16129805|ref|NP_416366.1|, length 491\n",
      "Record gi|16129806|ref|NP_416367.1|, length 289\n",
      "Record gi|16129807|ref|NP_416368.1|, length 480\n",
      "Record gi|16129808|ref|NP_416369.1|, length 323\n",
      "Record gi|90111345|ref|NP_416370.2|, length 440\n",
      "Record gi|90111346|ref|NP_416371.4|, length 310\n",
      "Record gi|16129811|ref|NP_416372.1|, length 251\n",
      "Record gi|16129812|ref|NP_416373.1|, length 261\n",
      "Record gi|16129813|ref|NP_416374.1|, length 336\n",
      "Record gi|16129814|ref|NP_416375.1|, length 203\n",
      "Record gi|226524725|ref|YP_002791244.1|, length 21\n",
      "Record gi|90111347|ref|NP_416376.2|, length 200\n",
      "Record gi|16129816|ref|NP_416377.1|, length 173\n",
      "Record gi|16129817|ref|NP_416378.1|, length 246\n",
      "Record gi|16129818|ref|NP_416379.1|, length 150\n",
      "Record gi|16129819|ref|NP_416380.1|, length 590\n",
      "Record gi|90111348|ref|NP_416381.2|, length 188\n",
      "Record gi|16129821|ref|NP_416382.1|, length 272\n",
      "Record gi|90111349|ref|NP_416383.4|, length 131\n",
      "Record gi|16129823|ref|NP_416384.1|, length 247\n",
      "Record gi|16129824|ref|NP_416385.1|, length 323\n",
      "Record gi|90111350|ref|NP_416386.4|, length 809\n",
      "Record gi|16129826|ref|NP_416387.1|, length 366\n",
      "Record gi|90111351|ref|YP_025309.2|, length 248\n",
      "Record gi|90111352|ref|NP_416389.4|, length 188\n",
      "Record gi|16129828|ref|NP_416390.1|, length 577\n",
      "Record gi|90111353|ref|NP_416391.4|, length 162\n",
      "Record gi|16129830|ref|NP_416392.1|, length 130\n",
      "Record gi|16129831|ref|NP_416393.1|, length 692\n",
      "Record gi|16129832|ref|NP_416394.1|, length 382\n",
      "Record gi|16129833|ref|NP_416395.1|, length 214\n",
      "Record gi|16129834|ref|NP_416396.1|, length 129\n",
      "Record gi|16129835|ref|NP_416397.1|, length 349\n",
      "Record gi|16129836|ref|NP_416398.1|, length 286\n",
      "Record gi|16129837|ref|NP_416399.1|, length 533\n",
      "Record gi|16129838|ref|NP_416400.1|, length 553\n",
      "Record gi|16129839|ref|NP_416401.1|, length 167\n",
      "Record gi|16129840|ref|NP_416402.1|, length 654\n",
      "Record gi|16129841|ref|NP_416403.1|, length 308\n",
      "Record gi|16129842|ref|NP_416404.1|, length 295\n",
      "Record gi|16129843|ref|NP_416405.1|, length 192\n",
      "Record gi|145698279|ref|NP_416406.4|, length 116\n",
      "Record gi|16129845|ref|NP_416407.1|, length 167\n",
      "Record gi|16129846|ref|NP_416408.1|, length 91\n",
      "Record gi|16129847|ref|NP_416409.1|, length 142\n",
      "Record gi|16129848|ref|NP_416410.1|, length 474\n",
      "Record gi|16129849|ref|NP_416411.1|, length 266\n",
      "Record gi|145698280|ref|YP_026162.2|, length 328\n",
      "Record gi|16129850|ref|NP_416413.1|, length 504\n",
      "Record gi|16129851|ref|NP_416414.1|, length 329\n",
      "Record gi|16129852|ref|NP_416415.1|, length 167\n",
      "Record gi|94541093|ref|NP_416416.2|, length 83\n",
      "Record gi|226524726|ref|YP_002791245.1|, length 28\n",
      "Record gi|16129854|ref|NP_416417.1|, length 107\n",
      "Record gi|16129855|ref|NP_416418.1|, length 165\n",
      "Record gi|16129856|ref|NP_416419.1|, length 79\n",
      "Record gi|16129857|ref|NP_416420.1|, length 403\n",
      "Record gi|16129858|ref|NP_416421.1|, length 221\n",
      "Record gi|16129859|ref|NP_416422.1|, length 182\n",
      "Record gi|90111354|ref|NP_416423.4|, length 610\n",
      "Record gi|16129861|ref|NP_416424.1|, length 218\n",
      "Record gi|16129862|ref|NP_416425.1|, length 74\n",
      "Record gi|16129863|ref|NP_416426.1|, length 240\n",
      "Record gi|16129864|ref|NP_416427.1|, length 250\n",
      "Record gi|16129865|ref|NP_416428.1|, length 222\n",
      "Record gi|90111355|ref|NP_416429.4|, length 328\n",
      "Record gi|16129867|ref|NP_416430.1|, length 266\n",
      "Record gi|90111356|ref|NP_416431.2|, length 183\n",
      "Record gi|49176170|ref|NP_416432.3|, length 239\n",
      "Record gi|16129870|ref|NP_416433.1|, length 498\n",
      "Record gi|16129871|ref|NP_416434.1|, length 468\n",
      "Record gi|16129872|ref|NP_416435.1|, length 136\n",
      "Record gi|16129873|ref|NP_416436.1|, length 121\n",
      "Record gi|16129874|ref|NP_416437.1|, length 495\n",
      "Record gi|16129875|ref|NP_416438.1|, length 137\n",
      "Record gi|16129876|ref|NP_416439.1|, length 401\n",
      "Record gi|16129877|ref|NP_416440.1|, length 77\n",
      "Record gi|49176171|ref|YP_025310.1|, length 222\n",
      "Record gi|16129879|ref|NP_416442.1|, length 159\n",
      "Record gi|16129884|ref|NP_416447.1|, length 104\n",
      "Record gi|16129885|ref|NP_416448.1|, length 552\n",
      "Record gi|16129886|ref|NP_416449.1|, length 331\n",
      "Record gi|90111357|ref|NP_416450.2|, length 228\n",
      "Record gi|16129888|ref|NP_416451.1|, length 457\n",
      "Record gi|16129889|ref|NP_416452.1|, length 147\n",
      "Record gi|16129890|ref|NP_416453.1|, length 375\n",
      "Record gi|16129891|ref|NP_416454.1|, length 154\n",
      "Record gi|16129892|ref|NP_416455.1|, length 334\n",
      "Record gi|16129893|ref|NP_416456.1|, length 137\n",
      "Record gi|90111358|ref|NP_416457.4|, length 121\n",
      "Record gi|16129895|ref|NP_416458.1|, length 245\n",
      "Record gi|16129896|ref|NP_416459.1|, length 89\n",
      "Record gi|16129897|ref|NP_416460.1|, length 261\n",
      "Record gi|16129898|ref|NP_416461.1|, length 207\n",
      "Record gi|16129899|ref|NP_416462.1|, length 62\n",
      "Record gi|90111359|ref|NP_416463.4|, length 75\n",
      "Record gi|16129901|ref|NP_416464.1|, length 271\n",
      "Record gi|90111360|ref|NP_416465.2|, length 564\n",
      "Record gi|16129903|ref|NP_416466.1|, length 60\n",
      "Record gi|16129904|ref|NP_416467.1|, length 305\n",
      "Record gi|16129905|ref|NP_416468.1|, length 306\n",
      "Record gi|16129906|ref|NP_416469.1|, length 156\n",
      "Record gi|16129907|ref|NP_416470.1|, length 472\n",
      "Record gi|16129908|ref|NP_416471.1|, length 231\n",
      "Record gi|90111361|ref|NP_416472.2|, length 121\n",
      "Record gi|16129913|ref|NP_416476.1|, length 283\n",
      "Record gi|16129914|ref|NP_416477.1|, length 452\n",
      "Record gi|90111362|ref|NP_416478.2|, length 223\n",
      "Record gi|16129916|ref|NP_416479.1|, length 137\n",
      "Record gi|16129917|ref|NP_416480.1|, length 334\n",
      "Record gi|16129918|ref|NP_416481.1|, length 211\n",
      "Record gi|16129919|ref|NP_416482.1|, length 216\n",
      "Record gi|90111363|ref|NP_416483.4|, length 176\n",
      "Record gi|90111364|ref|NP_416484.2|, length 265\n",
      "Record gi|145698281|ref|NP_416485.4|, length 2358\n",
      "Record gi|16129925|ref|NP_416488.1|, length 438\n",
      "Record gi|16129926|ref|NP_416489.1|, length 484\n",
      "Record gi|16129927|ref|NP_416490.1|, length 238\n",
      "Record gi|226524727|ref|NP_416491.2|, length 495\n",
      "Record gi|16129929|ref|NP_416492.1|, length 316\n",
      "Record gi|16129930|ref|NP_416493.1|, length 305\n",
      "Record gi|16129931|ref|NP_416494.1|, length 310\n",
      "Record gi|16129932|ref|NP_416495.1|, length 359\n",
      "Record gi|16129933|ref|NP_416496.1|, length 247\n",
      "Record gi|16129934|ref|NP_416497.1|, length 181\n",
      "Record gi|16129935|ref|NP_416498.1|, length 338\n",
      "Record gi|16129937|ref|NP_416500.1|, length 301\n",
      "Record gi|145698282|ref|YP_026163.2|, length 121\n",
      "Record gi|49176177|ref|YP_026164.1|, length 1039\n",
      "Record gi|90111368|ref|NP_416505.2|, length 510\n",
      "Record gi|16129943|ref|NP_416506.1|, length 148\n",
      "Record gi|16129944|ref|NP_416507.1|, length 73\n",
      "Record gi|16129945|ref|NP_416508.1|, length 122\n",
      "Record gi|16129946|ref|NP_416509.1|, length 124\n",
      "Record gi|145698283|ref|NP_416511.4|, length 109\n",
      "Record gi|16129949|ref|NP_416512.1|, length 352\n",
      "Record gi|16129950|ref|NP_416513.1|, length 157\n",
      "Record gi|162135904|ref|NP_416514.4|, length 388\n",
      "Record gi|16129952|ref|NP_416515.1|, length 475\n",
      "Record gi|16129953|ref|NP_416516.1|, length 75\n",
      "Record gi|16129954|ref|NP_416517.1|, length 352\n",
      "Record gi|90111370|ref|NP_416518.2|, length 452\n",
      "Record gi|226524728|ref|YP_002791246.1|, length 20\n",
      "Record gi|90111371|ref|NP_416519.4|, length 309\n",
      "Record gi|16129957|ref|NP_416520.1|, length 274\n",
      "Record gi|94541116|ref|YP_588458.1|, length 84\n",
      "Record gi|90111372|ref|NP_416521.2|, length 83\n",
      "Record gi|16129959|ref|NP_416522.1|, length 16\n",
      "Record gi|16129960|ref|NP_416523.1|, length 299\n",
      "Record gi|16129961|ref|NP_416524.1|, length 434\n",
      "Record gi|16129962|ref|NP_416525.1|, length 356\n",
      "Record gi|90111373|ref|NP_416526.4|, length 355\n",
      "Record gi|16129964|ref|NP_416527.1|, length 196\n",
      "Record gi|90111374|ref|NP_416528.2|, length 245\n",
      "Record gi|16129966|ref|NP_416529.1|, length 258\n",
      "Record gi|16129967|ref|NP_416530.1|, length 203\n",
      "Record gi|90111375|ref|NP_416531.4|, length 326\n",
      "Record gi|16129969|ref|NP_416532.1|, length 388\n",
      "Record gi|16129970|ref|NP_416533.1|, length 468\n",
      "Record gi|16129971|ref|NP_416535.1|, length 338\n",
      "Record gi|16129972|ref|NP_416536.1|, length 372\n",
      "Record gi|16129973|ref|NP_416537.1|, length 196\n",
      "Record gi|16129974|ref|NP_416538.1|, length 330\n",
      "Record gi|16129975|ref|NP_416539.1|, length 388\n",
      "Record gi|16129976|ref|NP_416540.1|, length 367\n",
      "Record gi|16129977|ref|NP_416541.1|, length 415\n",
      "Record gi|16129978|ref|NP_416542.1|, length 185\n",
      "Record gi|16129979|ref|NP_416543.1|, length 293\n",
      "Record gi|16129980|ref|NP_416544.1|, length 299\n",
      "Record gi|16129981|ref|NP_416545.1|, length 361\n",
      "Record gi|16129982|ref|NP_416546.1|, length 297\n",
      "Record gi|16129983|ref|NP_416547.1|, length 464\n",
      "Record gi|16129984|ref|NP_416548.1|, length 406\n",
      "Record gi|16129985|ref|NP_416549.1|, length 426\n",
      "Record gi|16129986|ref|NP_416550.1|, length 492\n",
      "Record gi|16129987|ref|NP_416551.1|, length 464\n",
      "Record gi|16129988|ref|NP_416552.1|, length 456\n",
      "Record gi|16129989|ref|NP_416553.1|, length 478\n",
      "Record gi|16129990|ref|NP_416554.1|, length 407\n",
      "Record gi|90111376|ref|NP_416555.2|, length 159\n",
      "Record gi|16129992|ref|NP_416556.1|, length 321\n",
      "Record gi|16129993|ref|NP_416557.1|, length 373\n",
      "Record gi|16129994|ref|NP_416558.1|, length 182\n",
      "Record gi|16129995|ref|NP_416559.1|, length 248\n",
      "Record gi|16129996|ref|NP_416560.1|, length 405\n",
      "Record gi|16129997|ref|NP_416561.1|, length 405\n",
      "Record gi|16129998|ref|NP_416562.1|, length 162\n",
      "Record gi|16129999|ref|NP_416563.1|, length 279\n",
      "Record gi|90111377|ref|NP_416564.4|, length 720\n",
      "Record gi|16130001|ref|NP_416565.1|, length 147\n",
      "Record gi|16130002|ref|NP_416566.1|, length 379\n",
      "Record gi|90111378|ref|NP_416567.4|, length 527\n",
      "Record gi|16130004|ref|NP_416568.1|, length 617\n",
      "Record gi|16130005|ref|NP_416569.1|, length 193\n",
      "Record gi|90111379|ref|NP_416570.2|, length 213\n",
      "Record gi|16130007|ref|NP_416571.1|, length 1105\n",
      "Record gi|16130008|ref|NP_416572.1|, length 282\n",
      "Record gi|90111380|ref|NP_416573.4|, length 450\n",
      "Record gi|16130010|ref|NP_416574.1|, length 648\n",
      "Record gi|16130011|ref|NP_416575.1|, length 153\n",
      "Record gi|16130012|ref|NP_416576.1|, length 253\n",
      "Record gi|16130013|ref|NP_416577.1|, length 219\n",
      "Record gi|226524729|ref|YP_002791247.1|, length 19\n",
      "Record gi|226524730|ref|YP_002791248.1|, length 18\n",
      "Record gi|90111381|ref|NP_416578.2|, length 415\n",
      "Record gi|16130015|ref|NP_416579.1|, length 1040\n",
      "Record gi|16130016|ref|NP_416580.1|, length 1025\n",
      "Record gi|16130017|ref|NP_416581.1|, length 471\n",
      "Record gi|16130018|ref|NP_416582.1|, length 467\n",
      "Record gi|16130019|ref|NP_416583.1|, length 240\n",
      "Record gi|90111382|ref|NP_416584.4|, length 110\n",
      "Record gi|16130021|ref|NP_416585.1|, length 453\n",
      "Record gi|16130022|ref|NP_416586.1|, length 72\n",
      "Record gi|90111383|ref|NP_416589.2|, length 105\n",
      "Record gi|16130026|ref|NP_416590.1|, length 299\n",
      "Record gi|226524731|ref|NP_416592.2|, length 99\n",
      "Record gi|16130028|ref|NP_416593.1|, length 288\n",
      "Record gi|16130029|ref|NP_416594.1|, length 346\n",
      "Record gi|16130031|ref|NP_416596.1|, length 94\n",
      "Record gi|16130032|ref|NP_416597.1|, length 150\n",
      "Record gi|16130033|ref|NP_416598.1|, length 420\n",
      "Record gi|90111384|ref|NP_416599.2|, length 284\n",
      "Record gi|90111385|ref|NP_416600.4|, length 350\n",
      "Record gi|16130036|ref|NP_416601.1|, length 425\n",
      "Record gi|16130037|ref|NP_416602.1|, length 334\n",
      "Record gi|16130038|ref|NP_416603.1|, length 321\n",
      "Record gi|16130039|ref|NP_416604.1|, length 248\n",
      "Record gi|90111386|ref|NP_416605.2|, length 272\n",
      "Record gi|16130041|ref|NP_416606.1|, length 266\n",
      "Record gi|16130042|ref|NP_416607.1|, length 262\n",
      "Record gi|16130043|ref|NP_416608.1|, length 90\n",
      "Record gi|16130044|ref|NP_416609.1|, length 274\n",
      "Record gi|90111387|ref|NP_416610.4|, length 112\n",
      "Record gi|16130046|ref|NP_416611.1|, length 344\n",
      "Record gi|16130047|ref|NP_416612.1|, length 826\n",
      "Record gi|16130048|ref|NP_416613.1|, length 239\n",
      "Record gi|16130049|ref|NP_416614.1|, length 180\n",
      "Record gi|16130050|ref|NP_416615.1|, length 93\n",
      "Record gi|90111388|ref|NP_416616.4|, length 369\n",
      "Record gi|16130052|ref|NP_416617.1|, length 677\n",
      "Record gi|16130056|ref|NP_416621.1|, length 1210\n",
      "Record gi|94541136|ref|YP_588459.1|, length 105\n",
      "Record gi|90111389|ref|NP_416623.2|, length 362\n",
      "Record gi|16130058|ref|NP_416624.1|, length 759\n",
      "Record gi|16130059|ref|NP_416625.1|, length 378\n",
      "Record gi|90111391|ref|NP_416627.2|, length 153\n",
      "Record gi|16130062|ref|NP_416628.1|, length 156\n",
      "Record gi|90111392|ref|NP_416629.4|, length 239\n",
      "Record gi|16130064|ref|NP_416630.1|, length 561\n",
      "Record gi|16130065|ref|NP_416631.1|, length 243\n",
      "Record gi|94541117|ref|YP_588460.1|, length 35\n",
      "Record gi|16130066|ref|NP_416632.1|, length 243\n",
      "Record gi|16130067|ref|NP_416633.1|, length 308\n",
      "Record gi|16130068|ref|NP_416634.1|, length 385\n",
      "Record gi|16130069|ref|NP_416635.1|, length 305\n",
      "Record gi|16130070|ref|NP_416636.1|, length 765\n",
      "Record gi|16130071|ref|NP_416637.1|, length 571\n",
      "Record gi|162135905|ref|NP_416638.4|, length 310\n",
      "Record gi|90111394|ref|NP_416639.2|, length 195\n",
      "Record gi|90111395|ref|NP_416640.2|, length 192\n",
      "Record gi|16130075|ref|NP_416641.1|, length 253\n",
      "Record gi|226524732|ref|YP_002791249.1|, length 27\n",
      "Record gi|16130078|ref|NP_416645.1|, length 315\n",
      "Record gi|16130079|ref|NP_416646.1|, length 132\n",
      "Record gi|16130080|ref|NP_416647.1|, length 231\n",
      "Record gi|16130081|ref|NP_416648.1|, length 294\n",
      "Record gi|16130082|ref|NP_416649.1|, length 239\n",
      "Record gi|16130083|ref|NP_416650.1|, length 79\n",
      "Record gi|16130084|ref|NP_416651.1|, length 412\n",
      "Record gi|90111398|ref|NP_416652.4|, length 411\n",
      "Record gi|16130086|ref|NP_416653.1|, length 336\n",
      "Record gi|16130087|ref|NP_416654.1|, length 506\n",
      "Record gi|16130088|ref|NP_416655.1|, length 332\n",
      "Record gi|16130089|ref|NP_416656.1|, length 346\n",
      "Record gi|16130090|ref|NP_416657.1|, length 385\n",
      "Record gi|16130091|ref|NP_416658.1|, length 222\n",
      "Record gi|16130092|ref|NP_416659.1|, length 278\n",
      "Record gi|16130093|ref|NP_416660.1|, length 663\n",
      "Record gi|16130094|ref|NP_416661.1|, length 489\n",
      "Record gi|16130095|ref|NP_416662.1|, length 293\n",
      "Record gi|16130096|ref|NP_416663.1|, length 349\n",
      "Record gi|16130097|ref|NP_416664.1|, length 285\n",
      "Record gi|16130098|ref|NP_416665.1|, length 362\n",
      "Record gi|16130099|ref|NP_416666.1|, length 416\n",
      "Record gi|16130100|ref|NP_416667.1|, length 313\n",
      "Record gi|49176193|ref|NP_416668.3|, length 219\n",
      "Record gi|16130102|ref|NP_416669.1|, length 416\n",
      "Record gi|16130103|ref|NP_416670.1|, length 312\n",
      "Record gi|16130104|ref|NP_416671.1|, length 313\n",
      "Record gi|16130105|ref|NP_416672.1|, length 563\n",
      "Record gi|16130106|ref|NP_416673.1|, length 312\n",
      "Record gi|16130107|ref|NP_416674.1|, length 376\n",
      "Record gi|16130108|ref|NP_416675.1|, length 393\n",
      "Record gi|94541118|ref|YP_588461.1|, length 84\n",
      "Record gi|90111399|ref|NP_416676.4|, length 190\n",
      "Record gi|16130110|ref|NP_416677.1|, length 488\n",
      "Record gi|16130111|ref|NP_416678.1|, length 328\n",
      "Record gi|90111400|ref|NP_416679.4|, length 237\n",
      "Record gi|16130113|ref|NP_416680.1|, length 188\n",
      "Record gi|16130114|ref|NP_416681.1|, length 518\n",
      "Record gi|90111401|ref|NP_416682.4|, length 604\n",
      "Record gi|16130116|ref|NP_416683.1|, length 364\n",
      "Record gi|16130117|ref|NP_416684.1|, length 341\n",
      "Record gi|16130118|ref|NP_416685.1|, length 529\n",
      "Record gi|16130119|ref|NP_416686.1|, length 114\n",
      "Record gi|16130120|ref|NP_416687.1|, length 396\n",
      "Record gi|16130121|ref|NP_416688.1|, length 231\n",
      "Record gi|16130122|ref|NP_416689.1|, length 586\n",
      "Record gi|16130123|ref|NP_416690.1|, length 94\n",
      "Record gi|16130124|ref|NP_416691.1|, length 335\n",
      "Record gi|16130125|ref|NP_416692.1|, length 75\n",
      "Record gi|16130126|ref|NP_416693.1|, length 586\n",
      "Record gi|16130129|ref|NP_416696.1|, length 338\n",
      "Record gi|16130130|ref|NP_416697.1|, length 215\n",
      "Record gi|16130131|ref|NP_416698.1|, length 350\n",
      "Record gi|16130132|ref|NP_416699.1|, length 185\n",
      "Record gi|16130133|ref|NP_416700.1|, length 647\n",
      "Record gi|16130134|ref|NP_416701.1|, length 159\n",
      "Record gi|16130135|ref|NP_416702.1|, length 69\n",
      "Record gi|16130136|ref|NP_416703.1|, length 245\n",
      "Record gi|16130137|ref|NP_416704.1|, length 220\n",
      "Record gi|90111402|ref|NP_416705.2|, length 207\n",
      "Record gi|16130139|ref|NP_416706.1|, length 200\n",
      "Record gi|90111403|ref|NP_416707.4|, length 149\n",
      "Record gi|16130141|ref|NP_416708.1|, length 287\n",
      "Record gi|16130142|ref|NP_416709.1|, length 231\n",
      "Record gi|16130143|ref|NP_416710.1|, length 828\n",
      "Record gi|16130144|ref|NP_416711.1|, length 87\n",
      "Record gi|16130145|ref|NP_416712.1|, length 164\n",
      "Record gi|16130146|ref|NP_416713.1|, length 162\n",
      "Record gi|16130147|ref|NP_416714.1|, length 548\n",
      "Record gi|16130148|ref|NP_416715.1|, length 547\n",
      "Record gi|16130149|ref|NP_416716.1|, length 216\n",
      "Record gi|16130150|ref|NP_416717.1|, length 354\n",
      "Record gi|16130151|ref|NP_416718.1|, length 351\n",
      "Record gi|16130152|ref|NP_416719.1|, length 367\n",
      "Record gi|16130153|ref|NP_416720.1|, length 890\n",
      "Record gi|16130154|ref|NP_416721.1|, length 216\n",
      "Record gi|145698285|ref|NP_416722.2|, length 949\n",
      "Record gi|16130156|ref|NP_416723.1|, length 608\n",
      "Record gi|16130157|ref|NP_416724.1|, length 461\n",
      "Record gi|16130158|ref|NP_416725.1|, length 220\n",
      "Record gi|16130159|ref|NP_416726.1|, length 216\n",
      "Record gi|16130160|ref|NP_416727.1|, length 440\n",
      "Record gi|16130161|ref|NP_416728.1|, length 394\n",
      "Record gi|16130162|ref|NP_416729.1|, length 258\n",
      "Record gi|16130163|ref|NP_416730.1|, length 549\n",
      "Record gi|90111404|ref|NP_416732.4|, length 207\n",
      "Record gi|90111405|ref|NP_416733.4|, length 562\n",
      "Record gi|16130166|ref|NP_416734.1|, length 875\n",
      "Record gi|16130167|ref|NP_416735.1|, length 240\n",
      "Record gi|16130168|ref|NP_416736.1|, length 1250\n",
      "Record gi|16130169|ref|NP_416737.1|, length 761\n",
      "Record gi|16130170|ref|NP_416738.1|, length 376\n",
      "Record gi|16130171|ref|NP_416739.1|, length 84\n",
      "Record gi|16130172|ref|NP_416740.1|, length 216\n",
      "Record gi|16130174|ref|NP_416742.1|, length 358\n",
      "Record gi|16130175|ref|NP_416743.1|, length 452\n",
      "Record gi|16130176|ref|NP_416744.1|, length 542\n",
      "Record gi|16130177|ref|NP_416745.1|, length 419\n",
      "Record gi|16130178|ref|NP_416746.1|, length 396\n",
      "Record gi|16130179|ref|NP_416747.1|, length 299\n",
      "Record gi|145698287|ref|YP_001165324.1|, length 61\n",
      "Record gi|16130180|ref|NP_416748.1|, length 267\n",
      "Record gi|90111406|ref|NP_416749.4|, length 429\n",
      "Record gi|226524733|ref|NP_416750.2|, length 401\n",
      "Record gi|16130183|ref|NP_416751.1|, length 260\n",
      "Record gi|16130184|ref|NP_416752.1|, length 400\n",
      "Record gi|90111407|ref|NP_416753.4|, length 180\n",
      "Record gi|16130186|ref|NP_416754.1|, length 141\n",
      "Record gi|16130187|ref|NP_416755.1|, length 200\n",
      "Record gi|145698288|ref|NP_416756.5|, length 379\n",
      "Record gi|16130189|ref|NP_416757.1|, length 322\n",
      "Record gi|16130190|ref|NP_416758.1|, length 660\n",
      "Record gi|16130191|ref|NP_416759.1|, length 296\n",
      "Record gi|16130192|ref|NP_416760.1|, length 550\n",
      "Record gi|94541119|ref|YP_588462.1|, length 111\n",
      "Record gi|90111409|ref|NP_416761.4|, length 128\n",
      "Record gi|90111410|ref|NP_416762.2|, length 88\n",
      "Record gi|16130195|ref|NP_416763.1|, length 451\n",
      "Record gi|16130196|ref|NP_416764.1|, length 320\n",
      "Record gi|16130197|ref|NP_416765.1|, length 285\n",
      "Record gi|16130198|ref|NP_416766.1|, length 252\n",
      "Record gi|16130199|ref|NP_416767.1|, length 556\n",
      "Record gi|90111411|ref|NP_416768.4|, length 431\n",
      "Record gi|16130201|ref|NP_416769.1|, length 101\n",
      "Record gi|16130202|ref|NP_416770.1|, length 153\n",
      "Record gi|90111412|ref|NP_416771.4|, length 305\n",
      "Record gi|16130204|ref|NP_416772.1|, length 403\n",
      "Record gi|16130205|ref|NP_416773.1|, length 575\n",
      "Record gi|90111413|ref|NP_416774.2|, length 323\n",
      "Record gi|16130207|ref|NP_416775.1|, length 167\n",
      "Record gi|16130208|ref|NP_416776.1|, length 238\n",
      "Record gi|226524734|ref|NP_416777.2|, length 140\n",
      "Record gi|345452721|ref|NP_416778.2|, length 282\n",
      "Record gi|145698289|ref|NP_416779.2|, length 485\n",
      "Record gi|16130212|ref|NP_416780.1|, length 509\n",
      "Record gi|16130213|ref|NP_416781.1|, length 613\n",
      "Record gi|16130214|ref|NP_416782.1|, length 100\n",
      "Record gi|16130215|ref|NP_416783.1|, length 184\n",
      "Record gi|16130216|ref|NP_416784.1|, length 180\n",
      "Record gi|16130217|ref|NP_416785.1|, length 325\n",
      "Record gi|145698290|ref|NP_416786.4|, length 908\n",
      "Record gi|16130219|ref|NP_416787.1|, length 445\n",
      "Record gi|16130220|ref|NP_416788.1|, length 166\n",
      "Record gi|145698291|ref|NP_416789.2|, length 596\n",
      "Record gi|16130222|ref|NP_416790.1|, length 220\n",
      "Record gi|49176207|ref|NP_416791.3|, length 147\n",
      "Record gi|16130224|ref|NP_416792.1|, length 312\n",
      "Record gi|16130225|ref|NP_416793.1|, length 405\n",
      "Record gi|16130226|ref|NP_416794.1|, length 199\n",
      "Record gi|16130227|ref|NP_416795.1|, length 610\n",
      "Record gi|90111414|ref|NP_416796.2|, length 216\n",
      "Record gi|90111415|ref|NP_416797.2|, length 164\n",
      "Record gi|16130230|ref|NP_416798.1|, length 151\n",
      "Record gi|16130231|ref|NP_416799.1|, length 400\n",
      "Record gi|16130232|ref|NP_416800.1|, length 714\n",
      "Record gi|90111416|ref|NP_416801.2|, length 506\n",
      "Record gi|16130234|ref|NP_416802.1|, length 180\n",
      "Record gi|49176208|ref|NP_416803.3|, length 184\n",
      "Record gi|16130236|ref|NP_416804.1|, length 214\n",
      "Record gi|16130237|ref|NP_416805.1|, length 215\n",
      "Record gi|16130238|ref|NP_416806.1|, length 120\n",
      "Record gi|16130239|ref|NP_416807.1|, length 297\n",
      "Record gi|16130240|ref|NP_416808.1|, length 296\n",
      "Record gi|16130241|ref|NP_416809.1|, length 257\n",
      "Record gi|16130242|ref|NP_416810.1|, length 238\n",
      "Record gi|16130243|ref|NP_416811.1|, length 228\n",
      "Record gi|16130244|ref|NP_416812.1|, length 260\n",
      "Record gi|16130245|ref|NP_416813.1|, length 260\n",
      "Record gi|16130246|ref|NP_416814.1|, length 189\n",
      "Record gi|16130247|ref|NP_416815.1|, length 505\n",
      "Record gi|16130248|ref|NP_416816.1|, length 162\n",
      "Record gi|90111417|ref|NP_416817.2|, length 220\n",
      "Record gi|16130250|ref|NP_416818.1|, length 422\n",
      "Record gi|16130251|ref|NP_416819.1|, length 304\n",
      "Record gi|16130252|ref|NP_416820.1|, length 219\n",
      "Record gi|16130253|ref|NP_416821.1|, length 270\n",
      "Record gi|16130254|ref|NP_416822.1|, length 337\n",
      "Record gi|16130255|ref|NP_416823.1|, length 378\n",
      "Record gi|16130256|ref|NP_416824.1|, length 331\n",
      "Record gi|16130257|ref|NP_416825.1|, length 392\n",
      "Record gi|16130258|ref|NP_416826.1|, length 406\n",
      "Record gi|90111418|ref|NP_416827.4|, length 668\n",
      "Record gi|16130260|ref|NP_416828.1|, length 92\n",
      "Record gi|16130261|ref|NP_416829.1|, length 182\n",
      "Record gi|16130262|ref|NP_416830.1|, length 269\n",
      "Record gi|16130263|ref|NP_416831.1|, length 274\n",
      "Record gi|16130264|ref|NP_416832.1|, length 361\n",
      "Record gi|90111419|ref|NP_416833.4|, length 310\n",
      "Record gi|16130266|ref|NP_416834.1|, length 183\n",
      "Record gi|90111420|ref|NP_416835.2|, length 273\n",
      "Record gi|16130268|ref|NP_416836.1|, length 179\n",
      "Record gi|16130269|ref|NP_416837.1|, length 162\n",
      "Record gi|16130270|ref|NP_416838.1|, length 170\n",
      "Record gi|16130271|ref|NP_416839.1|, length 250\n",
      "Record gi|16130272|ref|NP_416841.1|, length 187\n",
      "Record gi|16130273|ref|NP_416842.1|, length 161\n",
      "Record gi|16130274|ref|NP_416843.1|, length 714\n",
      "Record gi|16130275|ref|NP_416844.1|, length 436\n",
      "Record gi|90111421|ref|NP_416845.2|, length 94\n",
      "Record gi|145698292|ref|NP_416846.2|, length 446\n",
      "Record gi|16130278|ref|NP_416847.1|, length 352\n",
      "Record gi|16130279|ref|NP_416848.1|, length 251\n",
      "Record gi|16130280|ref|NP_416849.1|, length 310\n",
      "Record gi|16130281|ref|NP_416850.1|, length 385\n",
      "Record gi|16130282|ref|NP_416851.1|, length 120\n",
      "Record gi|16130283|ref|NP_416852.1|, length 306\n",
      "Record gi|16130284|ref|NP_416853.1|, length 443\n",
      "Record gi|16130286|ref|NP_416855.1|, length 146\n",
      "Record gi|16130289|ref|NP_416858.1|, length 164\n",
      "Record gi|145698293|ref|NP_416860.4|, length 120\n",
      "Record gi|16130292|ref|NP_416861.1|, length 274\n",
      "Record gi|90111424|ref|NP_416862.2|, length 178\n",
      "Record gi|16130294|ref|NP_416863.1|, length 120\n",
      "Record gi|16130295|ref|NP_416864.1|, length 101\n",
      "Record gi|94541120|ref|YP_588463.1|, length 66\n",
      "Record gi|16130296|ref|NP_416865.1|, length 311\n",
      "Record gi|16130297|ref|NP_416866.1|, length 445\n",
      "Record gi|16130298|ref|NP_416867.1|, length 442\n",
      "Record gi|16130299|ref|NP_416868.1|, length 512\n",
      "Record gi|16130300|ref|NP_416869.1|, length 387\n",
      "Record gi|16130301|ref|NP_416870.1|, length 204\n",
      "Record gi|16130302|ref|NP_416871.1|, length 1197\n",
      "Record gi|162135906|ref|NP_416872.4|, length 381\n",
      "Record gi|16130304|ref|NP_416873.1|, length 314\n",
      "Record gi|16130305|ref|NP_416874.1|, length 564\n",
      "Record gi|16130306|ref|NP_416875.1|, length 416\n",
      "Record gi|16130307|ref|NP_416876.1|, length 211\n",
      "Record gi|16130308|ref|NP_416877.1|, length 91\n",
      "Record gi|16130309|ref|NP_416878.1|, length 80\n",
      "Record gi|90111426|ref|NP_416879.4|, length 306\n",
      "Record gi|226524735|ref|YP_002791250.1|, length 23\n",
      "Record gi|16130311|ref|NP_416880.1|, length 412\n",
      "Record gi|16130312|ref|NP_416881.1|, length 565\n",
      "Record gi|16130313|ref|NP_416882.1|, length 244\n",
      "Record gi|16130314|ref|NP_416883.1|, length 285\n",
      "Record gi|16130315|ref|NP_416884.1|, length 831\n",
      "Record gi|16130316|ref|NP_416885.1|, length 345\n",
      "Record gi|16130317|ref|NP_416886.1|, length 361\n",
      "Record gi|16130318|ref|NP_416887.1|, length 415\n",
      "Record gi|16130319|ref|NP_416888.1|, length 108\n",
      "Record gi|16130320|ref|NP_416889.1|, length 321\n",
      "Record gi|16130321|ref|NP_416890.1|, length 418\n",
      "Record gi|16130322|ref|NP_416891.1|, length 108\n",
      "Record gi|16130324|ref|NP_416893.1|, length 412\n",
      "Record gi|16130325|ref|NP_416894.1|, length 400\n",
      "Record gi|90111427|ref|NP_416895.2|, length 370\n",
      "Record gi|90111428|ref|NP_416896.4|, length 729\n",
      "Record gi|90111429|ref|NP_416897.2|, length 114\n",
      "Record gi|226524736|ref|NP_416898.3|, length 130\n",
      "Record gi|16130330|ref|NP_416899.1|, length 471\n",
      "Record gi|16130331|ref|NP_416900.1|, length 294\n",
      "Record gi|16130332|ref|NP_416901.1|, length 418\n",
      "Record gi|16130333|ref|NP_416902.1|, length 277\n",
      "Record gi|16130334|ref|NP_416903.1|, length 254\n",
      "Record gi|16130335|ref|NP_416904.1|, length 308\n",
      "Record gi|16130336|ref|NP_416905.1|, length 332\n",
      "Record gi|94541121|ref|YP_588464.1|, length 72\n",
      "Record gi|16130337|ref|NP_416906.1|, length 671\n",
      "Record gi|16130338|ref|NP_416907.1|, length 328\n",
      "Record gi|16130339|ref|NP_416908.1|, length 253\n",
      "Record gi|16130340|ref|NP_416909.1|, length 323\n",
      "Record gi|16130341|ref|NP_416910.1|, length 85\n",
      "Record gi|16130342|ref|NP_416911.1|, length 575\n",
      "Record gi|16130343|ref|NP_416912.1|, length 169\n",
      "Record gi|16130344|ref|NP_416913.1|, length 283\n",
      "Record gi|16130345|ref|NP_416914.1|, length 124\n",
      "Record gi|16130346|ref|NP_416915.1|, length 244\n",
      "Record gi|16130347|ref|NP_416916.1|, length 303\n",
      "Record gi|16130348|ref|NP_416917.1|, length 365\n",
      "Record gi|90111430|ref|YP_026168.2|, length 291\n",
      "Record gi|16130349|ref|NP_416919.1|, length 277\n",
      "Record gi|16130350|ref|NP_416920.1|, length 338\n",
      "Record gi|90111431|ref|NP_416921.4|, length 263\n",
      "Record gi|16130352|ref|NP_416922.1|, length 285\n",
      "Record gi|16130353|ref|NP_416923.1|, length 298\n",
      "Record gi|16130354|ref|NP_416924.1|, length 474\n",
      "Record gi|90111432|ref|NP_416925.2|, length 434\n",
      "Record gi|90111433|ref|NP_416926.4|, length 299\n",
      "Record gi|16130357|ref|NP_416927.1|, length 191\n",
      "Record gi|90111434|ref|NP_416928.4|, length 149\n",
      "Record gi|90111435|ref|NP_416929.4|, length 141\n",
      "Record gi|16130360|ref|NP_416930.1|, length 289\n",
      "Record gi|16130361|ref|NP_416931.1|, length 299\n",
      "Record gi|16130362|ref|NP_416932.1|, length 350\n",
      "Record gi|90111436|ref|NP_416933.4|, length 166\n",
      "Record gi|16130364|ref|NP_416934.1|, length 219\n",
      "Record gi|16130365|ref|NP_416935.1|, length 295\n",
      "Record gi|90111437|ref|NP_416936.4|, length 453\n",
      "Record gi|145698295|ref|NP_416937.2|, length 402\n",
      "Record gi|16130368|ref|NP_416938.1|, length 213\n",
      "Record gi|16130369|ref|NP_416939.1|, length 81\n",
      "Record gi|145698296|ref|NP_416940.2|, length 122\n",
      "Record gi|16130371|ref|NP_416941.1|, length 138\n",
      "Record gi|16130372|ref|NP_416942.1|, length 197\n",
      "Record gi|145698297|ref|NP_416943.2|, length 125\n",
      "Record gi|16130374|ref|NP_416944.1|, length 130\n",
      "Record gi|145698298|ref|NP_416945.2|, length 269\n",
      "Record gi|16130376|ref|NP_416946.1|, length 467\n",
      "Record gi|16130377|ref|NP_416947.1|, length 408\n",
      "Record gi|90111438|ref|NP_416948.4|, length 395\n",
      "Record gi|16130379|ref|NP_416949.1|, length 278\n",
      "Record gi|16130380|ref|NP_416950.1|, length 467\n",
      "Record gi|16130381|ref|NP_416951.1|, length 95\n",
      "Record gi|90111439|ref|NP_416952.2|, length 97\n",
      "Record gi|16130383|ref|NP_416953.1|, length 338\n",
      "Record gi|16130384|ref|NP_416954.1|, length 267\n",
      "Record gi|16130385|ref|NP_416955.1|, length 233\n",
      "Record gi|16130386|ref|NP_416956.1|, length 159\n",
      "Record gi|90111440|ref|NP_416957.4|, length 111\n",
      "Record gi|16130388|ref|NP_416958.1|, length 759\n",
      "Record gi|16130389|ref|NP_416959.1|, length 316\n",
      "Record gi|16130390|ref|NP_416960.1|, length 667\n",
      "Record gi|16130391|ref|NP_416961.1|, length 347\n",
      "Record gi|16130392|ref|NP_416962.1|, length 191\n",
      "Record gi|16130393|ref|NP_416963.1|, length 659\n",
      "Record gi|16130394|ref|NP_416964.1|, length 566\n",
      "Record gi|16130395|ref|NP_416965.1|, length 1037\n",
      "Record gi|145698299|ref|YP_001165325.1|, length 19\n",
      "Record gi|16130396|ref|NP_416966.1|, length 118\n",
      "Record gi|16130397|ref|NP_416967.1|, length 375\n",
      "Record gi|94541122|ref|YP_588465.1|, length 66\n",
      "Record gi|90111441|ref|NP_416968.2|, length 232\n",
      "Record gi|16130399|ref|NP_416969.1|, length 671\n",
      "Record gi|16130400|ref|NP_416970.1|, length 287\n",
      "Record gi|16130401|ref|NP_416971.1|, length 237\n",
      "Record gi|90111442|ref|NP_416972.4|, length 344\n",
      "Record gi|16130403|ref|NP_416973.1|, length 292\n",
      "Record gi|90111443|ref|NP_416974.4|, length 190\n",
      "Record gi|16130405|ref|NP_416975.1|, length 156\n",
      "Record gi|90111444|ref|NP_416976.4|, length 205\n",
      "Record gi|16130407|ref|NP_416977.1|, length 672\n",
      "Record gi|90111445|ref|NP_416978.4|, length 315\n",
      "Record gi|16130409|ref|NP_416979.1|, length 479\n",
      "Record gi|16130410|ref|NP_416980.1|, length 216\n",
      "Record gi|16130411|ref|NP_416981.1|, length 526\n",
      "Record gi|16130412|ref|NP_416982.1|, length 555\n",
      "Record gi|16130413|ref|NP_416983.1|, length 181\n",
      "Record gi|16130414|ref|NP_416984.1|, length 252\n",
      "Record gi|90111446|ref|NP_416985.4|, length 137\n",
      "Record gi|90111447|ref|NP_416986.4|, length 670\n",
      "Record gi|16130417|ref|NP_416987.1|, length 282\n",
      "Record gi|16130418|ref|NP_416988.1|, length 353\n",
      "Record gi|16130419|ref|NP_416989.1|, length 487\n",
      "Record gi|16130420|ref|NP_416990.1|, length 119\n",
      "Record gi|226524737|ref|NP_416991.2|, length 233\n",
      "Record gi|16130422|ref|NP_416992.1|, length 429\n",
      "Record gi|90111448|ref|NP_416993.2|, length 208\n",
      "Record gi|16130424|ref|NP_416994.1|, length 345\n",
      "Record gi|16130425|ref|NP_416995.1|, length 212\n",
      "Record gi|16130426|ref|NP_416996.1|, length 688\n",
      "Record gi|16130427|ref|NP_416997.1|, length 513\n",
      "Record gi|16130428|ref|NP_416998.1|, length 747\n",
      "Record gi|16130429|ref|NP_416999.1|, length 63\n",
      "Record gi|16130430|ref|NP_417000.1|, length 172\n",
      "Record gi|16130431|ref|NP_417001.1|, length 179\n",
      "Record gi|16130432|ref|NP_417002.1|, length 525\n",
      "Record gi|16130433|ref|NP_417003.1|, length 488\n",
      "Record gi|16130434|ref|NP_417004.1|, length 456\n",
      "Record gi|90111449|ref|NP_417005.2|, length 71\n",
      "Record gi|90111450|ref|NP_417006.2|, length 490\n",
      "Record gi|16130437|ref|NP_417007.1|, length 392\n",
      "Record gi|16130438|ref|NP_417008.1|, length 206\n",
      "Record gi|16130439|ref|NP_417009.1|, length 424\n",
      "Record gi|16130440|ref|NP_417010.1|, length 372\n",
      "Record gi|16130441|ref|NP_417011.1|, length 337\n",
      "Record gi|16130442|ref|NP_417012.1|, length 384\n",
      "Record gi|16130443|ref|NP_417013.1|, length 143\n",
      "Record gi|16130444|ref|NP_417014.1|, length 770\n",
      "Record gi|16130445|ref|NP_417015.1|, length 1653\n",
      "Record gi|90111451|ref|NP_417016.4|, length 281\n",
      "Record gi|90111452|ref|NP_417017.4|, length 258\n",
      "Record gi|90111453|ref|NP_417018.4|, length 427\n",
      "Record gi|16130449|ref|NP_417019.1|, length 66\n",
      "Record gi|16130450|ref|NP_417020.1|, length 111\n",
      "Record gi|16130451|ref|NP_417021.1|, length 616\n",
      "Record gi|16130452|ref|NP_417022.1|, length 171\n",
      "Record gi|16130453|ref|NP_417023.1|, length 107\n",
      "Record gi|16130454|ref|NP_417024.1|, length 128\n",
      "Record gi|49176235|ref|YP_026169.1|, length 404\n",
      "Record gi|16130456|ref|NP_417026.1|, length 162\n",
      "Record gi|16130457|ref|NP_417027.1|, length 246\n",
      "Record gi|16130458|ref|NP_417028.1|, length 267\n",
      "Record gi|226524738|ref|NP_417029.4|, length 284\n",
      "Record gi|90111454|ref|NP_417030.2|, length 426\n",
      "Record gi|16130461|ref|NP_417031.1|, length 379\n",
      "Record gi|16130462|ref|NP_417032.1|, length 296\n",
      "Record gi|556503836|ref|YP_008592385.1|, length 21\n",
      "Record gi|16130463|ref|NP_417033.1|, length 453\n",
      "Record gi|16130464|ref|NP_417034.1|, length 172\n",
      "Record gi|16130465|ref|NP_417035.1|, length 106\n",
      "Record gi|16130466|ref|NP_417036.1|, length 270\n",
      "Record gi|16130467|ref|NP_417037.1|, length 400\n",
      "Record gi|145698300|ref|NP_417038.4|, length 140\n",
      "Record gi|16130469|ref|NP_417039.1|, length 290\n",
      "Record gi|90111455|ref|NP_417040.2|, length 353\n",
      "Record gi|16130471|ref|NP_417041.1|, length 332\n",
      "Record gi|16130472|ref|NP_417042.1|, length 503\n",
      "Record gi|16130473|ref|NP_417043.1|, length 327\n",
      "Record gi|162135907|ref|NP_417044.4|, length 1093\n",
      "Record gi|90111457|ref|NP_417045.4|, length 397\n",
      "Record gi|16130476|ref|NP_417046.1|, length 417\n",
      "Record gi|16130477|ref|NP_417047.1|, length 396\n",
      "Record gi|16130478|ref|NP_417048.1|, length 112\n",
      "Record gi|16130479|ref|NP_417049.1|, length 444\n",
      "Record gi|16130480|ref|NP_417050.1|, length 237\n",
      "Record gi|90111458|ref|NP_417051.2|, length 475\n",
      "Record gi|49176239|ref|YP_026170.1|, length 1295\n",
      "Record gi|171701683|ref|NP_417053.2|, length 518\n",
      "Record gi|145698301|ref|NP_417054.2|, length 167\n",
      "Record gi|90111459|ref|NP_417055.4|, length 211\n",
      "Record gi|90111460|ref|NP_417056.2|, length 282\n",
      "Record gi|16130487|ref|NP_417057.1|, length 86\n",
      "Record gi|226524739|ref|YP_002791251.1|, length 26\n",
      "Record gi|16130488|ref|NP_417058.1|, length 126\n",
      "Record gi|16130489|ref|NP_417059.1|, length 243\n",
      "Record gi|16130490|ref|NP_417060.1|, length 242\n",
      "Record gi|16130491|ref|NP_417061.1|, length 301\n",
      "Record gi|16130492|ref|NP_417062.1|, length 226\n",
      "Record gi|16130493|ref|NP_417063.1|, length 324\n",
      "Record gi|16130494|ref|NP_417064.1|, length 599\n",
      "Record gi|16130495|ref|NP_417065.1|, length 159\n",
      "Record gi|16130496|ref|NP_417066.1|, length 318\n",
      "Record gi|16130497|ref|NP_417067.1|, length 216\n",
      "Record gi|16130498|ref|NP_417068.1|, length 191\n",
      "Record gi|16130499|ref|NP_417069.1|, length 540\n",
      "Record gi|90111461|ref|NP_417070.2|, length 245\n",
      "Record gi|16130501|ref|NP_417071.1|, length 444\n",
      "Record gi|90111462|ref|NP_417072.4|, length 293\n",
      "Record gi|16130503|ref|NP_417073.1|, length 195\n",
      "Record gi|16130504|ref|NP_417074.1|, length 127\n",
      "Record gi|16130505|ref|NP_417075.1|, length 229\n",
      "Record gi|16130506|ref|NP_417076.1|, length 345\n",
      "Record gi|16130507|ref|NP_417077.1|, length 139\n",
      "Record gi|90111463|ref|NP_417078.4|, length 232\n",
      "Record gi|16130509|ref|NP_417079.1|, length 886\n",
      "Record gi|90111464|ref|NP_417080.4|, length 451\n",
      "Record gi|145698302|ref|NP_417081.2|, length 107\n",
      "Record gi|16130512|ref|NP_417082.1|, length 432\n",
      "Record gi|16130513|ref|NP_417083.1|, length 857\n",
      "Record gi|16130514|ref|NP_417084.1|, length 243\n",
      "Record gi|16130515|ref|NP_417085.1|, length 326\n",
      "Record gi|16130516|ref|NP_417086.1|, length 245\n",
      "Record gi|16130518|ref|NP_417088.1|, length 113\n",
      "Record gi|16130519|ref|NP_417089.1|, length 15\n",
      "Record gi|16130520|ref|NP_417090.1|, length 386\n",
      "Record gi|16130521|ref|NP_417091.1|, length 373\n",
      "Record gi|16130522|ref|NP_417092.1|, length 356\n",
      "Record gi|90111465|ref|NP_417093.2|, length 121\n",
      "Record gi|16130524|ref|NP_417094.1|, length 172\n",
      "Record gi|16130525|ref|NP_417095.1|, length 408\n",
      "Record gi|16130526|ref|NP_417096.1|, length 160\n",
      "Record gi|16130527|ref|NP_417097.1|, length 115\n",
      "Record gi|16130528|ref|NP_417098.1|, length 255\n",
      "Record gi|90111466|ref|NP_417099.4|, length 182\n",
      "Record gi|16130530|ref|NP_417100.1|, length 82\n",
      "Record gi|16130531|ref|NP_417101.1|, length 453\n",
      "Record gi|145698303|ref|NP_417102.4|, length 263\n",
      "Record gi|145698304|ref|YP_026171.3|, length 428\n",
      "Record gi|16130533|ref|NP_417104.1|, length 197\n",
      "Record gi|16130534|ref|NP_417105.1|, length 292\n",
      "Record gi|49176247|ref|YP_026172.1|, length 553\n",
      "Record gi|90111468|ref|NP_417107.2|, length 113\n",
      "Record gi|90111469|ref|NP_417108.4|, length 96\n",
      "Record gi|16130538|ref|NP_417109.1|, length 158\n",
      "Record gi|16130539|ref|NP_417110.1|, length 160\n",
      "Record gi|16130540|ref|NP_417111.1|, length 413\n",
      "Record gi|16130541|ref|NP_417112.1|, length 318\n",
      "Record gi|16130542|ref|NP_417113.1|, length 70\n",
      "Record gi|16130543|ref|NP_417114.1|, length 469\n",
      "Record gi|16130544|ref|NP_417115.1|, length 208\n",
      "Record gi|16130545|ref|NP_417116.1|, length 729\n",
      "Record gi|16130546|ref|NP_417117.1|, length 538\n",
      "Record gi|16130547|ref|NP_417118.1|, length 87\n",
      "Record gi|16130548|ref|NP_417119.1|, length 357\n",
      "Record gi|90111470|ref|NP_417120.2|, length 123\n",
      "Record gi|49176249|ref|YP_026173.1|, length 287\n",
      "Record gi|16130551|ref|NP_417122.1|, length 273\n",
      "Record gi|16130552|ref|NP_417123.1|, length 233\n",
      "Record gi|90111471|ref|YP_026174.2|, length 78\n",
      "Record gi|90111472|ref|NP_417124.4|, length 147\n",
      "Record gi|16130554|ref|NP_417125.1|, length 155\n",
      "Record gi|16130557|ref|NP_417129.1|, length 567\n",
      "Record gi|16130558|ref|NP_417130.1|, length 152\n",
      "Record gi|16130559|ref|NP_417131.1|, length 160\n",
      "Record gi|94541123|ref|YP_588466.1|, length 66\n",
      "Record gi|16130560|ref|NP_417132.1|, length 105\n",
      "Record gi|16130561|ref|NP_417133.1|, length 109\n",
      "Record gi|90111474|ref|NP_417134.2|, length 1526\n",
      "Record gi|90111476|ref|NP_417145.4|, length 325\n",
      "Record gi|90111477|ref|NP_417146.2|, length 422\n",
      "Record gi|16130575|ref|NP_417147.1|, length 482\n",
      "Record gi|16130576|ref|NP_417148.1|, length 426\n",
      "Record gi|16130577|ref|NP_417149.1|, length 466\n",
      "Record gi|90111478|ref|NP_417150.4|, length 220\n",
      "Record gi|16130579|ref|NP_417151.1|, length 149\n",
      "Record gi|16130580|ref|NP_417152.1|, length 52\n",
      "Record gi|16130581|ref|NP_417153.1|, length 99\n",
      "Record gi|16130582|ref|NP_417154.1|, length 174\n",
      "Record gi|16130583|ref|NP_417155.1|, length 134\n",
      "Record gi|16130584|ref|NP_417156.1|, length 149\n",
      "Record gi|16130585|ref|NP_417157.1|, length 114\n",
      "Record gi|459201372|ref|NP_417158.4|, length 109\n",
      "Record gi|16130587|ref|NP_417159.1|, length 81\n",
      "Record gi|16130588|ref|NP_417160.1|, length 136\n",
      "Record gi|16130589|ref|NP_417161.1|, length 714\n",
      "Record gi|16130590|ref|NP_417162.1|, length 319\n",
      "Record gi|16130591|ref|NP_417163.1|, length 400\n",
      "Record gi|16130592|ref|NP_417164.1|, length 354\n",
      "Record gi|16130593|ref|NP_417165.1|, length 330\n",
      "Record gi|16130594|ref|NP_417167.1|, length 245\n",
      "Record gi|16130595|ref|NP_417168.1|, length 111\n",
      "Record gi|16130596|ref|NP_417169.1|, length 176\n",
      "Record gi|16130597|ref|NP_417170.1|, length 390\n",
      "Record gi|16130598|ref|NP_417171.1|, length 512\n",
      "Record gi|16130599|ref|NP_417172.1|, length 171\n",
      "Record gi|16130600|ref|NP_417173.1|, length 518\n",
      "Record gi|16130601|ref|NP_417174.1|, length 142\n",
      "Record gi|16130602|ref|NP_417175.1|, length 188\n",
      "Record gi|16130603|ref|NP_417176.1|, length 61\n",
      "Record gi|16130604|ref|NP_417177.1|, length 876\n",
      "Record gi|16130605|ref|NP_417178.1|, length 166\n",
      "Record gi|16130606|ref|NP_417179.1|, length 353\n",
      "Record gi|16130607|ref|NP_417180.1|, length 165\n",
      "Record gi|16130608|ref|NP_417181.1|, length 361\n",
      "Record gi|49176260|ref|YP_026180.1|, length 187\n",
      "Record gi|49176261|ref|YP_026181.1|, length 319\n",
      "Record gi|16130611|ref|NP_417184.1|, length 123\n",
      "Record gi|16130612|ref|NP_417185.1|, length 259\n",
      "Record gi|16130613|ref|NP_417186.1|, length 119\n",
      "Record gi|16130614|ref|NP_417187.1|, length 257\n",
      "Record gi|90111480|ref|NP_417188.4|, length 321\n",
      "Record gi|90111481|ref|NP_417189.2|, length 504\n",
      "Record gi|16130617|ref|NP_417190.1|, length 479\n",
      "Record gi|16130618|ref|NP_417191.1|, length 377\n",
      "Record gi|16130619|ref|NP_417192.1|, length 750\n",
      "Record gi|16130620|ref|NP_417193.1|, length 175\n",
      "Record gi|345452722|ref|NP_417194.2|, length 336\n",
      "Record gi|49176263|ref|YP_026182.1|, length 485\n",
      "Record gi|16130623|ref|NP_417196.1|, length 474\n",
      "Record gi|16130624|ref|NP_417197.1|, length 156\n",
      "Record gi|16130625|ref|NP_417198.1|, length 136\n",
      "Record gi|16130626|ref|NP_417199.1|, length 255\n",
      "Record gi|16130627|ref|NP_417200.1|, length 180\n",
      "Record gi|16130628|ref|NP_417201.1|, length 569\n",
      "Record gi|16130629|ref|NP_417202.1|, length 307\n",
      "Record gi|16130630|ref|NP_417203.1|, length 608\n",
      "Record gi|16130631|ref|NP_417204.1|, length 203\n",
      "Record gi|16130632|ref|NP_417205.1|, length 153\n",
      "Record gi|16130633|ref|NP_417206.1|, length 116\n",
      "Record gi|16130634|ref|NP_417207.1|, length 290\n",
      "Record gi|16130635|ref|NP_417208.1|, length 90\n",
      "Record gi|16130636|ref|NP_417209.1|, length 373\n",
      "Record gi|226524740|ref|NP_417210.2|, length 336\n",
      "Record gi|16130638|ref|NP_417211.1|, length 692\n",
      "Record gi|16130639|ref|NP_417212.1|, length 117\n",
      "Record gi|16130640|ref|NP_417213.1|, length 853\n",
      "Record gi|16130641|ref|NP_417214.1|, length 218\n",
      "Record gi|226524741|ref|NP_417215.2|, length 255\n",
      "Record gi|16130643|ref|NP_417216.1|, length 302\n",
      "Record gi|16130644|ref|NP_417217.1|, length 388\n",
      "Record gi|16130645|ref|NP_417218.1|, length 212\n",
      "Record gi|16130646|ref|NP_417219.1|, length 258\n",
      "Record gi|16130647|ref|NP_417220.1|, length 454\n",
      "Record gi|16130648|ref|NP_417221.1|, length 330\n",
      "Record gi|16130649|ref|NP_417222.1|, length 379\n",
      "Record gi|16130650|ref|NP_417223.1|, length 208\n",
      "Record gi|16130651|ref|NP_417224.1|, length 253\n",
      "Record gi|16130652|ref|NP_417225.1|, length 349\n",
      "Record gi|16130653|ref|NP_417226.1|, length 159\n",
      "Record gi|16130654|ref|NP_417227.1|, length 236\n",
      "Record gi|16130655|ref|NP_417228.1|, length 103\n",
      "Record gi|16130656|ref|NP_417229.1|, length 107\n",
      "Record gi|16130657|ref|NP_417230.1|, length 201\n",
      "Record gi|16130658|ref|NP_417231.1|, length 475\n",
      "Record gi|16130659|ref|NP_417232.1|, length 302\n",
      "Record gi|16130660|ref|NP_417233.1|, length 345\n",
      "Record gi|90111482|ref|NP_417234.2|, length 94\n",
      "Record gi|16130662|ref|NP_417235.1|, length 305\n",
      "Record gi|16130663|ref|NP_417236.1|, length 199\n",
      "Record gi|90111483|ref|NP_417237.2|, length 224\n",
      "Record gi|16130665|ref|NP_417238.1|, length 363\n",
      "Record gi|16130666|ref|NP_417239.1|, length 160\n",
      "Record gi|16130667|ref|NP_417240.1|, length 502\n",
      "Record gi|16130668|ref|NP_417241.1|, length 888\n",
      "Record gi|16130669|ref|NP_417242.1|, length 244\n",
      "Record gi|16130670|ref|NP_417243.1|, length 570\n",
      "Record gi|16130671|ref|NP_417244.1|, length 599\n",
      "Record gi|16130672|ref|NP_417245.1|, length 121\n",
      "Record gi|90111484|ref|NP_417246.4|, length 423\n",
      "Record gi|90111485|ref|NP_417247.2|, length 86\n",
      "Record gi|16130675|ref|NP_417248.1|, length 191\n",
      "Record gi|90111486|ref|NP_417249.4|, length 286\n",
      "Record gi|90111487|ref|NP_417250.4|, length 259\n",
      "Record gi|90111488|ref|NP_417251.2|, length 445\n",
      "Record gi|49176267|ref|YP_026183.1|, length 484\n",
      "Record gi|90111489|ref|NP_417254.4|, length 261\n",
      "Record gi|16130682|ref|NP_417255.1|, length 425\n",
      "Record gi|16130683|ref|NP_417256.1|, length 492\n",
      "Record gi|16130684|ref|NP_417257.1|, length 223\n",
      "Record gi|226524742|ref|YP_002791252.1|, length 46\n",
      "Record gi|90111490|ref|NP_417258.4|, length 290\n",
      "Record gi|16130686|ref|NP_417259.1|, length 432\n",
      "Record gi|16130687|ref|NP_417260.1|, length 545\n",
      "Record gi|16130688|ref|NP_417261.1|, length 263\n",
      "Record gi|16130689|ref|NP_417262.1|, length 111\n",
      "Record gi|16130690|ref|NP_417263.1|, length 82\n",
      "Record gi|16130691|ref|NP_417264.1|, length 744\n",
      "Record gi|16130692|ref|NP_417265.1|, length 433\n",
      "Record gi|16130693|ref|NP_417266.1|, length 918\n",
      "Record gi|16130694|ref|NP_417267.1|, length 446\n",
      "Record gi|16130695|ref|NP_417268.1|, length 446\n",
      "Record gi|16130696|ref|NP_417269.1|, length 450\n",
      "Record gi|16130697|ref|NP_417270.1|, length 149\n",
      "Record gi|16130698|ref|NP_417271.1|, length 260\n",
      "Record gi|16130699|ref|NP_417272.1|, length 109\n",
      "Record gi|16130700|ref|NP_417273.1|, length 181\n",
      "Record gi|16130701|ref|NP_417274.1|, length 282\n",
      "Record gi|16130702|ref|NP_417275.1|, length 454\n",
      "Record gi|16130703|ref|NP_417276.1|, length 429\n",
      "Record gi|16130704|ref|NP_417277.1|, length 455\n",
      "Record gi|226524743|ref|NP_417278.4|, length 251\n",
      "Record gi|345452723|ref|NP_417279.2|, length 382\n",
      "Record gi|16130707|ref|NP_417280.1|, length 215\n",
      "Record gi|16130708|ref|NP_417281.1|, length 438\n",
      "Record gi|16130709|ref|NP_417282.1|, length 591\n",
      "Record gi|16130710|ref|NP_417283.1|, length 482\n",
      "Record gi|16130711|ref|NP_417284.1|, length 140\n",
      "Record gi|16130712|ref|NP_417285.1|, length 243\n",
      "Record gi|16130713|ref|NP_417286.1|, length 366\n",
      "Record gi|16130714|ref|NP_417287.1|, length 131\n",
      "Record gi|16130715|ref|NP_417288.1|, length 305\n",
      "Record gi|90111491|ref|NP_417289.2|, length 75\n",
      "Record gi|16130717|ref|NP_417290.1|, length 401\n",
      "Record gi|16130718|ref|NP_417291.1|, length 147\n",
      "Record gi|16130719|ref|NP_417292.1|, length 268\n",
      "Record gi|16130720|ref|NP_417293.1|, length 365\n",
      "Record gi|90111492|ref|NP_417294.4|, length 417\n",
      "Record gi|16130722|ref|NP_417295.1|, length 443\n",
      "Record gi|16130723|ref|NP_417296.1|, length 608\n",
      "Record gi|16130724|ref|NP_417297.1|, length 1180\n",
      "Record gi|16130725|ref|NP_417298.1|, length 962\n",
      "Record gi|16130726|ref|NP_417299.1|, length 1122\n",
      "Record gi|16130727|ref|NP_417300.1|, length 107\n",
      "Record gi|90111493|ref|NP_417301.4|, length 135\n",
      "Record gi|16130729|ref|NP_417302.1|, length 187\n",
      "Record gi|16130730|ref|NP_417303.1|, length 156\n",
      "Record gi|16130731|ref|NP_417304.1|, length 264\n",
      "Record gi|16130732|ref|NP_417305.1|, length 291\n",
      "Record gi|16130733|ref|NP_417306.1|, length 748\n",
      "Record gi|16130734|ref|NP_417307.1|, length 176\n",
      "Record gi|16130735|ref|NP_417308.1|, length 229\n",
      "Record gi|16130736|ref|NP_417309.1|, length 237\n",
      "Record gi|16130737|ref|NP_417310.1|, length 72\n",
      "Record gi|16130738|ref|NP_417311.1|, length 346\n",
      "Record gi|16130739|ref|NP_417312.1|, length 397\n",
      "Record gi|16130740|ref|NP_417313.1|, length 719\n",
      "Record gi|16130741|ref|NP_417314.1|, length 343\n",
      "Record gi|16130742|ref|NP_417315.1|, length 420\n",
      "Record gi|16130743|ref|NP_417316.1|, length 311\n",
      "Record gi|16130744|ref|NP_417317.1|, length 230\n",
      "Record gi|16130745|ref|NP_417318.1|, length 472\n",
      "Record gi|16130746|ref|NP_417319.1|, length 253\n",
      "Record gi|16130747|ref|NP_417320.1|, length 278\n",
      "Record gi|90111494|ref|NP_417321.2|, length 393\n",
      "Record gi|16130749|ref|NP_417322.1|, length 409\n",
      "Record gi|90111495|ref|NP_417323.2|, length 210\n",
      "Record gi|16130751|ref|NP_417324.1|, length 269\n",
      "Record gi|90111496|ref|NP_417325.4|, length 160\n",
      "Record gi|226524744|ref|YP_002791253.1|, length 26\n",
      "Record gi|90111497|ref|NP_417326.2|, length 141\n",
      "Record gi|16130755|ref|NP_417328.1|, length 163\n",
      "Record gi|16130756|ref|NP_417329.1|, length 458\n",
      "Record gi|16130757|ref|NP_417330.1|, length 72\n",
      "Record gi|16130763|ref|NP_417337.1|, length 301\n",
      "Record gi|145698306|ref|NP_417338.3|, length 121\n",
      "Record gi|90111503|ref|NP_417341.4|, length 251\n",
      "Record gi|16130768|ref|NP_417342.1|, length 752\n",
      "Record gi|16130769|ref|NP_417343.1|, length 292\n",
      "Record gi|16130770|ref|NP_417344.1|, length 159\n",
      "Record gi|16130771|ref|NP_417345.1|, length 592\n",
      "Record gi|49176276|ref|YP_026185.1|, length 396\n",
      "Record gi|16130773|ref|NP_417347.1|, length 398\n",
      "Record gi|16130774|ref|NP_417348.1|, length 403\n",
      "Record gi|90111504|ref|NP_417349.4|, length 461\n",
      "Record gi|16130776|ref|NP_417350.1|, length 310\n",
      "Record gi|16130777|ref|NP_417351.1|, length 541\n",
      "Record gi|90111505|ref|NP_417352.2|, length 256\n",
      "Record gi|16130779|ref|NP_417353.1|, length 192\n",
      "Record gi|16130780|ref|NP_417354.1|, length 1032\n",
      "Record gi|90111506|ref|NP_417355.4|, length 442\n",
      "Record gi|16130782|ref|NP_417356.1|, length 259\n",
      "Record gi|16130783|ref|NP_417357.1|, length 956\n",
      "Record gi|90111507|ref|NP_417358.2|, length 466\n",
      "Record gi|16130785|ref|NP_417359.1|, length 439\n",
      "Record gi|49176279|ref|YP_026186.1|, length 455\n",
      "Record gi|90111508|ref|NP_417362.4|, length 162\n",
      "Record gi|90111509|ref|NP_417363.2|, length 639\n",
      "Record gi|90111510|ref|NP_417364.2|, length 482\n",
      "Record gi|226524745|ref|YP_002791254.1|, length 41\n",
      "Record gi|16130791|ref|NP_417365.1|, length 182\n",
      "Record gi|16130792|ref|NP_417366.1|, length 505\n",
      "Record gi|16130793|ref|NP_417367.1|, length 365\n",
      "Record gi|16130794|ref|NP_417368.1|, length 577\n",
      "Record gi|16130795|ref|NP_417369.1|, length 236\n",
      "Record gi|16130796|ref|NP_417370.1|, length 298\n",
      "Record gi|16130797|ref|NP_417371.1|, length 173\n",
      "Record gi|16130798|ref|NP_417372.1|, length 135\n",
      "Record gi|16130799|ref|NP_417373.1|, length 88\n",
      "Record gi|16130800|ref|NP_417374.1|, length 326\n",
      "Record gi|16130801|ref|NP_417375.1|, length 219\n",
      "Record gi|16130802|ref|NP_417376.1|, length 103\n",
      "Record gi|16130803|ref|NP_417377.1|, length 479\n",
      "Record gi|16130804|ref|NP_417378.1|, length 247\n",
      "Record gi|16130805|ref|NP_417379.1|, length 957\n",
      "Record gi|16130806|ref|NP_417380.1|, length 129\n",
      "Record gi|16130807|ref|NP_417381.1|, length 364\n",
      "Record gi|16130808|ref|NP_417382.1|, length 400\n",
      "Record gi|16130809|ref|NP_417383.1|, length 392\n",
      "Record gi|16130810|ref|NP_417384.1|, length 441\n",
      "Record gi|90111511|ref|NP_417385.4|, length 192\n",
      "Record gi|16130812|ref|NP_417386.1|, length 109\n",
      "Record gi|49176282|ref|NP_417387.3|, length 182\n",
      "Record gi|226524746|ref|YP_002791255.1|, length 19\n",
      "Record gi|16130814|ref|NP_417388.1|, length 410\n",
      "Record gi|16130815|ref|NP_417389.1|, length 219\n",
      "Record gi|16130817|ref|NP_417391.1|, length 297\n",
      "Record gi|16130818|ref|NP_417392.1|, length 714\n",
      "Record gi|16130819|ref|NP_417393.1|, length 331\n",
      "Record gi|90111512|ref|NP_417394.4|, length 261\n",
      "Record gi|16130821|ref|NP_417395.1|, length 492\n",
      "Record gi|90111513|ref|NP_417396.4|, length 298\n",
      "Record gi|16130823|ref|NP_417397.1|, length 246\n",
      "Record gi|16130824|ref|NP_417398.1|, length 211\n",
      "Record gi|16130825|ref|NP_417399.1|, length 286\n",
      "Record gi|16130826|ref|NP_417400.1|, length 359\n",
      "Record gi|16130827|ref|NP_417401.1|, length 387\n",
      "Record gi|16130828|ref|NP_417402.1|, length 339\n",
      "Record gi|16130829|ref|NP_417403.1|, length 237\n",
      "Record gi|16130830|ref|NP_417404.1|, length 169\n",
      "Record gi|16130831|ref|NP_417405.1|, length 321\n",
      "Record gi|49176285|ref|YP_026187.1|, length 425\n",
      "Record gi|16130834|ref|NP_417408.1|, length 462\n",
      "Record gi|16130835|ref|NP_417409.1|, length 147\n",
      "Record gi|49176286|ref|YP_026188.1|, length 663\n",
      "Record gi|90111514|ref|NP_417411.2|, length 252\n",
      "Record gi|16130838|ref|NP_417412.1|, length 306\n",
      "Record gi|16130839|ref|NP_417413.1|, length 658\n",
      "Record gi|145698307|ref|NP_417414.2|, length 43\n",
      "Record gi|16130841|ref|NP_417415.1|, length 71\n",
      "Record gi|16130843|ref|NP_417417.1|, length 384\n",
      "Record gi|16130844|ref|NP_417418.1|, length 464\n",
      "Record gi|16130845|ref|NP_417419.1|, length 165\n",
      "Record gi|16130846|ref|NP_417420.1|, length 235\n",
      "Record gi|90111515|ref|NP_417421.4|, length 243\n",
      "Record gi|16130848|ref|NP_417422.1|, length 316\n",
      "Record gi|90111516|ref|NP_417423.4|, length 187\n",
      "Record gi|16130850|ref|NP_417424.1|, length 138\n",
      "Record gi|90111517|ref|NP_417425.2|, length 326\n",
      "Record gi|16130852|ref|NP_417426.1|, length 234\n",
      "Record gi|16130853|ref|NP_417427.1|, length 188\n",
      "Record gi|90111518|ref|NP_417428.2|, length 96\n",
      "Record gi|16130855|ref|NP_417429.1|, length 197\n",
      "Record gi|16130856|ref|NP_417430.1|, length 378\n",
      "Record gi|16130857|ref|NP_417431.1|, length 335\n",
      "Record gi|16130858|ref|NP_417432.1|, length 348\n",
      "Record gi|16130859|ref|NP_417433.1|, length 239\n",
      "Record gi|90111519|ref|NP_417434.4|, length 108\n",
      "Record gi|16130861|ref|NP_417435.1|, length 239\n",
      "Record gi|16130862|ref|NP_417436.1|, length 350\n",
      "Record gi|16130863|ref|NP_417437.1|, length 91\n",
      "Record gi|90111520|ref|NP_417438.2|, length 359\n",
      "Record gi|90111521|ref|NP_417439.4|, length 418\n",
      "Record gi|90111522|ref|NP_417440.4|, length 711\n",
      "Record gi|16130867|ref|NP_417441.1|, length 235\n",
      "Record gi|16130868|ref|NP_417442.1|, length 178\n",
      "Record gi|16130871|ref|NP_417445.1|, length 136\n",
      "Record gi|90111523|ref|NP_417446.4|, length 269\n",
      "Record gi|49176293|ref|YP_026189.1|, length 1520\n",
      "Record gi|16130875|ref|NP_417449.1|, length 560\n",
      "Record gi|16130876|ref|NP_417450.1|, length 723\n",
      "Record gi|16130877|ref|NP_417451.1|, length 134\n",
      "Record gi|49176294|ref|YP_026190.1|, length 407\n",
      "Record gi|49176295|ref|YP_026191.1|, length 350\n",
      "Record gi|16130879|ref|NP_417453.1|, length 499\n",
      "Record gi|16130880|ref|NP_417454.1|, length 254\n",
      "Record gi|16130882|ref|NP_417456.1|, length 338\n",
      "Record gi|49176296|ref|YP_026192.1|, length 355\n",
      "Record gi|16130884|ref|NP_417458.1|, length 252\n",
      "Record gi|16130885|ref|NP_417459.1|, length 237\n",
      "Record gi|16130886|ref|NP_417460.1|, length 230\n",
      "Record gi|16130887|ref|NP_417461.1|, length 499\n",
      "Record gi|16130888|ref|NP_417462.1|, length 619\n",
      "Record gi|90111524|ref|NP_417463.4|, length 288\n",
      "Record gi|16130890|ref|NP_417464.1|, length 82\n",
      "Record gi|16130891|ref|NP_417465.1|, length 113\n",
      "Record gi|16130892|ref|NP_417466.1|, length 162\n",
      "Record gi|16130893|ref|NP_417467.1|, length 164\n",
      "Record gi|16130894|ref|NP_417468.1|, length 567\n",
      "Record gi|16130895|ref|NP_417469.1|, length 392\n",
      "Record gi|16130896|ref|NP_417470.1|, length 328\n",
      "Record gi|16130897|ref|NP_417471.1|, length 372\n",
      "Record gi|16130898|ref|NP_417472.1|, length 95\n",
      "Record gi|16130899|ref|NP_417474.1|, length 346\n",
      "Record gi|16130900|ref|NP_417475.1|, length 164\n",
      "Record gi|16130901|ref|NP_417476.1|, length 294\n",
      "Record gi|16130903|ref|NP_417478.1|, length 141\n",
      "Record gi|16130904|ref|NP_417479.1|, length 244\n",
      "Record gi|16130906|ref|NP_417481.1|, length 395\n",
      "Record gi|16130907|ref|NP_417482.1|, length 219\n",
      "Record gi|90111526|ref|NP_417483.2|, length 318\n",
      "Record gi|16130909|ref|NP_417484.1|, length 387\n",
      "Record gi|90111527|ref|NP_417485.4|, length 275\n",
      "Record gi|49176301|ref|YP_026195.1|, length 308\n",
      "Record gi|16130912|ref|NP_417487.1|, length 85\n",
      "Record gi|49176302|ref|YP_026196.1|, length 739\n",
      "Record gi|16130913|ref|NP_417489.1|, length 470\n",
      "Record gi|16130914|ref|NP_417490.1|, length 245\n",
      "Record gi|16130915|ref|NP_417491.1|, length 752\n",
      "Record gi|16130916|ref|NP_417492.1|, length 535\n",
      "Record gi|16130917|ref|NP_417493.1|, length 131\n",
      "Record gi|16130918|ref|NP_417494.1|, length 98\n",
      "Record gi|49176303|ref|YP_026197.1|, length 160\n",
      "Record gi|16130920|ref|NP_417496.1|, length 130\n",
      "Record gi|16130921|ref|NP_417497.1|, length 219\n",
      "Record gi|16130922|ref|NP_417498.1|, length 449\n",
      "Record gi|16130923|ref|NP_417499.1|, length 110\n",
      "Record gi|16130924|ref|NP_417500.1|, length 193\n",
      "Record gi|16130925|ref|NP_417501.1|, length 104\n",
      "Record gi|16130926|ref|NP_417502.1|, length 630\n",
      "Record gi|16130927|ref|NP_417503.1|, length 193\n",
      "Record gi|16130928|ref|NP_417504.1|, length 275\n",
      "Record gi|16130929|ref|NP_417505.1|, length 140\n",
      "Record gi|16130930|ref|NP_417506.1|, length 209\n",
      "Record gi|90111528|ref|NP_417507.2|, length 493\n",
      "Record gi|145698308|ref|NP_417509.3|, length 223\n",
      "Record gi|16130934|ref|NP_417510.1|, length 386\n",
      "Record gi|226524747|ref|NP_417511.2|, length 262\n",
      "Record gi|16130936|ref|NP_417512.1|, length 257\n",
      "Record gi|16130937|ref|NP_417513.1|, length 217\n",
      "Record gi|90111530|ref|NP_417514.4|, length 96\n",
      "Record gi|16130939|ref|NP_417515.1|, length 183\n",
      "Record gi|145698309|ref|NP_417516.3|, length 121\n",
      "Record gi|16130941|ref|NP_417517.1|, length 301\n",
      "Record gi|90111532|ref|NP_417519.4|, length 249\n",
      "Record gi|16130944|ref|NP_417520.1|, length 354\n",
      "Record gi|16130945|ref|NP_417521.1|, length 66\n",
      "Record gi|16130946|ref|NP_417522.1|, length 209\n",
      "Record gi|16130947|ref|NP_417523.1|, length 553\n",
      "Record gi|226524748|ref|YP_002791256.1|, length 19\n",
      "Record gi|226524749|ref|YP_002791257.1|, length 19\n",
      "Record gi|16130948|ref|NP_417524.1|, length 477\n",
      "Record gi|16130949|ref|NP_417525.1|, length 946\n",
      "Record gi|16130950|ref|NP_417526.1|, length 433\n",
      "Record gi|16130951|ref|NP_417527.1|, length 206\n",
      "Record gi|16130952|ref|NP_417528.1|, length 412\n",
      "Record gi|16130953|ref|NP_417529.1|, length 273\n",
      "Record gi|90111533|ref|NP_417530.2|, length 122\n",
      "Record gi|16130955|ref|NP_417531.1|, length 205\n",
      "Record gi|16130956|ref|NP_417532.1|, length 310\n",
      "Record gi|16130957|ref|NP_417533.1|, length 303\n",
      "Record gi|16130958|ref|NP_417534.1|, length 201\n",
      "Record gi|16130959|ref|NP_417535.1|, length 487\n",
      "Record gi|16130960|ref|NP_417536.1|, length 337\n",
      "Record gi|16130961|ref|NP_417537.1|, length 71\n",
      "Record gi|16130962|ref|NP_417538.1|, length 581\n",
      "Record gi|16130963|ref|NP_417539.1|, length 613\n",
      "Record gi|16130964|ref|NP_417540.1|, length 168\n",
      "Record gi|16130965|ref|NP_417541.1|, length 254\n",
      "Record gi|16130966|ref|NP_417542.1|, length 207\n",
      "Record gi|16130967|ref|NP_417543.1|, length 506\n",
      "Record gi|145698310|ref|NP_417544.5|, length 459\n",
      "Record gi|16130969|ref|NP_417545.1|, length 110\n",
      "Record gi|16130970|ref|NP_417546.1|, length 327\n",
      "Record gi|49176308|ref|YP_026199.1|, length 1030\n",
      "Record gi|16130972|ref|NP_417548.1|, length 149\n",
      "Record gi|49176309|ref|YP_026200.1|, length 477\n",
      "Record gi|16130974|ref|NP_417550.1|, length 356\n",
      "Record gi|16130975|ref|NP_417551.1|, length 783\n",
      "Record gi|16130976|ref|NP_417552.1|, length 672\n",
      "Record gi|16130977|ref|NP_417553.1|, length 138\n",
      "Record gi|16130978|ref|NP_417554.1|, length 104\n",
      "Record gi|90111535|ref|NP_417555.4|, length 378\n",
      "Record gi|90111536|ref|NP_417556.2|, length 167\n",
      "Record gi|16130981|ref|NP_417557.1|, length 230\n",
      "Record gi|145698311|ref|NP_417558.2|, length 328\n",
      "Record gi|49176311|ref|YP_026201.1|, length 321\n",
      "Record gi|16130984|ref|NP_417560.1|, length 414\n",
      "Record gi|16130985|ref|NP_417561.1|, length 183\n",
      "Record gi|16130986|ref|NP_417562.1|, length 495\n",
      "Record gi|16130987|ref|NP_417563.1|, length 470\n",
      "Record gi|16130988|ref|NP_417564.1|, length 472\n",
      "Record gi|90111537|ref|NP_417565.2|, length 258\n",
      "Record gi|16130990|ref|NP_417566.1|, length 220\n",
      "Record gi|16130991|ref|NP_417567.1|, length 127\n",
      "Record gi|90111538|ref|NP_417568.2|, length 122\n",
      "Record gi|16130993|ref|NP_417569.1|, length 101\n",
      "Record gi|16130994|ref|NP_417570.1|, length 134\n",
      "Record gi|16130995|ref|NP_417571.1|, length 99\n",
      "Record gi|90111539|ref|NP_417572.2|, length 130\n",
      "Record gi|16130997|ref|NP_417573.1|, length 328\n",
      "Record gi|16130998|ref|NP_417574.1|, length 121\n",
      "Record gi|16130999|ref|NP_417575.1|, length 118\n",
      "Record gi|16131000|ref|NP_417576.1|, length 298\n",
      "Record gi|16131001|ref|NP_417577.1|, length 233\n",
      "Record gi|90111540|ref|NP_417578.2|, length 54\n",
      "Record gi|49176312|ref|YP_026202.1|, length 436\n",
      "Record gi|145698312|ref|YP_026203.3|, length 443\n",
      "Record gi|49176314|ref|YP_026204.1|, length 454\n",
      "Record gi|90111542|ref|NP_417583.4|, length 129\n",
      "Record gi|49176316|ref|YP_026205.1|, length 764\n",
      "Record gi|145698313|ref|NP_417585.2|, length 402\n",
      "Record gi|16131009|ref|NP_417586.1|, length 443\n",
      "Record gi|16131010|ref|NP_417587.1|, length 329\n",
      "Record gi|16131011|ref|NP_417588.1|, length 312\n",
      "Record gi|162135908|ref|NP_417589.4|, length 72\n",
      "Record gi|90111544|ref|NP_417590.2|, length 179\n",
      "Record gi|16131014|ref|NP_417591.1|, length 395\n",
      "Record gi|145698314|ref|NP_417593.4|, length 381\n",
      "Record gi|145698315|ref|NP_417594.3|, length 294\n",
      "Record gi|16131018|ref|NP_417595.1|, length 256\n",
      "Record gi|16131019|ref|NP_417596.1|, length 444\n",
      "Record gi|16131020|ref|NP_417597.1|, length 523\n",
      "Record gi|16131021|ref|NP_417598.1|, length 111\n",
      "Record gi|16131022|ref|NP_417599.1|, length 154\n",
      "Record gi|16131023|ref|NP_417600.1|, length 269\n",
      "Record gi|16131024|ref|NP_417601.1|, length 426\n",
      "Record gi|90111546|ref|NP_417602.4|, length 157\n",
      "Record gi|16131028|ref|NP_417605.1|, length 384\n",
      "Record gi|16131029|ref|NP_417606.1|, length 286\n",
      "Record gi|16131030|ref|NP_417607.1|, length 158\n",
      "Record gi|16131031|ref|NP_417608.1|, length 267\n",
      "Record gi|16131032|ref|NP_417609.1|, length 263\n",
      "Record gi|16131033|ref|NP_417610.1|, length 251\n",
      "Record gi|16131034|ref|NP_417611.1|, length 194\n",
      "Record gi|16131035|ref|NP_417612.1|, length 231\n",
      "Record gi|16131036|ref|NP_417613.1|, length 838\n",
      "Record gi|16131037|ref|NP_417614.1|, length 363\n",
      "Record gi|16131038|ref|NP_417615.1|, length 286\n",
      "Record gi|16131039|ref|NP_417616.1|, length 678\n",
      "Record gi|16131040|ref|NP_417617.1|, length 131\n",
      "Record gi|16131041|ref|NP_417618.1|, length 196\n",
      "Record gi|16131042|ref|NP_417619.1|, length 191\n",
      "Record gi|16131043|ref|NP_417620.1|, length 346\n",
      "Record gi|90111547|ref|NP_417621.2|, length 211\n",
      "Record gi|90111548|ref|NP_417622.2|, length 172\n",
      "Record gi|16131046|ref|NP_417623.1|, length 147\n",
      "Record gi|16131047|ref|NP_417624.1|, length 100\n",
      "Record gi|16131048|ref|NP_417625.1|, length 167\n",
      "Record gi|16131049|ref|NP_417626.1|, length 174\n",
      "Record gi|16131050|ref|NP_417627.1|, length 331\n",
      "Record gi|90111549|ref|NP_417628.2|, length 292\n",
      "Record gi|16131052|ref|NP_417629.1|, length 335\n",
      "Record gi|16131053|ref|NP_417630.1|, length 414\n",
      "Record gi|90111550|ref|NP_417631.2|, length 629\n",
      "Record gi|226524750|ref|YP_002791258.1|, length 26\n",
      "Record gi|16131055|ref|NP_417632.1|, length 294\n",
      "Record gi|145698316|ref|NP_417633.4|, length 711\n",
      "Record gi|16131057|ref|NP_417634.1|, length 89\n",
      "Record gi|16131058|ref|NP_417635.1|, length 314\n",
      "Record gi|16131059|ref|NP_417636.1|, length 133\n",
      "Record gi|16131060|ref|NP_417637.1|, length 890\n",
      "Record gi|16131061|ref|NP_417638.1|, length 495\n",
      "Record gi|226524751|ref|NP_417639.6|, length 150\n",
      "Record gi|16131063|ref|NP_417640.1|, length 447\n",
      "Record gi|90111552|ref|NP_417641.4|, length 541\n",
      "Record gi|16131065|ref|NP_417642.1|, length 110\n",
      "Record gi|16131066|ref|NP_417643.1|, length 445\n",
      "Record gi|90111553|ref|NP_417644.4|, length 282\n",
      "Record gi|16131068|ref|NP_417645.1|, length 644\n",
      "Record gi|16131069|ref|NP_417646.1|, length 209\n",
      "Record gi|16131070|ref|NP_417647.1|, length 97\n",
      "Record gi|90111554|ref|NP_417648.4|, length 158\n",
      "Record gi|16131072|ref|NP_417649.1|, length 477\n",
      "Record gi|16131073|ref|NP_417650.1|, length 390\n",
      "Record gi|16131074|ref|NP_417651.1|, length 321\n",
      "Record gi|16131075|ref|NP_417652.1|, length 85\n",
      "Record gi|16131076|ref|NP_417653.1|, length 103\n",
      "Record gi|16131077|ref|NP_417654.1|, length 323\n",
      "Record gi|16131078|ref|NP_417655.1|, length 92\n",
      "Record gi|16131079|ref|NP_417656.1|, length 419\n",
      "Record gi|90111555|ref|NP_417657.2|, length 84\n",
      "Record gi|90111556|ref|NP_417658.4|, length 97\n",
      "Record gi|16131082|ref|NP_417659.1|, length 211\n",
      "Record gi|16131083|ref|NP_417660.1|, length 183\n",
      "Record gi|16131084|ref|NP_417661.1|, length 260\n",
      "Record gi|16131085|ref|NP_417662.1|, length 269\n",
      "Record gi|16131086|ref|NP_417663.1|, length 325\n",
      "Record gi|16131087|ref|NP_417664.1|, length 328\n",
      "Record gi|16131088|ref|NP_417665.1|, length 188\n",
      "Record gi|16131089|ref|NP_417666.1|, length 191\n",
      "Record gi|16131090|ref|NP_417667.1|, length 185\n",
      "Record gi|16131091|ref|NP_417668.1|, length 241\n",
      "Record gi|16131092|ref|NP_417669.1|, length 477\n",
      "Record gi|16131093|ref|NP_417670.1|, length 95\n",
      "Record gi|16131094|ref|NP_417671.1|, length 163\n",
      "Record gi|16131095|ref|NP_417672.1|, length 284\n",
      "Record gi|16131096|ref|NP_417673.1|, length 90\n",
      "Record gi|16131097|ref|NP_417674.1|, length 210\n",
      "Record gi|16131098|ref|NP_417675.1|, length 242\n",
      "Record gi|90111557|ref|NP_417676.2|, length 217\n",
      "Record gi|49176326|ref|YP_026207.1|, length 778\n",
      "Record gi|16131101|ref|NP_417678.1|, length 309\n",
      "Record gi|308209621|ref|NP_417679.2|, length 1486\n",
      "Record gi|16131103|ref|NP_417680.1|, length 472\n",
      "Record gi|16131104|ref|NP_417681.1|, length 254\n",
      "Record gi|16131105|ref|NP_417682.1|, length 224\n",
      "Record gi|16131106|ref|NP_417683.1|, length 793\n",
      "Record gi|16131108|ref|NP_417685.1|, length 338\n",
      "Record gi|16131109|ref|NP_417686.1|, length 238\n",
      "Record gi|16131110|ref|NP_417687.1|, length 375\n",
      "Record gi|16131111|ref|NP_417688.1|, length 154\n",
      "Record gi|90111558|ref|NP_417689.4|, length 291\n",
      "Record gi|16131113|ref|NP_417690.1|, length 229\n",
      "Record gi|90111559|ref|NP_417691.4|, length 496\n",
      "Record gi|16131115|ref|NP_417692.1|, length 297\n",
      "Record gi|49176329|ref|NP_417693.3|, length 263\n",
      "Record gi|16131117|ref|NP_417694.1|, length 455\n",
      "Record gi|16131118|ref|NP_417695.1|, length 165\n",
      "Record gi|16131119|ref|NP_417696.1|, length 212\n",
      "Record gi|16131120|ref|NP_417697.1|, length 130\n",
      "Record gi|16131121|ref|NP_417698.1|, length 142\n",
      "Record gi|16131122|ref|NP_417699.1|, length 375\n",
      "Record gi|90111560|ref|NP_417700.2|, length 132\n",
      "Record gi|16131124|ref|NP_417701.1|, length 455\n",
      "Record gi|16131125|ref|NP_417702.1|, length 355\n",
      "Record gi|16131126|ref|NP_417703.1|, length 312\n",
      "Record gi|16131127|ref|NP_417704.1|, length 156\n",
      "Record gi|90111561|ref|NP_417705.2|, length 87\n",
      "Record gi|16131129|ref|NP_417706.1|, length 90\n",
      "Record gi|16131130|ref|NP_417707.1|, length 655\n",
      "Record gi|16131131|ref|NP_417708.1|, length 310\n",
      "Record gi|90111562|ref|NP_417709.2|, length 67\n",
      "Record gi|16131133|ref|NP_417710.1|, length 309\n",
      "Record gi|16131134|ref|NP_417711.1|, length 481\n",
      "Record gi|49176330|ref|YP_026208.1|, length 1266\n",
      "Record gi|90111563|ref|NP_417713.2|, length 489\n",
      "Record gi|16131136|ref|NP_417714.1|, length 197\n",
      "Record gi|16131137|ref|NP_417715.1|, length 162\n",
      "Record gi|16131138|ref|NP_417716.1|, length 367\n",
      "Record gi|90111564|ref|NP_417717.2|, length 347\n",
      "Record gi|16131140|ref|NP_417718.1|, length 646\n",
      "Record gi|16131141|ref|NP_417719.1|, length 324\n",
      "Record gi|16131143|ref|NP_417721.1|, length 156\n",
      "Record gi|16131144|ref|NP_417722.1|, length 449\n",
      "Record gi|16131145|ref|NP_417723.1|, length 80\n",
      "Record gi|90111565|ref|NP_417724.4|, length 483\n",
      "Record gi|16131147|ref|NP_417725.1|, length 293\n",
      "Record gi|16131148|ref|NP_417726.1|, length 321\n",
      "Record gi|16131149|ref|NP_417727.1|, length 98\n",
      "Record gi|90111566|ref|NP_417728.4|, length 294\n",
      "Record gi|16131151|ref|NP_417729.1|, length 59\n",
      "Record gi|16131152|ref|NP_417730.1|, length 220\n",
      "Record gi|16131153|ref|NP_417731.1|, length 385\n",
      "Record gi|16131154|ref|NP_417732.1|, length 1034\n",
      "Record gi|16131155|ref|NP_417733.1|, length 73\n",
      "Record gi|49176333|ref|YP_026209.1|, length 393\n",
      "Record gi|90111567|ref|NP_417736.2|, length 367\n",
      "Record gi|16131159|ref|NP_417737.1|, length 252\n",
      "Record gi|90111568|ref|NP_417738.4|, length 184\n",
      "Record gi|16131161|ref|NP_417739.1|, length 85\n",
      "Record gi|16131162|ref|NP_417740.1|, length 272\n",
      "Record gi|16131163|ref|NP_417741.1|, length 190\n",
      "Record gi|49176335|ref|YP_026210.1|, length 180\n",
      "Record gi|16131165|ref|NP_417743.1|, length 157\n",
      "Record gi|49176336|ref|YP_026211.1|, length 374\n",
      "Record gi|16131166|ref|NP_417745.1|, length 169\n",
      "Record gi|16131167|ref|NP_417746.1|, length 315\n",
      "Record gi|16131168|ref|NP_417747.1|, length 429\n",
      "Record gi|16131169|ref|NP_417748.1|, length 458\n",
      "Record gi|16131170|ref|NP_417749.1|, length 136\n",
      "Record gi|94541124|ref|YP_588467.1|, length 72\n",
      "Record gi|16131171|ref|NP_417751.1|, length 141\n",
      "Record gi|16131172|ref|NP_417752.1|, length 122\n",
      "Record gi|16131173|ref|NP_417753.1|, length 127\n",
      "Record gi|16131174|ref|NP_417754.1|, length 329\n",
      "Record gi|16131175|ref|NP_417755.1|, length 206\n",
      "Record gi|16131176|ref|NP_417756.1|, length 129\n",
      "Record gi|16131177|ref|NP_417757.1|, length 118\n",
      "Record gi|16131178|ref|NP_417758.1|, length 38\n",
      "Record gi|16131179|ref|NP_417759.1|, length 443\n",
      "Record gi|16131180|ref|NP_417760.1|, length 144\n",
      "Record gi|16131181|ref|NP_417761.1|, length 59\n",
      "Record gi|16131182|ref|NP_417762.1|, length 167\n",
      "Record gi|16131183|ref|NP_417763.1|, length 117\n",
      "Record gi|16131184|ref|NP_417764.1|, length 177\n",
      "Record gi|16131185|ref|NP_417765.1|, length 130\n",
      "Record gi|16131186|ref|NP_417766.1|, length 101\n",
      "Record gi|16131187|ref|NP_417767.1|, length 179\n",
      "Record gi|16131188|ref|NP_417768.1|, length 104\n",
      "Record gi|16131189|ref|NP_417769.1|, length 123\n",
      "Record gi|16131190|ref|NP_417770.1|, length 84\n",
      "Record gi|16131191|ref|NP_417771.1|, length 63\n",
      "Record gi|16131192|ref|NP_417772.1|, length 136\n",
      "Record gi|16131193|ref|NP_417773.1|, length 233\n",
      "Record gi|16131194|ref|NP_417774.1|, length 110\n",
      "Record gi|16131195|ref|NP_417775.1|, length 92\n",
      "Record gi|16131196|ref|NP_417776.1|, length 273\n",
      "Record gi|16131197|ref|NP_417777.1|, length 100\n",
      "Record gi|16131198|ref|NP_417778.1|, length 201\n",
      "Record gi|16131199|ref|NP_417779.1|, length 209\n",
      "Record gi|16131200|ref|NP_417780.1|, length 103\n",
      "Record gi|16131201|ref|NP_417781.1|, length 139\n",
      "Record gi|16131202|ref|NP_417782.1|, length 489\n",
      "Record gi|16131203|ref|NP_417783.1|, length 271\n",
      "Record gi|90111569|ref|NP_417784.4|, length 650\n",
      "Record gi|16131205|ref|NP_417785.1|, length 493\n",
      "Record gi|16131206|ref|NP_417786.1|, length 398\n",
      "Record gi|16131207|ref|NP_417787.1|, length 145\n",
      "Record gi|16131208|ref|NP_417788.1|, length 169\n",
      "Record gi|90111570|ref|NP_417789.4|, length 125\n",
      "Record gi|16131210|ref|NP_417790.1|, length 195\n",
      "Record gi|16131211|ref|NP_417791.1|, length 327\n",
      "Record gi|90111571|ref|NP_417792.2|, length 387\n",
      "Record gi|90111572|ref|NP_417793.2|, length 153\n",
      "Record gi|16131214|ref|NP_417794.1|, length 225\n",
      "Record gi|16131215|ref|NP_417795.1|, length 158\n",
      "Record gi|16131216|ref|NP_417796.1|, length 64\n",
      "Record gi|16131217|ref|NP_417797.1|, length 897\n",
      "Record gi|16131218|ref|NP_417798.1|, length 394\n",
      "Record gi|16131219|ref|NP_417799.1|, length 704\n",
      "Record gi|16131220|ref|NP_417800.1|, length 179\n",
      "Record gi|16131221|ref|NP_417801.1|, length 124\n",
      "Record gi|16131222|ref|NP_417802.1|, length 95\n",
      "Record gi|16131223|ref|NP_417803.1|, length 119\n",
      "Record gi|16131224|ref|NP_417804.1|, length 128\n",
      "Record gi|90111573|ref|NP_417805.4|, length 240\n",
      "Record gi|16131226|ref|NP_417806.1|, length 270\n",
      "Record gi|16131227|ref|NP_417807.1|, length 72\n",
      "Record gi|16131228|ref|NP_417808.1|, length 196\n",
      "Record gi|94541125|ref|YP_588468.1|, length 66\n",
      "Record gi|16131229|ref|NP_417809.1|, length 601\n",
      "Record gi|16131230|ref|NP_417810.1|, length 184\n",
      "Record gi|16131231|ref|NP_417811.1|, length 637\n",
      "Record gi|16131232|ref|NP_417812.1|, length 340\n",
      "Record gi|16131233|ref|NP_417813.1|, length 72\n",
      "Record gi|16131234|ref|NP_417814.1|, length 289\n",
      "Record gi|16131235|ref|NP_417815.1|, length 134\n",
      "Record gi|16131236|ref|NP_417816.1|, length 210\n",
      "Record gi|90111574|ref|NP_417817.2|, length 700\n",
      "Record gi|16131238|ref|NP_417818.1|, length 406\n",
      "Record gi|16131239|ref|NP_417819.1|, length 187\n",
      "Record gi|16131240|ref|NP_417820.1|, length 200\n",
      "Record gi|16131241|ref|NP_417821.1|, length 55\n",
      "Record gi|16131242|ref|NP_417822.1|, length 190\n",
      "Record gi|16131243|ref|NP_417823.1|, length 393\n",
      "Record gi|16131244|ref|NP_417824.1|, length 847\n",
      "Record gi|16131245|ref|NP_417825.1|, length 108\n",
      "Record gi|90111575|ref|YP_026212.2|, length 268\n",
      "Record gi|16131246|ref|NP_417827.1|, length 457\n",
      "Record gi|16131247|ref|NP_417828.1|, length 55\n",
      "Record gi|90111576|ref|NP_417829.2|, length 445\n",
      "Record gi|162135909|ref|NP_417830.4|, length 340\n",
      "Record gi|49176341|ref|YP_026213.1|, length 276\n",
      "Record gi|16131252|ref|NP_417833.1|, length 261\n",
      "Record gi|162135910|ref|NP_417834.6|, length 243\n",
      "Record gi|16131254|ref|NP_417835.1|, length 361\n",
      "Record gi|16131255|ref|NP_417836.1|, length 434\n",
      "Record gi|90111579|ref|NP_417837.2|, length 117\n",
      "Record gi|16131257|ref|NP_417838.1|, length 292\n",
      "Record gi|16131258|ref|NP_417839.1|, length 408\n",
      "Record gi|16131259|ref|NP_417840.1|, length 387\n",
      "Record gi|90111580|ref|NP_417841.2|, length 120\n",
      "Record gi|49176343|ref|YP_026214.1|, length 301\n",
      "Record gi|16131262|ref|NP_417843.1|, length 334\n",
      "Record gi|16131263|ref|NP_417844.1|, length 252\n",
      "Record gi|16131264|ref|NP_417845.1|, length 225\n",
      "Record gi|16131265|ref|NP_417846.1|, length 278\n",
      "Record gi|16131266|ref|NP_417847.1|, length 428\n",
      "Record gi|16131267|ref|NP_417848.1|, length 362\n",
      "Record gi|90111581|ref|YP_026215.2|, length 173\n",
      "Record gi|16131268|ref|NP_417850.1|, length 412\n",
      "Record gi|90111582|ref|NP_417851.4|, length 134\n",
      "Record gi|16131270|ref|NP_417852.1|, length 146\n",
      "Record gi|16131271|ref|NP_417853.1|, length 179\n",
      "Record gi|90111583|ref|NP_417854.4|, length 259\n",
      "Record gi|162135911|ref|NP_417855.4|, length 850\n",
      "Record gi|16131274|ref|NP_417856.1|, length 186\n",
      "Record gi|16131275|ref|NP_417857.1|, length 711\n",
      "Record gi|90111585|ref|NP_417858.4|, length 222\n",
      "Record gi|16131277|ref|NP_417859.1|, length 133\n",
      "Record gi|90111586|ref|NP_417860.2|, length 292\n",
      "Record gi|16131279|ref|NP_417861.1|, length 574\n",
      "Record gi|16131280|ref|NP_417862.1|, length 540\n",
      "Record gi|16131281|ref|NP_417863.1|, length 450\n",
      "Record gi|16131282|ref|NP_417864.1|, length 239\n",
      "Record gi|90111587|ref|YP_026216.2|, length 158\n",
      "Record gi|90111588|ref|NP_417866.4|, length 773\n",
      "Record gi|16131284|ref|NP_417867.1|, length 75\n",
      "Record gi|16131285|ref|NP_417868.1|, length 773\n",
      "Record gi|16131286|ref|NP_417869.1|, length 78\n",
      "Record gi|16131287|ref|NP_417870.1|, length 292\n",
      "Record gi|16131288|ref|NP_417871.1|, length 256\n",
      "Record gi|90111589|ref|NP_417872.2|, length 227\n",
      "Record gi|16131290|ref|NP_417873.1|, length 191\n",
      "Record gi|49176350|ref|YP_026217.1|, length 438\n",
      "Record gi|16131292|ref|NP_417875.1|, length 694\n",
      "Record gi|49176351|ref|YP_026218.1|, length 797\n",
      "Record gi|16131294|ref|NP_417877.1|, length 901\n",
      "Record gi|49176352|ref|YP_026219.1|, length 338\n",
      "Record gi|16131295|ref|NP_417879.1|, length 408\n",
      "Record gi|16131296|ref|NP_417880.1|, length 532\n",
      "Record gi|49176353|ref|YP_026220.1|, length 276\n",
      "Record gi|16131299|ref|NP_417883.1|, length 108\n",
      "Record gi|16131300|ref|NP_417884.1|, length 501\n",
      "Record gi|16131302|ref|NP_417886.1|, length 815\n",
      "Record gi|16131303|ref|NP_417887.1|, length 477\n",
      "Record gi|16131304|ref|NP_417888.1|, length 431\n",
      "Record gi|16131305|ref|NP_417889.1|, length 657\n",
      "Record gi|16131306|ref|NP_417890.1|, length 728\n",
      "Record gi|16131307|ref|NP_417891.1|, length 367\n",
      "Record gi|16131308|ref|NP_417892.1|, length 197\n",
      "Record gi|49176355|ref|YP_026221.1|, length 446\n",
      "Record gi|90111591|ref|NP_417894.2|, length 175\n",
      "Record gi|49176356|ref|YP_026222.1|, length 331\n",
      "Record gi|16131311|ref|NP_417896.1|, length 231\n",
      "Record gi|16131312|ref|NP_417897.1|, length 345\n",
      "Record gi|16131313|ref|NP_417898.1|, length 162\n",
      "Record gi|16131314|ref|NP_417899.1|, length 392\n",
      "Record gi|16131316|ref|NP_417901.1|, length 91\n",
      "Record gi|16131317|ref|NP_417902.1|, length 167\n",
      "Record gi|16131318|ref|NP_417903.1|, length 94\n",
      "Record gi|16131319|ref|NP_417904.1|, length 580\n",
      "Record gi|16131320|ref|NP_417905.1|, length 146\n",
      "Record gi|16131321|ref|NP_417906.1|, length 247\n",
      "Record gi|90111593|ref|NP_417907.2|, length 356\n",
      "Record gi|16131323|ref|NP_417908.1|, length 281\n",
      "Record gi|16131324|ref|NP_417909.1|, length 295\n",
      "Record gi|16131325|ref|NP_417910.1|, length 438\n",
      "Record gi|90111594|ref|NP_417911.4|, length 237\n",
      "Record gi|16131327|ref|NP_417912.1|, length 255\n",
      "Record gi|16131328|ref|NP_417913.1|, length 425\n",
      "Record gi|16131329|ref|NP_417914.1|, length 308\n",
      "Record gi|16131330|ref|NP_417915.1|, length 369\n",
      "Record gi|16131331|ref|NP_417916.1|, length 127\n",
      "Record gi|49176358|ref|YP_026223.1|, length 367\n",
      "Record gi|16131333|ref|NP_417918.1|, length 284\n",
      "Record gi|16131334|ref|NP_417919.1|, length 352\n",
      "Record gi|16131335|ref|NP_417920.1|, length 222\n",
      "Record gi|16131336|ref|NP_417921.1|, length 497\n",
      "Record gi|16131337|ref|NP_417922.1|, length 198\n",
      "Record gi|16131338|ref|NP_417923.1|, length 89\n",
      "Record gi|16131339|ref|NP_417924.1|, length 119\n",
      "Record gi|16131340|ref|NP_417925.1|, length 208\n",
      "Record gi|16131341|ref|NP_417926.1|, length 732\n",
      "Record gi|16131342|ref|NP_417927.1|, length 81\n",
      "Record gi|16131343|ref|NP_417928.1|, length 221\n",
      "Record gi|90111595|ref|NP_417929.4|, length 185\n",
      "Record gi|90111596|ref|NP_417930.2|, length 405\n",
      "Record gi|90111597|ref|NP_417931.4|, length 349\n",
      "Record gi|16131347|ref|NP_417932.1|, length 195\n",
      "Record gi|16131348|ref|NP_417933.1|, length 524\n",
      "Record gi|16131349|ref|NP_417934.1|, length 314\n",
      "Record gi|16131350|ref|NP_417935.1|, length 277\n",
      "Record gi|16131351|ref|NP_417936.1|, length 254\n",
      "Record gi|16131352|ref|NP_417937.1|, length 268\n",
      "Record gi|16131353|ref|NP_417938.1|, length 133\n",
      "Record gi|49176361|ref|YP_026224.1|, length 1411\n",
      "Record gi|90111598|ref|NP_417940.2|, length 127\n",
      "Record gi|16131356|ref|NP_417941.1|, length 378\n",
      "Record gi|226524752|ref|NP_417942.6|, length 374\n",
      "Record gi|49176363|ref|YP_026225.1|, length 911\n",
      "Record gi|16131359|ref|NP_417944.1|, length 355\n",
      "Record gi|16131360|ref|NP_417945.1|, length 540\n",
      "Record gi|226524753|ref|NP_417948.4|, length 350\n",
      "Record gi|16131364|ref|NP_417949.1|, length 400\n",
      "Record gi|16131365|ref|NP_417950.1|, length 499\n",
      "Record gi|16131366|ref|NP_417951.1|, length 111\n",
      "Record gi|16131367|ref|NP_417952.1|, length 144\n",
      "Record gi|16131368|ref|NP_417953.1|, length 489\n",
      "Record gi|16131369|ref|NP_417954.1|, length 250\n",
      "Record gi|16131370|ref|NP_417955.1|, length 680\n",
      "Record gi|16131371|ref|NP_417956.1|, length 280\n",
      "Record gi|16131372|ref|NP_417957.1|, length 450\n",
      "Record gi|635341466|ref|YP_001165328.2|, length 27\n",
      "Record gi|16131373|ref|NP_417958.1|, length 117\n",
      "Record gi|90111602|ref|NP_417959.4|, length 429\n",
      "Record gi|16131375|ref|NP_417960.1|, length 141\n",
      "Record gi|16131377|ref|NP_417962.1|, length 338\n",
      "Record gi|90111603|ref|NP_417963.4|, length 188\n",
      "Record gi|16131379|ref|NP_417964.1|, length 176\n",
      "Record gi|16131380|ref|NP_417965.1|, length 215\n",
      "Record gi|162135912|ref|NP_417966.4|, length 108\n",
      "Record gi|16131382|ref|NP_417967.1|, length 110\n",
      "Record gi|16131383|ref|NP_417968.1|, length 190\n",
      "Record gi|16131384|ref|NP_417969.1|, length 175\n",
      "Record gi|16131385|ref|NP_417970.1|, length 385\n",
      "Record gi|16131386|ref|NP_417971.1|, length 1037\n",
      "Record gi|16131387|ref|NP_417972.1|, length 242\n",
      "Record gi|16131388|ref|NP_417973.1|, length 274\n",
      "Record gi|16131389|ref|NP_417974.1|, length 466\n",
      "Record gi|16131390|ref|NP_417975.1|, length 465\n",
      "Record gi|16131391|ref|NP_417976.1|, length 549\n",
      "Record gi|16131392|ref|NP_417977.1|, length 200\n",
      "Record gi|145698322|ref|NP_417978.2|, length 299\n",
      "Record gi|16131394|ref|NP_417979.1|, length 337\n",
      "Record gi|16131395|ref|NP_417980.1|, length 440\n",
      "Record gi|226524754|ref|NP_417981.2|, length 686\n",
      "Record gi|90111605|ref|NP_417982.2|, length 255\n",
      "Record gi|90111606|ref|NP_417983.2|, length 309\n",
      "Record gi|16131399|ref|NP_417984.1|, length 498\n",
      "Record gi|16131400|ref|NP_417985.1|, length 428\n",
      "Record gi|226524755|ref|NP_417986.4|, length 662\n",
      "Record gi|226524756|ref|YP_026226.4|, length 1157\n",
      "Record gi|16131403|ref|NP_417988.1|, length 368\n",
      "Record gi|16131404|ref|NP_417989.1|, length 779\n",
      "Record gi|90111609|ref|NP_417990.4|, length 872\n",
      "Record gi|16131407|ref|NP_417992.1|, length 62\n",
      "Record gi|16131408|ref|NP_417993.1|, length 523\n",
      "Record gi|90111610|ref|NP_417994.2|, length 63\n",
      "Record gi|16131410|ref|NP_417995.1|, length 559\n",
      "Record gi|49176368|ref|YP_026227.1|, length 35\n",
      "Record gi|16131411|ref|NP_417996.1|, length 423\n",
      "Record gi|16131412|ref|NP_417997.1|, length 334\n",
      "Record gi|16131413|ref|NP_417998.1|, length 327\n",
      "Record gi|16131414|ref|NP_417999.1|, length 300\n",
      "Record gi|16131415|ref|NP_418000.1|, length 339\n",
      "Record gi|16131416|ref|NP_418001.1|, length 535\n",
      "Record gi|90111611|ref|NP_418002.2|, length 563\n",
      "Record gi|16131418|ref|NP_418003.1|, length 402\n",
      "Record gi|90111612|ref|NP_418004.4|, length 232\n",
      "Record gi|16131420|ref|NP_418005.1|, length 187\n",
      "Record gi|16131421|ref|NP_418006.1|, length 146\n",
      "Record gi|145698325|ref|NP_418007.3|, length 777\n",
      "Record gi|49176370|ref|YP_026228.1|, length 219\n",
      "Record gi|90111614|ref|NP_418009.2|, length 324\n",
      "Record gi|90111615|ref|NP_418010.2|, length 236\n",
      "Record gi|16131426|ref|NP_418011.1|, length 96\n",
      "Record gi|16131427|ref|NP_418012.1|, length 70\n",
      "Record gi|49176371|ref|YP_026229.1|, length 50\n",
      "Record gi|16131428|ref|NP_418014.1|, length 173\n",
      "Record gi|16131429|ref|NP_418015.1|, length 283\n",
      "Record gi|16131430|ref|NP_418016.1|, length 689\n",
      "Record gi|16131431|ref|NP_418017.1|, length 303\n",
      "Record gi|94541126|ref|YP_588469.1|, length 99\n",
      "Record gi|16131432|ref|NP_418018.1|, length 331\n",
      "Record gi|90111616|ref|NP_418019.4|, length 145\n",
      "Record gi|90111617|ref|NP_418020.2|, length 113\n",
      "Record gi|16131435|ref|NP_418021.1|, length 484\n",
      "Record gi|16131436|ref|NP_418022.1|, length 440\n",
      "Record gi|16131437|ref|NP_418023.1|, length 330\n",
      "Record gi|16131438|ref|NP_418024.1|, length 513\n",
      "Record gi|16131439|ref|NP_418025.1|, length 393\n",
      "Record gi|16131440|ref|NP_418026.1|, length 392\n",
      "Record gi|49176373|ref|YP_026230.1|, length 274\n",
      "Record gi|16131442|ref|NP_418028.1|, length 676\n",
      "Record gi|49176374|ref|YP_026231.1|, length 417\n",
      "Record gi|16131444|ref|NP_418030.1|, length 157\n",
      "Record gi|16131445|ref|NP_418031.1|, length 282\n",
      "Record gi|16131446|ref|NP_418032.1|, length 332\n",
      "Record gi|16131447|ref|NP_418033.1|, length 155\n",
      "Record gi|16131448|ref|NP_418034.1|, length 157\n",
      "Record gi|49176375|ref|YP_026232.1|, length 425\n",
      "Record gi|16131450|ref|NP_418036.1|, length 328\n",
      "Record gi|16131451|ref|NP_418037.1|, length 498\n",
      "Record gi|16131452|ref|NP_418038.1|, length 220\n",
      "Record gi|90111618|ref|NP_418039.2|, length 286\n",
      "Record gi|16131454|ref|NP_418040.1|, length 231\n",
      "Record gi|16131455|ref|NP_418041.1|, length 246\n",
      "Record gi|16131456|ref|NP_418042.1|, length 324\n",
      "Record gi|16131457|ref|NP_418043.1|, length 378\n",
      "Record gi|16131458|ref|NP_418044.1|, length 107\n",
      "Record gi|90111619|ref|NP_418045.4|, length 512\n",
      "Record gi|49176377|ref|YP_026233.1|, length 383\n",
      "Record gi|16131461|ref|NP_418047.1|, length 614\n",
      "Record gi|16131462|ref|NP_418048.1|, length 463\n",
      "Record gi|16131463|ref|NP_418049.1|, length 202\n",
      "Record gi|16131464|ref|NP_418050.1|, length 1377\n",
      "Record gi|16131465|ref|NP_418051.1|, length 280\n",
      "Record gi|16131467|ref|NP_418053.1|, length 153\n",
      "Record gi|16131468|ref|NP_418054.1|, length 378\n",
      "Record gi|16131469|ref|NP_418055.1|, length 120\n",
      "Record gi|16131470|ref|NP_418056.1|, length 637\n",
      "Record gi|16131471|ref|NP_418057.1|, length 382\n",
      "Record gi|16131472|ref|NP_418058.1|, length 195\n",
      "Record gi|94541127|ref|YP_588470.1|, length 69\n",
      "Record gi|16131473|ref|NP_418059.1|, length 120\n",
      "Record gi|16131474|ref|NP_418060.1|, length 551\n",
      "Record gi|16131475|ref|NP_418061.1|, length 258\n",
      "Record gi|16131476|ref|NP_418062.1|, length 396\n",
      "Record gi|16131477|ref|NP_418063.1|, length 157\n",
      "Record gi|16131478|ref|NP_418064.1|, length 273\n",
      "Record gi|16131479|ref|NP_418065.1|, length 339\n",
      "Record gi|16131480|ref|NP_418066.1|, length 155\n",
      "Record gi|16131481|ref|NP_418067.1|, length 83\n",
      "Record gi|16131482|ref|NP_418068.1|, length 143\n",
      "Record gi|16131483|ref|NP_418069.1|, length 514\n",
      "Record gi|162135913|ref|NP_418070.6|, length 419\n",
      "Record gi|90111621|ref|NP_418071.4|, length 319\n",
      "Record gi|16131486|ref|NP_418072.1|, length 344\n",
      "Record gi|16131487|ref|NP_418073.1|, length 341\n",
      "Record gi|16131488|ref|NP_418074.1|, length 398\n",
      "Record gi|90111622|ref|NP_418075.2|, length 285\n",
      "Record gi|16131490|ref|NP_418076.1|, length 310\n",
      "Record gi|16131491|ref|NP_418077.1|, length 348\n",
      "Record gi|16131492|ref|NP_418078.1|, length 319\n",
      "Record gi|16131493|ref|NP_418079.1|, length 419\n",
      "Record gi|16131494|ref|NP_418080.1|, length 357\n",
      "Record gi|16131495|ref|NP_418081.1|, length 283\n",
      "Record gi|16131496|ref|NP_418082.1|, length 232\n",
      "Record gi|16131497|ref|NP_418083.1|, length 338\n",
      "Record gi|16131498|ref|NP_418084.1|, length 339\n",
      "Record gi|226524757|ref|NP_418085.2|, length 359\n",
      "Record gi|16131500|ref|NP_418086.1|, length 311\n",
      "Record gi|16131501|ref|NP_418087.1|, length 265\n",
      "Record gi|16131502|ref|NP_418088.1|, length 374\n",
      "Record gi|16131503|ref|NP_418089.1|, length 344\n",
      "Record gi|16131504|ref|NP_418090.1|, length 425\n",
      "Record gi|16131505|ref|NP_418091.1|, length 159\n",
      "Record gi|16131506|ref|NP_418092.1|, length 269\n",
      "Record gi|16131507|ref|NP_418093.1|, length 55\n",
      "Record gi|16131508|ref|NP_418094.1|, length 78\n",
      "Record gi|162135914|ref|NP_418095.4|, length 222\n",
      "Record gi|90111624|ref|NP_418096.4|, length 406\n",
      "Record gi|16131511|ref|NP_418097.1|, length 151\n",
      "Record gi|90111625|ref|NP_418098.4|, length 198\n",
      "Record gi|16131513|ref|NP_418099.1|, length 213\n",
      "Record gi|16131515|ref|NP_418101.1|, length 287\n",
      "Record gi|90111626|ref|NP_418102.2|, length 274\n",
      "Record gi|90111627|ref|NP_418103.4|, length 205\n",
      "Record gi|90111628|ref|NP_418104.2|, length 560\n",
      "Record gi|16131519|ref|NP_418105.1|, length 207\n",
      "Record gi|16131520|ref|NP_418106.1|, length 91\n",
      "Record gi|16131521|ref|NP_418107.1|, length 702\n",
      "Record gi|16131522|ref|NP_418108.1|, length 229\n",
      "Record gi|16131523|ref|NP_418109.1|, length 693\n",
      "Record gi|16131524|ref|NP_418110.1|, length 401\n",
      "Record gi|16131525|ref|NP_418111.1|, length 463\n",
      "Record gi|16131526|ref|NP_418112.1|, length 569\n",
      "Record gi|16131527|ref|NP_418113.1|, length 772\n",
      "Record gi|90111629|ref|NP_418114.4|, length 460\n",
      "Record gi|16131529|ref|NP_418115.1|, length 394\n",
      "Record gi|16131530|ref|NP_418116.1|, length 307\n",
      "Record gi|16131531|ref|NP_418117.1|, length 272\n",
      "Record gi|94541128|ref|YP_588471.1|, length 97\n",
      "Record gi|145698327|ref|NP_418118.2|, length 396\n",
      "Record gi|90111630|ref|NP_418119.2|, length 150\n",
      "Record gi|90111631|ref|NP_418120.2|, length 444\n",
      "Record gi|16131535|ref|NP_418121.1|, length 588\n",
      "Record gi|16131536|ref|NP_418122.1|, length 463\n",
      "Record gi|90111632|ref|NP_418123.2|, length 439\n",
      "Record gi|90111633|ref|NP_418124.4|, length 500\n",
      "Record gi|16131539|ref|NP_418125.1|, length 196\n",
      "Record gi|16131540|ref|NP_418126.1|, length 96\n",
      "Record gi|16131541|ref|NP_418127.1|, length 562\n",
      "Record gi|16131542|ref|NP_418128.1|, length 32\n",
      "Record gi|145698329|ref|YP_001165331.1|, length 29\n",
      "Record gi|90111634|ref|NP_418129.2|, length 394\n",
      "Record gi|16131544|ref|NP_418130.1|, length 165\n",
      "Record gi|16131545|ref|NP_418131.1|, length 120\n",
      "Record gi|16131546|ref|NP_418132.1|, length 115\n",
      "Record gi|16131547|ref|NP_418133.1|, length 149\n",
      "Record gi|16131548|ref|NP_418134.1|, length 497\n",
      "Record gi|16131549|ref|NP_418135.1|, length 571\n",
      "Record gi|226524758|ref|NP_418136.2|, length 297\n",
      "Record gi|16131552|ref|NP_418139.1|, length 238\n",
      "Record gi|90111636|ref|NP_418140.4|, length 553\n",
      "Record gi|90111637|ref|NP_418141.2|, length 142\n",
      "Record gi|16131555|ref|NP_418142.1|, length 137\n",
      "Record gi|90111638|ref|NP_418143.2|, length 110\n",
      "Record gi|308209622|ref|NP_418144.3|, length 415\n",
      "Record gi|90111640|ref|NP_418145.2|, length 354\n",
      "Record gi|90111641|ref|NP_418146.4|, length 430\n",
      "Record gi|49176390|ref|YP_026237.1|, length 382\n",
      "Record gi|49176391|ref|YP_026238.1|, length 205\n",
      "Record gi|16131561|ref|NP_418148.1|, length 292\n",
      "Record gi|49176392|ref|YP_026239.1|, length 229\n",
      "Record gi|49176393|ref|YP_026240.1|, length 218\n",
      "Record gi|16131565|ref|NP_418152.1|, length 270\n",
      "Record gi|90111642|ref|NP_418153.4|, length 132\n",
      "Record gi|49176395|ref|YP_026241.1|, length 804\n",
      "Record gi|16131568|ref|NP_418155.1|, length 357\n",
      "Record gi|16131569|ref|NP_418156.1|, length 366\n",
      "Record gi|16131570|ref|NP_418157.1|, length 467\n",
      "Record gi|16131571|ref|NP_418158.1|, length 46\n",
      "Record gi|16131572|ref|NP_418159.1|, length 119\n",
      "Record gi|94541129|ref|YP_588472.1|, length 85\n",
      "Record gi|16131573|ref|NP_418161.1|, length 548\n",
      "Record gi|16131574|ref|NP_418162.1|, length 454\n",
      "Record gi|16131575|ref|NP_418163.1|, length 24\n",
      "Record gi|90111643|ref|NP_418164.4|, length 471\n",
      "Record gi|16131577|ref|NP_418165.1|, length 415\n",
      "Record gi|16131578|ref|NP_418166.1|, length 391\n",
      "Record gi|16131579|ref|NP_418167.1|, length 319\n",
      "Record gi|226524759|ref|NP_418168.2|, length 249\n",
      "Record gi|16131581|ref|NP_418169.1|, length 188\n",
      "Record gi|16131582|ref|NP_418170.1|, length 445\n",
      "Record gi|16131583|ref|NP_418171.1|, length 221\n",
      "Record gi|16131584|ref|NP_418172.1|, length 155\n",
      "Record gi|16131585|ref|NP_418173.1|, length 195\n",
      "Record gi|49176397|ref|YP_026242.1|, length 240\n",
      "Record gi|145698330|ref|NP_418175.3|, length 389\n",
      "Record gi|16131588|ref|NP_418176.1|, length 538\n",
      "Record gi|16131589|ref|NP_418177.1|, length 470\n",
      "Record gi|16131590|ref|NP_418178.1|, length 625\n",
      "Record gi|16131591|ref|NP_418179.1|, length 278\n",
      "Record gi|16131592|ref|NP_418180.1|, length 241\n",
      "Record gi|16131593|ref|NP_418181.1|, length 257\n",
      "Record gi|16131594|ref|NP_418182.1|, length 296\n",
      "Record gi|16131595|ref|NP_418183.1|, length 319\n",
      "Record gi|16131596|ref|NP_418184.1|, length 346\n",
      "Record gi|16131597|ref|NP_418185.1|, length 609\n",
      "Record gi|16131598|ref|NP_418186.1|, length 456\n",
      "Record gi|16131599|ref|NP_418187.1|, length 139\n",
      "Record gi|16131600|ref|NP_418188.1|, length 460\n",
      "Record gi|16131601|ref|NP_418189.1|, length 287\n",
      "Record gi|16131602|ref|NP_418190.1|, length 513\n",
      "Record gi|16131603|ref|NP_418191.1|, length 177\n",
      "Record gi|16131604|ref|NP_418192.1|, length 156\n",
      "Record gi|16131605|ref|NP_418193.1|, length 79\n",
      "Record gi|16131606|ref|NP_418194.1|, length 271\n",
      "Record gi|90111645|ref|NP_418195.2|, length 126\n",
      "Record gi|16131608|ref|NP_418196.1|, length 207\n",
      "Record gi|16131609|ref|NP_418197.1|, length 629\n",
      "Record gi|16131610|ref|NP_418198.1|, length 147\n",
      "Record gi|16131611|ref|NP_418199.1|, length 152\n",
      "Record gi|16131612|ref|NP_418200.1|, length 330\n",
      "Record gi|49176398|ref|YP_026243.1|, length 483\n",
      "Record gi|90111646|ref|NP_418202.4|, length 498\n",
      "Record gi|49176400|ref|YP_026244.1|, length 622\n",
      "Record gi|90111647|ref|NP_418204.2|, length 139\n",
      "Record gi|16131617|ref|NP_418205.1|, length 501\n",
      "Record gi|16131618|ref|NP_418206.1|, length 321\n",
      "Record gi|16131619|ref|NP_418207.1|, length 296\n",
      "Record gi|16131620|ref|NP_418208.1|, length 309\n",
      "Record gi|16131621|ref|NP_418209.1|, length 330\n",
      "Record gi|16131622|ref|NP_418210.1|, length 475\n",
      "Record gi|49176401|ref|YP_026245.1|, length 230\n",
      "Record gi|49176402|ref|YP_026246.1|, length 279\n",
      "Record gi|16131624|ref|NP_418213.1|, length 112\n",
      "Record gi|145698331|ref|NP_418214.2|, length 506\n",
      "Record gi|16131626|ref|NP_418215.1|, length 32\n",
      "Record gi|226524760|ref|YP_002791259.1|, length 16\n",
      "Record gi|16131627|ref|NP_418217.1|, length 87\n",
      "Record gi|49176403|ref|YP_026247.1|, length 309\n",
      "Record gi|49176404|ref|YP_026248.1|, length 616\n",
      "Record gi|16131630|ref|NP_418220.1|, length 514\n",
      "Record gi|16131631|ref|NP_418221.1|, length 297\n",
      "Record gi|16131632|ref|NP_418222.1|, length 491\n",
      "Record gi|16131633|ref|NP_418223.1|, length 93\n",
      "Record gi|49176407|ref|YP_026251.1|, length 673\n",
      "Record gi|49176408|ref|YP_026252.1|, length 494\n",
      "Record gi|16131636|ref|NP_418227.1|, length 421\n",
      "Record gi|67005950|ref|NP_418228.2|, length 109\n",
      "Record gi|16131639|ref|NP_418230.1|, length 419\n",
      "Record gi|16131640|ref|NP_418231.1|, length 367\n",
      "Record gi|90111649|ref|NP_418232.2|, length 348\n",
      "Record gi|49176409|ref|YP_026253.1|, length 376\n",
      "Record gi|49176410|ref|YP_026254.1|, length 420\n",
      "Record gi|49176411|ref|YP_026255.1|, length 355\n",
      "Record gi|16131645|ref|NP_418236.1|, length 293\n",
      "Record gi|145698332|ref|YP_026256.2|, length 224\n",
      "Record gi|16131647|ref|NP_418238.1|, length 376\n",
      "Record gi|16131648|ref|NP_418239.1|, length 416\n",
      "Record gi|49176413|ref|YP_026257.1|, length 359\n",
      "Record gi|16131649|ref|NP_418241.1|, length 450\n",
      "Record gi|16131650|ref|NP_418242.1|, length 246\n",
      "Record gi|49176414|ref|YP_026258.1|, length 461\n",
      "Record gi|49176415|ref|YP_026259.1|, length 411\n",
      "Record gi|16131653|ref|NP_418245.1|, length 551\n",
      "Record gi|16131654|ref|NP_418246.1|, length 398\n",
      "Record gi|16131655|ref|NP_418247.1|, length 393\n",
      "Record gi|16131656|ref|NP_418248.1|, length 246\n",
      "Record gi|49176416|ref|YP_026260.1|, length 313\n",
      "Record gi|16131658|ref|NP_418250.1|, length 848\n",
      "Record gi|16131659|ref|NP_418251.1|, length 106\n",
      "Record gi|94541130|ref|YP_588473.1|, length 67\n",
      "Record gi|90111650|ref|NP_418254.2|, length 274\n",
      "Record gi|16131662|ref|NP_418255.1|, length 235\n",
      "Record gi|16131663|ref|NP_418256.1|, length 298\n",
      "Record gi|16131664|ref|NP_418257.1|, length 238\n",
      "Record gi|16131665|ref|NP_418258.1|, length 720\n",
      "Record gi|49176417|ref|YP_026261.1|, length 254\n",
      "Record gi|16131666|ref|NP_418260.1|, length 316\n",
      "Record gi|16131667|ref|NP_418261.1|, length 126\n",
      "Record gi|90111651|ref|NP_418262.2|, length 126\n",
      "Record gi|49176418|ref|YP_026262.1|, length 296\n",
      "Record gi|90111652|ref|NP_418264.4|, length 155\n",
      "Record gi|16131671|ref|NP_418265.1|, length 289\n",
      "Record gi|162135918|ref|YP_026263.3|, length 609\n",
      "Record gi|49176421|ref|YP_026264.1|, length 206\n",
      "Record gi|49176422|ref|YP_026265.1|, length 206\n",
      "Record gi|49176423|ref|YP_026266.1|, length 340\n",
      "Record gi|49176424|ref|YP_026267.1|, length 266\n",
      "Record gi|16131676|ref|NP_418271.1|, length 299\n",
      "Record gi|16131677|ref|NP_418272.1|, length 317\n",
      "Record gi|16131678|ref|NP_418273.1|, length 753\n",
      "Record gi|49176425|ref|YP_026268.1|, length 271\n",
      "Record gi|16131680|ref|NP_418275.1|, length 253\n",
      "Record gi|16131681|ref|NP_418276.1|, length 475\n",
      "Record gi|49176426|ref|YP_026269.1|, length 251\n",
      "Record gi|16131683|ref|NP_418278.1|, length 201\n",
      "Record gi|16131684|ref|NP_418279.1|, length 546\n",
      "Record gi|90111653|ref|NP_418280.4|, length 89\n",
      "Record gi|49176428|ref|YP_026270.1|, length 171\n",
      "Record gi|16131687|ref|NP_418282.1|, length 258\n",
      "Record gi|49176429|ref|YP_026271.1|, length 260\n",
      "Record gi|16131688|ref|NP_418284.1|, length 162\n",
      "Record gi|16131689|ref|NP_418285.1|, length 497\n",
      "Record gi|16131690|ref|NP_418286.1|, length 233\n",
      "Record gi|49176430|ref|YP_026272.1|, length 387\n",
      "Record gi|16131692|ref|NP_418288.1|, length 729\n",
      "Record gi|16131693|ref|NP_418289.1|, length 443\n",
      "Record gi|90111654|ref|NP_418290.4|, length 204\n",
      "Record gi|49176432|ref|YP_026273.1|, length 483\n",
      "Record gi|16131696|ref|NP_418292.1|, length 181\n",
      "Record gi|145698333|ref|NP_418293.2|, length 175\n",
      "Record gi|16131698|ref|NP_418294.1|, length 194\n",
      "Record gi|16131699|ref|NP_418295.1|, length 89\n",
      "Record gi|16131700|ref|NP_418296.1|, length 328\n",
      "Record gi|16131701|ref|NP_418297.1|, length 208\n",
      "Record gi|90111655|ref|NP_418298.4|, length 476\n",
      "Record gi|16131703|ref|NP_418299.1|, length 310\n",
      "Record gi|16131704|ref|NP_418300.1|, length 928\n",
      "Record gi|145698334|ref|NP_418301.3|, length 210\n",
      "Record gi|16131706|ref|NP_418302.1|, length 169\n",
      "Record gi|90111657|ref|NP_418303.2|, length 457\n",
      "Record gi|226524761|ref|YP_002791260.1|, length 36\n",
      "Record gi|16131708|ref|NP_418304.1|, length 469\n",
      "Record gi|16131709|ref|NP_418305.1|, length 349\n",
      "Record gi|16131710|ref|NP_418306.1|, length 469\n",
      "Record gi|49176434|ref|YP_026274.1|, length 607\n",
      "Record gi|16131712|ref|NP_418308.1|, length 236\n",
      "Record gi|16131713|ref|NP_418309.1|, length 326\n",
      "Record gi|16131714|ref|NP_418310.1|, length 421\n",
      "Record gi|16131715|ref|NP_418311.1|, length 230\n",
      "Record gi|49176435|ref|YP_026275.1|, length 467\n",
      "Record gi|145698335|ref|NP_418313.4|, length 461\n",
      "Record gi|16131718|ref|NP_418314.1|, length 678\n",
      "Record gi|49176437|ref|NP_418315.3|, length 308\n",
      "Record gi|90111658|ref|NP_418316.4|, length 413\n",
      "Record gi|16131721|ref|NP_418317.1|, length 292\n",
      "Record gi|16131722|ref|NP_418318.1|, length 298\n",
      "Record gi|90111659|ref|NP_418319.2|, length 298\n",
      "Record gi|90111660|ref|NP_418320.2|, length 261\n",
      "Record gi|90111661|ref|NP_418321.4|, length 199\n",
      "Record gi|16131726|ref|NP_418322.1|, length 290\n",
      "Record gi|16131727|ref|NP_418323.1|, length 145\n",
      "Record gi|16131728|ref|NP_418324.1|, length 329\n",
      "Record gi|145698336|ref|NP_418325.3|, length 70\n",
      "Record gi|90111663|ref|NP_418326.2|, length 72\n",
      "Record gi|16131731|ref|NP_418327.1|, length 309\n",
      "Record gi|16131732|ref|NP_418328.1|, length 211\n",
      "Record gi|16131733|ref|NP_418329.1|, length 300\n",
      "Record gi|16131734|ref|NP_418330.1|, length 1016\n",
      "Record gi|16131735|ref|NP_418331.1|, length 277\n",
      "Record gi|16131736|ref|NP_418332.1|, length 351\n",
      "Record gi|16131737|ref|NP_418333.1|, length 582\n",
      "Record gi|16131738|ref|NP_418334.1|, length 356\n",
      "Record gi|90111664|ref|NP_418335.4|, length 483\n",
      "Record gi|16131740|ref|NP_418336.1|, length 148\n",
      "Record gi|16131741|ref|NP_418337.1|, length 104\n",
      "Record gi|16131742|ref|NP_418338.1|, length 274\n",
      "Record gi|49176441|ref|YP_026276.1|, length 419\n",
      "Record gi|16131744|ref|NP_418340.1|, length 489\n",
      "Record gi|16131745|ref|NP_418341.1|, length 278\n",
      "Record gi|171701684|ref|NP_418342.2|, length 282\n",
      "Record gi|16131747|ref|NP_418343.1|, length 344\n",
      "Record gi|49176442|ref|NP_418344.3|, length 206\n",
      "Record gi|90111665|ref|NP_418345.2|, length 327\n",
      "Record gi|90111666|ref|NP_418346.2|, length 224\n",
      "Record gi|16131751|ref|NP_418347.1|, length 457\n",
      "Record gi|16131752|ref|NP_418348.1|, length 232\n",
      "Record gi|49176443|ref|YP_026277.1|, length 166\n",
      "Record gi|16131753|ref|NP_418350.1|, length 300\n",
      "Record gi|16131754|ref|NP_418351.1|, length 320\n",
      "Record gi|16131755|ref|NP_418352.1|, length 329\n",
      "Record gi|16131756|ref|NP_418353.1|, length 251\n",
      "Record gi|16131757|ref|NP_418354.1|, length 255\n",
      "Record gi|16131758|ref|NP_418355.1|, length 199\n",
      "Record gi|16131759|ref|NP_418356.1|, length 146\n",
      "Record gi|16131760|ref|NP_418357.1|, length 99\n",
      "Record gi|16131761|ref|NP_418358.1|, length 142\n",
      "Record gi|16131762|ref|NP_418359.1|, length 248\n",
      "Record gi|16131763|ref|NP_418360.1|, length 336\n",
      "Record gi|16131764|ref|NP_418361.1|, length 502\n",
      "Record gi|16131765|ref|NP_418362.1|, length 281\n",
      "Record gi|16131766|ref|NP_418363.1|, length 81\n",
      "Record gi|16131767|ref|NP_418364.1|, length 161\n",
      "Record gi|16131768|ref|NP_418365.1|, length 308\n",
      "Record gi|16131769|ref|NP_418366.1|, length 443\n",
      "Record gi|16131770|ref|NP_418367.1|, length 176\n",
      "Record gi|16131771|ref|NP_418368.1|, length 319\n",
      "Record gi|16131772|ref|NP_418369.1|, length 341\n",
      "Record gi|16131773|ref|NP_418370.1|, length 732\n",
      "Record gi|16131774|ref|NP_418371.1|, length 70\n",
      "Record gi|16131775|ref|NP_418372.1|, length 202\n",
      "Record gi|16131776|ref|NP_418373.1|, length 105\n",
      "Record gi|16131777|ref|NP_418374.1|, length 386\n",
      "Record gi|16131778|ref|NP_418375.1|, length 810\n",
      "Record gi|16131779|ref|NP_418376.1|, length 296\n",
      "Record gi|16131780|ref|NP_418377.1|, length 726\n",
      "Record gi|90111667|ref|NP_418378.4|, length 301\n",
      "Record gi|16131782|ref|NP_418379.1|, length 205\n",
      "Record gi|90111668|ref|NP_418380.4|, length 367\n",
      "Record gi|16131784|ref|NP_418381.1|, length 220\n",
      "Record gi|49176446|ref|YP_026278.1|, length 833\n",
      "Record gi|16131787|ref|NP_418384.1|, length 359\n",
      "Record gi|16131788|ref|NP_418385.1|, length 106\n",
      "Record gi|16131789|ref|NP_418386.1|, length 765\n",
      "Record gi|49176447|ref|NP_418387.3|, length 292\n",
      "Record gi|16131791|ref|NP_418388.1|, length 113\n",
      "Record gi|16131792|ref|NP_418389.1|, length 283\n",
      "Record gi|16131793|ref|NP_418390.1|, length 577\n",
      "Record gi|16131794|ref|NP_418391.1|, length 883\n",
      "Record gi|16131795|ref|NP_418392.1|, length 383\n",
      "Record gi|16131796|ref|NP_418393.1|, length 334\n",
      "Record gi|145698337|ref|NP_418394.3|, length 258\n",
      "Record gi|16131798|ref|NP_418395.1|, length 457\n",
      "Record gi|16131799|ref|NP_418396.1|, length 305\n",
      "Record gi|90111670|ref|NP_418397.2|, length 466\n",
      "Record gi|145698338|ref|NP_418398.2|, length 215\n",
      "Record gi|16131802|ref|NP_418399.1|, length 119\n",
      "Record gi|16131803|ref|NP_418400.1|, length 366\n",
      "Record gi|16131804|ref|NP_418401.1|, length 614\n",
      "Record gi|90111671|ref|NP_418402.2|, length 285\n",
      "Record gi|16131806|ref|NP_418403.1|, length 342\n",
      "Record gi|16131807|ref|NP_418404.1|, length 321\n",
      "Record gi|16131808|ref|NP_418405.1|, length 316\n",
      "Record gi|16131810|ref|NP_418407.1|, length 394\n",
      "Record gi|16131811|ref|NP_418408.1|, length 127\n",
      "Record gi|16131812|ref|NP_418409.1|, length 181\n",
      "Record gi|16131813|ref|NP_418410.1|, length 142\n",
      "Record gi|16131814|ref|NP_418411.1|, length 234\n",
      "Record gi|16131815|ref|NP_418412.1|, length 165\n",
      "Record gi|16131816|ref|NP_418413.1|, length 121\n",
      "Record gi|16131817|ref|NP_418414.1|, length 1342\n",
      "Record gi|16131818|ref|NP_418415.1|, length 1407\n",
      "Record gi|16131819|ref|NP_418416.1|, length 179\n",
      "Record gi|16131820|ref|NP_418417.1|, length 377\n",
      "Record gi|33347812|ref|NP_418418.2|, length 256\n",
      "Record gi|49176448|ref|YP_026279.1|, length 66\n",
      "Record gi|90111672|ref|NP_418420.4|, length 251\n",
      "Record gi|16131823|ref|NP_418421.1|, length 211\n",
      "Record gi|16131824|ref|NP_418422.1|, length 631\n",
      "Record gi|16131825|ref|NP_418423.1|, length 158\n",
      "Record gi|49176450|ref|YP_026280.1|, length 257\n",
      "Record gi|16131827|ref|NP_418425.1|, length 354\n",
      "Record gi|90111673|ref|NP_418426.2|, length 223\n",
      "Record gi|16131829|ref|NP_418427.1|, length 196\n",
      "Record gi|16131830|ref|NP_418428.1|, length 90\n",
      "Record gi|16131831|ref|NP_418429.1|, length 231\n",
      "Record gi|90111674|ref|NP_418430.2|, length 141\n",
      "Record gi|16131833|ref|NP_418431.1|, length 465\n",
      "Record gi|16131834|ref|NP_418432.1|, length 441\n",
      "Record gi|16131835|ref|NP_418433.1|, length 429\n",
      "Record gi|16131836|ref|NP_418434.1|, length 529\n",
      "Record gi|16131837|ref|NP_418435.1|, length 127\n",
      "Record gi|16131838|ref|NP_418436.1|, length 147\n",
      "Record gi|16131839|ref|NP_418437.1|, length 309\n",
      "Record gi|16131840|ref|NP_418438.1|, length 533\n",
      "Record gi|16131841|ref|NP_418439.1|, length 434\n",
      "Record gi|16131842|ref|NP_418440.1|, length 578\n",
      "Record gi|16131843|ref|NP_418441.1|, length 728\n",
      "Record gi|90111675|ref|NP_418442.2|, length 274\n",
      "Record gi|16131845|ref|NP_418443.1|, length 1227\n",
      "Record gi|16131846|ref|NP_418444.1|, length 543\n",
      "Record gi|16131847|ref|NP_418445.1|, length 229\n",
      "Record gi|16131848|ref|NP_418446.1|, length 290\n",
      "Record gi|16131849|ref|NP_418447.1|, length 90\n",
      "Record gi|16131850|ref|NP_418448.1|, length 449\n",
      "Record gi|16131851|ref|NP_418449.1|, length 549\n",
      "Record gi|16131852|ref|NP_418450.1|, length 80\n",
      "Record gi|90111676|ref|NP_418451.4|, length 212\n",
      "Record gi|16131854|ref|NP_418452.1|, length 245\n",
      "Record gi|16131855|ref|NP_418453.1|, length 698\n",
      "Record gi|145698339|ref|YP_001165332.1|, length 92\n",
      "Record gi|16131856|ref|NP_418454.1|, length 136\n",
      "Record gi|16131857|ref|NP_418455.1|, length 491\n",
      "Record gi|16131858|ref|NP_418456.1|, length 296\n",
      "Record gi|16131859|ref|NP_418457.1|, length 514\n",
      "Record gi|16131860|ref|NP_418458.1|, length 396\n",
      "Record gi|16131861|ref|NP_418459.1|, length 371\n",
      "Record gi|16131862|ref|NP_418460.1|, length 446\n",
      "Record gi|16131863|ref|NP_418461.1|, length 306\n",
      "Record gi|90111677|ref|NP_418463.4|, length 165\n",
      "Record gi|16131866|ref|NP_418464.1|, length 290\n",
      "Record gi|90111678|ref|NP_418465.4|, length 807\n",
      "Record gi|16131868|ref|NP_418466.1|, length 122\n",
      "Record gi|16131869|ref|NP_418467.1|, length 202\n",
      "Record gi|162135919|ref|NP_418468.3|, length 459\n",
      "Record gi|16131871|ref|NP_418469.1|, length 69\n",
      "Record gi|90111679|ref|NP_418470.4|, length 171\n",
      "Record gi|16131873|ref|NP_418471.1|, length 84\n",
      "Record gi|16131874|ref|NP_418472.1|, length 235\n",
      "Record gi|635341467|ref|NP_418473.3|, length 345\n",
      "Record gi|90111680|ref|NP_418474.4|, length 80\n",
      "Record gi|16131877|ref|NP_418475.1|, length 327\n",
      "Record gi|16131878|ref|NP_418476.1|, length 471\n",
      "Record gi|16131879|ref|NP_418477.1|, length 359\n",
      "Record gi|16131880|ref|NP_418478.1|, length 397\n",
      "Record gi|145698341|ref|YP_001165333.1|, length 67\n",
      "Record gi|16131881|ref|NP_418479.1|, length 237\n",
      "Record gi|16131882|ref|NP_418480.1|, length 138\n",
      "Record gi|16131883|ref|NP_418481.1|, length 118\n",
      "Record gi|16131884|ref|NP_418482.1|, length 940\n",
      "Record gi|16131885|ref|NP_418483.1|, length 178\n",
      "Record gi|90111681|ref|NP_418484.4|, length 93\n",
      "Record gi|16131887|ref|NP_418485.1|, length 528\n",
      "Record gi|16131888|ref|NP_418486.1|, length 107\n",
      "Record gi|16131889|ref|NP_418487.1|, length 154\n",
      "Record gi|16131890|ref|NP_418488.1|, length 449\n",
      "Record gi|16131891|ref|NP_418489.1|, length 549\n",
      "Record gi|16131892|ref|NP_418490.1|, length 430\n",
      "Record gi|16131893|ref|NP_418491.1|, length 549\n",
      "Record gi|16131894|ref|NP_418492.1|, length 104\n",
      "Record gi|16131895|ref|NP_418493.1|, length 652\n",
      "Record gi|16131896|ref|NP_418494.1|, length 478\n",
      "Record gi|90111682|ref|NP_418495.2|, length 188\n",
      "Record gi|16131898|ref|NP_418496.1|, length 223\n",
      "Record gi|16131899|ref|NP_418497.1|, length 318\n",
      "Record gi|16131900|ref|NP_418498.1|, length 552\n",
      "Record gi|16131901|ref|NP_418499.1|, length 127\n",
      "Record gi|16131902|ref|NP_418500.1|, length 198\n",
      "Record gi|16131903|ref|NP_418501.1|, length 437\n",
      "Record gi|16131904|ref|NP_418502.1|, length 229\n",
      "Record gi|16131905|ref|NP_418503.1|, length 715\n",
      "Record gi|16131906|ref|NP_418504.1|, length 488\n",
      "Record gi|90111683|ref|NP_418505.2|, length 683\n",
      "Record gi|16131908|ref|NP_418506.1|, length 343\n",
      "Record gi|145698342|ref|YP_001165334.1|, length 91\n",
      "Record gi|90111684|ref|NP_418507.2|, length 661\n",
      "Record gi|16131910|ref|NP_418508.1|, length 309\n",
      "Record gi|16131911|ref|NP_418509.1|, length 231\n",
      "Record gi|16131912|ref|NP_418510.1|, length 326\n",
      "Record gi|16131913|ref|NP_418511.1|, length 510\n",
      "Record gi|16131914|ref|NP_418512.1|, length 311\n",
      "Record gi|90111685|ref|NP_418513.4|, length 296\n",
      "Record gi|16131916|ref|NP_418514.1|, length 149\n",
      "Record gi|49176458|ref|YP_026281.1|, length 109\n",
      "Record gi|16131918|ref|NP_418516.1|, length 252\n",
      "Record gi|16131919|ref|NP_418517.1|, length 144\n",
      "Record gi|16131920|ref|NP_418518.1|, length 185\n",
      "Record gi|16131921|ref|NP_418519.1|, length 378\n",
      "Record gi|16131922|ref|NP_418520.1|, length 226\n",
      "Record gi|49176459|ref|YP_026282.1|, length 252\n",
      "Record gi|16131924|ref|NP_418522.1|, length 281\n",
      "Record gi|16131925|ref|NP_418523.1|, length 354\n",
      "Record gi|16131926|ref|NP_418524.1|, length 194\n",
      "Record gi|16131927|ref|NP_418525.1|, length 150\n",
      "Record gi|16131928|ref|NP_418526.1|, length 241\n",
      "Record gi|16131931|ref|NP_418529.1|, length 338\n",
      "Record gi|16131932|ref|NP_418530.1|, length 262\n",
      "Record gi|16131933|ref|NP_418531.1|, length 147\n",
      "Record gi|16131934|ref|NP_418532.1|, length 111\n",
      "Record gi|16131935|ref|NP_418533.1|, length 742\n",
      "Record gi|90111687|ref|NP_418534.4|, length 292\n",
      "Record gi|16131937|ref|NP_418535.1|, length 500\n",
      "Record gi|345452724|ref|YP_004831120.1|, length 29\n",
      "Record gi|16131938|ref|NP_418536.1|, length 363\n",
      "Record gi|16131939|ref|NP_418537.1|, length 222\n",
      "Record gi|90111688|ref|NP_418538.2|, length 547\n",
      "Record gi|16131941|ref|NP_418539.1|, length 445\n",
      "Record gi|16131942|ref|NP_418540.1|, length 253\n",
      "Record gi|221800782|ref|NP_418541.2|, length 755\n",
      "Record gi|16131944|ref|NP_418542.1|, length 302\n",
      "Record gi|16131945|ref|NP_418543.1|, length 451\n",
      "Record gi|345452725|ref|NP_418544.2|, length 473\n",
      "Record gi|16131947|ref|NP_418545.1|, length 209\n",
      "Record gi|16131948|ref|NP_418546.1|, length 548\n",
      "Record gi|16131949|ref|NP_418547.1|, length 446\n",
      "Record gi|16131950|ref|NP_418548.1|, length 239\n",
      "Record gi|16131951|ref|NP_418549.1|, length 543\n",
      "Record gi|16131952|ref|NP_418550.1|, length 76\n",
      "Record gi|16131953|ref|NP_418551.1|, length 90\n",
      "Record gi|16131954|ref|NP_418552.1|, length 98\n",
      "Record gi|94541131|ref|YP_588474.1|, length 57\n",
      "Record gi|16131955|ref|NP_418553.1|, length 505\n",
      "Record gi|16131956|ref|NP_418554.1|, length 485\n",
      "Record gi|16131957|ref|NP_418555.1|, length 715\n",
      "Record gi|16131958|ref|NP_418556.1|, length 444\n",
      "Record gi|16131959|ref|NP_418557.1|, length 512\n",
      "Record gi|90111689|ref|NP_418558.2|, length 191\n",
      "Record gi|16131961|ref|NP_418559.1|, length 565\n",
      "Record gi|16131962|ref|NP_418560.1|, length 112\n",
      "Record gi|16131963|ref|NP_418561.1|, length 433\n",
      "Record gi|90111690|ref|NP_418562.4|, length 478\n",
      "Record gi|90111691|ref|NP_418564.2|, length 158\n",
      "Record gi|16131966|ref|NP_418565.1|, length 418\n",
      "Record gi|16131967|ref|NP_418566.1|, length 97\n",
      "Record gi|16131968|ref|NP_418567.1|, length 548\n",
      "Record gi|90111692|ref|NP_418568.2|, length 117\n",
      "Record gi|16131970|ref|NP_418569.1|, length 289\n",
      "Record gi|16131971|ref|NP_418570.1|, length 342\n",
      "Record gi|16131972|ref|NP_418571.1|, length 188\n",
      "Record gi|49176462|ref|YP_026283.1|, length 41\n",
      "Record gi|49176463|ref|YP_026284.1|, length 48\n",
      "Record gi|90111693|ref|NP_418572.4|, length 105\n",
      "Record gi|16131974|ref|NP_418573.1|, length 177\n",
      "Record gi|16131975|ref|NP_418574.1|, length 377\n",
      "Record gi|16131976|ref|NP_418575.1|, length 119\n",
      "Record gi|16131977|ref|NP_418576.1|, length 131\n",
      "Record gi|16131978|ref|NP_418577.1|, length 244\n",
      "Record gi|16131979|ref|NP_418578.1|, length 602\n",
      "Record gi|90111694|ref|NP_418579.2|, length 325\n",
      "Record gi|90111695|ref|NP_418580.2|, length 500\n",
      "Record gi|16131982|ref|NP_418581.1|, length 104\n",
      "Record gi|16131983|ref|NP_418582.1|, length 104\n",
      "Record gi|16131984|ref|NP_418583.1|, length 1107\n",
      "Record gi|16131985|ref|NP_418584.1|, length 322\n",
      "Record gi|90111696|ref|NP_418585.4|, length 350\n",
      "Record gi|90111697|ref|NP_418586.4|, length 181\n",
      "Record gi|226524762|ref|YP_002791261.1|, length 17\n",
      "Record gi|16131988|ref|NP_418587.1|, length 379\n",
      "Record gi|16131989|ref|NP_418588.1|, length 515\n",
      "Record gi|16131990|ref|NP_418589.1|, length 153\n",
      "Record gi|16131991|ref|NP_418590.1|, length 445\n",
      "Record gi|16131992|ref|NP_418591.1|, length 615\n",
      "Record gi|16131993|ref|NP_418592.1|, length 316\n",
      "Record gi|16131994|ref|NP_418593.1|, length 102\n",
      "Record gi|16131995|ref|NP_418594.1|, length 426\n",
      "Record gi|16131996|ref|NP_418595.1|, length 419\n",
      "Record gi|16131997|ref|NP_418596.1|, length 334\n",
      "Record gi|16131998|ref|NP_418597.1|, length 65\n",
      "Record gi|16131999|ref|NP_418598.1|, length 432\n",
      "Record gi|16132000|ref|NP_418599.1|, length 141\n",
      "Record gi|90111698|ref|NP_418600.4|, length 813\n",
      "Record gi|16132002|ref|NP_418601.1|, length 243\n",
      "Record gi|16132003|ref|NP_418602.1|, length 133\n",
      "Record gi|16132004|ref|NP_418603.1|, length 232\n",
      "Record gi|16132005|ref|NP_418604.1|, length 219\n",
      "Record gi|16132006|ref|NP_418605.1|, length 132\n",
      "Record gi|16132007|ref|NP_418606.1|, length 212\n",
      "Record gi|16132008|ref|NP_418607.1|, length 387\n",
      "Record gi|162135915|ref|NP_418608.6|, length 541\n",
      "Record gi|90111700|ref|NP_418609.4|, length 91\n",
      "Record gi|90111701|ref|NP_418610.4|, length 109\n",
      "Record gi|16132012|ref|NP_418611.1|, length 249\n",
      "Record gi|16132013|ref|NP_418612.1|, length 251\n",
      "Record gi|90111702|ref|NP_418613.2|, length 354\n",
      "Record gi|90111703|ref|NP_418614.4|, length 465\n",
      "Record gi|16132016|ref|NP_418615.1|, length 101\n",
      "Record gi|16132017|ref|NP_418616.1|, length 154\n",
      "Record gi|16132018|ref|NP_418617.1|, length 216\n",
      "Record gi|16132019|ref|NP_418618.1|, length 284\n",
      "Record gi|16132020|ref|NP_418619.1|, length 228\n",
      "Record gi|16132021|ref|NP_418620.1|, length 91\n",
      "Record gi|296044035|ref|NP_418621.5|, length 131\n",
      "Record gi|16132023|ref|NP_418622.1|, length 104\n",
      "Record gi|16132024|ref|NP_418623.1|, length 75\n",
      "Record gi|16132025|ref|NP_418624.1|, length 149\n",
      "Record gi|16132026|ref|NP_418625.1|, length 264\n",
      "Record gi|162135916|ref|NP_418627.4|, length 212\n",
      "Record gi|90111705|ref|NP_418628.4|, length 206\n",
      "Record gi|16132030|ref|NP_418629.1|, length 470\n",
      "Record gi|16132031|ref|NP_418630.1|, length 220\n",
      "Record gi|226524763|ref|NP_418631.2|, length 321\n",
      "Record gi|16132033|ref|NP_418632.1|, length 286\n",
      "Record gi|90111706|ref|NP_418633.4|, length 126\n",
      "Record gi|16132035|ref|NP_418634.1|, length 647\n",
      "Record gi|16132036|ref|NP_418635.1|, length 246\n",
      "Record gi|49176474|ref|YP_026285.1|, length 312\n",
      "Record gi|16132038|ref|NP_418637.1|, length 184\n",
      "Record gi|90111707|ref|NP_418638.2|, length 68\n",
      "Record gi|16132040|ref|NP_418639.1|, length 447\n",
      "Record gi|16132041|ref|NP_418640.1|, length 212\n",
      "Record gi|16132042|ref|NP_418641.1|, length 577\n",
      "Record gi|16132043|ref|NP_418642.1|, length 1259\n",
      "Record gi|16132044|ref|NP_418643.1|, length 113\n",
      "Record gi|90111708|ref|NP_418645.2|, length 83\n",
      "Record gi|16132047|ref|NP_418646.1|, length 116\n",
      "Record gi|16132048|ref|NP_418647.1|, length 176\n",
      "Record gi|16132049|ref|NP_418648.1|, length 318\n",
      "Record gi|49176475|ref|YP_026286.1|, length 500\n",
      "Record gi|145698343|ref|NP_418651.3|, length 341\n",
      "Record gi|90111710|ref|NP_418652.2|, length 331\n",
      "Record gi|16132054|ref|NP_418653.1|, length 332\n",
      "Record gi|16132055|ref|NP_418654.1|, length 457\n",
      "Record gi|16132056|ref|NP_418655.1|, length 183\n",
      "Record gi|16132057|ref|NP_418656.1|, length 450\n",
      "Record gi|16132059|ref|NP_418658.1|, length 154\n",
      "Record gi|16132060|ref|NP_418659.1|, length 712\n",
      "Record gi|16132061|ref|NP_418660.1|, length 551\n",
      "Record gi|16132062|ref|NP_418661.1|, length 473\n",
      "Record gi|16132063|ref|NP_418662.1|, length 315\n",
      "Record gi|308209623|ref|YP_003933616.1|, length 17\n",
      "Record gi|16132064|ref|NP_418663.1|, length 898\n",
      "Record gi|90111711|ref|NP_418664.2|, length 128\n",
      "Record gi|16132066|ref|NP_418665.1|, length 153\n",
      "Record gi|16132067|ref|NP_418666.1|, length 311\n",
      "Record gi|16132068|ref|NP_418667.1|, length 44\n",
      "Record gi|16132070|ref|NP_418669.1|, length 131\n",
      "Record gi|16132071|ref|NP_418670.1|, length 237\n",
      "Record gi|90111712|ref|NP_418672.4|, length 197\n",
      "Record gi|90111713|ref|NP_418673.4|, length 150\n",
      "Record gi|90111714|ref|NP_418674.2|, length 604\n",
      "Record gi|16132076|ref|NP_418675.1|, length 334\n",
      "Record gi|16132077|ref|NP_418676.1|, length 138\n",
      "Record gi|49176479|ref|YP_026287.1|, length 167\n",
      "Record gi|49176480|ref|YP_026288.1|, length 398\n",
      "Record gi|16132080|ref|NP_418679.1|, length 951\n",
      "Record gi|16132081|ref|NP_418680.1|, length 147\n",
      "Record gi|16132082|ref|NP_418681.1|, length 503\n",
      "Record gi|16132083|ref|NP_418682.1|, length 366\n",
      "Record gi|90111715|ref|NP_418683.4|, length 360\n",
      "Record gi|16132085|ref|NP_418684.1|, length 500\n",
      "Record gi|16132086|ref|NP_418685.1|, length 332\n",
      "Record gi|16132087|ref|NP_418686.1|, length 439\n",
      "Record gi|16132088|ref|NP_418687.1|, length 254\n",
      "Record gi|16132089|ref|NP_418688.1|, length 343\n",
      "Record gi|16132090|ref|NP_418689.1|, length 187\n",
      "Record gi|90111716|ref|NP_418690.4|, length 339\n",
      "Record gi|145698344|ref|NP_418692.2|, length 121\n",
      "Record gi|16132094|ref|NP_418693.1|, length 301\n",
      "Record gi|16132098|ref|NP_418697.1|, length 109\n",
      "Record gi|16132099|ref|NP_418698.1|, length 442\n",
      "Record gi|90111718|ref|NP_418699.4|, length 405\n",
      "Record gi|90111719|ref|NP_418700.2|, length 372\n",
      "Record gi|157783151|ref|YP_001491548.1|, length 41\n",
      "Record gi|16132105|ref|NP_418704.1|, length 383\n",
      "Record gi|16132108|ref|NP_418707.1|, length 255\n",
      "Record gi|16132109|ref|NP_418708.1|, length 318\n",
      "Record gi|16132110|ref|NP_418709.1|, length 332\n",
      "Record gi|162135917|ref|NP_418710.4|, length 300\n",
      "Record gi|16132112|ref|NP_418711.1|, length 774\n",
      "Record gi|16132113|ref|NP_418712.1|, length 317\n",
      "Record gi|16132114|ref|NP_418713.1|, length 173\n",
      "Record gi|16132115|ref|NP_418714.1|, length 91\n",
      "Record gi|90111724|ref|NP_418715.2|, length 328\n",
      "Record gi|16132117|ref|NP_418716.1|, length 449\n",
      "Record gi|16132118|ref|NP_418717.1|, length 655\n",
      "Record gi|90111725|ref|NP_418718.4|, length 301\n",
      "Record gi|16132120|ref|NP_418719.1|, length 262\n",
      "Record gi|16132121|ref|NP_418720.1|, length 260\n",
      "Record gi|16132122|ref|NP_418721.1|, length 210\n",
      "Record gi|16132123|ref|NP_418722.1|, length 143\n",
      "Record gi|16132124|ref|NP_418723.1|, length 268\n",
      "Record gi|16132125|ref|NP_418724.1|, length 437\n",
      "Record gi|94541133|ref|YP_588476.1|, length 92\n",
      "Record gi|90111726|ref|NP_418725.4|, length 373\n",
      "Record gi|16132127|ref|NP_418726.1|, length 248\n",
      "Record gi|16132128|ref|NP_418727.1|, length 181\n",
      "Record gi|94541138|ref|YP_588477.1|, length 85\n",
      "Record gi|16132130|ref|NP_418729.1|, length 326\n",
      "Record gi|90111727|ref|NP_418730.4|, length 368\n",
      "Record gi|90111728|ref|NP_418731.2|, length 238\n",
      "Record gi|16132133|ref|NP_418732.1|, length 200\n",
      "Record gi|16132134|ref|NP_418733.1|, length 198\n",
      "Record gi|16132135|ref|NP_418734.1|, length 182\n",
      "Record gi|90111729|ref|NP_418735.2|, length 179\n",
      "Record gi|49176488|ref|NP_418736.3|, length 241\n",
      "Record gi|16132138|ref|NP_418737.1|, length 878\n",
      "Record gi|16132139|ref|NP_418738.1|, length 176\n",
      "Record gi|16132140|ref|NP_418739.1|, length 167\n",
      "Record gi|16132141|ref|NP_418740.1|, length 300\n",
      "Record gi|16132142|ref|NP_418741.1|, length 447\n",
      "Record gi|16132143|ref|NP_418742.1|, length 394\n",
      "Record gi|16132144|ref|NP_418743.1|, length 486\n",
      "Record gi|16132145|ref|NP_418744.1|, length 257\n",
      "Record gi|49176489|ref|NP_418745.3|, length 276\n",
      "Record gi|90111730|ref|NP_418746.4|, length 130\n",
      "Record gi|16132148|ref|NP_418747.1|, length 303\n",
      "Record gi|16132149|ref|NP_418748.1|, length 390\n",
      "Record gi|16132150|ref|NP_418749.1|, length 153\n",
      "Record gi|90111731|ref|NP_418750.2|, length 227\n",
      "Record gi|90111732|ref|NP_418751.4|, length 184\n",
      "Record gi|16132153|ref|NP_418752.1|, length 392\n",
      "Record gi|145698345|ref|NP_418753.3|, length 286\n",
      "Record gi|90111734|ref|NP_418754.2|, length 255\n",
      "Record gi|90111735|ref|NP_418755.4|, length 383\n",
      "Record gi|16132157|ref|NP_418756.1|, length 426\n",
      "Record gi|16132158|ref|NP_418757.1|, length 410\n",
      "Record gi|16132161|ref|NP_418760.1|, length 470\n",
      "Record gi|16132162|ref|NP_418761.1|, length 54\n",
      "Record gi|16132166|ref|NP_418765.1|, length 348\n",
      "Record gi|90111737|ref|NP_418766.4|, length 459\n",
      "Record gi|145698346|ref|NP_418767.4|, length 113\n",
      "Record gi|16132169|ref|NP_418768.1|, length 464\n",
      "Record gi|16132170|ref|NP_418769.1|, length 529\n",
      "Record gi|226524764|ref|NP_418770.2|, length 1170\n",
      "Record gi|16132172|ref|NP_418771.1|, length 304\n",
      "Record gi|90111738|ref|NP_418772.4|, length 318\n",
      "Record gi|16132174|ref|NP_418773.1|, length 67\n",
      "Record gi|90111739|ref|NP_418774.2|, length 716\n",
      "Record gi|16132176|ref|NP_418775.1|, length 551\n",
      "Record gi|16132177|ref|NP_418776.1|, length 453\n",
      "Record gi|49176497|ref|YP_026290.1|, length 304\n",
      "Record gi|226524765|ref|NP_418778.4|, length 340\n",
      "Record gi|90111741|ref|NP_418779.2|, length 763\n",
      "Record gi|90111742|ref|NP_418780.2|, length 164\n",
      "Record gi|16132182|ref|NP_418781.1|, length 245\n",
      "Record gi|16132183|ref|NP_418782.1|, length 179\n",
      "Record gi|145698348|ref|NP_418783.2|, length 157\n",
      "Record gi|90111743|ref|NP_418784.4|, length 256\n",
      "Record gi|16132186|ref|NP_418785.1|, length 241\n",
      "Record gi|345452726|ref|NP_418786.4|, length 225\n",
      "Record gi|16132188|ref|NP_418787.1|, length 262\n",
      "Record gi|94541134|ref|YP_588478.1|, length 78\n",
      "Record gi|16132189|ref|NP_418788.1|, length 343\n",
      "Record gi|16132190|ref|NP_418789.1|, length 137\n",
      "Record gi|16132191|ref|NP_418790.1|, length 148\n",
      "Record gi|16132192|ref|NP_418791.1|, length 225\n",
      "Record gi|16132193|ref|NP_418792.1|, length 529\n",
      "Record gi|16132194|ref|NP_418793.1|, length 201\n",
      "Record gi|145698350|ref|YP_588479.2|, length 53\n",
      "Record gi|16132195|ref|NP_418794.1|, length 357\n",
      "Record gi|90111745|ref|YP_026291.2|, length 259\n",
      "Record gi|16132196|ref|NP_418796.1|, length 287\n",
      "Record gi|16132197|ref|NP_418797.1|, length 516\n",
      "Record gi|16132198|ref|NP_418798.1|, length 259\n",
      "Record gi|16132199|ref|NP_418799.1|, length 440\n",
      "Record gi|16132200|ref|NP_418800.1|, length 407\n",
      "Record gi|16132201|ref|NP_418801.1|, length 239\n",
      "Record gi|16132202|ref|NP_418802.1|, length 443\n",
      "Record gi|16132203|ref|NP_418803.1|, length 338\n",
      "Record gi|16132204|ref|NP_418804.1|, length 214\n",
      "Record gi|16132205|ref|NP_418805.1|, length 322\n",
      "Record gi|16132206|ref|NP_418806.1|, length 460\n",
      "Record gi|90111746|ref|NP_418807.4|, length 410\n",
      "Record gi|16132208|ref|NP_418808.1|, length 555\n",
      "Record gi|90111747|ref|NP_418809.4|, length 645\n",
      "Record gi|16132210|ref|NP_418810.1|, length 108\n",
      "Record gi|90111748|ref|NP_418811.2|, length 170\n",
      "Record gi|16132212|ref|NP_418812.1|, length 215\n",
      "Record gi|16132213|ref|NP_418813.1|, length 289\n",
      "Record gi|16132214|ref|NP_418814.1|, length 157\n",
      "Record gi|16132215|ref|NP_418815.1|, length 229\n",
      "Record gi|16132216|ref|NP_418816.1|, length 474\n",
      "Record gi|16132217|ref|NP_418817.1|, length 450\n",
      "Record gi|16132218|ref|NP_418818.1|, length 238\n",
      "Record gi|16132219|ref|NP_418819.1|, length 46\n",
      "Record gi|16132220|ref|NP_418820.1|, length 228\n"
     ]
    }
   ],
   "source": [
    "from Bio import SeqIO\n",
    "filename = \"NC_000913.faa\"\n",
    "for record in SeqIO.parse(filename, \"fasta\"):\n",
    "    print(\"Record \" + record.id + \", length \" + str(len(record.seq)))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Excercise 1:** - Warm up - Count how many sequences are less than 100 amino acids long."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "420\n"
     ]
    }
   ],
   "source": [
    "#Solution\n",
    "count= 0\n",
    "for record in SeqIO.parse(filename, \"fasta\"):\n",
    "    if len(record.seq) < 100:\n",
    "        count = count + 1\n",
    "print(count)\n",
    "    \n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    " **Exercise 2:** - Warm up - Plot a histogram of the sequence length distribution"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAYgAAAEKCAYAAAAIO8L1AAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDIuMi4yLCBodHRwOi8vbWF0cGxvdGxpYi5vcmcvhp/UCwAAFSZJREFUeJzt3X2wZVV95vHvI2++QIom9HS10E6jtiaoSQsdRCEZjQ4CZqo1YQhWSjoOsVMlZELlZabFmZGpGapwRs1oRkm1kQgpAiERAglERIJSsRTsZqB5C9KRpqCnhSa+ZmJIaH/zx1kXDu2+r937nnvv+X6qbp191tl7n7V3n3ufXmvts3aqCkmS9va8UVdAkrQwGRCSpE4GhCSpkwEhSepkQEiSOhkQkqROBoQkqZMBIUnqZEBIkjod2NeOk6wCLgdWAAVsrqqPJrkQeA+wu616QVXd2LZ5H3AOsAf491V101TvceSRR9bq1av7OQBJWqK2bt36ZFUtn2693gICeBr4zaq6M8lhwNYkN7fXfqeqPjS8cpJjgbOAVwEvBj6f5BVVtWeyN1i9ejVbtmzpqfqStDQleWQm6/XWxVRVu6rqzrb8PeAB4KgpNlkPXFVVT1XVw8B24IS+6idJmtq8jEEkWQ28Fri9FZ2XZFuSS5Msa2VHAY8ObfYYUweKJKlHvQdEkkOBzwDnV9V3gUuAlwFrgV3Ah2e5v41JtiTZsnv37uk3kCTNSa8BkeQgBuFwRVVdA1BVj1fVnqr6AfBJnu1G2gmsGtr86Fb2HFW1uarWVdW65cunHWORJM1RbwGRJMCngAeq6iND5SuHVnsHcG9bvh44K8khSY4B1gB39FU/SdLU+ryK6STgXcA9Se5qZRcA70yylsGlrzuAXwWoqvuSXA3cz+AKqHOnuoJJktSv3gKiqv4aSMdLN06xzUXARX3VSZI0c36TWpLUyYCQJHXqcwxCs7R60w3PLO+4+G0jrIkk2YKQJE3CgJAkdTIgJEmdHIOYJ44vSFpsbEFIkjoZEJKkTgaEJKmTASFJ6mRASJI6GRCSpE4GhCSpkwEhSerkF+VGwC/NSVoMbEFIkjoZEJKkTnYx7Wd2H0laKmxBSJI6GRCSpE4GhCSpkwEhSepkQEiSOhkQkqROXuY6C17CKmmc2IKQJHWyBdGj4RaHJC02BsQCZXeWpFGzi0mS1MmAkCR1sotpP3CsQdJSZAtCktTJgJAkdeotIJKsSnJrkvuT3Jfk11v5EUluTvJQe1zWypPkY0m2J9mW5Li+6iZJml6fLYingd+sqmOBE4FzkxwLbAJuqao1wC3tOcBpwJr2sxG4pMe6SZKm0VtAVNWuqrqzLX8PeAA4ClgPXNZWuwx4e1teD1xeA18BDk+ysq/6SZKmNi9XMSVZDbwWuB1YUVW72kvfAFa05aOAR4c2e6yV7WIJ8wooSQtV74PUSQ4FPgOcX1XfHX6tqgqoWe5vY5ItSbbs3r17P9ZUkjSs14BIchCDcLiiqq5pxY9PdB21xyda+U5g1dDmR7ey56iqzVW1rqrWLV++vL/KS9KY6/MqpgCfAh6oqo8MvXQ9sKEtbwCuGyo/u13NdCLwnaGuKEnSPOtzDOIk4F3APUnuamUXABcDVyc5B3gEOLO9diNwOrAd+Afg3T3WTZI0jd4Coqr+GsgkL7+5Y/0Czu2rPpKk2XEupjny6iNJS51TbUiSOhkQkqROdjFNw64kSePKFoQkqZMBIUnqZEBIkjoZEJKkTgaEJKmTASFJ6mRASJI6GRCSpE4GhCSpkwEhSepkQEiSOhkQkqROBoQkqZMBIUnqZEBIkjp5P4gO3gNCkmxBSJImYUBIkjoZEJKkTgaEJKmTASFJ6mRASJI6GRCSpE5+D2IRGP5exo6L3zbCmkgaJ7YgJEmdDAhJUicDQpLUyTGIRcbxCEnzxRaEJKmTASFJ6tRbQCS5NMkTSe4dKrswyc4kd7Wf04dee1+S7UkeTPLWvuolSZqZPlsQnwZO7Sj/napa235uBEhyLHAW8Kq2zSeSHNBj3SRJ0+gtIKrqNuCbM1x9PXBVVT1VVQ8D24ET+qqbJGl6oxiDOC/JttYFtayVHQU8OrTOY63shyTZmGRLki27d+/uu66SNLbmOyAuAV4GrAV2AR+e7Q6qanNVrauqdcuXL9/f9ZMkNfMaEFX1eFXtqaofAJ/k2W6kncCqoVWPbmWSpBGZ14BIsnLo6TuAiSucrgfOSnJIkmOANcAd81k3SdJz9fZN6iRXAm8EjkzyGPAB4I1J1gIF7AB+FaCq7ktyNXA/8DRwblXt6atukqTp9RYQVfXOjuJPTbH+RcBFfdVHkjQ7fpNaktRpRgGR5KSZlEmSlo6ZdjH9LnDcDMo0j5zZVVKfpgyIJK8H3gAsT/IbQy/9COBUGJK0hE3XgjgYOLStd9hQ+XeBM/qqlCRp9KYMiKr6IvDFJJ+uqkfmqU6SpAVgpmMQhyTZDKwe3qaqfraPSmnfODYhaX+YaUD8CfB7wO8DfoFNksbATAPi6aq6pNeaSJIWlJkGxJ8neS9wLfDURGFVzfR+D+rZcLeSJO0PMw2IDe3xt4fKCnjp/q2OJGmhmFFAVNUxfVdEkrSwzCggkpzdVV5Vl+/f6kiSFoqZdjH91NDy84E3A3cCBoQkLVEz7WL6teHnSQ4HruqlRpKkBWGu033/P8BxCUlawmY6BvHnDK5agsEkfT8OXN1XpSRJozfTMYgPDS0/DTxSVY/1UB/tZ067IWmuZtTF1Cbt+xsGM7ouA/6pz0pJkkZvpneUOxO4A/i3wJnA7Umc7luSlrCZdjG9H/ipqnoCIMly4PPAn/ZVMUnSaM30KqbnTYRD83ez2FaStAjNtAXx2SQ3AVe2578I3NhPlSRJC8F096R+ObCiqn47yc8DJ7eXvgxc0XflJEmjM10L4n8B7wOoqmuAawCSvKa99m96rZ0kaWSmG0dYUVX37F3Yylb3UiNJ0oIwXUAcPsVrL9ifFZEkLSzTBcSWJO/ZuzDJrwBb+6mSJGkhmG4M4nzg2iS/xLOBsA44GHhHnxWTJI3WlAFRVY8Db0jyJuDVrfiGqvqr3msmSRqpmd4P4lbg1p7rIklaQPw2tCSpkwEhSepkQEiSOvUWEEkuTfJEknuHyo5IcnOSh9rjslaeJB9Lsj3JtiTH9VUvSdLM9NmC+DRw6l5lm4BbqmoNcEt7DnAasKb9bAQu6bFekqQZ6C0gquo24Jt7Fa8HLmvLlwFvHyq/vAa+AhyeZGVfdZMkTW++xyBWVNWutvwNYEVbPgp4dGi9x1rZD0myMcmWJFt2797dX00lacyNbJC6qgqoOWy3uarWVdW65cuX91AzSRLMf0A8PtF11B4n7lK3E1g1tN7RrUySNCLzHRDXAxva8gbguqHys9vVTCcC3xnqipIkjcBMbzk6a0muBN4IHJnkMeADwMXA1UnOAR4Bzmyr3wicDmwH/gF4d1/1kiTNTG8BUVXvnOSlN3esW8C5fdVFA6s33fDM8o6L3zbCmkhaDPwmtSSpkwEhSepkQEiSOhkQkqROBoQkqZMBIUnqZEBIkjoZEJKkTgaEJKmTASFJ6mRASJI6GRCSpE4GhCSpkwEhSepkQEiSOhkQkqROBoQkqZMBIUnqZEBIkjoZEJKkTgaEJKmTASFJ6mRASJI6HTjqCmj0Vm+64ZnlHRe/bYQ1kbSQ2IKQJHWyBcFz/wctSRqwBSFJ6mRASJI6GRCSpE6OQYwpx10kTccWhCSpkwEhSepkQEiSOo1kDCLJDuB7wB7g6apal+QI4I+B1cAO4Myq+tYo6idJGu0g9Zuq6smh55uAW6rq4iSb2vP/2NebO0grSVNbSF1M64HL2vJlwNtHWBdJGnujCogCPpdka5KNrWxFVe1qy98AVoymapIkGF0X08lVtTPJvwBuTvI3wy9WVSWprg1boGwEeMlLXtJ/TSVpTI0kIKpqZ3t8Ism1wAnA40lWVtWuJCuBJybZdjOwGWDdunWdIaK5c+pvSRPmvYspyYuSHDaxDJwC3AtcD2xoq20ArpvvukmSnjWKFsQK4NokE+//R1X12SRfBa5Ocg7wCHDmCOomSWrmPSCq6uvAT3aU/x3w5vmujySp20K6zFWStIAYEJKkTgaEJKmTASFJ6mRASJI6GRCSpE4GhCSpkwEhSepkQEiSOhkQkqROo7yjnBYZZ3qVxosBoUl5W1ZpvNnFJEnqZAtC+8yuJ2lpsgUhSepkC0LzzhaHtDgYENqv/OMvLR0GhOZkJlc4GRbS4uYYhCSpkwEhSepkQEiSOhkQkqRODlJrXjioLS0+tiAkSZ1sQWiknBBQWrgMCC0qdkNJ88eA0KJlWEj9cgxCktTJFoQWJFsH0ugZEFrwHMiWRsOA0NiztSJ1cwxCktTJFoSWHFsE0v5hC0KS1GnBtSCSnAp8FDgA+P2qunjEVdIiMNlA9mxbE7Y+pGctqIBIcgDwceBfA48BX01yfVXdP9qaaamZ7ZVRc7mSarKAmSyE9iWcDDb1YUEFBHACsL2qvg6Q5CpgPWBAaFGbSQtnJtvuyx//PoJJS9tCC4ijgEeHnj8GvG5EddESMx+thv21/WyDY7Lyvlsihsv8mu/znarq/U1mKskZwKlV9Svt+buA11XVeUPrbAQ2tqevBB6cw1sdCTy5j9Vd7Mb9HIz78YPnAMb3HPzLqlo+3UoLrQWxE1g19PzoVvaMqtoMbN6XN0myparW7cs+FrtxPwfjfvzgOQDPwXQW2mWuXwXWJDkmycHAWcD1I66TJI2lBdWCqKqnk5wH3MTgMtdLq+q+EVdLksbSggoIgKq6Ebix57fZpy6qJWLcz8G4Hz94DsBzMKUFNUgtSVo4FtoYhCRpgRirgEhyapIHk2xPsmnU9elTkh1J7klyV5ItreyIJDcneag9LmvlSfKxdl62JTlutLWfmySXJnkiyb1DZbM+5iQb2voPJdkwimOZq0nOwYVJdrbPwl1JTh967X3tHDyY5K1D5YvydyXJqiS3Jrk/yX1Jfr2Vj9XnYL+pqrH4YTDo/bfAS4GDgbuBY0ddrx6Pdwdw5F5l/wPY1JY3AR9sy6cDfwkEOBG4fdT1n+Mx/wxwHHDvXI8ZOAL4entc1paXjfrY9vEcXAj8Vse6x7bfg0OAY9rvxwGL+XcFWAkc15YPA77WjnOsPgf762ecWhDPTONRVf8ETEzjMU7WA5e15cuAtw+VX14DXwEOT7JyFBXcF1V1G/DNvYpne8xvBW6uqm9W1beAm4FT+6/9/jHJOZjMeuCqqnqqqh4GtjP4PVm0vytVtauq7mzL3wMeYDBDw1h9DvaXcQqIrmk8jhpRXeZDAZ9LsrV9+xxgRVXtasvfAFa05aV8bmZ7zEv1XJzXulAuneheYYmfgySrgdcCt+PnYE7GKSDGzclVdRxwGnBukp8ZfrEG7eixuoRtHI+5uQR4GbAW2AV8eLTV6V+SQ4HPAOdX1XeHXxvjz8GsjVNATDuNx1JSVTvb4xPAtQy6DR6f6Dpqj0+01ZfyuZntMS+5c1FVj1fVnqr6AfBJBp8FWKLnIMlBDMLhiqq6phWP/edgLsYpIMZmGo8kL0py2MQycApwL4PjnbgaYwNwXVu+Hji7XdFxIvCdoeb4YjfbY74JOCXJstYVc0orW7T2Gk96B4PPAgzOwVlJDklyDLAGuINF/LuSJMCngAeq6iNDL43952BORj1KPp8/DK5Y+BqDKzTeP+r69HicL2Vw5cndwH0Txwr8KHAL8BDweeCIVh4GN2r6W+AeYN2oj2GOx30lgy6Uf2bQZ3zOXI4Z+HcMBmy3A+8e9XHth3Pwh+0YtzH4g7hyaP33t3PwIHDaUPmi/F0BTmbQfbQNuKv9nD5un4P99eM3qSVJncapi0mSNAsGhCSpkwEhSepkQEiSOhkQkqROBoQWhCTvb7Nvbmszjr5u1HXaF0k+neSMHvZ7wdDy6uFZW6fZ7vwkZ7flLySZ8X2YkyxP8tnZ11aLnQGhkUvyeuDnGMzC+RPAW3juPDh61gXTr/JcSQ5kcE3/H83lDatqN7AryUlz2V6LlwGhhWAl8GRVPQVQVU9W1f8FSHJ8ki+2SQdvGpou4fgkd7ef/znxP+kkv5zkf0/sOMlfJHljWz4lyZeT3JnkT9p8PRP3zvivrfyeJD/Wyg9N8getbFuSX5hqP5OZ4hi+kOSDSe5I8rUkP93KX5jk6gzuaXBtktuTrEtyMfCC1sK6ou3+gCSfbK2vzyV5QUcVfha4s6qeHip7V9vPvUlOaO97YZI/bMf2UJL3DK3/Z8AvTfPvqCXGgNBC8DlgVfsj+Ykk/wqemVPnd4Ezqup44FLgorbNHwC/VlU/OZM3SHIk8J+At9RgEsMtwG8MrfJkK78E+K1W9p8ZTL3wmtay+asZ7Gfv953qGAAOrKoTgPOBD7Sy9wLfqqpjWx2OB6iqTcD3q2ptVU38sV4DfLyqXgV8G/iFjmqcBGzdq+yFVbW2vdelQ+U/wSBQXg/8lyQvbuVbgJ+e7Di1NB046gpIVfX3SY5n8AfoTcAfZ3AXsy3Aq4GbB1PscACDro7DgcNrcO8DGEwlcdo0b3MigxvHfKnt62Dgy0OvT0zqthX4+bb8FgbzEE3U81tJfm6a/eztlV3HMMn7rm7LJwMfbe95b5JtU+z/4aq6q2Mfw1YyuC/CsCvb/m9L8iPtnAJcV1XfB76f5FYGE/v9GYPJ7V6MxooBoQWhqvYAXwC+kOQeBhOqbQXuq6rXD6879Mesy9M8t2X8/InNGNwA5p2TbPdUe9zD1L8X0+2na/0fOoY5vO9knhpa3gN0dTF9n2fPw4S959ipacqf3/ajMWIXk0YuySuTrBkqWgs8wmACueVtEJskByV5VVV9G/h2kpPb+sN94zuAtUmel2QVz05t/RXgpCQvb/t6UZJXTFO1m4Fzh+q5bA776TyGad73S8CZbf1jgdcMvfbPrdtqNh4AXr5X2S+2/Z/MoBvtO618fZLnJ/lR4I0MZnYFeAXPzgKrMWFAaCE4FLisDcpuY9CFc2ENbnd5BvDBJHczmJnzDW2bdwMfT3IXg/+lT/gS8DBwP/AxYOL2k7uBXwaubO/xZeDHpqnXfweWtYHcu4E3zXY/0xzDZD7BIFTub3W4D5j4A74Z2DY0SD0Tf8ngXtXD/jHJ/wF+j8GMrxO2AbcyCML/NnGxAIOuvxtm8Z5aApzNVYteBreW/IuqevWIq7JfJDkAOKiq/jHJyxhMT/3KFjZz3ee1wH+oqoemWOdC4O+r6kMdr90GrK/B/Zk1JhyDkBaeFwK3tq6kAO/dl3BoNjEYrJ40ICaTZDnwEcNh/NiCkCR1cgxCktTJgJAkdTIgJEmdDAhJUicDQpLUyYCQJHX6/yoK1Y5qUJD5AAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<Figure size 432x288 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "#Solution\n",
    "\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt\n",
    "lenghts=[len(record.seq) for record in SeqIO.parse(filename, \"fasta\")]\n",
    "plt.hist(lenghts, bins=100)\n",
    "plt.xlabel(\"Sequence length (bp)\")\n",
    "plt.ylabel(\"Count\")\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "In the next example we'll check all the protein sequences start with a methionine (represented as the letter \"M\" in the standard IUPAC single letter amino acid code), and count how many records fail this.\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found 0 records in NC_000913.faa which did not start with M\n"
     ]
    }
   ],
   "source": [
    "from Bio import SeqIO\n",
    "filename = \"NC_000913.faa\"\n",
    "bad = 0\n",
    "for record in SeqIO.parse(filename, \"fasta\"):\n",
    "    if not record.seq.startswith(\"M\"):\n",
    "        bad = bad + 1\n",
    "        print(record.id + \" starts \" + record.seq[0])\n",
    "print(\"Found \" + str(bad) + \" records in \" + filename + \" which did not start with M\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Excercise 3**: Modify the script above  to print out the description of the problem records, not just the identifier for the the potato protein file *PGSC_DM_v3.4_pep_representative.fasta*.\n",
    "\n",
    "- What did you notice about these record descriptions? \n",
    "- Can you think of any reasons why there could be so many genes/proteins with a problem at the start?"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Different File Formats"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The Biopython SeqIO module supports quite a few other important sequence file formats (see the table on the SeqIO wiki page)."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "If you work with genomes, you'll probably have to deal with annotated files in the EMBL or GenBank format. Let's try this with the E. coli K12 GenBank file, NC_000913.gbk, based on the previous example:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "genbank_record = SeqIO.read(\"NC_000913.gbk\", \"genbank\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "NC_000913.3 length 4641652\n"
     ]
    }
   ],
   "source": [
    "print(genbank_record.id + \" length \" + str(len(genbank_record)))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Writing sequence files "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We can also write our own sequence using the  **Bio.SeqIO.write()** function."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This is a function taking three arguments:\n",
    "some SeqRecord objects, a handle or filename to write to, and a sequence format."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "2"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from Bio.Seq import Seq\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "from Bio.Alphabet import generic_protein\n",
    "\n",
    "record1 = SeqRecord(Seq(\"ALKHTYIKKKLLLMN\", generic_protein), id =\"1|1|\", description =\"protein 1\")\n",
    "record2 = SeqRecord(Seq(\"LTHKLLL\", generic_protein),id=\"2|2|\" ,description = \"protein 2\")\n",
    "\n",
    "my_records = [record1, record2]\n",
    "\n",
    "from Bio import SeqIO\n",
    "SeqIO.write(my_records, \"my_example.faa\", \"fasta\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">1|1| protein 1\r\n",
      "ALKHTYIKKKLLLMN\r\n",
      ">2|2| protein 2\r\n",
      "LTHKLLL\r\n"
     ]
    }
   ],
   "source": [
    "! head -10 my_example.faa\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Accessing public databases"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "If, rather than getting sequence record from a file, we wish to get data directly from a database then there are a few helper functions in BioPython that allow easy access to some large database"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "For example, we can read a SWISSPROT record using the ExPASy:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "from Bio import ExPASy"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio import SeqIO\n",
    "socketObj = ExPASy.get_sprot_raw('ACM1_HUMAN')\n",
    "proteinObj = SeqIO.read(socketObj, \"swiss\")\n",
    "socketObj.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "RecName: Full=Muscarinic acetylcholine receptor M1;\n",
      "MNTSAPPAVSPNITVLAPGKGPWQVAFIGITTGLLSLATVTGNLLVLISFKVNTELKTVNNYFLLSLACADLIIGTFSMNLYTTYLLMGHWALGTLACDLWLALDYVASNASVMNLLLISFDRYFSVTRPLSYRAKRTPRRAALMIGLAWLVSFVLWAPAILFWQYLVGERTVLAGQCYIQFLSQPIITFGTAMAAFYLPVTVMCTLYWRIYRETENRARELAALQGSETPGKGGGSSSSSERSQPGAEGSPETPPGRCCRCCRAPRLLQAYSWKEEEEEDEGSMESLTSSEGEEPGSEVVIKMPMVDPEAQAPTKQPPRSSPNTVKRPTKKGRDRAGKGQKPRGKEQLAKRKTFSLVKEKKAARTLSAILLAFILTWTPYNIMVLVSTFCKDCVPETLWELGYWLCYVNSTINPMCYALCNKAFRDTFRLLLLCRWDKRRWRKIPKRPGSVHRTPSRQC\n"
     ]
    }
   ],
   "source": [
    "print(proteinObj.description)\n",
    "print(proteinObj.seq)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Another example: searching, downloading, and parsing GenBank records"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "- We make a query and find out the ids of the records to retrieve.\n",
    "- check how many records there are\n",
    "- download the list of GenBank identifiers\n",
    "- use the GenBank Identifiers to download the GenBank Records"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "152\n"
     ]
    }
   ],
   "source": [
    "from Bio import Entrez\n",
    "Entrez.email = \"anna.bochicchio@fau.de\" #always tell NCBI who you are\n",
    "socketObj = Entrez.egquery(term=\"HUMAN AND ACM1\")\n",
    "record = Entrez.read(socketObj)\n",
    "for row in record[\"eGQueryResult\"]:\n",
    "    if row[\"DbName\"]==\"nuccore\":\n",
    "        print(row[\"Count\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['1446035441', '330443753', '254584897', '50313009', '299788702', '1238276357', '1220444777', '1140446665', '1174756741', '255712900', '1188067496', '329138979', '1147169515', '1134518374', '1134518373', '1134518372', '1134518371', '1134518370', '1134518369', '1134518368']"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "socketObj = Entrez.esearch(db=\"nuccore\", term=\"HUMAN AND ACM1\")\n",
    "record = Entrez.read(socketObj)\n",
    "gi_list = record[\"IdList\"]\n",
    "gi_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1446035441,330443753,254584897,50313009,299788702,1238276357,1220444777,1140446665,1174756741,255712900,1188067496,329138979,1147169515,1134518374,1134518373,1134518372,1134518371,1134518370,1134518369,1134518368\n"
     ]
    }
   ],
   "source": [
    "gi_str =\",\".join(gi_list) \n",
    "print(gi_str) \n",
    "socketObj = Entrez.efetch(db=\"nuccore\", id=gi_str, rettype=\"gb\", retmode=\"text\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<_io.TextIOWrapper encoding='latin-1'>"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "socketObj"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Working with sequence features"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Most of the time GenBank files contain a single record for a single chromosome or plasmid, so we'll generally use the SeqIO.read(...) function. Remember the second argument is the file format, so if we start from the code to read in a FASTA file:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 299,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "gi|556503834|ref|NC_000913.3|\n",
      "4641652\n",
      "0\n"
     ]
    }
   ],
   "source": [
    "from Bio import SeqIO\n",
    "\n",
    "record = SeqIO.read(\"NC_000913.fna\", \"fasta\")\n",
    "print(record.id)\n",
    "print(len(record))\n",
    "print(len(record.features))\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now switch the filename and the format:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 300,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "NC_000913.3\n",
      "4641652\n",
      "22996\n"
     ]
    }
   ],
   "source": [
    "record = SeqIO.read(\"NC_000913.gbk\", \"genbank\")\n",
    "print(record.id)\n",
    "print(len(record))\n",
    "print(len(record.features))\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "So what is this new .features thing? It is a Python list, containing a Biopython SeqFeature object for each feature in the GenBank file. For instance,"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 302,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "type: gene\n",
      "location: [336:2799](+)\n",
      "qualifiers:\n",
      "    Key: db_xref, Value: ['EcoGene:EG10998', 'GeneID:945803']\n",
      "    Key: gene, Value: ['thrA']\n",
      "    Key: gene_synonym, Value: ['ECK0002; Hs; JW0001; thrA1; thrA2; thrD']\n",
      "    Key: locus_tag, Value: ['b0002']\n",
      "\n"
     ]
    }
   ],
   "source": [
    "my_gene = record.features[3]\n",
    "print(my_gene)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "There are three key properties, .type which is a string like gene or CDS, .location which describes where on the genome this feature is, and .qualifiers which is a Python dictionary full of all the annotation for the feature (things like gene identifiers)."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Feature Locations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 303,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['b0002']\n",
      "[336:2799](+)\n",
      "336\n",
      "2799\n",
      "1\n"
     ]
    }
   ],
   "source": [
    "from Bio import SeqIO\n",
    "record = SeqIO.read(\"NC_000913.gbk\", \"genbank\")\n",
    "my_gene = record.features[3]\n",
    "print(my_gene.qualifiers[\"locus_tag\"])\n",
    "\n",
    "print(my_gene.location)\n",
    "\n",
    "print(my_gene.location.start)\n",
    "\n",
    "print(my_gene.location.end)\n",
    "\n",
    "print(my_gene.location.strand)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This was a very simple location on the forward strand, if it had been on the reverse strand you'd need to take the reverse-complement. Also if the location had been a more complicated compound location like a join (used for eukaryotic genes where the CDS is made up of several exons), then the location would have-sub parts to consider."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "All these complications are taken care of for you via the .extract(...) method which takes the full length parent record's sequence as an argument:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 304,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ATGCGAGTGTTGAAGTTCGGCGGTACATCAGTGGCAAATGCAGAACGTTTTCTGCGTGTTGCCGATATTCTGGAAAGCAATGCCAGGCAGGGGCAGGTGGCCACCGTCCTCTCTGCCCCCGCCAAAATCACCAACCACCTGGTGGCGATGATTGAAAAAACCATTAGCGGCCAGGATGCTTTACCCAATATCAGCGATGCCGAACGTATTTTTGCCGAACTTTTGACGGGACTCGCCGCCGCCCAGCCGGGGTTCCCGCTGGCGCAATTGAAAACTTTCGTCGATCAGGAATTTGCCCAAATAAAACATGTCCTGCATGGCATTAGTTTGTTGGGGCAGTGCCCGGATAGCATCAACGCTGCGCTGATTTGCCGTGGCGAGAAAATGTCGATCGCCATTATGGCCGGCGTATTAGAAGCGCGCGGTCACAACGTTACTGTTATCGATCCGGTCGAAAAACTGCTGGCAGTGGGGCATTACCTCGAATCTACCGTCGATATTGCTGAGTCCACCCGCCGTATTGCGGCAAGCCGCATTCCGGCTGATCACATGGTGCTGATGGCAGGTTTCACCGCCGGTAATGAAAAAGGCGAACTGGTGGTGCTTGGACGCAACGGTTCCGACTACTCTGCTGCGGTGCTGGCTGCCTGTTTACGCGCCGATTGTTGCGAGATTTGGACGGACGTTGACGGGGTCTATACCTGCGACCCGCGTCAGGTGCCCGATGCGAGGTTGTTGAAGTCGATGTCCTACCAGGAAGCGATGGAGCTTTCCTACTTCGGCGCTAAAGTTCTTCACCCCCGCACCATTACCCCCATCGCCCAGTTCCAGATCCCTTGCCTGATTAAAAATACCGGAAATCCTCAAGCACCAGGTACGCTCATTGGTGCCAGCCGTGATGAAGACGAATTACCGGTCAAGGGCATTTCCAATCTGAATAACATGGCAATGTTCAGCGTTTCTGGTCCGGGGATGAAAGGGATGGTCGGCATGGCGGCGCGCGTCTTTGCAGCGATGTCACGCGCCCGTATTTCCGTGGTGCTGATTACGCAATCATCTTCCGAATACAGCATCAGTTTCTGCGTTCCACAAAGCGACTGTGTGCGAGCTGAACGGGCAATGCAGGAAGAGTTCTACCTGGAACTGAAAGAAGGCTTACTGGAGCCGCTGGCAGTGACGGAACGGCTGGCCATTATCTCGGTGGTAGGTGATGGTATGCGCACCTTGCGTGGGATCTCGGCGAAATTCTTTGCCGCACTGGCCCGCGCCAATATCAACATTGTCGCCATTGCTCAGGGATCTTCTGAACGCTCAATCTCTGTCGTGGTAAATAACGATGATGCGACCACTGGCGTGCGCGTTACTCATCAGATGCTGTTCAATACCGATCAGGTTATCGAAGTGTTTGTGATTGGCGTCGGTGGCGTTGGCGGTGCGCTGCTGGAGCAACTGAAGCGTCAGCAAAGCTGGCTGAAGAATAAACATATCGACTTACGTGTCTGCGGTGTTGCCAACTCGAAGGCTCTGCTCACCAATGTACATGGCCTTAATCTGGAAAACTGGCAGGAAGAACTGGCGCAAGCCAAAGAGCCGTTTAATCTCGGGCGCTTAATTCGCCTCGTGAAAGAATATCATCTGCTGAACCCGGTCATTGTTGACTGCACTTCCAGCCAGGCAGTGGCGGATCAATATGCCGACTTCCTGCGCGAAGGTTTCCACGTTGTCACGCCGAACAAAAAGGCCAACACCTCGTCGATGGATTACTACCATCAGTTGCGTTATGCGGCGGAAAAATCGCGGCGTAAATTCCTCTATGACACCAACGTTGGGGCTGGATTACCGGTTATTGAGAACCTGCAAAATCTGCTCAATGCAGGTGATGAATTGATGAAGTTCTCCGGCATTCTTTCTGGTTCGCTTTCTTATATCTTCGGCAAGTTAGACGAAGGCATGAGTTTCTCCGAGGCGACCACGCTGGCGCGGGAAATGGGTTATACCGAACCGGACCCGCGAGATGATCTTTCTGGTATGGATGTGGCGCGTAAACTATTGATTCTCGCTCGTGAAACGGGACGTGAACTGGAGCTGGCGGATATTGAAATTGAACCTGTGCTGCCCGCAGAGTTTAACGCCGAGGGTGATGTTGCCGCTTTTATGGCGAATCTGTCACAACTCGACGATCTCTTTGCCGCGCGCGTGGCGAAGGCCCGTGATGAAGGAAAAGTTTTGCGCTATGTTGGCAATATTGATGAAGATGGCGTCTGCCGCGTGAAGATTGCCGAAGTGGATGGTAATGATCCGCTGTTCAAAGTGAAAAATGGCGAAAACGCCCTGGCCTTCTATAGCCACTATTATCAGCCGCTGCCGTTGGTACTGCGCGGATATGGTGCGGGCAATGACGTTACAGCTGCCGGTGTCTTTGCTGATCTGCTACGTACCCTCTCATGGAAGTTAGGAGTCTGA\n"
     ]
    }
   ],
   "source": [
    "gene_seq = my_gene.extract(record.seq)\n",
    "len(gene_seq)\n",
    "print(gene_seq)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Exercise 4:** Finish the following script by setting an appropriate feature name like the locus tag or GI number (use the .qualifiers or .dbxrefs information) to extract all the coding sequences from the GenBank file:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio import SeqIO\n",
    "record = SeqIO.read(\"NC_000913.gbk\", \"genbank\")\n",
    "output_handle = open(\"NC_000913_cds.fasta\", \"w\")\n",
    "count = 0\n",
    "for feature in record.features:\n",
    "    if feature.type == \"CDS\":\n",
    "        count = count + 1\n",
    "        feature_name = \"...\" # Use feature.qualifiers or feature.dbxrefs here\n",
    "        feature_seq = feature.extract(record.seq)\n",
    "        # Simple FASTA output without line wrapping:\n",
    "        output_handle.write(\">\" + feature_name + \"\\n\" + str(feature_seq) + \"\\n\")\n",
    "output_handle.close()\n",
    "print(str(count) + \" CDS sequences extracted\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Check your sequences using the NCBI FASTA file NC_000913.ffn."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Feature Lengths"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "The length of Biopython's SeqFeature objects (and the location objects) is defined as the length of the sequence region they describe (i.e. how many bases are included; or for protein annotation how many amino acids)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 320,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total length of genome is 4641652\n"
     ]
    }
   ],
   "source": [
    "from Bio import SeqIO\n",
    "record = SeqIO.read(\"NC_000913.gbk\", \"genbank\")\n",
    "print(\"Total length of genome is \" + str(len(record)))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "** Exercise 5**: Write a code to calculate to calculate the total length of the genes and of the coding sequences (CDS) within the NC_000913.gbk genome. Give a separate count for the \"gene\" and \"CDS\" feature type. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 316,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "4319 CDS sequences extracted\n",
      "Total length of all CDS is 4089653\n",
      "4497 gene sequences extracted\n",
      "Total length of all gene is 4137209\n"
     ]
    }
   ],
   "source": [
    "#Solution\n",
    "from Bio import SeqIO\n",
    "record = SeqIO.read(\"NC_000913.gbk\", \"genbank\")\n",
    "output_handle = open(\"NC_000913_cds.fasta\", \"w\")\n",
    "total_CDS = 0\n",
    "total_gene = 0\n",
    "count_CDS = 0\n",
    "count_gene = 0\n",
    "for feature in record.features:\n",
    "    if feature.type == \"CDS\":\n",
    "        count_CDS = count_CDS +1\n",
    "        total_CDS = total_CDS + len(feature)\n",
    "    elif feature.type == \"gene\":\n",
    "        count_gene = count_gene +1\n",
    "        total_gene = total_gene + len(feature)\n",
    "\n",
    "print(str(count_CDS) + \" CDS sequences extracted\")\n",
    "print(\"Total length of all CDS is \" + str(total_CDS))\n",
    "print(str(count_gene) + \" gene sequences extracted\")\n",
    "print(\"Total length of all gene is \" + str(total_gene))\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "What proportion of the genome is annotated as gene coding?"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 318,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "88.10770389507873"
      ]
     },
     "execution_count": 318,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#Solution:\n",
    "4089653 * 100.0 / 4641652"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Exercise 6:** Extend the previous script to also count the number of features of each type, and report this and the average length of that feature type. e.g.\n"
   ]
  },
  {
   "cell_type": "raw",
   "metadata": {},
   "source": [
    "Total length of genome is 4641652\n",
    "misc_feature\n",
    " - total number: 13686\n",
    " - total length: 6136082\n",
    " - average length: 448.347362268\n",
    "mobile_element\n",
    " - total number: 49\n",
    " - total length: 50131\n",
    " - average length: 1023.08163265\n",
    "..."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 322,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total length of genome is 4641652\n",
      "source\n",
      " - total number: 1\n",
      " - total length: 1\n",
      " - average length: 1.0\n",
      "gene\n",
      " - total number: 4497\n",
      " - total length: 4137144\n",
      " - average length: 919.9786524349566\n",
      "CDS\n",
      " - total number: 4319\n",
      " - total length: 4089588\n",
      " - average length: 946.8830747858301\n",
      "misc_feature\n",
      " - total number: 13547\n",
      " - total length: 6004688\n",
      " - average length: 443.2485421126449\n",
      "repeat_region\n",
      " - total number: 355\n",
      " - total length: 28919\n",
      " - average length: 81.46197183098592\n",
      "mobile_element\n",
      " - total number: 49\n",
      " - total length: 50128\n",
      " - average length: 1023.0204081632653\n",
      "ncRNA\n",
      " - total number: 65\n",
      " - total length: 8225\n",
      " - average length: 126.53846153846153\n",
      "STS\n",
      " - total number: 49\n",
      " - total length: 16145\n",
      " - average length: 329.48979591836735\n",
      "rRNA\n",
      " - total number: 22\n",
      " - total length: 30542\n",
      " - average length: 1388.2727272727273\n",
      "tRNA\n",
      " - total number: 89\n",
      " - total length: 6780\n",
      " - average length: 76.17977528089888\n",
      "tmRNA\n",
      " - total number: 2\n",
      " - total length: 15\n",
      " - average length: 7.5\n",
      "rep_origin\n",
      " - total number: 1\n",
      " - total length: 1\n",
      " - average length: 1.0\n"
     ]
    }
   ],
   "source": [
    "#Solution\n",
    "from Bio import SeqIO\n",
    "record = SeqIO.read(\"NC_000913.gbk\", \"genbank\")\n",
    "print(\"Total length of genome is \" + str(len(record)))\n",
    "totals = dict()\n",
    "counts = dict()\n",
    "for feature in record.features:\n",
    "    if feature.type in totals:\n",
    "        totals[feature.type] = totals[feature.type] + len(feature)\n",
    "        counts[feature.type] = counts[feature.type] + 1\n",
    "    else:\n",
    "        #First time to see this feature type\n",
    "        totals[feature.type] = 1\n",
    "        counts[feature.type] = 1\n",
    "for f_type in totals:\n",
    "    print(f_type)\n",
    "    print(\" - total number: \" + str(counts[f_type]))\n",
    "    print(\" - total length: \" + str(totals[f_type]))\n",
    "    ave_len = totals[f_type] / counts[f_type]\n",
    "    print(\" - average length: \" + str(ave_len))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Sequence alignments"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true
   },
   "source": [
    "The **AlignIO** module which as the name suggests is for alignment input and ouput. \n",
    "This is focused on dealing with multiple sequence alignments of the kind typically used in phylogenetics - a separate **SearchIO** module targets pairwise alignments generated by search tools like BLAST."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We're going to look at a small seed alignment for one of the PFAM domains, the A2L zinc ribbon domain (A2L_zn_ribbon; PF08792). This was picked almost at random - it is small enough to see the entire alignment on screen, and has some obvious gap-rich columns."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "From the alignments tab on the Pfam webpage, you can download the raw alignment in several different formats (Selex, Stockholm, FASTA, and MSF). Biopython is able to work with FASTA (very simple) and Stockholm format (richly annotated)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 351,
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio import AlignIO\n",
    "\n",
    "alignment = AlignIO.read(\"PF08792_seed.sth\", \"stockholm\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "As you might guess from using SeqIO.convert(...) and SeqIO.write(...), there are matching AlignIO.convert() and AlignIO.write(...) functions."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 352,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1"
      ]
     },
     "execution_count": 352,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from Bio import AlignIO\n",
    "input_filename = \"PF08792_seed.sth\"\n",
    "output_filename = \"PF08792_seed_converted.fasta\"\n",
    "AlignIO.convert(input_filename, \"stockholm\", output_filename, \"fasta\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 358,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">Q4A276_EHV8U/148-180\r\n",
      "NDDSKCIK--CGGPVLMQ--AARSLLNCQECGYSAAV\r\n",
      ">Y494R_PBCV1/148-181\r\n",
      "DQIYTCT---CGGQMELWVNSTQSDLVCNECGATQPY\r\n",
      ">VF385_ASFB7/145-177\r\n",
      "LKYKECKY--CHTDMVFN--TTQFGLQCPNCGCIQEL\r\n",
      ">Q8QNH7_ESV1K/101-133\r\n",
      "SKEWICEV--CNKELVYI--RKDAERVCPDCGLSHPY\r\n",
      ">VF282_IIV6/150-181\r\n",
      "SIPVVCT---CGNDKDFY--KDDDIYICQLCNAETVK\r\n"
     ]
    }
   ],
   "source": [
    "! head -10 PF08792_seed_converted.fasta\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "** Excercise 7:** Pairwise Sequence identity: \n",
    "- Calculate the sequence identity (expressed in percentage) between every pair of sequences of the PF08792_seed_converted.fasta data set. \n",
    "- Calculate the mean and the standard deviation of the percentages\n",
    "- Filter the pairs of sequences that have a sequence identity (statistically) significantly deviating from the mean\n",
    "- Save on a fasta file the filtered sequence ordered according to their sequence identity percentage"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "For this exercise you might find useful to know that:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 197,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[0 5 7 4 0]\n",
      "[ True False False False  True]\n",
      "[0 0]\n"
     ]
    }
   ],
   "source": [
    "a=np.array([0,5,7,4,0])\n",
    "print(a)\n",
    "print(a[:]==0)\n",
    "print(a[a[:]==0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 356,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "The sequence identity between 1 and 2 is 32.432432432432435 %\n",
      "The sequence identity between 1 and 3 is 27.027027027027028 %\n",
      "The sequence identity between 1 and 4 is 27.027027027027028 %\n",
      "The sequence identity between 1 and 5 is 27.027027027027028 %\n",
      "The sequence identity between 1 and 6 is 27.027027027027028 %\n",
      "The sequence identity between 1 and 7 is 24.324324324324323 %\n",
      "The sequence identity between 1 and 8 is 21.62162162162162 %\n",
      "The sequence identity between 1 and 9 is 21.62162162162162 %\n",
      "The sequence identity between 1 and 10 is 24.324324324324323 %\n",
      "The sequence identity between 1 and 11 is 24.324324324324323 %\n",
      "The sequence identity between 1 and 12 is 24.324324324324323 %\n",
      "The sequence identity between 2 and 3 is 32.432432432432435 %\n",
      "The sequence identity between 2 and 4 is 27.027027027027028 %\n",
      "The sequence identity between 2 and 5 is 27.027027027027028 %\n",
      "The sequence identity between 2 and 6 is 16.216216216216218 %\n",
      "The sequence identity between 2 and 7 is 21.62162162162162 %\n",
      "The sequence identity between 2 and 8 is 18.91891891891892 %\n",
      "The sequence identity between 2 and 9 is 21.62162162162162 %\n",
      "The sequence identity between 2 and 10 is 21.62162162162162 %\n",
      "The sequence identity between 2 and 11 is 16.216216216216218 %\n",
      "The sequence identity between 2 and 12 is 24.324324324324323 %\n",
      "The sequence identity between 3 and 4 is 32.432432432432435 %\n",
      "The sequence identity between 3 and 5 is 27.027027027027028 %\n",
      "The sequence identity between 3 and 6 is 27.027027027027028 %\n",
      "The sequence identity between 3 and 7 is 21.62162162162162 %\n",
      "The sequence identity between 3 and 8 is 27.027027027027028 %\n",
      "The sequence identity between 3 and 9 is 24.324324324324323 %\n",
      "The sequence identity between 3 and 10 is 24.324324324324323 %\n",
      "The sequence identity between 3 and 11 is 32.432432432432435 %\n",
      "The sequence identity between 3 and 12 is 21.62162162162162 %\n",
      "The sequence identity between 4 and 5 is 27.027027027027028 %\n",
      "The sequence identity between 4 and 6 is 29.72972972972973 %\n",
      "The sequence identity between 4 and 7 is 24.324324324324323 %\n",
      "The sequence identity between 4 and 8 is 21.62162162162162 %\n",
      "The sequence identity between 4 and 9 is 21.62162162162162 %\n",
      "The sequence identity between 4 and 10 is 21.62162162162162 %\n",
      "The sequence identity between 4 and 11 is 21.62162162162162 %\n",
      "The sequence identity between 4 and 12 is 35.13513513513514 %\n",
      "The sequence identity between 5 and 6 is 35.13513513513514 %\n",
      "The sequence identity between 5 and 7 is 27.027027027027028 %\n",
      "The sequence identity between 5 and 8 is 21.62162162162162 %\n",
      "The sequence identity between 5 and 9 is 24.324324324324323 %\n",
      "The sequence identity between 5 and 10 is 24.324324324324323 %\n",
      "The sequence identity between 5 and 11 is 21.62162162162162 %\n",
      "The sequence identity between 5 and 12 is 24.324324324324323 %\n",
      "The sequence identity between 6 and 7 is 27.027027027027028 %\n",
      "The sequence identity between 6 and 8 is 27.027027027027028 %\n",
      "The sequence identity between 6 and 9 is 27.027027027027028 %\n",
      "The sequence identity between 6 and 10 is 24.324324324324323 %\n",
      "The sequence identity between 6 and 11 is 27.027027027027028 %\n",
      "The sequence identity between 6 and 12 is 21.62162162162162 %\n",
      "The sequence identity between 7 and 8 is 64.86486486486487 %\n",
      "The sequence identity between 7 and 9 is 67.56756756756756 %\n",
      "The sequence identity between 7 and 10 is 59.45945945945946 %\n",
      "The sequence identity between 7 and 11 is 48.648648648648646 %\n",
      "The sequence identity between 7 and 12 is 21.62162162162162 %\n",
      "The sequence identity between 8 and 9 is 81.08108108108108 %\n",
      "The sequence identity between 8 and 10 is 64.86486486486487 %\n",
      "The sequence identity between 8 and 11 is 67.56756756756756 %\n",
      "The sequence identity between 8 and 12 is 27.027027027027028 %\n",
      "The sequence identity between 9 and 10 is 70.27027027027027 %\n",
      "The sequence identity between 9 and 11 is 59.45945945945946 %\n",
      "The sequence identity between 9 and 12 is 27.027027027027028 %\n",
      "The sequence identity between 10 and 11 is 67.56756756756756 %\n",
      "The sequence identity between 10 and 12 is 21.62162162162162 %\n",
      "The sequence identity between 11 and 12 is 18.91891891891892 %\n",
      "0.01652982199448161\n"
     ]
    }
   ],
   "source": [
    "#solution \n",
    "import timeit\n",
    "import Bio \n",
    "from Bio import AlignIO\n",
    "import numpy as np\n",
    "\n",
    "start_time = timeit.default_timer()\n",
    "\n",
    "def calcSeqIdentity(seqA, seqB):\n",
    "    numPlaces = min(len(seqA), len(seqB))\n",
    "    score = 0.0\n",
    "    for i in range(numPlaces):\n",
    "        if seqA[i] == seqB[i]:\n",
    "            score += 1.0\n",
    "    return 100.0 * score / numPlaces\n",
    "\n",
    "\n",
    "filename = \"PF08792_seed_converted.fasta\"\n",
    "alignment = AlignIO.read(\"PF08792_seed_converted.fasta\", \"fasta\")\n",
    "\n",
    "x_dim = int(((len(alignment))**2 - (len(alignment)))/2)\n",
    "y_dim = 3\n",
    "array = np.zeros((x_dim, y_dim))\n",
    "count=0\n",
    "for i in range(len(alignment) - 1):\n",
    "        for j in range(i + 1, len(alignment)): \n",
    "                result= calcSeqIdentity(alignment[i].seq, alignment[j].seq)\n",
    "                print('The sequence identity between', i + 1,'and', j + 1, 'is', calcSeqIdentity(alignment[i].seq, alignment[j].seq),'%')\n",
    "                array[count, :]= [i, j, result]\n",
    "                count+=1\n",
    "average = np.mean(array[:,2])\n",
    "std = np.std(array[:,2])\n",
    "\n",
    "array[array[: ,2] > average + 2*std, :]\n",
    "\n",
    "elapsed = timeit.default_timer() - start_time\n",
    "print(elapsed)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Sequence pairs in an alignment can have a score, even when they are not identical,  which describe how similar two sequences are. This similarity is defined considering how *substitutable* one residue type is for another; in other words how likely they are to have been swapped or exchanged for one another.\n",
    "Residues that commonly swap for one another are deemed to be similar and give high scores, while those that rarely swap are dissimilar and give low scores.\n",
    "\n",
    "\n",
    "The substitutability of one residue for another is stored in a two-dimensional array, commonly called **substitution matrix** or **similarity matrix**."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We will define our similarity matrix using a **python dictionary**\n",
    "\n",
    "A **dictionary** is a collection which is unordered, changeable and indexed. \n",
    "In Python dictionaries are written with curly brackets, and they have keys and values.\n",
    "For example:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 202,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'brand': 'Ford', 'model': 'Mustang', 'year': 1964}\n"
     ]
    }
   ],
   "source": [
    "thisdict = {\n",
    "  \"brand\": \"Ford\",\n",
    "  \"model\": \"Mustang\",\n",
    "  \"year\": 1964\n",
    "}\n",
    "print(thisdict)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "You can access the items of a dictionary by referring to its key name:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 206,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Mustang'"
      ]
     },
     "execution_count": 206,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "x = thisdict[\"model\"]\n",
    "x"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Or also use the values() function to return values of a dictionary:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 207,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Ford\n",
      "Mustang\n",
      "1964\n"
     ]
    }
   ],
   "source": [
    "for x in thisdict.values():\n",
    "    print(x)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Adding an item to the dictionary is done by using a new index key and assigning a value to it:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 208,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'brand': 'Ford', 'model': 'Mustang', 'year': 1964, 'color': 'red'}\n"
     ]
    }
   ],
   "source": [
    "thisdict ={\n",
    "  \"brand\": \"Ford\",\n",
    "  \"model\": \"Mustang\",\n",
    "  \"year\": 1964\n",
    "}\n",
    "thisdict[\"color\"] = \"red\"\n",
    "print(thisdict)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "There are several methods to remove items from a dictionary:"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "1. The **del** and the **pop()** keyword both remove the item with the specified key name:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 209,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'brand': 'Ford', 'year': 1964}\n"
     ]
    }
   ],
   "source": [
    "thisdict = {\n",
    "  \"brand\": \"Ford\",\n",
    "  \"model\": \"Mustang\",\n",
    "  \"year\": 1964\n",
    "}\n",
    "del thisdict[\"model\"]\n",
    "print(thisdict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 210,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'brand': 'Ford', 'year': 1964}\n"
     ]
    }
   ],
   "source": [
    "thisdict = {\n",
    "  \"brand\": \"Ford\",\n",
    "  \"model\": \"Mustang\",\n",
    "  \"year\": 1964\n",
    "}\n",
    "thisdict.pop(\"model\")\n",
    "print(thisdict)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The del keyword can also delete the dictionary completely:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 213,
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'thisdict' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-213-bc3eef664a96>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      5\u001b[0m }\n\u001b[1;32m      6\u001b[0m \u001b[0;32mdel\u001b[0m \u001b[0mthisdict\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 7\u001b[0;31m \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mthisdict\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;31m#this will cause an error because \"thislist\" no longer exists.\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m: name 'thisdict' is not defined"
     ]
    }
   ],
   "source": [
    "thisdict ={\n",
    "  \"brand\": \"Ford\",\n",
    "  \"model\": \"Mustang\",\n",
    "  \"year\": 1964\n",
    "}\n",
    "del thisdict\n",
    "print(thisdict) #this will cause an error because \"thislist\" no longer exists."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "2. The **popitem()** method removes the last inserted item "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 211,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'brand': 'Ford', 'model': 'Mustang'}\n"
     ]
    }
   ],
   "source": [
    "thisdict ={\n",
    "  \"brand\": \"Ford\",\n",
    "  \"model\": \"Mustang\",\n",
    "  \"year\": 1964\n",
    "}\n",
    "thisdict.popitem()\n",
    "print(thisdict)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "3. The **clear ()** method empties the dictionary:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 214,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{}\n"
     ]
    }
   ],
   "source": [
    "thisdict ={\n",
    "  \"brand\": \"Ford\",\n",
    "  \"model\": \"Mustang\",\n",
    "  \"year\": 1964\n",
    "}\n",
    "thisdict.clear()\n",
    "print(thisdict)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Below there are examples of substitution matrices.\n",
    "\n",
    "* The first one - DNA_1 -is very simple and would give scores as if you were measuring sequence identity (i.e., a score of 1 were residues are identical, 0 elsewhere.). \n",
    "* Rather than scoring DNA for matches we could also score for complementarity (i.e., using Watson-Crick's pairing rules), with 1 for A:T or C:G matches \n",
    "and -1 for mismatches. Expressed as Python dictionary this would correspond to the example called REV_COMP\n",
    "*  Going to a more complex example, in the DNA_2 matrix identicatl residues give a score of 1 but non identitcal -3. The latter indicate that there is a mismatch between the sequence. A score of 0 is here used to indicate indifference\n",
    "* The last example, raprestents one of the most widely used similarity matrix, called BLOSUM62 (to read more about it: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC50453/pdf/pnas01096-0363.pdf ).                                                                               "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 134,
   "metadata": {},
   "outputs": [],
   "source": [
    "DNA_1 = {'G': { 'G':1, 'C':0, 'A':0, 'T':0 },\n",
    "         'C': { 'G':0, 'C':1, 'A':0, 'T':0 },\n",
    "         'A': { 'G':0, 'C':0, 'A':1, 'T':0 },\n",
    "         'T': { 'G':0, 'C':0, 'A':0, 'T':1 }}\n",
    "\n",
    "          \n",
    "REV_COMP = {'G': { 'G':-1, 'C': 1, 'A':-1, 'T':-1 },\n",
    "            'C': { 'G': 1, 'C':-1, 'A':-1, 'T':-1 },\n",
    "            'A': { 'G':-1, 'C':-1, 'A':-1, 'T': 1 },\n",
    "            'T': { 'G':-1, 'C':-1, 'A': 1, 'T':-1 }}\n",
    "\n",
    "          \n",
    "DNA_2 = {'G': { 'G': 1, 'C':-3, 'A':-3, 'T':-3, 'N':0 },\n",
    "         'C': { 'G':-3, 'C': 1, 'A':-3, 'T':-3, 'N':0 },\n",
    "         'A': { 'G':-3, 'C':-3, 'A': 1, 'T':-3, 'N':0 },\n",
    "         'T': { 'G':-3, 'C':-3, 'A':-3, 'T': 1, 'N':0 },\n",
    "         'N': { 'G': 0, 'C': 0, 'A': 0, 'T': 0, 'N':0 }}  \n",
    "\n",
    "BLOSUM62 = {'A':{'A': 4,'R':-1,'N':-2,'D':-2,'C': 0,'Q':-1,'E':-1,'G': 0,'H':-2,'I':-1,\n",
    "                 'L':-1,'K':-1,'M':-1,'F':-2,'P':-1,'S': 1,'T': 0,'W':-3,'Y':-2,'V': 0,'X':0},\n",
    "            'R':{'A':-1,'R': 5,'N': 0,'D':-2,'C':-3,'Q': 1,'E': 0,'G':-2,'H': 0,'I':-3,\n",
    "                 'L':-2,'K': 2,'M':-1,'F':-3,'P':-2,'S':-1,'T':-1,'W':-3,'Y':-2,'V':-3,'X':0},\n",
    "            'N':{'A':-2,'R': 0,'N': 6,'D': 1,'C':-3,'Q': 0,'E': 0,'G': 0,'H': 1,'I':-3,\n",
    "                 'L':-3,'K': 0,'M':-2,'F':-3,'P':-2,'S': 1,'T': 0,'W':-4,'Y':-2,'V':-3,'X':0},\n",
    "            'D':{'A':-2,'R':-2,'N': 1,'D': 6,'C':-3,'Q': 0,'E': 2,'G':-1,'H':-1,'I':-3,\n",
    "                 'L':-4,'K':-1,'M':-3,'F':-3,'P':-1,'S': 0,'T':-1,'W':-4,'Y':-3,'V':-3,'X':0},\n",
    "            'C':{'A': 0,'R':-3,'N':-3,'D':-3,'C': 9,'Q':-3,'E':-4,'G':-3,'H':-3,'I':-1,\n",
    "                 'L':-1,'K':-3,'M':-1,'F':-2,'P':-3,'S':-1,'T':-1,'W':-2,'Y':-2,'V':-1,'X':0},\n",
    "            'Q':{'A':-1,'R': 1,'N': 0,'D': 0,'C':-3,'Q': 5,'E': 2,'G':-2,'H': 0,'I':-3,\n",
    "                 'L':-2,'K': 1,'M': 0,'F':-3,'P':-1,'S': 0,'T':-1,'W':-2,'Y':-1,'V':-2,'X':0},\n",
    "            'E':{'A':-1,'R': 0,'N': 0,'D': 2,'C':-4,'Q': 2,'E': 5,'G':-2,'H': 0,'I':-3,\n",
    "                 'L':-3,'K': 1,'M':-2,'F':-3,'P':-1,'S': 0,'T':-1,'W':-3,'Y':-2,'V':-2,'X':0},\n",
    "            'G':{'A': 0,'R':-2,'N': 0,'D':-1,'C':-3,'Q':-2,'E':-2,'G': 6,'H':-2,'I':-4,\n",
    "                 'L':-4,'K':-2,'M':-3,'F':-3,'P':-2,'S': 0,'T':-2,'W':-2,'Y':-3,'V':-3,'X':0},\n",
    "            'H':{'A':-2,'R': 0,'N': 1,'D':-1,'C':-3,'Q': 0,'E': 0,'G':-2,'H': 8,'I':-3,\n",
    "                 'L':-3,'K':-1,'M':-2,'F':-1,'P':-2,'S':-1,'T':-2,'W':-2,'Y': 2,'V':-3,'X':0},\n",
    "            'I':{'A':-1,'R':-3,'N':-3,'D':-3,'C':-1,'Q':-3,'E':-3,'G':-4,'H':-3,'I': 4,\n",
    "                 'L': 2,'K':-3,'M': 1,'F': 0,'P':-3,'S':-2,'T':-1,'W':-3,'Y':-1,'V': 3,'X':0},\n",
    "            'L':{'A':-1,'R':-2,'N':-3,'D':-4,'C':-1,'Q':-2,'E':-3,'G':-4,'H':-3,'I': 2,\n",
    "                 'L': 4,'K':-2,'M': 2,'F': 0,'P':-3,'S':-2,'T':-1,'W':-2,'Y':-1,'V': 1,'X':0},\n",
    "            'K':{'A':-1,'R': 2,'N': 0,'D':-1,'C':-3,'Q': 1,'E': 1,'G':-2,'H':-1,'I':-3,\n",
    "                 'L':-2,'K': 5,'M':-1,'F':-3,'P':-1,'S': 0,'T':-1,'W':-3,'Y':-2,'V':-2,'X':0},\n",
    "            'M':{'A':-1,'R':-1,'N':-2,'D':-3,'C':-1,'Q': 0,'E':-2,'G':-3,'H':-2,'I': 1,\n",
    "                 'L': 2,'K':-1,'M': 5,'F': 0,'P':-2,'S':-1,'T':-1,'W':-1,'Y':-1,'V': 1,'X':0},\n",
    "            'F':{'A':-2,'R':-3,'N':-3,'D':-3,'C':-2,'Q':-3,'E':-3,'G':-3,'H':-1,'I': 0,\n",
    "                 'L': 0,'K':-3,'M': 0,'F': 6,'P':-4,'S':-2,'T':-2,'W': 1,'Y': 3,'V':-1,'X':0},\n",
    "            'P':{'A':-1,'R':-2,'N':-2,'D':-1,'C':-3,'Q':-1,'E':-1,'G':-2,'H':-2,'I':-3,\n",
    "                 'L':-3,'K':-1,'M':-2,'F':-4,'P': 7,'S':-1,'T':-1,'W':-4,'Y':-3,'V':-2,'X':0},\n",
    "            'S':{'A': 1,'R':-1,'N': 1,'D': 0,'C':-1,'Q': 0,'E': 0,'G': 0,'H':-1,'I':-2,\n",
    "                 'L':-2,'K': 0,'M':-1,'F':-2,'P':-1,'S': 4,'T': 1,'W':-3,'Y':-2,'V':-2,'X':0},\n",
    "            'T':{'A': 0,'R':-1,'N': 0,'D':-1,'C':-1,'Q':-1,'E':-1,'G':-2,'H':-2,'I':-1,\n",
    "                 'L':-1,'K':-1,'M':-1,'F':-2,'P':-1,'S': 1,'T': 5,'W':-2,'Y':-2,'V': 0,'X':0},\n",
    "            'W':{'A':-3,'R':-3,'N':-4,'D':-4,'C':-2,'Q':-2,'E':-3,'G':-2,'H':-2,'I':-3,\n",
    "                 'L':-2,'K':-3,'M':-1,'F': 1,'P':-4,'S':-3,'T':-2,'W':11,'Y': 2,'V':-3,'X':0},\n",
    "            'Y':{'A':-2,'R':-2,'N':-2,'D':-3,'C':-2,'Q':-1,'E':-2,'G':-3,'H': 2,'I':-1,\n",
    "                 'L':-1,'K':-2,'M':-1,'F': 3,'P':-3,'S':-2,'T':-2,'W': 2,'Y': 7,'V':-1,'X':0},\n",
    "            'V':{'A': 0,'R':-3,'N':-3,'D':-3,'C':-1,'Q':-2,'E':-2,'G':-3,'H':-3,'I': 3,\n",
    "                 'L': 1,'K':-2,'M': 1,'F':-1,'P':-2,'S':-2,'T': 0,'W':-3,'Y':-1,'V': 4,'X':0},\n",
    "            'X':{'A': 0,'R': 0,'N': 0,'D': 0,'C': 0,'Q': 0,'E': 0,'G': 0,'H': 0,'I': 0,\n",
    "                 'L': 0,'K': 0,'M': 0,'F': 0,'P': 0,'S': 0,'T': 0,'W': 0,'Y': 0,'V': 0,'X':0}}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "The following function will now consider a substitution matrix and use it to calculate an overall similarity score. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 143,
   "metadata": {},
   "outputs": [],
   "source": [
    "def calcSeqSimilarity(seqA, seqB, simMatrix):\n",
    "\n",
    "    numPlaces = min(len(seqA), len(seqB))\n",
    "  \n",
    "    totalScore = 0.0\n",
    "  \n",
    "    for i in range(numPlaces):\n",
    "        residueA = seqA[i]\n",
    "        residueB = seqB[i]\n",
    "  \n",
    "        totalScore += simMatrix[residueA][residueB]\n",
    "\n",
    "    return totalScore"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 144,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3.0\n"
     ]
    }
   ],
   "source": [
    "# Test with pre-defined substitution matrices\n",
    "  # DNA example\n",
    "print(calcSeqSimilarity('AGCATCGCTCT', 'AGCATCGTTTT', DNA_2))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 226,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "8.0\n",
      "33.333333333333336\n"
     ]
    }
   ],
   "source": [
    "#Protein example\n",
    "print(calcSeqSimilarity ('ALIGNMENT', 'ALINGEMTN', BLOSUM62))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Exercise 8:** Modify the above function to take into account the presence of gaps ('-') in the sequences without modifying the BLOSUM62 substitution matrix. \n",
    "Gaps are generally undesirable but are tolerable if the subsequent alignment matches well.\n",
    "\n",
    "Introduce different gap penalties depending on whether there is an insertion of a new gap or an extension of an existing one. \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 269,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "32.0\n",
      "-3.0\n"
     ]
    }
   ],
   "source": [
    "#Solution\n",
    "\n",
    "def calcSeqSimilarityGaps(seqA, seqB, simMatrix, insert=8, extend=4):\n",
    "\n",
    "    totalScore = 0.0\n",
    "    \n",
    "    n = min(len(seqA), len(seqB))\n",
    "    \n",
    "    for i in range(n):\n",
    "        \n",
    "        residueA = seqA[i]\n",
    "        residueB = seqB[i]\n",
    "        \n",
    "        if '-' not in (residueA, residueB):\n",
    "            simScore = simMatrix[residueA][residueB]\n",
    "        elif (i > 0) and ('-' in (seqA[i-1]), seqB[i-1]):\n",
    "            simScore = - extend\n",
    "        else:\n",
    "            simScore = - insert\n",
    "            \n",
    "        totalScore += simScore\n",
    "        \n",
    "    return totalScore\n",
    "\n",
    "#test\n",
    "print(calcSeqSimilarityGaps('ALIGDPPVENTS', 'ALIGN--MENTS', BLOSUM62))\n",
    "print(calcSeqSimilarityGaps('ALIGDPPVENTS', '--ALIGNMENTS', BLOSUM62))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Optimising pairwise alignment"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We have seen the basic principles of how it is possible to measure the match quality of an aligned pair of sequences. \n",
    "In general, however, one is interested in determining which alignment out of all the possible combinations is the best (highest scoring).\n",
    "Consider the following three examples:"
   ]
  },
  {
   "cell_type": "raw",
   "metadata": {},
   "source": [
    "ALIGNMENTS---           A--LIGN-MENTS        --ALIGN-MENTS  \n",
    "ANALIGDPVENTS           ANALIGDPVENTS        ANALIGDPVENTS"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The last alignment is the best, with the middle one a close second."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This problem is cleary more complex. The number of possible combinations grows very rapidly with the length of the sequence. Hence, more clever algorithm have been developed. All of them use the so called *dynamic programming* principle.\n",
    "\n",
    "\n",
    "The idea behind dynamic programming is that a big problem can be broken down into local-sub-problems that occur repeatedly, then the solution to each sub-problem only needs to be calculated once. \n",
    "\n",
    "In terms of sequence alignment the big problem is to find the highest scoring arrangement and the local sub-problems involve smaller sub-alignments.\n",
    "\n",
    "We will not cover this here, but we invite you to refer to Durbin et al. (https://pdfs.semanticscholar.org/2ed5/d6b35f8971fb9d7434a2683922c3bfcc058e.pdf) for in-depth information on sequence alignment algorithms.   "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "BioPython includes a built-in pairwise aligner that implements some of these algorithms: e.g. the Needleman-Wunsch, Smith-Waterman, Gotoh (three-state), and Waterman-Smith-Beyer global and local pairwise alignment algorithms. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 245,
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio import Align\n",
    "from Bio.SubsMat.MatrixInfo import blosum62\n",
    "aligner = Align.PairwiseAligner()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The PairwiseAligner object aligner  stores the alignment parameters to be used for the pairwise alignments."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 267,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "40.0\n",
      "40.0\n",
      "ALIGDPPVENTS\n",
      "||||XXXX||||\n",
      "ALIGN--MENTS\n",
      "\n"
     ]
    }
   ],
   "source": [
    "seqA = \"ALIGDPPVENTS\"\n",
    "seqB = \"ALIGN--MENTS\"\n",
    "aligner = Align.PairwiseAligner()\n",
    "aligner.substitution_matrix = blosum62\n",
    "aligner.extend_gap_score = -4\n",
    "aligner.open_gap_score = -8\n",
    "score = aligner.score(seqA, seqB)\n",
    "print(score)\n",
    "\n",
    "alignments = aligner.align(seqA, seqB)\n",
    "len(alignments)\n",
    "print(alignments[0].score)\n",
    "\n",
    "print(alignments[0])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Depending on the gap scoring parameters and mode, a PairwiseAligner object automatically chooses the appropriate algorithm to use for pairwise sequence alignment. To verify the selected algorithm, use"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 272,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Gotoh global alignment algorithm'"
      ]
     },
     "execution_count": 272,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "aligner.algorithm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 251,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--ALIGDPPVENTS\n",
      "--||||X--X||||\n",
      "--ALIGN--MENTS\n",
      "\n"
     ]
    }
   ],
   "source": [
    "alignments = aligner.align(seqA, seqB)\n",
    "for alignment in alignments:\n",
    "    print(alignment)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## BLAST"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "BLAST (Basic Local Alignment Search Tool) is one of the most widely used bioinformatics programs for sequence searching within a library or database of sequences, and to identify library sequences that resemble the query sequence above a certain threshold (e-value).\n",
    "\n",
    "Using a dynamic algorithm for this purpose would be very computational expensive. BLAST use a so called *heuristic* approach ( https://en.wikipedia.org/wiki/BLAST ) and it does **not** perform an optimal alignment. "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "You can use BLAST through BioPython:\n",
    "the function qblast() in the Bio.Blast.NCBIWWW module calls the online version of BLAST. \n",
    "\n",
    "\n",
    "* The first argument is the blast program to use for the search, as a lower case string. The options and descriptions of the programs are available at http://www.ncbi.nlm.nih.gov/BLAST/blast_program.shtml. Currently qblast only works with blastn, blastp, blastx, tblast and tblastx.\n",
    "\n",
    "* The second argument specifies the databases to search against. Again, the options for this are available on the NCBI web pages at http://www.ncbi.nlm.nih.gov/BLAST/blast_databases.shtml.\n",
    "\n",
    "* The third argument is a string containing your query sequence. This can either be the sequence itself, the sequence in fasta format, or an identifier like a GI number.\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The qblast function also take a number of other option arguments which are basically analogous to the different parameters you can set on the BLAST web page.\n",
    "\n",
    "* The argument url_base sets the base URL for running BLAST over the internet. By default it connects to the NCBI.\n",
    "\n",
    "* The qblast function can return the BLAST results in various formats, which you can choose with the optional format_type keyword: \"HTML\", \"Text\", \"ASN.1\", or \"XML\". The default is \"XML\", as that is the format expected by the parser\n",
    "\n",
    "* The argument expect sets the expectation or e-value threshold.\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 295,
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio.Blast import NCBIWWW\n",
    "result_handle = NCBIWWW.qblast(\"blastn\", \"nt\", \"8332126\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 290,
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio.Blast import NCBIXML\n",
    "blast_records = NCBIXML.parse(result_handle)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 294,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "****Alignment****\n",
      "sequence: gi|7595506|gb|AF206986.1| Pereskia aculeata 18S ribosomal RNA gene, complete sequence\n",
      "length: 1739\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|7595428|gb|AF206908.1| Ercilla volubilis 18S ribosomal RNA gene, complete sequence\n",
      "length: 1760\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|7595520|gb|AF207000.1| Portulaca grandiflora 18S ribosomal RNA gene, complete sequence\n",
      "length: 1730\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|7595419|gb|AF206899.1| Delosperma echinatum 18S ribosomal RNA gene, complete sequence\n",
      "length: 1756\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|307601301|gb|GQ497578.1| Claytonia perfoliata 18S ribosomal RNA gene, partial sequence\n",
      "length: 1790\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|307601300|gb|GQ497577.1| Corbichonia decumbens 18S ribosomal RNA gene, partial sequence\n",
      "length: 1797\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|307601290|gb|GQ497567.1| Basella alba 18S ribosomal RNA gene, partial sequence\n",
      "length: 1799\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|118918043|dbj|AB233556.1| Necepsia afzelii gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1736\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|6688996|emb|AJ235996.1| Phlox longifolia 18S rRNA gene\n",
      "length: 1766\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|307601297|gb|GQ497574.1| Halophytum ameghinoi 18S ribosomal RNA gene, partial sequence\n",
      "length: 1798\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "||||||||||||||||||||||||||||||||||||||||| |||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACAAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|1777649|gb|U42496.1|TEU42496 Tetragonia expansa 18S ribosomal RNA gene, partial sequence\n",
      "length: 1713\n",
      "e value: 0.0\n",
      "AGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTAAATCAGTTATA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "AGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTAAATCAGTTATA...\n",
      "****Alignment****\n",
      "sequence: gi|155029506|dbj|AB268095.1| Ditta myricoides gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1736\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|155029500|dbj|AB268089.1| Tragiella anomala gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1736\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|155029497|dbj|AB268086.1| Speranskia cantonensis gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1736\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|118918047|dbj|AB233560.1| Romanoa tamnoides gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1733\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|118918046|dbj|AB233559.1| Ricinus communis gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1736\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|118918037|dbj|AB233548.1| Chrozophora oblongifolia gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1736\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|118918033|dbj|AB233543.1| Adriana quadripartita gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1736\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|18654262|gb|L49293.1|HLXRGEA Phlox hoodii 18S ribosomal RNA gene, partial sequence\n",
      "length: 1736\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||| ||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGANTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|18654252|gb|L49282.1|GIIRGE Gilia capitata 18S ribosomal RNA gene, partial sequence\n",
      "length: 1747\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|343179099|emb|HE578879.1| Beta vulgaris subsp. vulgaris genomic DNA containing 26S-18S IGS and partial 18S gene, cultivar KWS2320, clone pZR18S\n",
      "length: 5066\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|307601309|gb|GQ497586.1| Sarcobatus vermiculatus 18S ribosomal RNA gene, partial sequence\n",
      "length: 1797\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "||||||||||||||||||||||||||||||||||||||||| |||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACAAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|17028177|gb|L49277.1|CBERGE Cobaea scandens 18S ribosomal RNA gene, partial sequence\n",
      "length: 1727\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|17028071|gb|L49294.1|PMXRGE Polemonium californicum 18S ribosomal RNA gene, partial sequence\n",
      "length: 1719\n",
      "e value: 0.0\n",
      "TTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTAAA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "TTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTAAA...\n",
      "****Alignment****\n",
      "sequence: gi|512388725|emb|HF565049.1| Portulaca oleracea subsp. nitida genomic DNA containing 18S rRNA gene\n",
      "length: 1131\n",
      "e value: 0.0\n",
      "AAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTAAATCAGTTAT...\n",
      "||||||| |||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "AAGATTACGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTAAATCAGTTAT...\n",
      "****Alignment****\n",
      "sequence: gi|155029511|dbj|AB268100.1| Klaineanthus gaboniae gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1736\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|155029499|dbj|AB268088.1| Tragia betonicifolia gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1736\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|119368144|dbj|AB233551.1| Dalechampia spathulata gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1715\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|18654264|gb|L49284.1|HSIRGE Halesia tetraptera 18S ribosomal RNA gene, partial sequence\n",
      "length: 1738\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|17028173|gb|L49273.1|BPDRGE Bonplandia geminiflora 18S ribosomal RNA gene, partial sequence\n",
      "length: 1735\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||| ||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAANCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|7595427|gb|AF206907.1| Epilobium angustifolium 18S ribosomal RNA gene, complete sequence\n",
      "length: 1736\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|155573952|dbj|AB268081.1| Melanolepis multiglandulosa gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1736\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|155029525|dbj|AB268116.1| Spirostachys africana gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1736\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|155029521|dbj|AB268110.1| Adenopeltis serrata gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1736\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|155029513|dbj|AB268102.1| Micrandropsis scleroxylon gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1736\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|155029510|dbj|AB268099.1| Hevea brasiliensis gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1733\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|155029498|dbj|AB268087.1| Sumbaviopsis albicans gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1736\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|119368152|dbj|AB233574.1| Hura crepitans gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1736\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|119368151|dbj|AB233573.1| Excoecaria cochinchinensis gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1709\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|119368148|dbj|AB233568.1| Manihot esculenta gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1736\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|18654199|gb|L49286.1|IPORGE Ipomopsis aggregata 18S ribosomal RNA gene, complete sequence\n",
      "length: 1727\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|17028150|gb|L49271.1|AHGRGE Acanthogilia gloriosa 18S ribosomal RNA gene, partial sequence\n",
      "length: 1728\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|6651249|gb|AF161008.1|AF161008 Arbutus unedo 18S ribosomal RNA gene, partial sequence >gi|7595373|gb|AF206853.1| Arbutus unedo 18S ribosomal RNA gene, complete sequence\n",
      "length: 1744\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|1777701|gb|U42793.1|PAU42793 Phytolacca americana 18S ribosomal RNA gene, partial sequence\n",
      "length: 1733\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||  ||| ||| |||||||||||||||||||||||||||||||||||||||||| |||||||||||||...\n",
      "GCTTGTCTCNNAGANTAANCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGMGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|916445649|gb|KT179718.1| Silene antirrhina voucher Kellar 1417 18S ribosomal RNA gene, partial sequence\n",
      "length: 1810\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|256373708|gb|FJ748880.1| Coptis chinensis 18S ribosomal RNA gene, partial sequence\n",
      "length: 1809\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|254553761|gb|FJ527842.1| Coptis chinensis isolate 1 18S ribosomal RNA gene, partial sequence >gi|254553762|gb|FJ527843.1| Coptis chinensis isolate 2 18S ribosomal RNA gene, partial sequence >gi|254553763|gb|FJ527844.1| Coptis deltoidea 18S ribosomal RNA gene, partial sequence\n",
      "length: 1785\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|118918038|dbj|AB233549.1| Clutia augustifolia gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1736\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|6687269|emb|AJ236021.1| Cephalanthus occidentalis 18S rRNA gene\n",
      "length: 1766\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "****Alignment****\n",
      "sequence: gi|118918042|dbj|AB233554.1| Macaranga tanarius gene for 18S ribosomal RNA, partial sequence\n",
      "length: 1736\n",
      "e value: 0.0\n",
      "GCTTGTCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n",
      "||||| |||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||...\n",
      "GCTTGCCTCAAAGATTAAGCCATGCATGTGTAAGTATGAACTAATTCAGACTGTGAAACTGCGAATGGCTCATTA...\n"
     ]
    }
   ],
   "source": [
    "E_VALUE_THRESH = 0.04\n",
    "\n",
    "\n",
    "for alignment in blast_record.alignments:\n",
    "    for hsp in alignment.hsps:  # HSP (high-scoring pair) \n",
    "        if hsp.expect < E_VALUE_THRESH:\n",
    "            print(\"****Alignment****\")\n",
    "            print(\"sequence:\", alignment.title)\n",
    "            print(\"length:\", alignment.length)\n",
    "            print(\"e value:\", hsp.expect)\n",
    "            print(hsp.query[0:75] + \"...\")\n",
    "            print(hsp.match[0:75] + \"...\")\n",
    "            print(hsp.sbjct[0:75] + \"...\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# PHYLOGENETIC THREE"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true
   },
   "source": [
    "The Bio.Phylo aims to provide a common way to work with phylogenetic trees independently of the source data format.\n",
    "\n",
    "Bio.Phylo is described in an open-access journal article(https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-209), which you might also find helpful.\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 359,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Q4A276_EHV8U/148-180\t0\n",
      "Q6TVP3_ORFSA/1-32\t0.7567567567567568\t0\n",
      "Q8QNH7_ESV1K/101-133\t0.7297297297297297\t0.7567567567567568\t0\n",
      "Q8V3K7_SWPV1/1-32\t0.7837837837837838\t0.3513513513513513\t0.7837837837837838\t0\n",
      "Q98271_MCV1/6-36\t0.7567567567567568\t0.4054054054054054\t0.7837837837837838\t0.3513513513513513\t0\n",
      "Q9EMK1_AMEPV/2-33\t0.7297297297297297\t0.7297297297297297\t0.7027027027027026\t0.7297297297297297\t0.7567567567567568\t0\n",
      "VF282_IIV6/150-181\t0.7297297297297297\t0.7297297297297297\t0.7297297297297297\t0.7837837837837838\t0.7567567567567568\t0.6486486486486487\t0\n",
      "VF385_ASFB7/145-177\t0.7297297297297297\t0.7837837837837838\t0.6756756756756757\t0.7297297297297297\t0.7567567567567568\t0.7297297297297297\t0.7297297297297297\t0\n",
      "VLTF3_FOWPN/3-33\t0.7567567567567568\t0.5135135135135135\t0.7837837837837838\t0.32432432432432434\t0.32432432432432434\t0.7297297297297297\t0.7837837837837838\t0.6756756756756757\t0\n",
      "VLTF3_VAR67/1-32\t0.7837837837837838\t0.32432432432432434\t0.7837837837837838\t0.18918918918918914\t0.29729729729729726\t0.7297297297297297\t0.7567567567567568\t0.7567567567567568\t0.4054054054054054\t0\n",
      "Y494R_PBCV1/148-181\t0.6756756756756757\t0.7837837837837838\t0.7297297297297297\t0.8108108108108107\t0.7837837837837838\t0.8378378378378378\t0.7297297297297297\t0.6756756756756757\t0.8378378378378378\t0.7837837837837838\t0\n",
      "YR429_MIMIV/213-247\t0.7567567567567568\t0.7837837837837838\t0.6486486486486487\t0.7297297297297297\t0.7837837837837838\t0.7837837837837838\t0.7567567567567568\t0.7837837837837838\t0.8108108108108107\t0.7297297297297297\t0.7567567567567568\t0\n",
      "\tQ4A276_EHV8U/148-180\tQ6TVP3_ORFSA/1-32\tQ8QNH7_ESV1K/101-133\tQ8V3K7_SWPV1/1-32\tQ98271_MCV1/6-36\tQ9EMK1_AMEPV/2-33\tVF282_IIV6/150-181\tVF385_ASFB7/145-177\tVLTF3_FOWPN/3-33\tVLTF3_VAR67/1-32\tY494R_PBCV1/148-181\tYR429_MIMIV/213-247\n"
     ]
    }
   ],
   "source": [
    "from Bio.Phylo.TreeConstruction import DistanceCalculator\n",
    "\n",
    "#calculate the distance matrix\n",
    "calculator = DistanceCalculator('identity')\n",
    "#adds distance matrix to the calculator object and returns it\n",
    "dm = calculator.get_distance(alignment)\n",
    "print(dm)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "and finally, we can construct a phylogenetic tree from the pairwise distances between all the sequences."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 360,
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio.Phylo.TreeConstruction import DistanceTreeConstructor\n",
    "\n",
    "#initialize a DistanceTreeConstructor object based on our distance calculator object\n",
    "constructor = DistanceTreeConstructor(calculator)\n",
    "\n",
    "#build the tree\n",
    "upgma_tree = constructor.build_tree(alignment)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 363,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Tree(rooted=False)\n",
      "    Clade(branch_length=0, name='Inner10')\n",
      "        Clade(branch_length=0.02301520270270266, name='Inner7')\n",
      "            Clade(branch_length=0.3416385135135134, name='Y494R_PBCV1/148-181')\n",
      "            Clade(branch_length=0.3340371621621623, name='Q4A276_EHV8U/148-180')\n",
      "        Clade(branch_length=0.01583614864864867, name='Inner9')\n",
      "            Clade(branch_length=0.04497466216216214, name='Inner5')\n",
      "                Clade(branch_length=0.34712837837837834, name='YR429_MIMIV/213-247')\n",
      "                Clade(branch_length=0.30152027027027034, name='Q8QNH7_ESV1K/101-133')\n",
      "            Clade(branch_length=0.017525337837837857, name='Inner8')\n",
      "                Clade(branch_length=0.0335022522522522, name='Inner6')\n",
      "                    Clade(branch_length=0.31520270270270273, name='VF282_IIV6/150-181')\n",
      "                    Clade(branch_length=0.33344594594594595, name='Q9EMK1_AMEPV/2-33')\n",
      "                Clade(branch_length=0.18440315315315317, name='Inner4')\n",
      "                    Clade(branch_length=0.029440154440154465, name='Inner3')\n",
      "                        Clade(branch_length=0.020270270270270285, name='Inner2')\n",
      "                            Clade(branch_length=0.17042042042042038, name='VLTF3_FOWPN/3-33')\n",
      "                            Clade(branch_length=0.15390390390390396, name='Q98271_MCV1/6-36')\n",
      "                        Clade(branch_length=0.06756756756756754, name='Inner1')\n",
      "                            Clade(branch_length=0.09324324324324323, name='VLTF3_VAR67/1-32')\n",
      "                            Clade(branch_length=0.09594594594594591, name='Q8V3K7_SWPV1/1-32')\n",
      "                    Clade(branch_length=0.1969111969111969, name='Q6TVP3_ORFSA/1-32')\n",
      "        Clade(branch_length=0.34184966216216217, name='VF385_ASFB7/145-177')\n"
     ]
    }
   ],
   "source": [
    "print(upgma_tree)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "And let's use the Phylo module to visualize the result!"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 365,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAaMAAAEKCAYAAAC/hjrSAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDIuMi4yLCBodHRwOi8vbWF0cGxvdGxpYi5vcmcvhp/UCwAAIABJREFUeJzsnXdc1WX//58X4MC90BA0EBeyDuDWBHKgdjsxFC01V/5SM8v1rdQss2GlSWb3bZqrG03MkblRnLfiOiBCiIMKt7hFkXH9/jjnfOLIdnAc1/PxOA/P5/pc4319os/7XOv9ElJKFAqFQqGwJFaWNkChUCgUCuWMFAqFQmFxlDNSKBQKhcVRzkihUCgUFkc5I4VCoVBYHOWMFAqFQmFxlDNSKBQKhcVRzkihUCgUFkc5I4VCoVBYHBtLG1AYqlWrJp2cnCxthkKhUDxVHDp06LKU0s7SdhSGp8IZOTk5cfDgQUuboVAoFE8VQog/LW1DYVHTdAqFQqGwOMoZKRQKhcLiKGekUCgUCoujnJFCoVAoLI5yRgqFQqGwOMoZKRQKhcLiKGekUCgUCoujnJFCoVAoLI5yRgqF4oHo27cvc+fO1a7379+Pp6cnGRkZWlrXrl1xd3fXrqOjo2nRogUeHh506dKFGzdumNX5119/Ua5cOb766istzdraGp1Oh7u7O126dOHatWt52pSUlIStrS06nY5GjRoxfPhwsrKyzNK9vLxo2bIlCQkJWrmoqCjatGlDgwYNaNiwIUOGDCEuLg5HR0eysrLM2tDpdERFRbFz5058fHywsbEhPDw8hy0dO3bkzJkzfPfdd9StWxchBJcvX86R78CBA1hbW5vVMX78eNzc3HB1deXtt99GSpmjXEpKCgEBAZQrV46RI0ea3QsLC8PDwwOgkRBioxCiGoAQ4lUhxDEhRJYQonGeD9ISSCmf+I+vr69UKBRPFufPn5fOzs7y4sWLMjMzUzZu3Fju2rVLu79y5UoZEhIi3dzctLTGjRvLyMhIKaWU8+fPlx9++KFZnT179pS9evWSM2bM0NLKli2rfe/fv7+cNm1anjadPn1aay89PV2+9NJLcuXKlWbpUkr5ww8/yP79+2v9qF27tty7d6+UUsqsrCy5YsUKef78edm8eXPNXimljI+Pl3Xq1NHaio6Olq+//rpcsWKFmR2pqamySZMmUkopDx8+LE+fPi1ffPFFeenSJbN8GRkZMiAgQHbq1EmrY8+ePbJly5YyIyNDZmRkyObNm8vt27fn6OutW7fkrl275Ny5c+WIESO09PT0dGlnZycvXbokgYPAl8BH0uDQXIEGQCTQWD4B73fTR42MFArFA1GjRg3Gjh3L+PHj+eGHH/D09KR169YA3Lp1i2+++YYPP/zQrExCQgJt2rQBoH379qxcuVK7t3r1aurUqYObm1uebbZo0YIzZ84Uyj4bGxtatmzJiRMncty7ceMGlStXBmDOnDkMGDCAFi1aACCEoFevXtSoUYOQkBCWLVumlVu2bBkhISGAIUyZp6cnVlY5X6ORkZH4+/sD4O3tTV6xNUNDQwkKCqJ69epamhCCu3fvcu/ePdLS0khPT6dGjRo5ypYtW5bWrVtTunRps3TTy/327dumpArAWeO9eCllAk8gT0VsuuLC9MejUChyJzIy0ux6+PDhLFq0iMjISLP4kZMmTeK9996jTJkyZvnd3d1Zu3Yt3bp1Y8WKFfz9998A3L59my+++IItW7aYTdFlJzMzk4iICAYPHlwoW1NTU4mIiODjjz8G4OTJk+h0Om7evElqair79+8HIDY2lgEDBuRaR3BwMN7e3oSGhmJjY8Py5ctZsWJFgW1v2LCB7t2755vnzJkzrFq1im3btnHgwAEtvUWLFgQEBGBvb4+UkpEjR+Lq6lqoPgOUKFGCuXPnmqbpPIFUYEShK7AQamRkxN/fH71eb2kzFIqnCisrK9588006depE1apVAdDr9Zw4cYIePXrkyL9gwQLmzJmDr68vN2/epGTJkgBMmTKFMWPGUK5cuRxl7ty5g06no2rVqly5coX27dvna5PJ6bRq1YpXXnmFTp06AeDi4oJer+fkyZPMmjWLYcOGFdi/F154ATc3NyIiItDr9ZQoUcJsDSwv9uzZo40S8+Kdd97hiy++wNra2iz9xIkTxMfHk5yczJkzZ9i2bRs7d+4ssE0T6enpzJ07lyNHjgDEGD//V+gKLIQaGWVDp9Pl+OWnUCjyx8rKymyq6n//+x+HDh3CycmJjIwMLl68iL+/P5GRkTRs2JDNmzcDcPz4cX7//XfAsPkhPDyc8ePHc+3aNaysrChdujQjR47E1tYWvV7P9evX+de//sWcOXN4++2387TH5HTyo2vXrrzxxhsAuLm5cejQIbp165ZrXtNUnWnariBOnTpFrVq1NEebFwcPHqRPnz4AXL58mfXr12NjY0NiYiLNmzfXHHOnTp3Yt28fKSkpTJ06FYAff/yRxo1z339g6ruLi4sp6RdgYoGGWxg1MlIoFI+U//f//h9nz54lKSmJ3bt3U79+fe1H3sWLFwHIyspi2rRpDB8+HIBdu3aRlJREUlIS77zzDu+//36OHWIVK1Zk9uzZfPXVV6Snpz+Ujbt379Ze1iNHjmTRokXatB3A0qVLOX/+PABBQUGsX7+e5cuXa84jPzZs2EDHjh0LzHf69Gmtz7169eL777+ne/fu1K5dmx07dpCRkUF6ejo7duzA1dWVHj16oNfr0ev1eToiAAcHB+Li4rh06ZIpqT0QX6BBFkY5I4VCUWyEhYVRv359GjZsSM2aNbXRSWHx9vbGy8vLbFNBYTFN33l5efH+++/z448/AoaNGMuWLWPs2LE0aNAAV1dXdu3aRYUKFQCoVKkSzZs3p0aNGjg7O2v1HThwAEdHR1asWMGbb76pbbzYuHGjmTOaPXs2jo6OJCcn4+npyZAhQ/K1s1evXri4uODh4YGXlxdeXl506dIl17xOTk68++67LFy4EEdHR+Li4qhZsyZTpkwxbRRpBOiA6QBCiB5CiGSgBfC7EGJTkR/kY0LIXPavP2k0btxYPm5xPdPmhezTdCZRv2rVqj22dhMSEujdu7d2ferUKT7++GPeeeedx9amQqF4PKSlpdGqVasnRgxUCHFISvlknSfKA7VmZEEyMzNp0KCBNsebmZmJg4NDrgu/CoXiyadUqVJPjCN62lDTdIUgKSkJV1dXhg4dipubGx06dODOnTuAYUQ1YcIEmjZtSv369dm1axdgcCzjxo2jSZMmeHp68u9//xswjLwCAgLo27evaeulRkREBC4uLrz44ovF20GF4inj6NGj6HQ6s0+zZs0sbZbiIVAjo0KSmJhIWFgY8+bNIzg4mJUrV/Laa68BkJGRQVRUFOvXr2fq1Kls3bqV+fPnU7FiRQ4cOKAN3Tt06AAYQo/ExsaazT+D+YE6hUKRNx4eHuooxjPGM+uMinqAVa/Xo9Pp8rzv7Oys3ff19SUpKUm717NnzxzpmzdvJiYmRos3df36dRITEylZsiRNmzbN4Yju3bvH2rVr+eyzz4pkt0KhUDwLPJPOyHSANT/nUlRKlSqlfbe2ttam6bLfs7a21oJESikJDQ0lMDDQrJ7IyEjKli2bo/4NGzbg4+OTa9gPhUKheNZ5Jp0RFP0A66MOBRQYGMjcuXN5+eWXKVGiBMePH8fBwSHP/GFhYWqKTqFQPLc8s87I0gwZMoSkpCR8fHyQUmJnZ8fq1atzzZuamsqWLVu0TQ4KhULxvPFMnjPK7czQ4yijUCgUTzJP0zkjtbVboVAUmuvXr9O/f39cXFxwcXGhX79+XL161SzPjRs3cHBwyBHO58iRIwgh2LTpn0P/f//9NwEBAbi6uuLm5sa3336r3evdu7e2bdvJyclsDTgmJoYWLVrg5uaGh4cHd+/ezdNmJycnPDw8tLpMce0GDhyYQxTPFA/O2dnZTHwPDIFNv/zyS9LT0xkwYAAeHh64urpqm46SkpJyBFH96KOPzKKQ/+9//2Po0KH5CuOZuF+YUK/X07x5c3Q6HY0bNyYqKirXctnF/Mg2+yWEqCiE+E0IEW0U2Hsj270BQohE4yf3EOaPmWJ3RkKIWkKI7UKIeOMDGV3cNigUigdj8ODB1KlTh5MnT3Ly5Enq1q3LwIEDzfJMmjQJPz+/HGXDwsJo3bo1YWFhWpqNjQ1ff/018fHx7Nu3jzlz5hAXFwfA8uXLtVhsQUFB2q7VjIwMXnvtNX744QeOHTtGZGQkJUqUyNfu7du3a3XNnj27wH726dPHLORQVlYW4eHh9O7dmxUrVpCWlsbRo0c5dOgQ//73v8121+aHKVRQ6dKl+eSTT/KUy/j1119zRDAfP348U6ZMQa/X8/HHHzN+/Phcy7Zq1YqtW7fmdl5xBBAnpfQC/IGvhRAlhRBVgClAM6ApMEUIUblQHXqEWGJklAG8J6V0BZoDI4QQjSxgh0KhKAInTpzg0KFDTJo0SUubPHky0dHR2iji0KFDXLhwQTtTZ0JKSXh4OAsXLmTz5s3aSMbe3h4fHx8Aypcvj6uraw7xPCklv/zyi7bBZ/PmzXh6euLl5QVA1apVc8gwPCz3i+rt3LkTJycnXnzxRYQQ3L59m4yMDO7cuUPJkiW1OHYFERERQbt27fIUxoO8hQmFEJpM+/Xr16lZs2aubeQj5ieB8sIwZCoHXMHwPg4Etkgpr0gprwJbgIIjvT5iin0Dg5TyHHDO+P2mECIecADiituW+9Hr9UpgT6HIRvY11Li4OHQ6ndmL39raGm9vb+Lj46lXrx7vvfceS5YsISIiwqyePXv24OzsjIuLC/7+/qxfv14b6ZhISkriyJEjOSIp7Nq1ixo1alCvXj3AID0hhCAwMJBLly7Rp0+fPEcJJgICAjS7BwwYwJgxYwAYN24c06ZNy5HfpOAaHR2tBWY1OcNevXqxZs0a7O3tSU1NZebMmVSpUkVzFHlx+fJlSpQoQcWKFfPNl5cw4axZswgMDGTs2LFkZWWxd+/efOvJhe+AtRhUX8sDvaWUWUIIB+DvbPmSMbyTixWLrhkJIZwAb2B/LveGCSEOCiEOZguF/tiIjIx8pOeSFIpnDSmlaR0iRzrA999/T+fOnalVq1aOPGFhYZr8Qp8+fcym6sAwGggKCmLWrFk5Rhn3H3vIyMhg9+7d/Pzzz+zevZtVq1blcH73k32azuSIAGbMmKGl3x/RwTQ6ysjIYM2aNbz66quAIYKKtbU1Z8+e5fTp03z99decOnUq12cDaOmbN2/OMWK8n/yECefOncvMmTP5+++/mTlzZqEVb7MRCOiBmhgieX8nhKgA5GZ4se9ss9jWbiFEOWAl8I6UMsdPCinlf4D/gGE3XXHYpHbSKRR54+bmxpEjR8jKytLE9LKysoiJicHHx4cVK1awa9cuvv/+e27dusW9e/coV64cn376KStXrmTt2rV8+umnSClJSUnh5s2blC9fnvT0dIKCgujXr1+O0VJGRga//vorhw4d0tIcHR3x8/PToul37tyZw4cP07Zt20fa35CQEDp06ICfnx+enp5Ur14dgP/+97907NiREiVKUL16dS1Kd+fOnXNs5rhy5YoWbWXDhg28++67+baZnzDhokWLtA0er776qiZFERgYyIULF2jcuLEmi5EHbwCfS8OvhxNCiNNAQwwjIf9s+RyByEI9pEeIRUZGQogSGBzRz1LKXy1hg0KhKBp169bF29vbbFpr2rRptG3bltq1a/Pzzz/z119/kZSUxFdffUX//v35/PPP2bp1K15eXvz9998kJSXx559/EhQUxOrVq5FSMnjwYFxdXXN9UW/dupWGDRvi6OiopQUGBhITE0NqaioZGRns2LGDRo0e/bKzi4sLVatWZeLEiWYjs9q1a7Nt2zaklNy+fZt9+/bRsGFDypUrh729vTZKu3LlChs3bqR169ZIKYmJiSlw9iU/YcKaNWuyY8cOALZt26ZNW27atAm9Xl+QIwL4C2gLIISoATQATgGbgA5CiMrGjQsdjGnFSrGPjIyLZ/OBeCnlN8XdvkKheHAWLFjAqFGjqFu3LtevX6dJkyb89ttv+ZYJCwvLMe0UFBTE3LlzcXZ2ZsmSJdrWa4Dp06fTuXNnIPfgwZUrV+bdd9+lSZMmCCHo3Lkzr7zySr42ZF8z8vT0ZPHixYXqb0hICP/3f/9nZv+IESN44403cHd3R0rJG2+8gaenJwCLFy9mxIgRvPfeewBMmTIFFxcXDh48iLe3t9lUnpOTEzdu3ODevXusXr2azZs35+tU582bx+jRo8nIyKB06dL85z//yTXf7Nmz+fLLL01KtY2EED9KKYcAnwALhRBHMUzNTZBSXgYQQnwCHDBW8bGU8kqhHtAjpNgPvQohWgO7gKNAljH5fSnl+rzKFMehV4VCUTQSEhLo3LkzoaGhmvNQ5M60adOoW7duoWTLHyVP06FXS+ym203uC2YKheIpokGDBpw8edLSZjwV3L9NW5ETFZtOoVA8EzRr1oy0tDSzNNMUoOLJRzkjhULxTLB/f44TIoqnCBWbTqFQKBQWRzkjhUKhUFgc5YwUCoVCYXGUM1IoFAqFxVHOSKFQKBQWRzkjhULxRNO3b1/mzp2rXe/fvx9PT08yMjI04TxPT0/8/Pz4888/zcpmZmbi7e3Nv/71Ly2tX79+NGjQAHd3dwYNGkR6ejoAV69epUePHnh6etK0aVNiY2PztUsIweuvv65dZ2RkYGdnp7W1cOFCTTjvo48+QgjBiRMntPwzZ85ECIHpQL+TkxOXL1/G39/fTIAQDBG733rrLe26Y8eOnDlzJs++/PHHH7Ro0QLARwgxNpvNpYUQUdkE9qbm0TedEOJ/xjwxQojeueQJFULcynY9UwihN36OCyGu5fsA70M5I4VC8UQzc+ZMZsyYwaVLl8jKymLkyJF8//332NgYTqZs376dmJgY/P39c8hBfPvtt7i6upql9evXjz/++IOjR49y584dLabb9OnT0el0xMTEsHjxYkaPzl/3s2zZssTGxnLnzh0AtmzZgoND3soLHh4eZhpJ4eHhuYb/uV9LCczDIt25c4crV67g4OCQZ1+qVKliEhG8cF/1acDLRoE9HdBRCNE8F3NTgf5SSjcM2kazhBCVTDeFEI2BStkLSCnHSCl1UkodEAoUKe6ockYKheKJpkaNGowdO5bx48fzww8/4OnpSevWrXPka9GihZkwX3JyMr///rsW3dpE586dEUIghKBp06YkJycDBr0mU+Tvhg0bkpSUxIUL97/LzenUqRO///47kFPq4n66d+/OmjVrADh16hQVK1bEzs4uR75evXqxbt067QBvUlISZ8+e1focGRmphTzLqy/Vq1enSZMmcJ8UhDRgGs2UMH5yxISTUh6XUiYav58FLgJ2AEIIa2AGkJ+IVAgQls/9HChnpFAonniGDx9OXFwcM2bM4Msvv8w1z8aNG+nevbt2/c477/Dll19qchf3k56ezpIlS+jY0SBq6uXlxa+/Gn7MR0VF8eeff2ov97wwyZPfvXuXmJiYHMKA2alQoQK1atUiNjaWsLAwevfOMfMFGJRrmzZtysaNGwHDqKh3795akNUNGzZoNufVl/wQQlgLIfQYHMwWKWW+p4WFEE2BkoAp9tNIYK1RKDW3/C8CzsC2Ao3JhorAUEwoBVmFovDcH+TYysqKN998k4MHD1K1alWzewEBAVy4cIHq1atr03Tr1q2jevXq+Pr65hkw+a233qJNmza89NJLAEycOJHRo0ej0+nw8PDA29tbmwrMC09PT5KSkggLCytUsFiT89q0aRMRERH89NNPueYzTdV169aNZcuWsWDBAu3enj17+Oqrr/LtS35IKTMBnXHabZUQwl1KmesCmRDCHlgCDDCqwtYEXsVc/yhHN4FwYzuFRo2MigF/f/8cKpIKhaJoWFlZ5TrK2b59O3/++Sdubm5MnjwZMLyw165di5OTE3369GHbtm289tprWpmpU6dy6dIlvvnmHxWbChUq8NNPP6HX61m8eDGXLl3ShPHyo2vXrowdOzbfKToTXbp0YcmSJdSuXTuHom12unfvTkREBIcPH+bOnTv4+PgAhum9WrVqUbJkyXz7UhiklNcwiOh1FEI0y7b5oCuAUQX2d+BDKeU+YzFvoC4Gcb4koIwQ4sR9VfehiFN0oEZGxYZOp1OSFgrFY8LW1pZZs2bh4eHBhx9+yGeffcZnn30GGEZZX331FUuXLgXgxx9/1EYm2Z3btWvXKFOmDCVLluTHH3+kTZs2+ToME4MGDaJixYp4eHgU+P+4ra0tX3zxBfXr1883X7ly5fD392fQoEFmTu7+Kbq8+pIXQgg7IF1KeU0IYQu0A74wTtXpsuUrCawCFkspV5jSpZS/Ay9ky3dLSlk323UDoDLwvwKNuQ81MlIoFM8E9vb2hISEMGfOnHzzDR8+nAsXLtCiRQt0Oh0ff/wxAPHx8bi5udGwYUM2bNigSXwXhKOjY4E777LTp08fbaSTHyEhIURHR5tpIG3cuNHMGeXVl/Pnz5vUcWsAHwohko0jHXtguxAiBoOY3hYp5bpcmg8G2gADs42Y8pepNZoNLJMPIJRX7OJ6D8LTLq73pNmjUCiePtLS0mjVqhVFeRcqcT1FgTg5OXHw4EGqVav22NspX7481tbW2NjYFOkPWaFQPDmUKlXqmf7/VzmjZ5TMzEysra0BwwLv43Z6CsWzSEpKinb2KDsRERE5dvUpHg7ljCxMUlISnTp1onXr1uzduxcHBwfWrFmDra0t/v7+NGvWjO3bt3Pt2jXmz5/PSy+9RGZmJhMnTiQyMpK0tDRGjBjBm2++SWRkJFOnTsXe3h69Xk9cXJylu6dQPNVUrVpV7YQtJp5rZ1RcZ3/0ej06Xd5rf4mJiYSFhTFv3jyCg4NZuXKltg01IyODqKgo1q9fz9SpU9m6dSvz58+nYsWKHDhwQJtH7tChA2A4rBcbG6ttSRVC0KFDB4QQvPnmmwwbNuzxd1ihUCiKyHPrjExnf/JzEsWFs7OzZoevry9JSUnavZ49e+ZI37x5MzExMYSHhwNw/fp1EhMTKVmyJE2bNjU7G7Fnzx5q1qzJxYsXad++PQ0bNqRNmzbF0zGFQqEoJM+tM4LiO/tT0AisVKlS2ndra2st8GL2e9bW1mRkZAAgpSQ0NJTAwECzeiIjIylbtqxZWs2aNQFDrKoePXoQFRWlnJFCoXjiUOeMnkICAwOZO3euFi7++PHj3L59O0e+27dvc/PmTe375s2bcXd3L1ZbFQqFojA81yOjp5UhQ4aQlJSEj48PUkrs7OxYvXp1jnwXLlygR48egGHtqW/fvoUKpKhQKBTFzXN76LU4D6KqQ68KhcISPE2HXi02TWcMY35ECJFbKAqFQvGEc/36dfr374+LiwsuLi7069ePq1evavfHjx+Pm5sbrq6uvP3225h++OZXLikpCSEEoaGhWj0jR45k4cKFAAwcOFDbuGOiXLlygOE8nU6n0z6lS5fOdcbAhL+/Pw0aNNDy9+rVC4CEhAT8/f3R6XS4uroybNgwbt++TdWqVbl+/bpZHd27d+eXX37RlFVLlSqVI6K2yT6A9evXU69ePf766y8Azp07p+2E7dixI5UqVTJTpQU4ffo0zZo1o169evTu3Zt79+4BsHPnTnx8fLCxscnxTLIjhGgjhDgshMgQQvTKlv6iEOKQMdTPMSHE8Gz3NmZTg/3BqGH0WLHkmtFoIN6C7SsUiodg8ODB1KlTh5MnT3Ly5Enq1q3LwIEDAdi7dy979uwhJiaG2NhYDhw4wI4dOwosB4bNNt9++6320i0sAQEB6PV69Ho927Zto0yZMtqLPi9+/vlnrYzphf72228zZswY9Ho98fHxjBo1irJly9KhQwcz53b9+nV2797Nv/71L01ZdezYsXk1RUREBKNGjWLjxo3Url0bMMSaM21EGjduHEuWLMlRbsKECYwZM4bExEQqV67M/PnzAahduzYLFy6kb9++BT2av4CBwH/vSz8HtDQqszYDJholIgCCjWqw7hhE9V4tqJGHxSLOSAjhCLwC/GiJ9hUKxcNx4sQJDh06xKRJk7S0yZMnEx0dTUJCAkII7t69y71790hLSyM9PZ0aNWoUWA7Azs6Otm3bsmjRoge2Lzw8nE6dOlGmTJkilz137pwpyChgkAuHnHLgq1atomPHjpQpU0ZTVi1RokSude7atYuhQ4fy+++/4+LioqVv3LiRTp06AdC2bVvKly9vVk5KybZt27RR24ABAzSH6OTkhKenZ4HRuqWUSVLKGCDrvvR7Uso042UpsvkDKeUN41cbDMJ6j309x1IbGGZhkKwtX1DGZwW9Xq8E9hRPNdnXPOPi4tDpdFrIKTAcP/D29iY+Pp7u3bsTEBCAvb09UkpGjhyJq6sra9euzbec6bzdxIkT6dSpE4MGDcphx7hx4zQRvbxYtmwZ7777boF96tevH7a2tgC0b9+eGTNmMGbMGF5++WVatmxJhw4deOONN6hUqRIdO3ZkyJAhpKSkULVqVZYtW8aoUaMKbCMtLY1u3boRGRlJw4YNtfTMzEwSEhJo1KhRnmVTUlKoVKmSJvLn6OhoJq3+sAghamHQLKoLjDNKjJvubQKaAhuAvOcBHxHFPjISQvwLuCilPFRAvmFCiINCiIOXLl0qJuseD5GRkU/E4VqF4lEhpdRksO9PB8PIKT4+nuTkZM6cOcO2bdvYuXNngeVMODs707RpU/773/tnlmDGjBna1FpuoXrOnTvH0aNHc5zDy43s03QzZswA4I033iA+Pp5XX32VyMhImjdvTlpaGiVLlqRr166Eh4dz+fJl9Hp9gdOAACVKlKBly5ba9JqJ/fv35ytTDjmfC5Dr83tQpJR/Syk9MTijAUKIGtnuBWKQnCgFvPzIGs0DS4yMWgFdhRCdgdJABSHEUinla9kzSSn/A/wHDLvpit/MR4vaSad4lnBzc+PIkSNkZWVp00RZWVnExMTg4+PD8uXLad68ubZ436lTJ/bt20fPnj3zLZeV9c9M0vvvv0+vXr2KfEj7l19+oUePHnlOmRUHDFSiAAAgAElEQVSGmjVrMmjQIAYNGoS7uzuxsbH4+voSEhLCtGnTkFLSrVu3QrVhZWXFL7/8Qrt27Zg+fTrvv/8+kFMoLzeqVavGtWvXyMjIwMbGhuTkZO0ge1588MEH/P7774XvLCClPCuEOAa8RLZRkJTyrhBiLdAN2FKkSotIsY+MpJT/J6V0lFI6YZCn3Xa/I1IoFE82devWxdvb22y6bNq0abRt25batWtTu3ZtduzYQUZGBunp6ezYsQNXV9cCy2WnYcOGNGrUiHXrirbhNiwsrFAS4HmxceNG7UD5+fPnSUlJwcHBATBskkhMTGTOnDlFaqNMmTKsW7eOn3/+WRshRURE5BoRPDtCCAICArTNFYsWLaJbt275lvn000/zHDXeV7ejUe0VIURlDAOFBCFEOSGEvTHdBugM/FGIbj4cUkqLfQB/YF1B+Xx9fWVR8PPzk35+fg+dR6FQ5M3Vq1fla6+9Jl1cXGS1atVkp06dZEZGhpRSyoyMDDls2DDZsGFD6erqKseMGVOocqdPn5Zubm5aXr1eL4UQ8qeffpJSSjlgwAC5YsUKMzvKli2rfT99+rSsWbOmzMzMLNB+Pz8/Wb9+fenl5SW9vLxk27ZtpZRSjhkzRtavX196enpKT09PuWTJErNyb7/9trS3tzdr49y5c9LBwUGWL19eVqxYUTo4OMjr16/nsO+vv/6STk5OcuHChTIgIMCs3tatW8tq1arJ0qVLSwcHB7lx40YppZQnT56UTZo0kS4uLrJXr17y7t27Ukopo6KipIODgyxTpoysUqWKbNSoUY4+AgeBJkAycBtIAY4ZbtEeiAGijf8OM6bXwKACGwMcA0IBG/mY/YE69KqmzxSKhyYhIYHOnTsTGhpK586dH3u5p52lS5eSnJzMxIkTH2s7T9OhVxUOSKFQPDQNGjTg5MmTxVbuacckEaP4B+WMFArFM02PHj04ffq0WdoXX3xRqN12iuJDOSOFQvFMs2rVKkuboCgESkJCoVAoFBZHOSOFQqFQWBzljBQKhUJhcZQzUigUCoXFUc5IoVAoFBZHOSOFQvHYGThwIP/+97/N0lavXq0ddLW2tjYTxktKSmLLli34+vri4eGBr68v27Zt08qGhYXh4eGBp6cnHTt25PLly4AhonfDhg3x9PSkR48eXLt2LU+bIiMjNSG7hQsXMnLkSCIjI2nRooVZvoyMDGrUqMG5c+cACA0NpUGDBri5uTF+/HizvL6+vty7d48PPviAWrVqmQnrmdqxs7PT+vnjj/+o6CxatIh69epRr169POUzUlJSCAgIoFy5cowcOdLsXl7PRAjxqlEkL0sI8cQegFXOSKFQPHbu1wICg8yDKb6bra2tWSRuJycnqlWrxm+//cbRo0dZtGgRr7/+OmBwDqNHj2b79u3ExMTg6enJd999BxhkIGJjY4mJiaF+/fp89tlnRbKzTZs2JCcnk5SUpKVt3boVd3d37O3t2b59O2vWrCEmJoZjx46ZieklJSXh4OBAyZIl6dKlC1FRUbm20bt3b62fQ4YMAeDKlStMnTqV/fv3ExUVxdSpU81Uc02ULl2aTz75JIeabH7PBIgFegI7i/QwihnljBQKxWOnXbt2/PHHH9roIjU1la1bt9K9e/c8y3h7e2sRqt3c3Lh79y5paWlaLLPbt28jpeTGjRtavg4dOmjaP82bNyc5OblIdlpZWfHqq6+yfPlyLS2705w7dy4TJ06kVKlSgEGV1kT2KNzNmzfH3t6+0O1u2rSJ9u3bU6VKFSpXrkz79u3ZuHFjjnxly5aldevWlC5d2iw9v2cipYyXUiYU2hgLoQ69WgAlsqd4Hsge99Ha2pqePXvyyy+/MHr0aNauXUtAQICmbHrnzh1N88vZ2TnHQdWVK1fi7e2tOYG5c+fi4eFB2bJlqVevHnPmzMnR/oIFC+jdu3eR7Q4JCWHYsGFMmDCBtLQ01q9fz8yZMwE4fvw4u3bt4oMPPqB06dJ89dVXNGnSBDBE+zbly4+VK1eyc+dO6tevz8yZM6lVqxZnzpyhVq1aWp6iiuiVKFEi12fy5ptvFrH3lkONjIoZf3//AkO7KxTPItmn6rKPNsB8mu5+R3Ts2DEmTJigrTmlp6czd+5cjhw5wtmzZ/H09MwxHffpp59iY2NDv379imxnkyZNuHXrFgkJCWzYsIHmzZtTuXJlwDAddvXqVfbt28eMGTMIDg5GSsm9e/dITk6mTp06+dbdpUsXkpKSiImJoV27dgwYMAB4eBG9wjyTJx01MrIAOp1ORQtXPHe0atWKc+fOER0dzd69e3OsIeVGcnIyPXr0YPHixbi4uABoP+ZM18HBwXz++edamUWLFrFu3ToiIiIeWBW1T58+LFu2jPj4eDOn6ejoSM+ePRFC0LRpU6ysrLh8+TIxMTG0bt26wHqrVq2qfR86dCgTJkzQ6s3+TkhOTsbf359Vq1YxdepUAH788UcaN859/0FBz+RpQI2MFApFsSCEIDg4mAEDBtC5c+cc6x73c+3aNV555RU+++wzWrVqpaU7ODgQFxfHpUuXANiyZQuurq6AYarsiy++YO3atZQpU+aBbQ0JCWHp0qVs27aNrl27aundu3fXdvUdP36ce/fuUa1aNTZu3EinTp0KrNe0Zgawdu1aze7AwEA2b97M1atXuXr1Kps3byYwMJAePXpoI8a8HBHk/0yeFpQzUigUxUZISAjR0dH06dOnwLzfffcdJ06c4JNPPtG2Ql+8eJGaNWsyZcoU2rRpg6enJ3q9XpPyHjlyJDdv3qR9+/bodDqGDx/+QHY2atSIMmXK8PLLL1O2bFktfdCgQZw6dQp3d3f69OnDokWLEEIQGRmJn5+flm/8+PE4OjqSmpqKo6MjH330EQCzZ8/Gzc0NLy8vZs+ezcKFCwGoUqUKkyZNokmTJjRp0oTJkydTpUqVXG1zcnLi3XffZeHChTg6OhIXF5fvMxFC9BBCJAMtgN+FEJse6KE8ZpS4XjFPl+XWrpOTEwcPHqRatWqPte1r164xZMgQYmNjEUKwYMGCHGcqFApF0UhOTmbo0KFs2LDB0qbkQInrPaUUl1NITk7G3d0dKSVDhw59rG0BZGZmYm1tzejRo+nYsSPh4eHcu3eP1NTUx962QvGs4+jo+EQ6oqcNNU1XjGRmZnL79m3OnTtHVFQU0dHRrFu3joyMDMBwaM7V1ZWhQ4fi5uZGhw4duHPnDmAYUU2YMIGmTZtSv359du3apdU5btw4mjRpgqenp7bjKDIykoCAAPr27YuHhwc3btxg586dDB48GICSJUtSqVIlCzwFhaJ42bRpk1l0B51OR48ePSxtluI+lDPKhcfpFFJTU6lQoQJlypTBxsYGPz8/sxFKYmIiI0aM4NixY1SqVImVK1dq9zIyMoiKimLWrFnaDpv58+dTsWJFDhw4wIEDB5g3b56mahkVFcWnn35KXFwcp06dws7OjjfeeANvb2+GDBnC7du3i+V5KhSWJDAw0Cy6Q27bxxWWR03T5UFiYiJhYWHMmzeP6tWr4+PjQ40aNdDr9Zw8eRIXFxfu3LlD165d8fLy4uzZs6Snp/Piiy9ia2vL2LFjWbBgAWlpaRw9epTGjRtTvXp1Dh06RGZmJikpKdja2rJ+/XptZASGA3+mw3++vr5mYUl69uyZI33z5s3ExMQQHh4OwPXr10lMTKRkyZI0bdoUZ2dnwODIDh8+TGhoKM2aNWP06NF8/vnnfPLJJ8XwNBUKhSJ/nllnpNfr8410oNfrtZd+bpicgr+/Pzdv3jTbUWNaUypfvjx3794F4OrVq9y+fVvbWpmZmcmdO3ewsrKifPny2NraAoaT6DVr1qR9+/aUK1cOLy8vjh07ptVtOmFuymsakWW/Z21trTkwKSWhoaEEBgaa2R8ZGWlms6OjI46OjjRr1gyAXr16PXXnEBQKxbOLmqbLg+xOoWbNmgwYMIDIyEh0Oh3z5s0jMjKStWvXUqNGDSIjI2nTpg2rVq3i1q1b3Lp1izt37hATE8OsWbNo3bo1kZGRWnl7e3sOHz7Mzp07qVKlCiVKlHhgOwMDA5k7dy7p6emA4exDbtNvL7zwArVq1SIhwRCiKiIigkaNGj1wuwqFQvEoeWZHRgVFOXjU8eFMTuHll1+mRIkSHD9+HAcHh1zz3rt3D4C//vqLX3/91WwEU1SGDBlCUlISPj4+SCmxs7Nj9erVueYNDQ2lX79+3Lt3jzp16vDTTz89cLsKhULxKHlmnVFxUxSncOzYMRo1akSJEiWYM2cObdu2BQzTf7GxsVq+7OHpszvWatWqaWtGVlZWTJ8+nenTp5u14e/vn8Ph6nQ6inJeS6FQKIoNU+jxJ/nj6+sri4Kfn5/08/N76DxFyVdYHnV9CsWzxrVr1+Trr78u69SpI+vUqSP79u0rr1y5ot0fP368dHNzk25ubnLZsmVaup+fn6xfv7708vKSXl5eMigoSEop5ZQpUyQgExMTtbzffPONBOSBAweklFK++OKL8tKlS1JKKQ8ePCidnJzk4cOHZXx8vGzevLksWbKknDFjRqHs//XXXyUg4+PjtbTTp09LQH744Yda2qVLl6SNjY0cMWKEZmfNmjU1+728vOTVq1fl9u3bZYUKFaROp5MNGzaUH330kbx165asUqWKvHbtmlnb3bp1k8uXL5dSSnnv3j0J3AZqAduBeOAYMFrm8p4FSgNRQLQx39Rs9+Yb02OAcKBcbnU8zMcia0ZCiEpCiHAhxB9CiHghhAoDoFAoABg8eDB16tTh5MmTnDx5krp16zJw4EAAfv/9dw4fPoxer2f//v3MmDGDGzduaGV//vlnbfu2aYcpgIeHh1lg1vDw8FzXTGNiYujVqxfLly/H29ubKlWqMHv2bLNZioIICwujdevWOQLB1qlTh3Xr1mnXK1aswM3NzSzPmDFjzLagm84CvvTSSxw5coSDBw+ydOlS/vjjDzp06GA2+3L9+nV2796tqdfu3r0b4BaQAbwnpXQFmgMjhBC5LRinAS9LKb0AHdBRCNHcZJqU0ktK6Qn8BYzMpfxDYakNDN8CG6WUDQEvDB5boVA855w4cYJDhw4xadIkLW3y5MlER0eTkJBAXFwcfn5+2NjYULZsWby8vHIVobuf7t27s2bNGgBOnTpFxYoVsbOzM8sTHx9P9+7dWbJkCU2bNgUM4nlNmjQp9CajW7dusWfPHubPn5/DGdna2uLq6qpNlS9fvpzg4OBC1WuibNmy+Pr6cvLkyRzquatWraJjx45agFjjc7khpTwnpTwMIKW8ieF9m2NB2zi4umW8LGH8SOO9GwDCEAbd1pT+KCn2NSMhRAWgDTAQQEp5D7hX3HZYkoK2nSsUzxPZ10Pj4uLQ6XRYW1tradbW1nh7exMfH4+XlxdTp07l3XffJTU1le3bt5uNcPr166cdo2jfvj0zZswAoEKFCtSqVYvY2FjWrFlD7969c2zg6datG0uXLi2UFERerF69mo4dO1K/fn2qVKnC4cOH8fHx0e6bpCleeOEF7ZjH2bNntfszZ85k6dKlAFSuXJnt27eb1Z+SksK+ffuYNGkS9erVY8iQIaSkpFC1alWWLVvGqFGjtLzGsjezlxdCOAHewP7c7BdCWAOHgLrAHCnl/mz3fgI6A3HAe0V+OAVQaGckhKiOYU4RACnlXw/YZh3gEvCTEMILQ8dHSynN9iMLIYYBwwBq1679gE09eURGRipHpFDkgZQyVw0i47oFHTp04MCBA7Rs2RI7OztatGihyYyDYZouL6kFkyPYtGkTEREROZxRu3bt+PHHHwkMDDRzhkUhLCyMd955R2svLCzMzBl17NiRSZMmUaNGjVxVaMeMGZPrlOCuXbvw9vbGysqKiRMnatN7Xbt2JTw8nKCgIPR6PR06dADg7NmzpqjfWaY6hBDlgJXAO6aRzv1IKTMBnRCiErBKCOEupYw13nvD6KxCgd7AI92OW6AzEkJ0Bb4GagIXgRcxDPPc8itXQJs+wCgp5X4hxLfARGBS9kxSyv8A/wFD1O4HbOuJRAnrKRS54+bmxpEjR8jKysLKyrCKkJWVRUxMjPZS/+CDD/jggw8A6Nu3L/Xq1StU3V26dGHcuHE0btyYChUq5Lj/3XffMXz4cN566y0tnFdRSElJYdu2bVpU/MzMTIQQfPnll1qekiVL4uvry9dff82xY8f47bffClX3Sy+9ZLbeZCIkJIRp06YhpaRbt27adOKGDRsIDAxk0yaDWoQQogQGR/SzlPJXY1otwGTAD1LKH0z1SimvCSEigY5AbLb0TCHEcmAcj9gZFWbN6BMMi17HpZTOQFtgz0O0mQwkZxv+hWNwTgqF4jmnbt26eHt7M23aNC1t2rRptG3bltq1a2uhtMCw2SAmJkYbDRSEra0tX3zxhebI7sfKyoqwsDASEhKYPHlykW0PDw+nf//+/PnnnyQlJfH333/j7Oxs2kig8d577/HFF1+Yqb4+KAEBASQmJjJnzhwzRdrsYn/GdZ75QLyU8htTHinl31JKnfHzgxDCzjgiQghhC7QD/hAG6marqwvwx0Mbfx+FmaZLl1KmCCGshBBWUsrtQogvHrRBKeV5IcTfQogGUsoEDM4t7kHrUygUzxYLFixg1KhR1K1bl+vXr9OkSRNtBJGens5LL70EGNaBli5dajZNl33NqFq1amzdutWs7oJE/UqVKsWaNWvw8/OjRo0aBAUF0bhxY27cuIGVlRWzZs0iLi4u15FVWFgYEydONEsLCgriv//9ryYvDobR3/276ExkXzMC8jyraMLKyoqgoCBWrFhBmzZtAEMossTERBo2bGjK1gp4HTgqhNAb096XUq6/rzp7YJFxKs4K+EVKuU4IYWVMrwAIDFu8/1++hj0ABYrrCSG2At2Bz4BqGKbqmkgpWz5wo0LogB+BksAp4A0p5dW88ltSXM9SInwKhQISEhLo3LkzoaGhdO7c2dLmPBXs3r2bpUuX8sMPPzxz4nrdgLvAGKAfUBH4+GEalVLqgafiASkUCsvRoEEDTp48aWkznipat279UDsCLUVhnNGLUkrTNNoiACGEPxD5mGxSKBSKJ5aUlBQthFd2IiIiHsk60PNKYZzRL0KIJcCXGLZ2f4lhVKOiJigUiueOqlWrotfrC86oKBKF2U3XDENso73AAeAshgUxhUKhUCgeCYVxRunAHQwhIEoDp6WUWfkXefJwcnLi8uXLj72dQYMGUb16ddzd3c3Sr1y5Qvv27alXrx7t27fn6tU892soFArFc0dhnNEBDM6oCdAaCBFChOdf5PkjMzMTgIEDB+YaK+vzzz+nbdu2JCYm0rZtW6WyqlAoFNkojDMaLKWcLKVMl1Kel1J2A9Y8bsMeF0lJSbi6upKQkMCBAwfo0KGDJu3t7+/PhAkTaNq0KfXr12fXrl2AIRTJuHHjaNKkCZ6entrp7MjISAICAujbty8eHh4AtGnTxhSGw4w1a9YwYMAAAAYMGFDg+QGFQqF4nihwA4OU8iDkiE2343Ea9bhJTExEp9PRoEEDKlWqxMqVK3nttdcAyMjIICoqivXr1zN16lQAzp07R8WKFTlw4ABpaWm0atVKO/UdFRVFbGwszs7O+bZ54cIF7O3tAbC3t+fixYuPsYcKhULxdFHgyEgI0UUIkQicxuCEkoANj9mux4qzszPlypUDwNfXV1NNBejZs2eO9KtXr7J48WJ0Oh3NmjUjJSWFxMREAJo2bVqgI1IoFApF/hRmmm4ajzY2ncUpVaqU9t3a2pqMjIwc9+5PDw0N1QSvTp8+rY2MypYtW6g2a9Sowblz5wDDSKt69eoP3Q+FQlE4Bg4cmCP46erVq7WoDqYfpyY+/vhjdDqdJmdh+j5nzhw+/PBDHBwctDRTrLsBAwbg5eWFh4cHwcHB3L5tJkRgRl51pKWlMWrUKFxcXKhXrx7du3fXJCZGjRrFd999p9XRtm1bhg8frl2PHj2a2bNnc+LECWxtbdHpdABuQog5pvhyQggphNBC+QghfhBCvJbt+iUhxFwhRAshRLQQQm/8t2tu/RBC9BRCxBjzHRBCtDSmOwshDhvTY4UQQ/N8GEYKtZtOSpkCaLHpMKgAPjdUrlyZuXPnkp6eDsDx48fz/UPLja5du7Jo0SIAFi1aRLdu3R65nQqFInfuF6IDWLZsmVlw0exMnjwZvV7PwYMHKV++vPZDdMSIEQCMGzdOS/v0008Bww/W6Ohojh49ygsvvMDcuXPztSm3OiZMmEBaWhrHjx8nMTGRV155haCgIABatmzJ3r17AcOGqWvXrhETE6PVt3fvXlq1Mpy6adCggeksVByG93UXY7YLwBghRF5LNB2BjRjiz/lKKXVAJ2CeMUbd/WwGvIz53sSotIDhCFALY3pLYJJxqSdPCuOMrhl1MHYCPxslH9ILUe6Zwd7enkaNGuHj44O7uztvvvmm2agpOyEhIbRo0YKEhAQcHR2ZP38+ABMnTmTLli3Uq1ePLVu25AioqFAoHh/t2rXjjz/+0GYnUlNT2bp1K927d39kbZiCp2ZlZXH37t1cdZny4+bNmyxdupRvvvlG01MaOtQwoNixYwetWrXSnFFMTAw6nQ5bW1tu3LjBnTt3SExMxMvL6/5qJfA/DGJ5AOeBXRgCp+bGy0CElDJVSml6ydka/83RISnlLflPgNOy/KMMmyalTDOml8Lga/J9IIWJwBANpGIem65cviWeQEzrP9WqVSM2NlYLgJpdyCp7MNRq1aqRlJSEv78/QgimT5/O9OnTzer09/fPIZQXFhaWa/tVq1YlIiLiofuheHJRoolPFtn/f7a2tqZnz5788ssvjB49mrVr1xIQEED58uUfqO4ZM2awcOFCAL766ivatWsHQP/+/dm4cSOenp58++23RaqjSpUqZuvZJho3bsyxY8fw8/MjMzOTs2fPsnfvXlq0aEH16tXZt28fpUqVwsfHxyyCuRErDA5mQra0z4C1QohF2TMKIWoAt03S48Ypt3kYNOz6GoX3ciCE6AV8iiGQduds6U7AWgyO8F0p5YX8nkdhRkYBUsosKWWGlHKRlHI2hjNHCoXCiL+/vwoR84STfaouvym6wpB9is3kiAAWL17MuXPncHFxYcWKFUWqIz+VW1O6aarO5IxatGihXbds+Y+QQkJCgmnNqAGwSkq5JVt9JwA9BrXW7AQCm7Ll2yuldMMQhecDIUTJ3PohpQyXUjYAemHQvzOlJ0kpPYF6wGAhRLX8nkeeIyPjItdbgIsQIibbrfI85RsYFIrHgU6nU1IjTzCtWrXi3LlzREdHs3fv3hxrSI8Ka2trgoODCQ0NpX///oUuV79+fU6dOsWtW7fMRkeHDx/m1VdfBf5xRnFxcbi6ulK9enXmzJlDyZIleeutt7QypjUjIUS8lPKTHI0ZRjL/BfZlS+sETL8/o5TymBDiHtBICNEGGGS81UFKeTFbvu1CiMVCiEpSymvZ0s8IIf7AEDQhzwOW+Y2M/oth0WuN8V/Tx1dK+Vo+5RQKheKJQwhBcHAwAwYMoHPnzpQuXbrgQoUkKyuLU6dOAYaRzG+//ZZd3K5QlC9fnr59+zJu3DiysgwR1xYsWEBmZiZ+fn6AwaGuWbOG6tWrY2VlhZ2dHRcvXmT//v20aFH42NVSymPASQwOCOPmBFcp5VHjtbNRZA8hhDOGqbY/pZSzs6nDXjTu0BPGfI2NdV8TQjgKIUob06tiCKx9PD+b8nRGUsrrxmFWiJTyz2yfK4XusUKhUDxBhISEEB0dnUPxNTU1FUdHR+3zzTff5FFD7mRmZvLaa6/h4eGBp6cnKSkpecqb58eXX36JlZUV9erVo27duqxevZpff/1Vu6/T6bhw4YKZ43Fzc8POzo5KlSoVtblpGIJgAzQFsiuY+gExRmXYcODNPARQg4FYY77Z/DP15w4cEEJEA9uBz7JJEeVKgUqvTwJK6VXxpJPb34mTkxMHDx6kWrV8p8ofir///pv+/ftz/vx5rKysGDZsGKNHj35s7SmeLgqr9CqE+AiIlVJaLO5oYXbTKRSKJ5DMzExsbGz4+uuv8fHx4ebNm/j6+tK+fXsaNWpkafMUTxFSyo8sbUNhdtMp7qO45ChmzpyJm5sb7u7uhISEcPfu3cfepuLRYwrOO3ToUNzc3AoVnDczM7NQwXnt7e3x8fEBDGsOrq6unDlzxjIdVeQgeyQH00dF7M8dNTJ6AsnMzOT8+fPMnj2buLg4bG1tCQ4OZtmyZQwcONDS5j03FOXckF6vN22lzZXExETCwsKYN28ewcHB+Qbn3bp1K/Pnzy9ycN6kpCSOHDlCs2bNit5ZxWNh8uTJTJ482dJmPBWokdFD8Dh/8YLhJXXnzh0yMjJITU2lZs2alunoc8ijPjfk7OysOavCBOfdvHlzkYLz3rp1i6CgIGbNmqVFAlAoniae65GRXq8v8NevJX/xjh07ltq1a2Nra0uHDh20fIrioSjnhgr6O7o/OK/pR0v2e9mD80opCQ0NJTAw0KyeyMjIHMF509PTCQoKol+/fppjUyieNp7bkVFkZGS+TqawPK5fvFevXmXNmjWcPn2as2fPcvv2bZYuXfrQ9iqeDgIDAwsVnFdKyeDBg3F1deXdd98tbjMVikfGcz0yKsyvXkv94t26dSvOzs7Y2dkBBse2d+9ebdSleLYZMmQISUlJ+Pj4IKXEzs4uV3XgPXv2sGTJEjw8PLQfRdOnT9ekERSKp4Xn2hlZAtMv3pdffpkSJUpw/PhxHBwccuSrXbs2+/btIzU1FVtbWyIiImjcuMDjAooniPuD85ooKDgvgJWVVaGC87Zu3Zqn4aygQlEQFpmmE0KMEUIcM4ouhZnCRjwPDBkypFByFM2aNaNXr174+Pjg4eFBVlYWw4YNs4DFCsWTz/Xr1+nfvz8uLi64uLjQr18/rl79J/qkOEoAACAASURBVGDAhAkTcHd3x93dneXLl2vpERER+Pj4oNPpaN26NSdOnADgm2++oVGjRnh6etK2bVv+/PNPALZv3262Tbt06dLaiPW7776jbt26CCEKPPqxcOFChBBmkfxXrVqFEILwcMO50/T0dCZOnEi9evVwd3enadOmbNiwoUChwEGDBlG9enXc3d1ztGsUzZtn/O4phPif8V18NLf3sBCiqVEgzySy1yPbvUpCiHAhxB9CiHghROHjEeWGlLJYP4ADBglzW+P1L8DA/Mr4+vrKouDn5yf9/PyKVKY46lI8PRT1v7v6O7EsQUFBcsqUKdr15MmTZdeuXaWUUq5bt062a9dOpqeny1u3bklfX195/fp1KaWU9erVk3FxcVJKKefMmSMHDBggpZRy27Zt8vbt21JKKb///nsZHByco82UlBRZuXJlLd/hw4fl6dOn5YsvvigvXbqUr70//fST9PDwkIMHD9bSgoODpZeXl1yxYoWUUsoJEybI/v37y7t370oppTx//rxcvny53Lhxo/T39zerr3fv3nLx4sVSSil37NghDx06JN3c3CRwUJq/f6cCQRhmxWIwCOMBVAWsZc73dRnAxvjdHriY7XoRMMT4vSRQ6f7yRflYagODDWBrVBssg0EVUKFQKIrMiRMnOHToEJMmTdLSJk+eTHR0NAkJCcTFxeHn54eNjQ1ly5bFy8uLjRs3AobgqTdu3AAMoyvT8YmAgADKlCkDQPPmzUlOTs7Rbnh4OJ06ddLyeXt74+TkVGi7X3rpJaKiokhPT+fWrVucOHFCW/dLTU1l3rx5hIaGamvPNWrUIDg4uEChwDZt2lClSpW8mm0LbAU6ADFSymgAKf9/e3ceH1V1/3/89SGobAIqS5EACWGHLOw7gkAEqlQQhIg/kMVqi6BYQf3iF9FSNyhUrd+qVUDERwJiWbQaLIKAIIhI2EUwCRAJJSCyh2yf3x8zc82eCSSZJHyej0cezNy5c+85mZCTc8+5562nNJe8Is0aslcJd3ieiFQHegHvuvdL0UwrdV+JEh8zUtdy4nOAI8Al4HNV/byky2FMUfPmVgFTNDKPte3bt4+wsDAnHRVck4batm3L/v37CQ0N5bnnnuPxxx/n4sWLrFu3zlku6Z133mHQoEFUrlyZ6tWrs2XLluyn4t1332XgwIE5tkdFRV3VDEYRoV+/fqxevZozZ84wePBg4uLiAFcD27Bhw1zvGbvSoEB3nlCqqp4RkWaAishqoDYQpaqv5PG+zsB8XCF7/09V00SkMZAELBCRUGA78Kiq5pzy6aUS7xmJyE3A74BA4FagqojkmCImIr8XkW9F5NukpKSSLqYxhVJUtwqYwtN8QukAwsPDGTRoEN26dSMiIoKuXbs6iajz5s3j008/JSEhgbFjx+ZoXBYvXsy3337L1KlTs2xPTExk9+7dOWbFFtbIkSOJiooqdNjfFQYFhgOeP/wr4soXGuX+d4iI9M3tTaq6VV0hex2Bp91jSxWBdsA/VLUtcAF4yusK5MIXs+n6AXGqmgQgIv8CugFZbqJR1beBt8G1andJF9KYwrKV3X2jdevW7Nixg4yMDCpUcP19nZGRwa5du5x1+6ZPn+5EOtx33300bdqUpKQkdu7c6SyfNGLECAYMGOAcd82aNfzlL39h/fr1WW7hAFi6dClDhgzhuuuuu6qyd+rUiT179lC5cmWaNWvmbG/SpAlHjhzh3LlzufZ4rjAocCDgycZIANar6kkAEfkUaOe+/Pase58JqurEJajqfhG5gCseIgFIUNWt7peXcZWNkS/GjI4AXUSkijuUqS+w3wflMMaUA02aNKFt27bMmjXL2TZr1iz69u1Lw4YNSU9P59SpUwDs2rWLXbt2ER4ezk033cSZM2f44QdX5tt//vMfWrZsCcCOHTt46KGHWLVqFXXq1MlxzsjIyKuKLc/sxRdfzDGFv0qVKowfP57JkyeTkpICuHpjnhvfCxsU6P5dG4Irbhxc8eIh7t/DFXHlF+1T1eX6a3jet+6QvYruYzTCFWMer6rHgaMi0tx9vL5AvnlFBfHFmNFWEVkGfAekATtw94CMMeZKzJ8/n0mTJtGkSRPOnDlDx44d+fjjjwHXFOmePXsCUL16dRYvXuxcpvvnP//JPffcQ4UKFbjpppuYP38+AFOnTuX8+fNO3HfDhg1ZtWoV4Lp/7OjRo076qsdrr73GK6+8wvHjxwkJCWHQoEG88847BZY9t/EocDWozzzzDK1ataJSpUpUrVqV559/3nk9IiKC2bNn51gFPCIigi+//NIzvTxERMYDO4Ed6r52qaqnRWQusA3XpIRPVfXfuRSjB/CUiKQCGcAfPb0pYBLwgYhcD8QCYwusbD6u2XA9XxzLlB2+CstLTk6mV69eXL58mbS0NIYNG8Zzzz1XbOcrjw4cOMCgQYN4/fXXr/mVKDzheiLyDHBIVb26nucLtgKDMaVEeno6N9xwA2vXrqVatWqkpqbSo0cPBg4cSJcuXXxdvDKjefPm/Pjjj74uRqmiqrMK3su3rtmFUo25UsUZHSIiVKtWDXBdXkpNTc11ppgpGxYsWJAjXG/ixIm+LlapZD0jY65AcUaHpKen0759ew4dOsTEiRMtLK8MGzt2LGPHXtVQyjXDGiNTLl3tzacF5VhdSXTIrl27nHXHzpw5w8GDB7n++utzhOX5+fkRExPDL7/8wpAhQ9izZ0+u64wZU57YZTpT7hR1SmtuskeHZF7sNr/okJiYGGJiYoiLi3N6RtnD8jxq1qxJ7969naVrjCnPrGdkyqXCpLTmpqiX9fE2OiQpKYnrrruOmjVrcunSJdasWcOTTz5ZpGUxpjSyxsiYEuBtWF5iYiJjxowhPT2djIwM7r33Xu68804flNiYkmX3GZXgsUzJKIrPzD53Ux547jPydTm8YWNGxphSraAwOc9UeI/nn3/emUbt5+fnPH7jjTd45plnqF+/vrPNs17dmDFjCA0NJTg4mHvvvZcLF3JffHrNmjXOag4eqamp1KlThxMnTgCQkpLCzTffnCXSAlypvM2bNyc0NJROnTqxa9cu57XLly8zYcIEmjdvTosWLZxe8+TJk52yNm3aNMcN1/379+f48eO8+uqrBAUFISL88kveSQ4i8p47JG+3iCwVkaru7dPcAXk7ReQ/ItIgz4MUl6sJQyqpLwvXM4VRFJ+Zfe6lR0FhclWrVs31fampqVqjRo0s26ZPn67z5s3Lsa8nbE9VddKkSTp79uxcj5mWlqb16tXTo0ePOts+/vhjDQ8Pd56vXLlSu3fvrk2bNs3y3u7du+uOHTtUVfXtt9/WAQMGOK/9z//8jxMOmJ6eridPnsxx7rlz5+qDDz7oPD9//rx27txZVV3BfvHx8Vq/fn09ffq0sw85w/WqZ3r8GvCE+/Ht/Bp4Ogn4QIv593r2L+sZGWNKtYLC5IqCJzcoIyOD5OTkPG809vPzY9iwYVmiy7NHOERGRvL4449Tt25dtm3blutxunbtyk8//eQ8X7hwoTNRpUKFCtxyyy053pN9cda1a9dy++23A65gv0aNGhVYT1U9CyAiFcgUlqeqa1X1knu3LYB/gQcrYjaBwZgyqjwH+WUeq7vSMLm8zJ49m4ULFwIwZ84c+vXrB8Do0aOJjo4mJCSEV199Nc/3R0REMHnyZP70pz+RnJzM6tWreeONNwC4cOEC69evZ8GCBRw/fpzIyEg6duyY4xjR0dFOY3ry5Emuv/56nn76aTZs2EDTpk35+9//Tu3atZ39Y2Nj+emnn7IszvrZZ58xcuTIQtdfRBYBA3DFjj+ayy7jgc8KfeCrZD0jY8qgkriXqjS5wjC5XE2dOtW538vTEAEsWrSIxMREgoKC+PDDD/N8f9euXTl16hQ//vgjn3zyCT179qRGjRoArFq1iv79+1OpUiWGDx/ORx99REZGhvPeESNG4O/vz7x585xlgdLS0oiPj6dPnz589913tG/fnmnTpmU5Z2RkJPfee6+T1wSwZcsWunXrVuj6q+pooB7wIzA882siMgYI5tfcoxJjPSNjyqirvZeqLLnCMLlC8/Pz49577+X1119n9OjRee43YsQIoqKi2LFjR45LdFu3biUgIACAEydOsGHDBqcXu2TJElq1asW0adOYNGkSS5cupU6dOlSpUoXBgwcDMHz48ByXIKOionj33Xed5wcOHCAwMNCJwshHMxGJAbao6sOejaqaLiJLcY0PLQIQkQHANOA2VU0p6MBFzXpGxphSr7BhcoWRkZFBbGws4JrQ9fHHH9OiRYt83xMREcGiRYvYsGGDcx/Y6dOn2bp1KwkJCcTHxxMfH89rr71GZGRklvdef/31vPDCC2zYsIEffviBChUqMHDgQGdR3S+++IJWrVo5++/du5dLly7RqVMnZ1t0dHSeOUjZ/KCuoLyHRaSCiDQGJ2zvLuB79/MOwBvAYP01r6hEWWNkjCkTIiIi2LlzZ45xkosXL+Lv7+98zZ1buCtM6enp3H///QQHBxMSEsKpU6ecKd95CQkJ4brrriM8PJzKlSsD8NFHH9G/f/8sUeR33303y5cvJzU1Ncv7q1SpwpQpU/jrX/8KuMaxpk+fTkhICFFRUbzyyivOvpGRkTnqHB0dnSUife7cufj7+3P8+HFat27NQw89lFux/YDFIrIb13jRLcBf3K/NAaoCH4lIjIgsz/cbUAzsptcSPJYpGdfCTa++Cv8DGDduHJ988gl16tRhz549xXouk5MngPGbb74pcF+76dUYU+6kp6cDrptQbfFW36lUqZJXDVFZY42RMeVMcYb/AfTq1Yubb77ZN5UrQZlXcvB8vfTSS74uVrlls+mMKYeKM/zvWjFjxgxmzJjh62JcM6wxMiYPMTExpfbGUl+G/xlTHOwynbkmBAQEcPKk9zNWv/zyy3x/2edHVdm+fTu7d+++ovcXhZII/zOmKFnPyJg8FGYmXXp6On5+foBrmm39+vU5e/Ysn3zySbGUzVfhf8YUF+sZmWtKcQ/uJyQk8O9//5sJEyb4poJXaMKECbRq1Yp27drRpk0bHnrooSy9qcwiIiLo2rUrBw4cwN/fP8vKAMZcKesZeaE0jx2YnAoaTynOwf3HHnuMV155hXPnzhV/RbPxjP/UqlUry/0/TzzxhPM4c2+vVq1aznsqVKjACy+8wAsvvJDlmL17987xs599RQFjioL1jApwNWMHpnS6ksH9RYsWERYWRufOnTl16hQHDx4EyDK477kRtH379iVXmWvImTNnGD16NEFBQQQFBTFq1ChOnz7tvD5t2jRat25Ny5YtmTx5MqqabzBfcnIynTp1IjQ0lNatW/Pss886+/Tu3RvPjfbx8fE0bdqU1atX88EHH2SZ6l2hQoV8F6ydP3++s7JDmzZtWLlyJTt37szyOyUyMpIqVao4qzTs3r2bkJAQpxyeQL7u3btz4MABZs6cydNPP53lPDExMbRs2RKA6dOn06BBgxyhgwAiUk9EPnc/jhaRX0Qkz2vJItJIRLa7V2XYKyIPu7dXEZF/i8j37u1XP+e9uIKSgPnACWBPpm03A/8BDrr/vcmbY/kyXM+UPbl9/o0aNdKkpCSNi4vT1q1bO9tnz57thJrddtttum3bNlVVTUpK0kaNGqmq6tChQzU6OjrHedatW6e//e1vnedPPfWU1q9fXxs1aqR169bVypUr66hRo4q2cm7X4s/4Pffc43xWqqozZszQwYMHq6rqpk2btFu3bpqWlqZpaWnapUsXXbduXb7BfBkZGXru3DlVVU1JSdFOnTrp119/raq//iwcPXpUmzVrpitXrsxRnl27dmlgYGCe5T169Kg2btxYf/nlF1VVPXfunMbGxmp6errWrFlTz549q6qqjzzyiLZt21a3bt2qqqpvvvmmPvzww1nKoar61ltv6V133aXff/99jvM++eST+vzzz6uq6tdff63Hjh3TqlWr5hauNxb4k/txX1zr032ief8evx64wf24GhAP3ApUAfpk2mcjMDCv43jzVZw9o4W4MjMyewr4QlWbAl+4nxtTqnkG9z1/uf7www+5xlK/+OKLziKZUVFR3H777SxevLiki1suHTp0iO3bt2eJ8p4xYwY7d+7kwIEDiAjJycmkpKRw+fJlUlNTqVu3br7BfCLi9B5SU1NJTU3NEqp3/PhxwsPDmTVrlrOidmbZw+6yO3HiBDfeeKNzjmrVqhEYGEiFChXo2LEjW7duBWD79u1MnDiRzZs3A7B58+ZcoyF69erFoUOHaN68OTVr1nTeD7B06VJn/bouXbpQr169vIo1AHdWkap+AeR7PVlVU1T1svvpDbivpqnqRVVd59kH+I6rDOQrtjEjVd0gIgHZNv8O6O1+/B7wJfBkcZXBmKIwYcIE4uPjadeuHapK7dq1WbFiha+LVe7HMjOPb+3bt4+wsDBnxiK4pqa3bduW/fv3c/fdd9OnTx/q1auHqvLII484l63yC+ZLT0+nffv2HDp0iIkTJ9K5c2fn+KNHj2bWrFkMH54l8sexZMkSVq5cmWf5Q0NDqVu3LoGBgfTt25ehQ4dy1113AdCtWzc2b95M165dqVChAr179+bpp5/mscceY/PmzVkuGXp8/PHHzkQZT75T586d2bJlC7fccgtNmzbN9/spIn5Ac1Xdl++OOd/XAPg30ASYqqrHsr1eE1cPK+9EQi+U9JhRXVVNBHD/WyevHUXk9yLyrYh8m5SUVGIFNOVTfHw8tWrVIiAgIMfg/syZMwHXL78OHVxrSuY2uL9792727NnDunXrqFGjBr17985z6nZ+rxWFa20sU1VzjQJ3Xybi0KFD7N+/n4SEBH766SfWrl3Lhg0bgPyD+fz8/IiJiSEhIcGZjOLRr18/3n//fS5evJjjvFu3bqVKlSq0adMmzzL7+fkRHR3NsmXLaNasGVOmTHF+1rp3787mzZv55ptv6NixI0FBQRw6dIikpCTOnz9P48aNneOMGjWKsLAwNm3axJw5cwAYOXIky5YtIyMjozBhg52BrQXulY2qHlXVEFyN0RgRqet5TUQqApHAa6oaW9hjZ1ZqZ9Op6tvA2+BatdvHxTGm1CmtK4oXh9atW7Njxw4yMjKctNOMjAx27dpFu3btWLJkCV26dHEuiQ0cOJAtW7bQq1cvr4L5atasSe/evYmOjnYamGnTprF48WKGDx/OypUrswTZedsAiAidOnWiU6dO9O/fn7FjxzJz5ky6dOnCtm3b+Oqrr+jatSsA/v7+REVF5bhE98EHHzh/JHk0aNCAgIAA1q9fz0cffcTXX3/tzbdxIJDvCrci0hnwzPiYoaqrPK+p6jER2Qv0BJa5N78NHFTVv3lTgPyUdM/ovyJSD1yzOnBNcDDGmHw1adKEtm3bMmvWLGfbrFmz6Nu3Lw0bNqRhw4asX7+etLQ0UlNTWb9+vXOZLq9gvqSkJH755RcALl26xJo1a3KE6s2bN4/q1aszfvx4pxeWkZHBhx9+mCNjKLtjx47x3XffOc9jYmJo1KgRADfeeCMNGjRg4cKFTmPUtWtX/va3v3kdJR4REcGUKVMICgrC39+r4Zq+uMbq86SqW9UVxhemqqtExF9EKgOIyE1Ad+CA+/ksoAbwmFcFLkBJN0argDHux2OAvC+4GmNMJvPnz+fgwYM0adKE2rVrs2XLFt58800Ahg0bRlBQEMHBwYSGhhIaGuqMz0DuwXyJiYn06dOHkJAQOnbsSP/+/Z3UVg8R4b333iMxMZFp06YBsGHDBvz9/bNcSstNamoqTzzxBC1atCAsLIwlS5bw6qu/Dqt0796dy5cv06BBA8DVGMXGxnrdGA0fPpy9e/fmaBSnTZuGv7+/5/JiiIjMFJHaQLKqns1Ut43Ah0BfEUkQkTtyOU1LYKuI7ATWA3NUdbeI+APTgVbAd+6p31d1p3exheuJSCSuyQq1gP8CzwIrgKVAQ+AIMFxVfy7oWL4M1zNlj33+5d+BAwcYNGgQr7/+OoMGDfJ1cUotT7ieiNwP+Ktqqc3AKM7ZdHldUO1bXOc0xlwbmjdvzo8//ujrYpQZqlrq7zEotRMYjDGmrOjcuTOXL1/Osu399993pmKbglljZIwxVynzDajmytjadMYYY3zOGiNjjDE+Z42RMcYYn7PGyBhjjM9ZY2SMMcbnrDEyxhjjc9YYGWPKnIJSX48cOUJ4eDgtW7akVatWxMfHM2TIEMLCwmjSpAk1atRw0lrDw8PzTU4NCAhwlhkKDw/n+PHjAAwYMMBJiX344YdJT0/Ps7yqyqxZs2jatCnNmjWjT58+7N2713ndc46QkBBuu+02Dh8+7Lzm5+eXJV02Pj6eixcvMmrUKIKDg2nTpg09evTg/PnzznuWL1/uWeW8UvayuBNe64vIIyJySERURGrlVXYReVdEdorILhFZJiLV3NsfF5F97u1fiEijPL8B3riaZL6S+rKkV1MY9vmXf/mlvqq6fgY+//xzVXUlrF64cMF5LXtCb0HJqZ6UYFXVp59+WidNmqSqqmfOnFFV1YyMDB06dKhGRkbmWd7XX39dBw4c6JRj9erV2rhxY7106VKOc8yYMUMnTJjgvLdq1ao5jvfCCy/olClTstQhOTnZeT58+HDt0aOHAsc0a3JrZeAb9+O2QACu9NZamsfvX6B6psdzgafcj/sAVdyP/wAsyesY3nxZz8gYU6YUlPq6b98+0tLS6N+/P+BKWK1SpUqexysoOTUzT9oqQPXq1QFIS0sjJSUl17wlj5dffpnXX3/dKUd4eDjdunXjgw8+yLFv165d+emnn/L7FpCYmEj9+vWz1OGGG24A4Pz582zatIl3330X4KZsb+2NK9QUVd2hqvH5nsi131kAcVWwMqDu7etU1RP2tIXSmvRqjC+V9xTUa01hUl/BlU80dOhQ4uLi6NevHy+99FKW/bPzNjn1k08+ybLEzx133ME333zDwIEDGTZsWK7HPnv2LBcuXCAoKCjL9g4dOmS5VOcRHR3N3Xff7Ty/dOmSE6QYGBjI8uXLGTduHOHh4Sxbtoy+ffsyZswYp7wrVqxgwIABNGvWDCBdRNqpqifLYiCuBasLRUQWAIOAfcCfctllPO448ytlPSNT7lxrKajXGi0g9TUtLY2NGzcyZ84ctm3bRmxsLAsXLsz3mAUlp/bp04ewsDDOnj2bZXxp9erVJCYmcvnyZdauXXtV9ejTpw916tRhzZo13Hfffc72ypUrExMTQ0xMDMuXLwcgLCyM2NhYpk6dys8//0zHjh2dhjgyMjJzr+5nIHNlugNfFaqgrrKOBW4F9gMjMr/mXhG8AzC7sMfNzHpGplyy+Ijyq6DU12PHjtG2bVsnb+juu+9my5YtjB8/Ps9jFpScum7dOmrVyn2Mv1KlSgwePJiVK1c6lwYzq169OlWrViU2NjZLBtJ3333HbbfdluUcVatW5YEHHmDGjBnMnTs33+9DtWrVGDp0KEOHDqVChQp8+umn1KlTh7Vr17Jnzx5PQ/cbYISITAMCgaOqmpLfcUVkNVAX+FZVnYwiVU0XkSXAVGCBe99+uHKNblPVy7kdz1vWMzLGlCkFpb527NiR06dPk5SUBMDatWtp1apVgcctTHLq+fPnSUxMBFw9sU8//TRHSmxmU6dOZfLkyVy6dAmANWvW8NVXX2XpAYGrF/S3v/2NRYsW8fPPeUe9bdq0yZk9mJKSwr59+2jUqBHLli1j9OjRHD58mPj4eIBdQBzQAy9ixwFU9Q51Jb1OEJcm4IwZ3QV8737eFldE+WBVverUbmuMjDFlTn6pr35+fsyZM4e+ffsSHByMqvLggw8WeMy8klNzc+HCBQYPHkxISAihoaHUqVOHhx9+OM/9J02aRMeOHQkODqZ58+b8+c9/ZuXKlVSuXDnHvvXq1SMiIoI33ngjz+P9+OOP3HbbbQQHB9O2bVs6dOjAPffcQ2RkJEOGDMm++0fAfcAAMjVGIjJZRBJwTTzYJSLv5HIqAd4Tkd3AbqAe8Lz7tdlANeBDd9LrqjwL7IViS3otSpb0aozJi6W+5i1T0usNwCZV7eDrMuXFxoyMMWWapb4WzD2eU2obIrDGyBhjiszEiRPZtGlTlm2PPvooY8eO9VGJyg5rjIwxpojkN85j8mcTGIwxxvicNUbGGGN8zhojY4wxPmeNkTHGGJ+zxsgYY4zPFVtjJCLzReSEiOzJtG22iHzvDmNaLiI1i+v8xhjvPfDAA7z11ltZtq1YscK5iTS3gLdvvvnGeR4aGuos4gkwb948WrduTZs2bYiIiCA5Odk5T2BgoPO+mJiYAsv2u9/9jq5du2bZduDAAXr37k1YWBgtW7bk97//PeC60T1zcF6/fv0AmDlzJvXr1ycsLIwWLVrwhz/8gYyMDABGjBjh7B8QEJBjkd327duTkpLC9OnTadCgAdWqVcu1nMuWLUNE8NygHx8fT+XKlZ1j57VCw6lTp+jTpw/VqlXjkUcecbafO3cuy/e8Vq1aPPbYYwBMmTLF2d6sWTNq1iwHv0qvJgwpvy+gF9AO2JNpWzhQ0f34ZeBlb45l4XrGFK/o6Gjt3bt3lm0jRozQRYsWqWruAW8XLlzQ1NRUVVU9duyY1q5dW1NTUzUhIUEDAgL04sWLquoKeluwYIGqqo4ZM0Y//PBDr8t1+vRp9ff31xYtWmhsbKyzPTw8XFesWOE837Vrl6rmDM7zePbZZ3X27Nmqqpqenq7du3fXtWvX5tjv8ccf1+eee855HhcXp3fddZeqqn799dd67NixXL8XZ8+e1Z49e2rnzp1127Ztzntbt25dYB3Pnz+vGzdu1H/84x86ceLEPPdr166drl+/Psf21157TceOHZvre3AtdurzgFRvvoqtZ6SqG3AtX5552+eqmuZ+etVhTMaYotGvXz++//57Z/HPixcvsmbNmiy5OtlVqVKFXxgmmwAADEJJREFUihVdtyomJydniUNIS0vj0qVLpKWlcfHiRW699dYrKtdHH33EXXfdxciRI4mKinK2JyYmZlnMNHPGUEFSUlJITk7mppuy5s6pKkuXLs0SH/HZZ58xYMAAALp06UK9evVyPeb//u//Mm3aNCpVypHyXaCqVavSo0ePfN978OBBTpw4Qc+ePXO8FhkZmSPyoizy5U2v44AlxXVwC1czJn+Z12708/Nj6NChLF26lEcffZRVq1bRp08fbrzxRiD3gDeArVu3Mm7cOA4fPsz7779PxYoVqV+/Pk888QQNGzakcuXKhIeHEx4e7pxr+vTpPP/88/Tt25eXXnrJSSjNTWRkJM8++yx169Zl2LBhTpbQlClTuP322+nWrRvh4eGMHTvWuVS1ceNGp6zDhw9n+vTpgOvS4eLFizl8+DADBw7McTlu48aN1K1bN0uoXnR0NPPmzcv3+7hjxw6OHj3KnXfeyZw5c7K8FhcXR9u2balevTqzZs3KtTHxRmRkJCNGjMiR43T48GHi4uK4/fbbr+i4pYlPJjCIyHQgDciZufvrPr8XkW9F5FvPUvDesnA1YwrPk3YK5AiYyy3gDaBz587s3buXbdu28eKLL5KcnMzp06dZuXIlcXFxHDt2jAsXLrB48WIAXnzxRb7//nu2bdvGzz//zMsvv5xnef773/9y6NAhevToQbNmzahYsSJ79riGoMeOHcv+/fsZPnw4X375JV26dOHyZVecTs+ePZ2yehoicDVgMTExnDhxggsXLmTpaUHOHkZKSgoJCQlZMoiyy8jIYMqUKfz1r3/N8Vq9evU4cuQIO3bsYO7cudx3332cPXs2z2PlJ7fAP8/2YcOG5ZtiW1aUeM9IRMYAdwJ93dc0c6WqbwNvg2vV7sKex1bsNqZwunfvTmJiIjt37mTz5s05flnnp2XLllStWpU9e/YQFxdHYGAgtWvXBmDo0KFs3ryZ+++/37nMdcMNNzB27NgcPYnMlixZwunTpwkMDARc8d1RUVFOjtGtt97KuHHjGDduHG3atHEaqoJcd911DBgwgA0bNjhxEWlpafzrX/9i+/btzn4bN26kR48e+R7r3Llz7Nmzx7kKc/z4cQYPHsyqVavo0KGD0+tr3749QUFB/PDDDxw9epTnnnsOgHfeeYcOHfJfv3Tnzp2kpaXRvn37HK9FRUWVmyWISrRnJCIDgCdxhTFdLMlzG2PyJyLce++9jBkzhkGDBhU4/hEXF0dammsI+PDhwxw4cICAgAAaNmzIli1buHjxIqrKF198QcuWLQGcMSlVZcWKFbRp0ybP40dGRhIdHU18fDzx8fFs377daSCjo6NJTU0FXA3AqVOnqF+/vlf1VFU2b95MUFCQs23NmjW0aNEiyzhUdHQ0AwcOzPdYNWrU4OTJk04Zu3Tp4jRESUlJpKenAxAbG8vBgwdp3LgxQ4YMcXpuBTVEnu9Dbr2iAwcOcPr06RwzDcuq4pzaHQl8DTQXkQQRGQ/8HbgR+I87jOnN4jq/MabwIiIi2Llzp1cBc1999RWhoaGEhYUxZMgQ/u///o9atWrRuXNnhg0bRrt27QgODiYjI8OZej1q1CiCg4MJDg7m5MmTPPPMM7keOz4+niNHjtClSxdnW2BgINWrV2fr1q18/vnntGnThtDQUO644w5mz57Nb37zm3zLO2/ePMLCwmjTpg1paWn88Y9/dF7L7TLYl19+mSUWfNq0afj7+3Px4kX8/f2ZOXNmvufbsGGDE743bNgw3nzzTW6++eZc9w0ICODxxx9n4cKF+Pv7s2/fPue17JMqPCIjIxk5cmSOcaSyqlyG6xljzNVISEjgwQcf5LPPPvN1Ua6KJ1zP1+Xwhq3AYIwx2fj7+5f5hqissTwjY4xPLViwgFdffTXLtu7du5ebgXnjHbtMZ4wx5ZRdpjPGGGMKwRojY4wxPmeNkTHGGJ+zxsgYY4zPWWNkjDHG56wxMsYY43NlYmq3iCQBhwv5tlrAyWIoTml0rdT1WqknWF3Lq5KuayNVrV2C57tiZaIxuhIi8m1ZmV9/ta6Vul4r9QSra3l1LdW1sOwynTHGGJ+zxsgYY4zPlefG6G1fF6AEXSt1vVbqCVbX8upaqmuhlNsxI2OMMWVHee4ZGWOMKSPKfGMkIgNE5ICIHBKRp3J5/QYRWeJ+fauIBJR8Ka+eF/XsJSLfiUiaiAzzRRmLihd1fVxE9onILhH5QkQa+aKcRcGLuj4sIrvdychfiUgrX5SzKBRU10z7DRMRFZEyO+vMi8/1ARFJcn+uMSIywRflLFVUtcx+AX7Aj0Bj4HpgJ9Aq2z5/BN50Px4JLPF1uYupngFACLAIGObrMhdzXfsAVdyP/1AWP9NC1LV6pseDgWhfl7u46ure70ZgA7AF6ODrchfj5/oA8Hdfl7U0fZX1nlEn4JCqxqpqChAF/C7bPr8D3nM/Xgb0lbIXGl9gPVU1XlV3ARm+KGAR8qau61T1ovvpFsC/hMtYVLyp69lMT6sCZXWQ15v/qwB/Bl4BkkuycEXM27qaTMp6Y1QfOJrpeYJ7W677qGoacAa4pURKV3S8qWd5Udi6jgfKaj60V3UVkYki8iOuX9KTS6hsRa3AuopIW6CBqn5SkgUrBt7+DN/jvtS8TEQalEzRSq+y3hjl1sPJ/pejN/uUduWhDt7yuq4icj/QAZhdrCUqPl7VVVXfUNUg4EngmWIvVfHIt64iUgGYB/ypxEpUfLz5XD8GAlQ1BFjDr1dvrlllvTFKADL/ReEPHMtrHxGpCNQAfi6R0hUdb+pZXnhVVxHpB0wHBqvq5RIqW1Er7OcaBdxdrCUqPgXV9UagDfCliMQDXYBVZXQSQ4Gfq6qeyvRz+0+gfQmVrdQq643RNqCpiASKyPW4JiisyrbPKmCM+/EwYK26RxDLEG/qWV4UWFf35Zy3cDVEJ3xQxqLiTV2bZnr6W+BgCZavKOVbV1U9o6q1VDVAVQNwjQUOVtVvfVPcq+LN51ov09PBwP4SLF+pVNHXBbgaqpomIo8Aq3HNYJmvqntF5HngW1VdBbwLvC8ih3D1iEb6rsRXxpt6ikhHYDlwE3CXiDynqq19WOwr4uVnOhuoBnzonotyRFUH+6zQV8jLuj7i7gWmAqf59Q+rMsXLupYLXtZ1sogMBtJw/V56wGcFLiVsBQZjjDE+V9Yv0xljjCkHrDEyxhjjc9YYGWOM8TlrjIwxxvicNUbGGGN8zhojU+aISICI7Cnhc35Z0A2YxVUuEektIt0yPV9Y1ldmNyY7a4xMuSUifr4uQxHpDXQraCdjyjJrjExZVVFE3su00GQVABGJF5EZIvIVMFxEHhSRbSKyU0Q+yrTfQhF5TUQ2i0hs5p6GiExzZwjtFJGXMp1zuIh8IyI/iEjP/AonIn4iMtt97l0i8pB7e293L2uZiHwvIh94VpEXkUHubV+5y/aJuPK3HgamuHNvPOftlVvZjSmrrDEyZVVz4G33QpNnceVWeSSrag9VjQL+paodVTUU15Ir4zPtVw/oAdwJvAQgIgNxrf/W2f2eVzLtX1FVOwGPAc8WUL7xwBlV7Qh0BB4UkUD3a23dx2iFK/Omu4hUwrXE0UBV7QHUBlc0CPAmME9Vw1R1Y15lN6Yss8bIlFVHVXWT+/FiXL+YPZZketxGRDaKyG5gFJB5iaQVqpqhqvuAuu5t/YAFnrwkVc28qO6/3P9uxxVmmJ9wYLSIxABbccWWeNaZ+0ZVE1Q1A4hxH6sFEKuqce59Igs4fm5lN6bMKtNr05lrWvZ1rDI/v5Dp8ULgblXdKSIP4Bp/8ci82rdk+jevNbI8+6dT8P8dASap6uosG0V6Zzuv51iFDXzMrezGlFnWMzJlVUMR6ep+HAF8lcd+NwKJInIdrp5RQT4HxmUaW7r5Csu3GviD+7yISDMRqZrP/t8Djd1jRAAjMr12Dlc9jCm3rDEyZdV+YIyI7AJuBv6Rx37/i+sy2X9w/cLPl6pG41ru/1v3JbYnrrB87wD7gO/c073fIp/elKpewjXuFe2efPFfXKnE4ApiG5JtAoMx5Yqt2m1MKSEi1VT1vHt23RvAQVWd5+tyGVMSrGdkTOnxoLs3thdXIvFbPi6PMSXGekbGGGN8znpGxhhjfM4aI2OMMT5njZExxhifs8bIGGOMz1ljZIwxxuesMTLGGONz/x9kcIaBN90qCQAAAABJRU5ErkJggg==\n",
      "text/plain": [
       "<Figure size 432x288 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "%matplotlib inline\n",
    "from Bio import Phylo\n",
    "import matplotlib.pyplot\n",
    "#draw the tree\n",
    "Phylo.draw(upgma_tree)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true
   },
   "source": [
    "The function draw support the display of different colors and branch widths in a tree"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 368,
   "metadata": {},
   "outputs": [],
   "source": [
    "tree = upgma_tree.as_phyloxml()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 369,
   "metadata": {},
   "outputs": [],
   "source": [
    "tree.root.color = (128, 128, 128)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Colors for a clade are treated as cascading down through the entire clade, so when we colorize the root here, it turns the whole tree gray. We can override that by assigning a different color lower down on the tree.\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 405,
   "metadata": {},
   "outputs": [],
   "source": [
    "mrca = tree.common_ancestor({\"name\": \"Inner10\"})\n",
    "mrca.color = \"salmon\"\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Color a clade (branch):"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 406,
   "metadata": {},
   "outputs": [],
   "source": [
    "tree.clade[0, 1].color = \"blue\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 407,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAaMAAAEKCAYAAAC/hjrSAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDIuMi4yLCBodHRwOi8vbWF0cGxvdGxpYi5vcmcvhp/UCwAAIABJREFUeJzs3XlclNX+wPHPAVwQd0FD0EDckG0AxTWFXFC7rpiKlvt2UzPL7VdpWWa37KZJZveWuRYumEvmjqKmV3EDRAhxocJdVFxQZDm/P2bmiZHdhXE579drXjHnOc/znGeud86c85zn+xVSShRFURTFnCzM3QBFURRFUZ2RoiiKYnaqM1IURVHMTnVGiqIoitmpzkhRFEUxO9UZKYqiKGanOiNFURTF7FRnpCiKopid6owURVEUs7MydwOKwtbWVjo5OZm7GYqiKE+Vw4cPX5FS2pm7HUXxVHRGTk5OHDp0yNzNUBRFeaoIIf4wdxuKSk3TKYqiKGanOiNFURTF7FRnpCiKopid6owURVEUs1OdkaIoimJ2qjNSFEVRzE51RoqiKIrZqc5IURRFMTvVGSmK8kD69evH/PnztfcHDhzA09OTzMxMraxr1664u7tr76Ojo2nevDkeHh506dKFGzdumBzzzz//pHz58nzxxRdamaWlJTqdDnd3d7p06cL169fzbVNSUhLW1tbodDoaNWrEqFGjyM7ONin38vKiRYsWJCQkaPtFRkbSunVrGjRoQMOGDRk2bBhxcXE4OjqSnZ1tcg6dTkdkZCS7d+/Gx8cHKysrwsLCcrWlY8eOnD17lq+//pq6desihODKlSu56h08eBBLS0uTY0yaNAk3NzdcXV158803kVLm2i8lJYWAgADKly/PmDFjTLaFhobi4eEB0EgIsVkIYQsghHhVCHFcCJEthGic7wdpDlLKJ/7l6+srFUV5sly4cEE6OzvLS5cuyaysLNm4cWO5Z88ebfvq1atlcHCwdHNz08oaN24sIyIipJRSLliwQL7//vsmx+zZs6fs1auXnDVrllZmY2Oj/T1gwAA5Y8aMfNt05swZ7XwZGRnypZdekqtXrzYpl1LKb7/9Vg4YMEC7jtq1a8t9+/ZJKaXMzs6Wq1atkhcuXJDNmjXT2iullPHx8bJOnTrauaKjo+Xrr78uV61aZdKOtLQ02aRJEymllEeOHJFnzpyRL774orx8+bJJvczMTBkQECA7deqkHWPv3r2yRYsWMjMzU2ZmZspmzZrJnTt35rrWW7duyT179sj58+fL0aNHa+UZGRnSzs5OXr58WQKHgM+BD6W+Q3MFGgARQGP5BHy/G19qZKQoygOpUaMGEyZMYNKkSXz77bd4enrSqlUrAG7dusWXX37J+++/b7JPQkICrVu3BqB9+/asXr1a27Z27Vrq1KmDm5tbvuds3rw5Z8+eLVL7rKysaNGiBSdPnsy17caNG1SpUgWAefPmMXDgQJo3bw6AEIJevXpRo0YNgoODWb58ubbf8uXLCQ4OBvRhyjw9PbGwyP01GhERgb+/PwDe3t7kF1szJCSEoKAgqlevrpUJIbh79y737t0jPT2djIwMatSokWtfGxsbWrVqRdmyZU3KjV/ut2/fNhZVBM4ZtsVLKRN4Aj0VselKSuaib8zdBEV5olkNesPk/ahRo1i8eDEREREm8SOnTp3KO++8Q7ly5Uzqu7u7s379erp168aqVav466+/ALh9+zafffYZ27ZtM5miyykrK4vw8HCGDh1apLampaURHh7ORx99BMCpU6fQ6XTcvHmTtLQ0Dhw4AEBsbCwDBw7M8xi9e/fG29ubkJAQrKysWLFiBatWrSr03Js2baJ79+4F1jl79ixr1qxhx44dHDx4UCtv3rw5AQEB2NvbI6VkzJgxuLq6FumaAUqVKsX8+fON03SeQBowusgHMBM1MjLIXPQN8kLRfnEpiqJnYWHByJEj6dSpE9WqVQMgKiqKkydP0qNHj1z1f/jhB+bNm4evry83b96kdOnSAHzwwQeMHz+e8uXL59rnzp076HQ6qlWrxtWrV2nfvn2BbTJ2Oi1btuSVV16hU6dOALi4uBAVFcWpU6eYM2cOI0aMKPT6XnjhBdzc3AgPDycqKopSpUqZ3APLz969e7VRYn7eeustPvvsMywtLU3KT548SXx8PMnJyZw9e5YdO3awe/fuQs9plJGRwfz58zl69ChAjOH1f0U+gJmokVEO4gWHXL/8FEUpmIWFhclU1f/+9z8OHz6Mk5MTmZmZXLp0CX9/fyIiImjYsCFbt24F4MSJE/z666+AfvFDWFgYkyZN4vr161hYWFC2bFnGjBmDtbU1UVFRpKam8o9//IN58+bx5ptv5tseY6dTkK5duzJ48GAA3NzcOHz4MN26dcuzrnGqzjhtV5jTp09Tq1YtraPNz6FDh+jbty8AV65cYePGjVhZWZGYmEizZs20jrlTp07s37+flJQUpk+fDsD3339P48Z5rz8wXruLi4uxaCUwpdCGm5kaGSmK8kj985//5Ny5cyQlJfHbb79Rv359IiIiALh06RIA2dnZzJgxg1GjRgGwZ88ekpKSSEpK4q233uLdd9/NtUKsUqVKzJ07ly+++IKMjIyHauNvv/2mfVmPGTOGxYsXa9N2AMuWLePChQsABAUFsXHjRlasWKF1HgXZtGkTHTt2LLTemTNntGvu1asX33zzDd27d6d27drs2rWLzMxMMjIy2LVrF66urvTo0YOoqCiioqLy7YgAHBwciIuL4/Lly8ai9kB8oQ0yM9UZKYpSYkJDQ6lfvz4NGzakZs2a2uikqLy9vfHy8jJZVFBUxuk7Ly8v3n33Xb7//ntAvxBj+fLlTJgwgQYNGuDq6sqePXuoWLEiAJUrV6ZZs2bUqFEDZ2dn7XgHDx7E0dGRVatWMXLkSG3hxebNm006o7lz5+Lo6EhycjKenp4MGzaswHb26tULFxcXPDw88PLywsvLiy5duuRZ18nJibfffptFixbh6OhIXFwcNWvW5IMPPjAuFGkE6ICZAEKIHkKIZKA58KsQYkuxP8jHRMg81q8/aRo3biwfd3I94+KFnNN0xqR+tra2j+28CQkJ9OnTR3t/+vRpPvroI956663Hdk5FUR6P9PR0WrZs+cQkAxVCHJZSPlnPE+VD3TMyo6ysLBo0aKDN8WZlZeHg4JDnjV9FUZ58ZcqUeWI6oqeNmqYrgqSkJFxdXRk+fDhubm506NCBO3fuAODv78/kyZPx8/Ojfv367NmzB9B3LBMnTqRJkyZ4enryn//8B9A/fxAQEEC/fv2MSy814eHhuLi48OKLL5bsBSrKU+bYsWPodDqTV9OmTc3dLOUhqJFRESUmJhIaGsp3331H7969Wb16Na+99hoAmZmZREZGsnHjRqZPn8727dtZsGABlSpV4uDBg9rQvUOHDoA+9EhsbKzJ/DOYPlCnKEr+PDw8Cl0xpzxdntnOqLgPsMoLZxEvOOS73dnZGZ1OB4Cvry9JSUnatp49e+Yq37p1KzExMVq8qdTUVBITEyldujR+fn65OqJ79+6xfv16Pv3002K1W1EU5VnwTHZGxgdYC+pciqtMmTLa35aWlto0Xc5tlpaWWpBIKSUhISEEBgaaHCciIgIbG5tcx9+0aRM+Pj55hv1QFEV51j2TnREU/wHWRx0KKDAwkPnz5/Pyyy9TqlQpTpw4gYND/p1jaGiomqJTFOW59cx2RuY2bNgwkpKS8PHxQUqJnZ0da9euzbNuWloa27Zt0xY5KIqiPG+eyeeM8npm6HHsoyiK8iR7mp4zUku7FUUpstTUVAYMGICLiwsuLi7079+fa9eumdS5ceMGDg4OucL5HD16FCEEW7b8/dD/X3/9RUBAAK6urri5ufHVV19p2/r06aMt23ZyctIWEAHExMTQvHlz3Nzc8PDw4O7du/m22cnJCQ8PD+1Yxrh2gwYNypUUzxgPztnZ2ST5HugDm37++edkZGQwcOBAPDw8cHV11RYdJSUl5Qqi+uGHH5pEIf/f//7H8OHDC0yMZ3R/YsKoqCiaNWuGTqejcePGREZG5rlfzmR+5Jj9EkJUEkL8IoSINiTYG5xj20AhRKLhlXcI88esxDsjIUQtIcROIUS84QMZV9JtUBTlwQwdOpQ6depw6tQpTp06Rd26dRk0aJBJnalTp9KmTZtc+4aGhtKqVStCQ0O1MisrK/79738THx/P/v37mTdvHnFxcQCsWLFCi8UWFBSkrVrNzMzktdde49tvv+X48eNERERQqlSpAtu9c+dO7Vhz584t9Dr79u1rEnIoOzubsLAw+vTpw6pVq0hPT+fYsWMcPnyY//znPyarawtiDBVUtmxZPv7443zTZfz888+5IphPmjSJDz74gKioKD766CMmTZqU574tW7Zk+/bteT2vOBqIk1J6Af7Av4UQpYUQVYEPgKaAH/CBEKJKkS7oETLHyCgTeEdK6Qo0A0YLIRqZoR2KohTDyZMnOXz4MFOnTtXKpk2bRnR0tDaKOHz4MBcvXtSeqTOSUhIWFsaiRYvYunWrNpKxt7fHx8cHgAoVKuDq6poreZ6UkpUrV2oLfLZu3YqnpydeXl4AVKtWLVcahod1f1K93bt34+TkxIsvvogQgtu3b5OZmcmdO3coXbq0FseuMOHh4bRr1y7fxHiQf2JCIYSWpj01NZWaNWvmeY4CkvlJoILQD5nKA1fRfx8HAtuklFellNeAbUDhkV4fsRJfwCClPA+cN/x9UwgRDzgAcSXdlvtFRcEEf3O3QlGeHIZg2wDExcWh0+lMvvgtLS3x9vYmPj6eevXq8c4777B06VLCw8NNjrN3716cnZ1xcXHB39+fjRs3aiMdo6SkJI4ePZorksKePXuoUaMG9erVA/SpJ4QQBAYGcvnyZfr27ZvvKMEoICBAa/fAgQMZP348ABMnTmTGjBm56hszuEZHR2uBWY2dYa9evVi3bh329vakpaUxe/ZsqlatqnUU+bly5QqlSpWiUqVKBdbLLzHhnDlzCAwMZMKECWRnZ7Nv374Cj5OHr4H16LO+VgD6SCmzhRAOwF856iWj/04uUWZdTSeEcAK8gQN5bBsBjACoXbv2Y2+L1aA3VEekKAWQUhrvQ+QqB/jmm2/o3LkztWrVylUnNDRUS7/Qt29fli5datIZ3bp1i6CgIObMmZNrlHH/Yw+ZmZn89ttvHDx4kHLlytG2bVt8fX1p27Ztvm3fuXNnngGPZ82aRa9evbT3OafGjKMjNzc31q1bp2WMjYyMxNLSknPnznHt2jVeeukl2rVrl+/ozPiZbd26NdeI8X7GxISzZ8/ONfU3f/58Zs+eTVBQECtXrmTo0KFs3769wOPdJxCIAl4GXIBtQog9QO7/UfWjqBJlts5ICFEeWA28JaXM9ZNCSvlf4L+gX01XEm3K+StQURRTbm5uHD16lOzsbC2ZXnZ2NjExMfj4+LBq1Sr27NnDN998w61bt7h37x7ly5fnk08+YfXq1axfv55PPvkEKSUpKSncvHmTChUqkJGRQVBQEP379881WsrMzOTnn3/m8OHDWpmjoyNt2rTROpfOnTtz5MiRAjujBxEcHEyHDh1o06YNnp6eVK9eHYCffvqJjh07UqpUKapXr65F6e7cuXOuxRxXr17Voq1s2rSJt99+u8BzFpSYcPHixdoCj1dffVVLRREYGMjFixdp3LixlhYjH4OBf0n9r4eTQogzQEP0IyH/HPUcgYgifUiPkFlW0wkhSqHviH6UUv5sjjYoilI8devWxdvb22Raa8aMGbRt25batWvz448/8ueff5KUlMQXX3zBgAED+Ne//sX27dvx8vLir7/+IikpiT/++IOgoCDWrl2LlJKhQ4fi6uqa5xf19u3badiwIY6OjlpZYGAgMTExpKWlkZmZya5du2jU6NHfdnZxcaFatWpMmTLFZGRWu3ZtduzYgZSS27dvs3//fho2bEj58uWxt7fXpiivXr3K5s2badWqFVJKYmJiTFYE5qWgxIQ1a9Zk165dAOzYsUObttyyZQtRUVGFdUQAfwJtAYQQNYAGwGlgC9BBCFHFsHChg6GsRJX4yMhw82wBEC+l/LKkz68oyoP74YcfGDt2LHXr1iU1NZUmTZrwyy+/FLhPaGhorrQoQUFBzJ8/H2dnZ5YuXaotvQaYOXMmnTt3BvIOHlylShXefvttmjRpghCCzp0788orrxTYhpz3jDw9PVmyZEmRrjc4OJj/+7//M2n/6NGjGTx4MO7u7kgpGTx4MJ6engAsWbKE0aNH88477wDwwQcf4OLiwqFDh/D29jaZ5nRycuLGjRvcu3ePtWvXsnXr1gI71e+++45x48aRmZlJ2bJl+e9//5tnvblz5/L5558bM9U2EkJ8L6UcBnwMLBJCHEM/NTdZSnkFQAjxMXDQcIiPpJRXi/QBPUIl/tCrEKIVsAc4BmQbit+VUm7Mb5+SeOhVUZTiSUhIoHPnzoSEhGidh5K3GTNmULdu3SKlLX+UnqaHXs2xmu438r5hpijKU6RBgwacOnXK3M14Kty/TFvJTcWmUxTlmdC0aVPS09NNyoxTgMqTT3VGiqI8Ew4cyPWEiPIUUbHpFEVRFLNTnZGiKIpidqozUhRFUcxOdUaKoiiK2anOSFEURTE71RkpivJE69evH/Pnz9feHzhwAE9PTzIzM7XEeZ6enrRp04Y//vjDZN+srCy8vb35xz/+oZX179+fBg0a4O7uzpAhQ8jIyADg2rVr9OjRA09PT/z8/IiNjS2wXUIIXn/9de19ZmYmdnZ22rkWLVqkJc778MMPEUJw8uRJrf7s2bMRQmB8oN/JyYkrV67g7+9vkoAQ9BG733jj74f4O3bsyNmzZ/O9lt9//53mzZsD+AghJuRoc1khRGSOBHvT87k2nRDif4Y6MUKIPnnUCRFC3MrxfrYQIsrwOiGEuF7gB3gf1RkpivJEmz17NrNmzeLy5ctkZ2czZswYvvnmG6ys9E+m7Ny5k5iYGPz9/XOlg/jqq69wdXU1Kevfvz+///47x44d486dO1pMt5kzZ6LT6YiJiWHJkiWMG1dw3k8bGxtiY2O5c+cOANu2bcPBIf/MCx4eHiY5ksLCwvIM/3N/LiUwDYt0584drl69ioODQ77XUrVqVWMSwYv3HT4deNmQYE8HdBRCNMujuWnAACmlG/rcRnOEEJWNG4UQjYHKOXeQUo6XUuqklDogBChW3FHVGSmK8kSrUaMGEyZMYNKkSXz77bd4enrSqlWrXPWaN29ukpgvOTmZX3/9VYtubdS5c2eEEAgh8PPzIzk5GdDnazJG/m7YsCFJSUlcvHj/d7mpTp068euvvwK5U13cr3v37qxbtw6A06dPU6lSJezs7HLV69WrFxs2bNAe4E1KSuLcuXPaNUdERODv71/gtVSvXp0mTZrAfakgpJ5xNFPK8MoVE05KeUJKmWj4+xxwCbADEEJYArOAgpJIBQOhBWzPRXVGiqI88UaNGkVcXByzZs3i888/z7PO5s2b6d69u/b+rbfe4vPPP9fSXdwvIyODpUuX0rGjPqmpl5cXP/+s/zEfGRnJH3/8oX2558eYnvzu3bvExMTkSgyYU8WKFalVqxaxsbGEhobSp0+umS9An7nWz8+PzZs3A/pRUZ8+fbQgq5s2bdLanN+1FEQIYSmEiELfwWyTUhb4tLAQwg8oDRhjP40B1hsSpeZV/0XAGdhRaGNyUBEYSogxeKuiKIW7P8ixhYUFI0eO5NChQ1SrVs1kW0BAABcvXqR69eraNN2GDRuoXr06vr6+WgqG+73xxhu0bt2al156CYApU6Ywbtw4dDodHh4eeHt7a1OB+fH09CQpKYnQ0NAiBYs1dl5btmwhPDychQsX5lnPOFXXrVs3li9fzg8//KBt27t3L1988UWB11IQKWUWoDNMu60RQrhLKfO8QSaEsAeWAgMNWWFrAq9imv8o12UCYYbzFJkaGZWAzEXfIC+cLbyioij5srCwyHOUs3PnTv744w/c3NyYNm0aoP/CXr9+PU5OTvTt25cdO3bw2muvaftMnz6dy5cv8+WXf2exqVixIgsXLiQqKoolS5Zw+fJlLTFeQbp27cqECRMKnKIz6tKlC0uXLqV27dq5Mtrm1L17d8LDwzly5Ah37tzBx8cH0E/v1apVi9KlSxd4LUUhpbyOPoleRyFE0xyLD7oCCCEqAr8C70sp9xt28wbqok/OlwSUE0KcvO/QfSnmFB2okVGJES84qJQWivKYWFtbM2fOHDw8PHj//ff59NNP+fTTTwH9PZYvvviCZcuWAfD9999rI5Ocndv169cpV64cpUuX5vvvv6d169YFdhhGQ4YMoVKlSnh4eOQ7CsvZzs8++4z69esXWK98+fL4+/szZMgQk07u/im6/K4lP0IIOyBDSnldCGENtAM+M0zV6XLUKw2sAZZIKVcZy6WUvwIv5Kh3S0pZN8f7BkAV4H+FNuY+amSkKMozwd7enuDgYObNm1dgvVGjRnHx4kWaN2+OTqfjo48+AiA+Ph43NzcaNmzIpk2btBTfhXF0dCx05V1Offv21UY6BQkODiY6OtokB9LmzZtNOqP8ruXChQvG7Lg1gPeFEMmGkY49sFMIEYM+md42KeWGPE7fG2gNDMoxYio4Ta2h2cBy+QCJ8ko8ud6DeNqT6z1p7VEU5emTnp5Oy5YtKc53oUqupxTKycmJQ4cOYWtr+9jPU6FCBSwtLbGysirWP2RFUZ4cZcqUeab//6s6o2dUVlYWlpaWgP4G7+Pu9BTlWZSSkqI9e5RTeHh4rlV9ysNRnZGZJSUl0alTJ1q1asW+fftwcHBg3bp1WFtb4+/vT9OmTdm5cyfXr19nwYIFvPTSS2RlZTFlyhQiIiJIT09n9OjRjBw5koiICKZPn469vT1RUVHExcWZ+/IU5alWrVo1oqKizN2M58Jz3RmV1LM/8sJZxAv5hwlJTEwkNDSU7777jt69e7N69WptGWpmZiaRkZFs3LiR6dOns337dhYsWEClSpU4ePCgNo/coUMHQP+wXmxsrLYkVQhBhw4dEEIwcuRIRowY8fgvWFEUpZie287I+OxPQZ1ESXF2dkan0y9U8fX1JSkpSdvWs2fPXOVbt24lJiaGsLAwAFJTU0lMTKR06dL4+fmZPBuxd+9eatasyaVLl2jfvj0NGzakdevWJXNhiqIoRfTcdkZQcs/+FDYCK1OmjPa3paWlFngx5zZLS0syMzMBkFISEhJCYGCgyXEiIiKwsbExKatZsyagj1XVo0cPIiMjVWekKMoTRz1n9BQKDAxk/vz5Wrj4EydOcPv27Vz1bt++zc2bN7W/t27diru7e4m2VVEUpSie65HR02rYsGEkJSXh4+ODlBI7OzvWrl2bq97Fixfp0aMHoL/31K9fvyIFUlQURSlpz+1DryX5IKp66FVRFHN4mh56Nds0nSGM+VEhRF6hKBRFecKlpqYyYMAAXFxccHFxoX///ly7dk3bPmnSJNzc3HB1deXNN9/E+MO3oP2SkpIQQhASEqIdZ8yYMSxatAiAQYMGaQt3jMqXLw/on6fT6XTaq2zZsnnOGBj5+/vToEEDrX6vXr0ASEhIwN/fH51Oh6urKyNGjOD27dtUq1aN1NRUk2N0796dlStXaplVy5QpkyuitrF9ABs3bqRevXr8+eefAJw/f15bCduxY0cqV65skpUW4MyZMzRt2pR69erRp08f7t27B8Du3bvx8fHBysoq12eSkxCitRDiiBAiUwjRK0f5i0KIw4ZQP8eFEKNybNucIxvst4YcRo+VOe8ZjQPizXh+RVEewtChQ6lTpw6nTp3i1KlT1K1bl0GDBgGwb98+9u7dS0xMDLGxsRw8eJBdu3YVuh/oF9t89dVX2pduUQUEBBAVFUVUVBQ7duygXLly2hd9fn788UdtH+MX+ptvvsn48eOJiooiPj6esWPHYmNjQ4cOHUw6t9TUVH777Tf+8Y9/aJlVJ0yYkN+pCA8PZ+zYsWzevJnatWsD+lhzxoVIEydOZOnSpbn2mzx5MuPHjycxMZEqVaqwYMECAGrXrs2iRYvo169fYR/Nn8Ag4Kf7ys8DLQyZWZsCUwwpIgB6G7LBuqNPqvdqYSd5WGbpjIQQjsArwPfmOL+iKA/n5MmTHD58mKlTp2pl06ZNIzo6moSEBIQQ3L17l3v37pGenk5GRgY1atQodD8AOzs72rZty+LFix+4fWFhYXTq1Ily5coVe9/z588bg4wC+nThkDsd+Jo1a+jYsSPlypXTMquWKlUqz2Pu2bOH4cOH8+uvv+Li4qKVb968mU6dOgHQtm1bKlSoYLKflJIdO3Zoo7aBAwdqHaKTkxOenp6FRuuWUiZJKWOA7PvK70kp0w1vy5CjP5BS3jD8aYU+sd5jv59jrgUMc9CnrK1QWMVnhbxwViXYU55qOe95xsXFodPptJBToH/8wNvbm/j4eLp3705AQAD29vZIKRkzZgyurq6sX7++wP2Mz9tNmTKFTp06MWTIkFztmDhxopZELz/Lly/n7bffLvSa+vfvj7W1NQDt27dn1qxZjB8/npdffpkWLVrQoUMHBg8eTOXKlenYsSPDhg0jJSWFatWqsXz5csaOHVvoOdLT0+nWrRsRERE0bNhQK8/KyiIhIYFGjRrlu29KSgqVK1fWkvw5OjqapFZ/WEKIWuhzFtUFJhpSjBu3bQH8gE1A/vOAj0iJj4yEEP8ALkkpDxdSb4QQ4pAQ4tDly5dLqHWPh9WgN56Ih2sV5VGRUmppsO8vB/3IKT4+nuTkZM6ePcuOHTvYvXt3ofsZOTs74+fnx08/3T+zBLNmzdKm1vIK1XP+/HmOHTuW6zm8vOScpps1axYAgwcPJj4+nldffZWIiAiaNWtGeno6pUuXpmvXroSFhXHlyhWioqIKnQYEKFWqFC1atNCm14wOHDhQYJpyyP25AHl+fg9KSvmXlNITfWc0UAhRI8e2QPQpJ8oALz+yk+bDHCOjlkBXIURnoCxQUQixTEr5Ws5KUsr/Av8F/Wq6km/mo6VW0inPEjc3N44ePUp2drY2TZSdnU1MTAw+Pj6sWLGCZs2aaTcWQCoVAAAgAElEQVTvO3XqxP79++nZs2eB+2Vn/z2T9O6779KrV69iP6S9cuVKevToke+UWVHUrFmTIUOGMGTIENzd3YmNjcXX15fg4GBmzJiBlJJu3boV6RwWFhasXLmSdu3aMXPmTN59910gd6K8vNja2nL9+nUyMzOxsrIiOTlZe5A9P++99x6//vpr0S8WkFKeE0IcB14ixyhISnlXCLEe6AZsK9ZBi6nER0ZSyv+TUjpKKZ3Qp6fdcX9HpCjKk61u3bp4e3ubTJfNmDGDtm3bUrt2bWrXrs2uXbvIzMwkIyODXbt24erqWuh+OTVs2JBGjRqxYUPxFtyGhoYWKQV4fjZv3qw9UH7hwgVSUlJwcNDPbAQEBJCYmMi8efOKdY5y5cqxYcMGfvzxR22EFB4enmdE8JyEEAQEBGiLKxYvXky3bt0K3OeTTz7Jd9R437EdDdleEUJUQT9QSBBClBdC2BvKrYDOwO9FuMyHI6U02wvwBzYUVs/X11cWR8bCeTJj4byHrqMoSv6uXbsmX3vtNeni4iJtbW1lp06dZGZmppRSyszMTDlixAjZsGFD6erqKsePH1+k/c6cOSPd3Ny0ulFRUVIIIRcuXCillHLgwIFy1apVJu2wsbHR/j5z5oysWbOmzMrKKrT9bdq0kfXr15deXl7Sy8tLtm3bVkop5fjx42X9+vWlp6en9PT0lEuXLjXZ780335T29vYm5zh//rx0cHCQFSpUkJUqVZIODg4yNTU1V/v+/PNP6eTkJBctWiQDAgJMjtuqVStpa2sry5YtKx0cHOTmzZullFKeOnVKNmnSRLq4uMhevXrJu3fvSimljIyMlA4ODrJcuXKyatWqslGjRrmuETgENAGSgdtACnBcv4n2QAwQbfjvCEN5DfRZYGOA40AIYCUfc3+gHnpV02eK8tASEhLo3LkzISEhdO7c+bHv97RbtmwZycnJTJky5bGe52l66FWFA1IU5aE1aNCAU6dOldh+Tztjihjlb6ozUhTlmdajRw/OnDljUvbZZ58VabWdUnJUZ6QoyjNtzZo15m6CUgQqhYSiKIpidqozUhRFUcxOdUaKoiiK2anOSFEURTE71RkpiqIoZqc6I0VRHrtBgwbxn//8x6Rs7dq12oOulpaWJonxkpKS2LZtG76+vnh4eODr68uOHTu0fUNDQ/Hw8MDT05OOHTty5coVQB/Ru2HDhnh6etKjRw+uX7+eb5siIiK0RHaLFi1izJgxRERE0Lx5c5N6mZmZ1KhRg/PnzwMQEhJCgwYNcHNzY9KkSSZ1fX19uXfvHu+99x61atUySaxnPI+dnZ12nd9//3cWncWLF1OvXj3q1auXb/qMlJQUAgICKF++PGPGjDHZlt9nIoR41ZAkL1sI8cQ+AKs6I0VRHrv7cwGBPs2DMb6btbW1SSRuJycnbG1t+eWXXzh27BiLFy/m9ddfB/Sdw7hx49i5cycxMTF4enry9ddfA/o0ELGxscTExFC/fn0+/fTTYrWzdevWJCcnk5SUpJVt374dd3d37O3t2blzJ+vWrSMmJobjx4+bJNNLSkrCwcGB0qVL06VLFyIjI/M8R58+fbTrHDZsGABXr15l+vTpHDhwgMjISKZPn26SNdeobNmyfPzxx7myyRb0mQCxQE9gd7E+jBKmOiNFUR67du3a8fvvv2uji7S0NLZv30737t3z3cfb21uLUO3m5sbdu3dJT0/XYpndvn0bKSU3btzQ6nXo0EHL/dOsWTOSk5OL1U4LCwteffVVVqxYoZXl7DTnz5/PlClTKFOmDKDPSmuUMwp3s2bNsLe3L/J5t2zZQvv27alatSpVqlShffv2bN68OVc9GxsbWrVqRdmyZU3KC/pMpJTxUsqEIjfGTNRDr2bg72/uFijK4xcR8ffflpaW9OzZk5UrVzJu3DjWr19PQECAltn0zp07WmI9Z2fnXA+qrl69Gm9vb60TmD9/Ph4eHtjY2FCvXj3mzZuX6/w//PADffr0KXa7g4ODGTFiBJMnTyY9PZ2NGzcye/ZsAE6cOMGePXt47733KFu2LF988QVNmjQB9NG+jfUKsnr1anbv3k39+vWZPXs2tWrV4uzZs9SqVUurU9wkeqVKlcrzMxk5cmQxr9581MiohPn7QyGR3RXlmZRzqi7naANMp+nu74iOHz/O5MmTtXtOGRkZzJ8/n6NHj3Lu3Dk8PT1zTcd98sknWFlZ0b9//2K3s0mTJty6dYuEhAQ2bdpEs2bNqFKlCqCfDrt27Rr79+9n1qxZ9O7dGykl9+7dIzk5mTp16hR47C5dupCUlERMTAzt2rVj4MCBwMMn0SvKZ/KkUyMjM9DpTH81KsrzoGXLlpw/f57o6Gj27duX6x5SXpKTk+nRowdLlizBxcUFQMvTY3zfu3dv/vWvf2n7LF68mA0bNhAeHv7AWVH79u3L8uXLiY+PN+k0HR0d6dmzJ0II/Pz8sLCw4MqVK8TExNCqVatCj1utWjXt7+HDhzN58mTtuBE5vhSSk5Px9/dnzZo1TJ8+HYDvv/+exo3zXn9Q2GfyNFAjI0VRSoQQgt69ezNw4EA6d+6c677H/a5fv84rr7zCp59+SsuWLbVyBwcH4uLiuHz5MgDbtm3D1dUV0E+VffbZZ6xfv55y5co9cFuDg4NZtmwZO3bsoGvXrlp59+7dtVV9J06c4N69e9ja2rJ582Y6depU6HGN98wA1q9fr7U7MDCQrVu3cu3aNa5du8bWrVsJDAykR48e2ogxv44ICv5MnhaqM1IUpcQEBwcTHR1N3759C6379ddfc/LkST7++GNtKfSlS5eoWbMmH3zwAa1bt8bT05OoqCgtlfeYMWO4efMm7du3R6fTMWrUqAdqZ6NGjShXrhwvv/wyNjY2WvmQIUM4ffo07u7u9O3bl8WLFyOEICIigjZt2mj1Jk2ahKOjI2lpaTg6OvLhhx8CMHfuXNzc3PDy8mLu3LksWrQIgKpVqzJ16lSaNGlCkyZNmDZtGlWrVs2zbU5OTrz99tssWrQIR0dH4uLiCvxMhBA9hBDJQHPgVyHElgf6UB4zlVyvhJPrGRcv5Jymc3Jy4tChQ9ja2j7Wc1+/fp1hw4YRGxuLEIIffvgh1zMViqIUT3JyMsOHD2fTpk3mbkouKrneU6qkOoXk5K9wd/8OKSXDhw9/rOcCyMrKwtLSknHjxtGxY0fCwsK4d+8eaWlpj/3civKsc3R0fCI7oqeNmqYrQVlZWdy+Hcv5898RGRlJdHQ0GzZsIDMzE9A/NOfq6srw4cNxc3OjQ4cO3LlzBwB/f38mT56Mn58f9evXZ8+ePdoxJ06cSJMmTfD09NRWHEVERBAQEEC/fv3w8PDgxo0b7N69m6FDhwJQunRpKleubIZPQVFK1pYtW0yiO+h0Onr06GHuZin3UZ1RHh5np5CWFk/Fis0oV64cVlZWtGnTxmSEkpiYyOjRozl+/DiVK1dm9erV2rbMzEwiIyOZM2eOtsJmwYIFVKpUiYMHD3Lw4EG+++47LatlZGQkn3zyCXFxcZw+fRo7OzsGDx6Mt7c3w4YN4/bt2yXyeSqKOQUGBppEd8hr+bhifqozykfOTiEqqjI+Pqu1Z4R++imTcuUiKVNmDl27TsffH1xdFxAWVgkbm4NUrnyQCRO+o2nTM7z1FuzeHcmpU59QvXocf/7pTmrqblJSUkhLS2Pjxo3ayAj0D/wZH/7z9fU1CUvSs2fPXOVbt25lyZIl6HQ6mjZtSkpKComJiQD4+fnh7OwM6DuyI0eO8M9//pOjR49iY2Pz1C39VBTl2fXM3jOSF85qixTy2y5ecMh3u7FT8PeHmzd9sbFJ0rbZ2uo7hQoVfLl7V19+7dpWbt+O4fLlMACyslK5cycRC4vSVKjgh7W1vlOwtHSlZs3JtG/fnvLly+Pl5cXx48e1YxufMNfXtdRGZDm3WVpaah2YlJKQkBACAwNN2h8REWGyCsjR0RFHR0eaNm0KQK9evVRnpCjKE0ONjPKRs1OoWdOSgQMziYjQP7D63XdliIiA9estqVFDX966tWTNmhBu3Yri1q0o7tw5Q0xMB+bMgVatbIiIQNvf3n4oR44cYffu3VStWpVSpUo9cDsDAwOZP38+GRkZgP7Zh7ym31544QVq1apFQoI+RFV4eDiNGjV64PMqiqI8Ss/syEi84FCkpd2PirFTePnllylVqhQnTpzAwSHvkde9e5eA6vz555/8/PPPJiOY4ho2bBhJSUn4+PggpcTOzo61a9fmWTckJIT+/ftz79496tSpw8KFCx/4vIqiKI/SM9sZlbTidArHjwfRqFEKpUqVYt68ebRt2xYAW1tbYmNjtXo5w9PnDBVia2ur3TOysLBg5syZzJw50+Qc/v7++N8XkVWn01Gc57UURVFKjDH0+JP88vX1lcWRsXCezFg476HrSCllmzb616PyqI+nKM+a69evy9dff13WqVNH1qlTR/br109evXpV2z5p0iTp5uYm3dzc5PLly7XyNm3ayPr160svLy/p5eUlg4KCpJRSfvDBBxKQiYmJWt0vv/xSAvLgwYNSSilffPFFefnyZSmllIcOHZJOTk7yyJEjMj4+XjZr1kyWLl1azpo1q0jt//nnnyUg4+PjtbIzZ85IQL7//vta2eXLl6WVlZUcPXq01s6aNWtq7ffy8pLXrl2TO3fulBUrVpQ6nU42bNhQfvjhh/LWrVuyatWq8vr16ybn7tatm1yxYoWUUsp79+5J4DZQC9gJxAPHgXEyj+9ZoCwQCUQb6k3PsW2BoTwGCAPK53WMh3mZ5Z6REKKyECJMCPG7ECJeCKHCACiKAsDQoUOpU6cOp06d4tSpU9StW5dBgwYB8Ouvv3LkyBGioqI4cOAAs2bN4saNG9q+P/74o7Z8OywsTCv38PAwCcwaFhaW5z3TmJgYevXqxYoVK/D29qZq1arMnTvXZJaiMKGhobRq1SpXINg6deqwYcMG7f2qVatwc3MzqTN+/HiTJejGZwFfeukljh49yqFDh1i2bBm///47HTp0MJl9SU1N5bffftOy1/72228At4BM4B0ppSvQDBgthMjrhnE68LKU0gvQAR2FEM2MTZNSekkpPYE/gTF57P9QzLWA4Stgs5SyIeCFvsdWFOU5d/LkSQ4fPszUqVO1smnTphEdHU1CQgJxcXG0adMGKysrbGxs8PLyyjMJ3f26d+/OunXrADh9+jSVKlXCzs7OpE58fDzdu3dn6dKl+Pn5AfrkeU2aNCnyIqNbt26xd+9eFixYkKszsra2xtXVVZsqX7FiBb179y7ScY1sbGzw9fXl1KlTubLnrlmzho4dO2oBYg2fyw0p5Xkp5REAKeVN9N+3uW5oGwZXtwxvSxle0rDtBoDQh0G3NpY/SiV+z0gIURFoDQwCkFLeA+6VdDvMKSpKJdhTFKOccRrj4uLQ6XRYWlpqZZaWlnh7exMfH4+XlxfTp0/n7bffJi0tjZ07d5qMcPr374+1tTWgT0E+a9YsACpWrEitWrWIjY1l3bp19OnTJ9cCnm7durFs2bIipYLIz9q1a+nYsSP169enatWqHDlyBB8fH227MTXFCy+8gKWlJTVr1uTcuXPa9tmzZ7Ns2TIAqlSpws6dO02On5KSwv79+5k6dSr16tVj2LBhpKSkUK1aNZYvX87YsWO1uoZ9b+bcXwjhBHgDB/JqvxDCEjgM1AXmSSkP5Ni2EOgMxAHvFPvDKUSROyMhRHX0c4oASCn/fMBz1gEuAwuFEF7oL3yclNJkPbIQYgQwAqB27doPeKonT0SE6ogUJT9SyjxzEBnuW9ChQwcOHjxIixYtsLOzo3nz5lqacdBP0+WXasHYEWzZsoXw8PBcnVG7du34/vvvCQwMNOkMiyM0NJS33npLO19oaKhJZ9SxY0emTp1KjRo18sxCO378+DynBPfs2YO3tzcWFhZMmTJFm97r2rUrYWFhBAUFERUVRYcOHQA4d+6cMep3tvEYQojywGrgLeNI535SyixAJ4SoDKwRQrhLKWMN2wYbOqsQoA/wSJfjFtoZCSG6Av8GagKXgBfRD/PcCtqvkHP6AGOllAeEEF8BU4CpOStJKf8L/Bf0Ubsf8FxPJJVYT1Hy5ubmxtGjR8nOzsbCQn8XITs7m5iYGO1L/b333uO9994DoF+/ftSrV69Ix+7SpQsTJ06kcePGVKxYMdf2r7/+mlGjRvHGG29o4byKIyUlhR07dmhR8bOyshBC8Pnnn2t1Spcuja+vL//+9785fvw4v/zyS5GO/dJLL5ncbzIKDg5mxowZSCnp1q2bNp24adMmAgMD2bJFny1CCFEKfUf0o5TyZ0NZLcDYgG+llN8ajyulvC6EiAA6ArE5yrOEECuAiTzizqgo94w+Rn/T64SU0hloC+x9iHMmA8k5hn9h6DsnRVGec3Xr1sXb25sZM2ZoZTNmzKBt27bUrl2brKwsUlJSAP1ig5iYGG00UBhra2s+++wzrSO7n4WFBaGhoSQkJDBt2rRitz0sLIwBAwbwxx9/kJSUxF9//YWzs7NxIYHmnXfe4bPPPjPJ+vqgAgICSExMZN68eSYZaXMm+zPc51kAxEspvzTWkVL+JaXUGV7fCiHsDCMihBDWQDvgd6FXN8exugC/P3Tj71OUaboMKWWKEMJCCGEhpdwphPjsQU8opbwghPhLCNFASpmAvnOLe9DjKYrybPnhhx8YO3YsdevWJTU1lSZNmmgjiIyMDF566SVAfx9o2bJlJtN0Oe8Z2drasn37dpNjF5bUr0yZMqxbt442bdpQo0YNgoKCaNy4MTdu3MDCwoI5c+YQFxeX58gqNDSUKVOmmJQFBQXx008/aenFQT/6u38VnVHOe0ZAvs8qGllYWBAUFMSqVato3bo1oA/anJiYSMOGDY3VWgKvA8eEEFGGsnellBvvO5w9sNgwFWcBrJRSbhBCWBjKKwIC/RLvfxbYsAdQaHI9IcR2oDvwKWCLfqquiZSyxQOfVAgd8D1QGjgNDJZSXsuvvjmT6+WVDE9RlJKRkJBA586dCQkJoXPnzuZuzlPht99+Y9myZXz77bfPXHK9bsBdYDzQH6gEfPQwJ5VSRgFPxQekKIr5NGjQgFOnTpm7GU+VVq1aPdSKQHMpSmf0opTSOI22GEAI4Q9EPKY2KYqiPLFSUlK0EF45hYeHP5L7QM+ronRGK4UQS4HP0S/t/hz9qEZFTVAU5blTrVo1oqKiCq+oFEtRVtM1RR/baB9wEDiH/oaYoiiKojwSRemMMoA76ENAlAXOSCmzC97lyePk5MSVK1ce+3mGDBlC9erVcXd3Nym/evUq7du3p169erRv355r1/Jdr6EoivLcKUpndBB9Z9QEaAUECyHCCt7l+ZOVlQXAoEGD8oyV9a9//Yu2bduSmJhI27ZtVZZVRVGUHIrSGQ2VUk6TUmZIKS9IKbsB6x53wx6XpKQkXF1dGbnwR7ze+5gOHTpoqb39/f2ZPHkyfn5+1K9fnz179gAgZRYTJ06kSZMmeHp6ak9nR0REEBAQQL9+/fDw8ACgdevWxjAcJtatW8fAgQMBGDhwYKHPDyiKojxPCl3AIKU8BLli0+16nI163BITE1kaPJH/DO5Pv40RrF69mtdeew2AzMxMIiMj2bhxI9OnTwe2c/78AipVqsTBgwdJT0+nZcuW2lPfkZGRxMbG4uzsXOA5L168iL29PQD29vZcunTpsV6joijK06Qosem6AF/y6GLTmZ2zszO62rUA8PX11bKmAvTs2dOk3NERrl3bypIlMVp+lNTUVBITEyldujR+fn6FdkSKoihKwYoyTTeDRxubzuzKlCmj/W1paUlmZmaubablkpCQEC3h1ZkzZ7SRkY2NTZHOWaNGDc6fPw/A+fPnqV69+iO4EkVRimLQoEG5gp+uXbtWi+pQvnx5k20fffQROp1OS2dh/HvevHm8//77ODg4aGXGWHcDBw7Ey8sLDw8Pevfuze3bJokITOR3jPT0dMaOHYuLiwv16tWje/fuWoqJsWPH8vXXX2vHaNu2LaNGjdLejxs3jrlz53Ly5Emsra3R6XQAbkKIecb4ckIIKYTQQvkIIb4VQryW4/1LQoj5QojmQohoIUSU4b9d87oOIURPIUSMod5BIUQLQ7mzEOKIoTxWCDE83w/DoEir6aSUKYAWmw59FsDnRpUqgcyfP5+MjAwATpw4UeA/tLx07dqVxYsXA7B48WK6dev2yNupKEre7k9EB7B8+XKT4KI5TZs2jaioKA4dOkSFChW0H6KjR48GYOLEiVrZJ598AkBISAjR0dEcO3aMF154gfnz5xfYpryOMXnyZNLT0zlx4gSJiYm88sorBAUFAdCiRQv27dsH6BdMXb9+nZiYGO14+/bto2VL/VM3DRo0MD4LFYf++7qLodpFYLwQIr9ZsY7AZvTx53yllDqgE/CdIUbd/bYCXoZ6IzFkWkD/CFBzQ3kLYKrhVk++itIZXTfkwdgN/GhI+ZBRhP2eGfb2w2jUqBE+Pj64u7szcuRIk9FUTsHBwTRv3pyEhAQcHR1ZsGABAFOmTGHbtm3Uq1ePbdu25QqoqCjK49OuXTt+//13bXYiLS2N7du3071790d2DmPw1OzsbO7evZtnXqaC3Lx5k2XLlvHll19q+ZSGD9cPKHbt2kXLli21zigmJgadToe1tTU3btzgzp07JCYm4uXldf9hJfA/9MnyAC4Ae9AHTs3Ly0C4lDJNSmn8krM2/DfXBUkpb8m/A5za8Hdm2HQpZbqhvAz6vqbAD6QoERiigTRMY9OVL3CPJ5DxvpCtrS2xsbFaoNSciawickRDtbW1JSkpCX9/EMKCmTNnMnPmTJNj+vv7439fprzQ0NA8z1+tWjXCw8Mf+jqUJ5dKmvhkyRnc2NLSkp49e7Jy5UrGjRvH+vXrCQgIoEKFCg907FmzZrFo0SIAvvjiC9q1awfAgAED2Lx5M56ennz11VfFOkbVqlVxdnbONWXYuHFjjh8/Tps2bcjKyuLcuXPs27eP5s2bU716dfbv30+ZMmXw8fExiWBuYIG+g5mco+xTYL0QYnHOikKIGsBtY+pxw5Tbd+jXCfQzJN7LRQjRC/gEfSDtzjnKnYD16DvCt6WUFwv6PIoyMgqQUmZLKTOllIullHPRP3OkKIqBv78+nbzy5Mo5VVfQFF1R5JxiM3ZEAEuWLOH8+fO4uLiwatWqYh2joCy3xnLjVJ2xM2revLn2vkWLvxMpJCQkGO8ZNQDWSCm35TjeSSAKfbbWnAKBLTnq7ZNSuqGPwvOeEKJ0XtchpQyTUjYAeqHPf2csT5JSegL1gKFCCNuCPo98R0aGm1xvAC5CiJgcmyrwlC9gUJTHQadTqUaeZC1btuT8+fNER0ezb9++XPeQHhVLS0t69+5NSEgIAwYMKPJ+9evX5/Tp09y6dctkdHTkyBFeffVV4O/OKC4uDldXV6pXr868efMoXbo0b7zxdzoc4z0jIUS8lPLjXCfTj2R+AvbnKOsEzLy/opTyuBDiHtBICNEaGGLY1EFKeSlHvZ1CiCVCiMpSyus5ys8KIX5HHzQh3wcsCxoZ/YT+ptc6w3+NL18p5WsF7KcoivLEEULQu3dvBg4cSOfOnSlbtmzhOxVRdnY2p0+fBvQjmV9++SVncrsiqVChAv369WPixIlkZ+sjrv3www9kZWXRpk0bQN+hrlu3jurVq2NhYYGdnR2XLl3iwIEDNG9e9NjVUsrjwCn0HRCGxQmuUspjhvfOhiR7CCGc0U+1/SGlnJsjO+wlwwo9YajX2HDs60IIRyFEWUN5NfSBtU8U1KZ8OyMpZaphmBUspfwjx+tqka9YURTlCRIcHEx0dHSujK9paWk4Ojpqry+//DKfI+QtKyuL1157DQ8PDzw9PUlJSck3vXlBPv/8cywsLKhXrx5169Zl7dq1/Pzzz9p2nU7HxYsXTToeNzc37OzsqFy5cnFPNwN9EGwAPyBnBtM2QIwhM2wYMDKfBKi9gVhDvbn8PfXnDhwUQkQDO4FPc6QiylOhmV6fBCrTq/Kky+vfiZOTE4cOHcLWtsCp8ofy119/MWDAAC5cuICFhQUjRoxg3Lhxj+18ytOlqJlehRAfArFSSrPFHS3KajpFUZ5AWVlZWFlZ8e9//xsfHx9u3ryJr68v7du3p1GjRuZunvIUkVJ+aO42FGU1nXKfkkpHMXv2bNzc3HB3dyc4OJi7d+8+9nMqj54xOO/w4cNxc3MrUnDerKyiBee1t7fHx8cH0N9zcHV15ezZs+a5UCWXnJEcjC8VsT9vamT0BMrKyuLChQvMnTuXuLg4rK2t6d27N8uXL2fQoEHmbt5zozjPDUVF6VfT5ScxMZHQ0FC+++47evfuXWBw3u3bt7NgQfGD8yYlJXH06FGaNm1a3EtVHpNp06Yxbdo0czfjqaBGRg/hcf7iBf2X1J07d8jMzCQtLY2aNWua50KfQ4/6uSFnZ2fjcx+FBucF2Lp1K0uWLEGn09G0aVNSUlJITEwEyDM4761btwgKCmLOnDlaJABFeZo81yOjqCiY4F94HXP94p0wYQK1a9fG2tqaDh06aPWUklGc54YKG0XdH5zX+KMl57acwXml1AfnDQwMNDlOREREruC8GRkZBAUF0b9/f61jU5SnzXM7MrIa9AYTogpeSVcUj+sX77Vr11i3bh1nzpzh3Llz3L59m2XLlj10e5WnQ2Bg0YLzSikZOnQorq6uvP322yXdTEV5ZJ7rkVFRfvWa6xfv9u3bcXZ2xs7ODtB3bPv27dNGXcqzbdiwYSQlJeHj44OUEjs7uzyzA+/du5elS5fi4eGh/SiaOXOmlhpBUZ4Wz3VnZA7GX7wvv/wypUqV4sSJEzg4OOSqV7t2bfbv309aWhrW1taEh4fTuHGhjwsoT5D7g/MaFRacF8DComjBeVu1asXT8KygouQHhU0AACAASURBVBTGLNN0QojxQojjhqRLocawEc+DYcOKlo6iadOm9OrVCx8fHzw8PMjOzmbEiBFmaLGiPPlSU1MZMGAALi4uuLi40L9/f65d+ztgwOTJk3F3d8fd3Z0VK1Zo5eHh4fj4+KDT6WjVqhUnT54E4Msvv6RRo0Z4enrStm1b/vjjDwB27txpsky7bNmy2oj166+/pm7dugghCn30Y9GiRQghTCL5r1mzBiGEllE6IyODKVOmUK9ePdzd3fHz82PTpk2FJgocMmQI1atXx93dPdd5DUnzvjP87SmE+J/hu/hYXt/DQgg/Q4I8Y5K9Hjm2VRZChAkhfhdCxAshih6PKC9SyhJ9AQ7AGcDa8H4lMKigfXx9fWVxZCycJzMWzivWPvlp00b/Up4vxf3fXf07Ma+goCD5wQcfaO+nTZsmu3btKqWUcsOGDbJdu3YyIyND3rp1S/r6+srU1FQppZT16tWTcXFxUkop582bJwcOHCillHLHjh3y9u3bUkopv/nmG9m7d+9c50xJSZFVqlTR6h05ckSeOXNGvvjii/Ly5csFtnfhwoXSw8NDDh06VCvr3bu39PLykqtWrZJSSjl58mQ5YMAAeffuXSmllBcuXJArVqyQmzdvlv7+/ibH69Onj1yyZImUUspdu3bJw4cPSzc3Nwkckqbfv9OBIPSzYjHoE+MBVAMsZe7v63KAleFve+BSjveLgWGGv0sDle/fvzgvcy1gsAKsDdkGy6HPCqgoilJsJ0+e5PDhw0ydOlUrmzZtGtHR0SQkJBAXF0ebNm2wsvr/9u48Pqrq/v/460NQgSAgshQJkBB2yMYaVoNAJFSoIFvEH8hitUVQrKB+8YtoqRsUqtRv1Sog4iMBoSxSDRZZBVklhF0wCRAJJSCyhZDt8/tjZq6TfQJJJhPO8/HIg5k7d+49JxNycu4597wr4+3tTVBQEDExMYBt8dTLly8Dtt6V4/aJ3r17U61aNQBCQ0NJSkrKc97ly5cTERFh7RcSEoKvr6/L5e7Zsye7du0iIyODq1evcuLECWvcLzU1lX/+85/Mnz/fGnuuX78+w4cPLzIosFevXtSuXbug0/YB1gPhQJyq7gdQ1QuaT16R5gzZq4I9PE9EagC9gI/t+6Wr00rdN6PMx4zUtpz4HOAUcB34WlW/LutyGEZJi401AXtlxXny0eHDhwkODrbSUcE2aSgkJIQjR44QFBTEq6++ynPPPUdqaiobN260lkv66KOPGDBgAFWrVqVGjRrs2LGD3D7++GMiIiLybI+Ojr6lGYwiQt++fVm3bh2XLl1i0KBBJCQkALYGtnHjxvneM3azQYH2PKEMVb0kIi0AFZF1QF0gWlXfLuB9XYAF2EL2/p+qZopIUyAFWCgiQcBe4BlVzTvl00Vl3jMSkXuA3wF+wH2At4jkmSImIr8XkT0isiclJaWsi2kYxbJpU+H3oxmlRwsJpQMIDw9nwIABdOvWjcjISLp27Wolos6bN48vv/ySpKQkxo4dm6dxWbJkCXv27GHq1Kk5ticnJ3PgwIE8s2KLa+TIkURHRxc77O8mgwLDAccf/pWx5QuNsv87WET65PcmVd2ptpC9TsBL9rGlykB74B+qGgJcA150uQL5cMdsur5AgqqmAIjIv4BuQI6baFT1Q+BDsK3aXdaFNIziMiu7u0fbtm3Zt28f2dnZVKpk+/s6OzubuLg4a92+6dOnW5EOjz76KM2bNyclJYX9+/dbyyeNGDGC/v37W8ddv349f/nLX9i8eXOOWzgAli1bxuDBg7njjjtuqeydO3fm4MGDVK1alRYtWljbmzVrxqlTp7hy5Uq+PZ6bDAqMABzZGEnAZlU9DyAiXwLt7ZffXrHvM0FVrbgEVT0iItewxUMkAUmqutP+8nJusTFyx5jRKSBURKrZQ5n6AEfcUA7DMCqAZs2aERISwqxZs6xts2bNok+fPjRu3JisrCwuXLgAQFxcHHFxcYSHh3PPPfdw6dIlfvjBlvn2n//8h9atWwOwb98+nnzySdasWUO9evXynDMqKuqWYsudvfHGG3mm8FerVo3x48czefJk0tPTAVtvzHHje3GDAu2/awOxxY2DLV480P57uDK2/KLDqrpSfw3P22MP2atsP0YTbDHmiap6FjgtIi3tx+sDFJpXVBR3jBntFJHlwPdAJrAPew/IMAzjZixYsIBJkybRrFkzLl26RKdOnfjiiy8A2xTpnj17AlCjRg2WLFliXab75z//ySOPPEKlSpW45557WLBgAQBTp07l6tWrVtx348aNWbNmDWC7f+z06dNW+qrDu+++y9tvv83Zs2cJDAxkwIABfPTRR0WWPb/xKLA1qC+//DJt2rShSpUqeHt789prr1mvR0ZGMnv27DyrgEdGRrJp0ybH9PJAERkP7Af2qf3apapeFJG5wG5skxK+VNV/51OMHsCLIpIBZAN/dPSmgEnAZyJyJxAPjC2ysoW4bcP1XGXC9W5P7grLS0tLo1evXty4cYPMzEyGDh3Kq6++Wmrnq4iOHTvGgAEDmD9//m2/EoUjXE9EXgZOqKpL1/PcwazAYBjlRFZWFnfddRcbNmygevXqZGRk0KNHDyIiIggNDXV38TxGy5Yt+fHHH91djHJFVWcVvZd73bYLpRrGzSrN6BARoXr16oDt8lJGRka+M8UMz7Bw4cI84XoTJ050d7HKJdMzMoybUJrRIVlZWXTo0IETJ04wceJEE5bnwcaOHcvYsbc0lHLbMI2RUSHd6s2nReVY3Ux0SFxcnLXu2KVLlzh+/Dh33nlnnrA8Ly8vYmNj+eWXXxg8eDAHDx7Md50xw6hIzGU6o8Ip6ZTW/OSODnFe7Law6JDY2FhiY2NJSEiweka5w/IcatWqRVhYmLV0jWFUZKZnZFRIxUlpzU9JL+vjanRISkoKd9xxB7Vq1eL69eusX7+eF154oWQLYxjlkGmMDKMMuBqWl5yczJgxY8jKyiI7O5vhw4fz0EMPuaHEhlG2zH1GRTD3GXmekvjMzOduVASO+4zcXQ5XmDEjwzDKtaLC5BxT4R1ee+01axq1l5eX9fi9997j5ZdfpmHDhtY2x3p1Y8aMISgoiICAAIYPH861a/kvPr1+/XprNQeHjIwM6tWrx7lz5wBIT0+ndu3aOSItwJbK27JlS4KCgujcuTNxcXHWazdu3GDChAm0bNmSVq1aWb3myZMnW2Vt3rx5nhuu+/Xrx9mzZ3nnnXfw9/dHRPjll4KTHETkE3tI3gERWSYi3vbt0+wBeftF5D8i0qjAg5SWWwlDKqsvE65nFEdJfGbmcy8/igqT8/b2zvd9GRkZWrNmzRzbpk+frvPmzcuzryNsT1V10qRJOnv27HyPmZmZqQ0aNNDTp09b27744gsNDw+3nq9evVq7d++uzZs3z/He7t276759+1RV9cMPP9T+/ftbr/3P//yPFQ6YlZWl58+fz3PuuXPn6hNPPGE9v3r1qnbp0kVVbcF+iYmJ2rBhQ7148aK1D3nD9Wo4PX4XeN7++AF+DTydBHympfx7PfeX6RkZhlGuFRUmVxIcuUHZ2dmkpaUVeKOxl5cXQ4cOzRFdnjvCISoqiueee4769euze/fufI/TtWtXfvrpJ+v5okWLrIkqlSpV4t57783zntyLs27YsIEHHngAsAX7NWnSpMh6quplABGphFNYnqpuUNXr9t12AD5FHqyEmQkMhuGhKnKQn/NY3c2GyRVk9uzZLFq0CIA5c+bQt29fAEaPHk1MTAyBgYG88847Bb4/MjKSyZMn86c//Ym0tDTWrVvHe++9B8C1a9fYvHkzCxcu5OzZs0RFRdGpU6c8x4iJibEa0/Pnz3PnnXfy0ksvsWXLFpo3b87f//536tata+0fHx/PTz/9lGNx1q+++oqRI0cWu/4ishjojy12/Jl8dhkPfFXsA98i0zMyDA9UFvdSlSc3GSaXr6lTp1r3ezkaIoDFixeTnJyMv78/n3/+eYHv79q1KxcuXODHH39k7dq19OzZk5o1awKwZs0a+vXrR5UqVRg2bBgrVqwgOzvbeu+IESPw8fFh3rx51rJAmZmZJCYm0rt3b77//ns6dOjAtGnTcpwzKiqK4cOHW3lNADt27KBbt27Frr+qjgYaAD8Cw5xfE5ExQAC/5h6VGdMzMgwPdav3UnmSmwyTKzYvLy+GDx/O/PnzGT16dIH7jRgxgujoaPbt25fnEt3OnTvx9fUF4Ny5c2zZsoUwezd26dKltGnThmnTpjFp0iSWLVtGvXr1qFatGoMGDQJg2LBheS5BRkdH8/HHH1vPjx07hp+fnxWFUYgWIhIL7FDVpxwbVTVLRJZhGx9aDCAi/YFpwP2qml7UgUua6RkZhlHuFTdMrjiys7OJj48HbBO6vvjiC1q1alXoeyIjI1m8eDFbtmyx7gO7ePEiO3fuJCkpicTERBITE3n33XeJiorK8d4777yT119/nS1btvDDDz9QqVIlIiIirEV1v/nmG9q0aWPtf+jQIa5fv07nzp2tbTExMQXmIOXyg9qC8p4SkUoi0hSssL2BwFH7847Ae8Ag/TWvqEyZxsgwDI8QGRnJ/v3784yTpKam4uPjY33NnVu8K0xZWVk89thjBAQEEBgYyIULF6wp3wUJDAzkjjvuIDw8nKpVqwKwYsUK+vXrlyOK/OGHH2blypVkZGTkeH+1atWYMmUKf/3rXwHbONb06dMJDAwkOjqat99+29o3KioqT51jYmJyRKTPnTsXHx8fzp49S9u2bXnyySfzK7YXsEREDmAbL7oX+Iv9tTmAN7BCRGJFZGWh34BSYG56LYK5+dHz3A43vbor/A9g3LhxrF27lnr16nHw4MFSPZeRlyOAcdeuXUXua256NQyjwsnKygJsN6GaxVvdp0qVKi41RJ7GNEaGUcGUZvgfQK9evahdu7Z7KleGnFdycHy9+eab7i5WhWVm0xlGBVSa4X+3ixkzZjBjxgx3F+O2YRojwyhAbGz5vbHUneF/hlEazGU647bg6+vL+fOuz1jdtKnwX/aFUc1i794QDhxwX/RDWYT/GUZJMj0jwyhAcWbSZWVl4eXlBcDcue/QsGFrLl++zNq1pVM2d4X/GUZpMT0j47ZS2oP7SUlJ/Pvf/2bChAnuqeBNmjBhAm3atKF9+/a0a9eOJ598MkdvyllkZCRdu3bl2LFj+Pj45FgZwDBulukZuaA8jx0YeRU1nlKag/vPPvssb7/9NleuXCn1eubmGP+pU6dOjvt/nn/+eevxJqfuXp06daz3VKpUiddff53XX389xzHDwsKspWwccq8oYBglwfSMinArYwdG+XQzg/uLFy8mODiYLl26cOHCBY4fPw6QY3DfcSNohw4dyq4yt5FLly4xevRo/P398ff3Z9SoUVy8eNF6fdq0abRt25bWrVszefJkVLXQYL60tDQ6d+5MUFAQbdu25ZVXXrH2CQsLw3GjfWJiIs2bN2fdunV89tlnOaZ6V6pUidhCVqxdsGCBtbJDu3btWL16Nfv377d+/sDWuFerVs1apeHAgQMEBgZa5XAE8nXv3p1jx44xc+ZMXnrppRzniY2NpXXr1gBMnz6dRo0a5QkdBBCRBiLytf1xjIj8IiIFXkwWkSYiste+KsMhEXnKvr2aiPxbRI7at9/6nPfSCkoCFgDngINO22oD/wGO2/+9x5VjuTNcz/A8+QXjNWnSRFNSUjQhIUHbtm1rbZ89e7YVanb//ffr7t27VVU1JSVFmzRpoqqqQ4YM0ZiYmDzn2bhxo/72t7+1nr/44ovasGFDbdKkidavX1+rVq2qo0aNKsmqWW7H8L9HHnnE+qxUVWfMmKGDBg1SVdVt27Zpt27dNDMzUzMzMzU0NFQ3btxYaDBfdna2XrlyRVVV09PTtXPnzvrdd9+p6q8/C6dPn9YWLVro6tWr85QnLi5O/fz8Cizv6dOntWnTpvrLL7+oquqVK1c0Pj5es7KytFatWnr58mVVVX366ac1JCREd+7cqaqq77//vj711FM5yqGq+sEHH+jAgQP16NGjec77wgsv6Guvvaaqqt99952eOXNGvb298wvXGwv8yf64D7b16dZqwb/H7wTusj+uDiQC9wHVgN5O+2wFIgo6jitfpdkzWoQtM8PZi8A3qtoc+Mb+3DDKNcfgvuMv1x9++CHfWOo33njDWiQzOjqaBx54gCVLlpR1cSukEydOsHfv3hxR3jNmzGD//v0cO3YMESEtLY309HRu3LhBRkYG9evXLzSYT0Ss3kNGRgYZGRk5QvXOnj1LeHg4s2bNslbUdpY77C63c+fOcffdd1vnqF69On5+flSqVIlOnTqxc+dOAPbu3cvEiRPZvn07ANu3b883GqJXr16cOHGCli1bUqtWLev9AMuWLbPWrwsNDaVBgwYFFas/9qwiVf0GKPR6sqqmq+oN+9O7sF9NU9VUVd3o2Af4nlsM5Cu1MSNV3SIivrk2/w4Isz/+BNgEvFBaZTCMkjBhwgQSExNp3749qkrdunVZtWqVu4tV4ccynWczHj58mODgYGvGItimpoeEhHDkyBEefvhhevfuTYMGDVBVnn76aeuyVWHBfFlZWXTo0IETJ04wceJEunTpYh1/9OjRzJo1i2HDckT+WJYuXcrq1asLLH9QUBD169fHz8+PPn36MGTIEAYOHAhAt27d2L59O127dqVSpUqEhYXx0ksv8eyzz7J9+/YclwwdvvjiC2uijCPfqUuXLuzYsYN7772X5s2bF/r9FBEvoKWqHi50x7zvawT8G2gGTFXVM7ler4Wth1VwIqELynrMqL6qJgPY/61X0I4i8nsR2SMie1JSUsqsgEbFlJiYSJ06dfD19c0zuD9z5kzANrjfsaNtTcn8BvcPHDjAwYMH2bhxIzVr1iQsLIy1BczdLuy1knC7jWWqar5R4PbLRJw4cYIjR46QlJTETz/9xIYNG9iyZQtQeDCfl5cXsbGxJCUlWZNRHPr27cunn35KampqnvPu3LmTatWq0a5duwLL7OXlRUxMDMuXL6dFixZMmTLF+lnr3r0727dvZ9euXXTq1Al/f39OnDhBSkoKV69epWnTptZxRo0aRXBwMNu2bWPOnDkAjBw5kuXLl5OdnV2csMEuwM4i98pFVU+raiC2xmiMiNR3vCYilYEo4F1VjS/usZ2V29l0qvoh8CHYVu12c3EMo9wpryuKl4a2bduyb98+srOzrbTT7Oxs4uLiaN++PUuXLiU0NNS6JBYREcGOHTvo1auXS8F8tWrVIiwsjJiYGKuBmTZtGkuWLGHYsGGsXr06R5Cdqw2AiNC5c2c6d+5Mv379GDt2LDNnziQ0NJTdu3fz7bff0rVrVwB8fHyIjo7Oc4nus88+s/5IcmjUqBG+vr5s3ryZFStW8N1337nybYwACl3hVkS6AI4ZHzNUdY3jNVU9IyKHgJ7AcvvmD4Hjqvo3VwpQmLLuGf1XRBqAbVYHtgkOhmEYhWrWrBkhISHMmjXL2jZr1iz69OlD48aNady4MZs3byYzM5OMjAw2b95sXaYrKJgvJSWFX375BYDr16+zfv36PKF68+bNo0aNGowfP97qhWVnZ/P555/nyRjK7cyZM3z//ffW89jYWJo0aQLA3XffTaNGjVi0aJHVGHXt2pW//e1vLkeJR0ZGMmXKFPz9/fHxcWm4pg+2sfoCqepOtYXxBavqGhHxEZGqACJyD9AdOGZ/PguoCTzrUoGLUNaN0RpgjP3xGKDgC66GYRhOFixYwPHjx2nWrBl169Zlx44dvP/++wAMHToUf39/AgICCAoKIigoyBqfgfyD+ZKTk+nduzeBgYF06tSJfv36WamtDiLCJ598QnJyMtOmTQNgy5Yt+Pj45LiUlp+MjAyef/55WrVqRXBwMEuXLuWdd34dVunevTs3btygUaNGgK0xio+Pd7kxGjZsGIcOHcrTKE6bNg0fHx/H5cVAEZkpInWBNFW97FS3rcDnQB8RSRKRB/M5TWtgp4jsBzYDc1T1gIj4ANOBNsD39qnft3Snd6mF64lIFLbJCnWA/wKvAKuAZUBj4BQwTFV/LupY7gzXMzxPeQ/GM27dsWPHGDBgAPPnz2fAgAHuLk655QjXE5HHAB9VLbcZGKU5m66gC6p9SuuchmHcHlq2bMmPP/7o7mJ4DFUt9/cYlNsJDIZhGJ6iS5cu3LhxI8e2Tz/91JqKbRTNNEaGYRi3yPkGVOPmmLXpDMMwDLczjZFhGIbhdqYxMgzDMNzONEaGYRiG25nGyDAMw3A70xgZhmEYbmcaI8MwPE5Rqa+nTp0iPDyc1q1b06ZNGxITExk8eDDBwcE0a9aMmjVrWmmt4eHhhSan+vr6WssMhYeHc/bsWQD69+9vpcQ+9dRTZGVlFVheVWXWrFk0b96cFi1a0Lt3bw4dOmS97jhHYGAg999/PydPnrRe8/LyypEum5iYSGpqKqNGjSIgIIB27drRo0cPrl69ar1n5cqVjlXOq+Quiz3htaGIPC0iJ0RERaROQWUXkY9FZL+IxInIchGpbt/+nIgctm//RkSaFPgNcMWtJPOV1ZdJejWK43ZMQb3dFJb6qmpLSP36669V1Zaweu3aNeu13Am9RSWnOlKCVVVfeuklnTRpkqqqXrp0SVVVs7OzdciQIRoVFVVgeefPn68RERFWOdatW6dNmzbV69ev5znHjBkzdMKECdZ7vb298xzv9ddf1ylTpuSoQ1pamvV82LBh2qNHDwXOaM7k1qrALvvjEMAXW3prHS3g9y9Qw+nxXOBF++PeQDX74z8ASws6hitfpmdkGIZHKSr19fDhw2RmZtKvXz/AlrBarVq1Ao9XVHKqM0faKkCNGjUAyMzMJD09Pd+8JYe33nqL+fPnW+UIDw+nW7dufPbZZ3n27dq1Kz/99FNh3wKSk5Np2LBhjjrcddddAFy9epVt27bx8ccfA9yT661h2EJNUdV9qppY6Ils+10GEFsFqwJq375RVR1hTzsor0mvhuFOFT0F9XZTnNRXsOUTDRkyhISEBPr27cubb76ZY//cXE1OXbt2bY4lfh588EF27dpFREQEQ4cOzffYly9f5tq1a/j7++fY3rFjxxyX6hxiYmJ4+OGHrefXr18n2J6k6Ofnx8qVKxk3bhzh4eEsX76cPn36MGbMGKu8q1aton///rRo0QIgS0Taq6ojyyIC24LVxSIiC4EBwGHgT/nsMh57nPnNMj0jo8K53VJQbzdaROprZmYmW7duZc6cOezevZv4+HgWLVpU6DGLSk7t3bs3wcHBXL58Ocf40rp160hOTubGjRts2LDhlurRu3dv6tWrx/r163n00Uet7VWrViU2NpbY2FhWrlwJQHBwMPHx8UydOpWff/6ZTp06WQ1xVFSUc6/uZ8C5Mt2Bb4tVUFtZxwL3AUeAEc6v2VcE7wjMLu5xnZmekVEhmfiIiquo1NczZ84QEhJi5Q09/PDD7Nixg/Hjxxd4zKKSUzdu3EidOvmP8VepUoVBgwaxevVq69Kgsxo1auDt7U18fHyODKTvv/+e+++/P8c5vL29efzxx5kxYwZz584t9PtQvXp1hgwZwpAhQ6hUqRJffvkl9erVY8OGDRw8eNDR0P0GGCEi0wA/4LSqphd2XBFZB9QH9qiqlVGkqlkishSYCiy079sXW67R/ap6I7/jucr0jAzD8ChFpb526tSJixcvkpKSAsCGDRto06ZNkcctTnLq1atXSU5OBmw9sS+//DJPSqyzqVOnMnnyZK5fvw7A+vXr+fbbb3P0gMDWC/rb3/7G4sWL+fnngqPetm3bZs0eTE9P5/DhwzRp0oTly5czevRoTp48SWJiIkAckAD0wIXYcQBVfVBtSa8TxKYZWGNGA4Gj9uch2CLKB6nqLad2m8bIMAyPU1jqq5eXF3PmzKFPnz4EBASgqjzxxBNFHrOg5NT8XLt2jUGDBhEYGEhQUBD16tXjqaeeKnD/SZMm0alTJwICAmjZsiV//vOfWb16NVWrVs2zb4MGDYiMjOS9994r8Hg//vgj999/PwEBAYSEhNCxY0ceeeQRoqKiGDx4cO7dVwCPAv1xaoxEZLKIJGGbeBAnIh/lcyoBPhGRA8ABoAHwmv212UB14HN70uuaAgvsglJLei1JJunVMIyCmNTXgjklvd4FbFPVju4uU0HMmJFhGB7NpL4WzT6eU24bIjCNkWEYRomZOHEi27Zty7HtmWeeYezYsW4qkecwjZFhGEYJKWycxyicmcBgGIZhuJ1pjAzDMAy3M42RYRiG4XamMTIMwzDczjRGhmEYhtuVWmMkIgtE5JyIHHTaNltEjtrDmFaKSK3SOr9hGK57/PHH+eCDD3JsW7VqlXUTaX4Bb7t27bKeBwUFWYt4AsybN4+2bdvSrl07IiMjSUtLs87j5+dnvS82NrbIsv3ud7+ja9euObYdO3aMsLAwgoODad26Nb///e8B2LRpU47gvL59+wIwc+ZMGjZsSHBwMK1ateIPf/gD2dnZAIwYMcLa39fX11oh26FDhw6kp6czffp0GjVqRPXq1fMt5/LlyxERHDfoJyYmUrVqVevYBa3QcOHCBXr37k316tV5+umnre1XrlzJ8T2vU6cOzz77LABTpkyxtrdo0YJatSrAr9JbCUMq7AvoBbQHDjptCwcq2x+/BbzlyrFMuJ5hlK6YmBgNCwvLsW3EiBG6ePFiVc0/4O3atWuakZGhqqpnzpzRunXrakZGhiYlJamvr6+mpqaqqi3obeHChaqqOmbMGP38889dLtfFixfVx8dHW7VqpfHx8db28PBwXbVqlfU8Li5OVfMG5zm88sorOnv2bFVVzcrK0u7du+uGDRvy7Pfcc8/pq6++aj1PSEjQgQMHqqrqd999p2fOnMn3e3H58mXt2bOndunSRXfv3m29t23btkXW8erVq7p161b9xz/+oRMnKVNdEgAADKdJREFUTixwv/bt2+vmzZvzbH/33Xd17Nix+b4H22Knbg9IdeWr1HpGqroF2/Llztu+VtVM+9NbDmMyDKNk9O3bl6NHj1qLf6amprJ+/focuTq5VatWjcqVbbcqpqWl5YhDyMzM5Pr162RmZpKamsp99913U+VasWIFAwcOZOTIkURHR1vbk5OTcyxm6pwxVJT09HTS0tK4556cuXOqyrJly3LER3z11Vf0798fgNDQUBo0aJDvMf/3f/+XadOmUaVKnpTvInl7e9OjR49C33v8+HHOnTtHz54987wWFRWVJ/LCE7nzptdxwNLSOrie/clao84wjLyc12708vJiyJAhLFu2jGeeeYY1a9bQu3dv7r77biD/gDeAnTt3Mm7cOE6ePMmnn35K5cqVadiwIc8//zyNGzematWqhIeHEx4ebp1r+vTpvPbaa/Tp04c333zTSijNT1RUFK+88gr169dn6NChVpbQlClTeOCBB+jWrRvh4eGMHTvWulS1detWq6zDhg1j+vTpgO3S4ZIlSzh58iQRERF5Lsdt3bqV+vXr5wjVi4mJYd68eYV+H/ft28fp06d56KGHmDNnTo7XEhISCAkJoUaNGsyaNSvfxsQVUVFRjBgxIk+O08mTJ0lISOCBBx64qeOWJ26ZwCAi04FMIG/m7q/7/F5E9ojIHsdS8K6q/Pgfkd80LHpHwzAsjrRTIE/AXH4BbwBdunTh0KFD7N69mzfeeIO0tDQuXrzI6tWrSUhI4MyZM1y7do0lS5YA8MYbb3D06FF2797Nzz//zFtvvVVgef773/9y4sQJevToQYsWLahcuTIHD9qGoMeOHcuRI0cYNmwYmzZtIjQ0lBs3bHE6PXv2tMrqaIjA1oDFxsZy7tw5rl27lqOnBXl7GOnp6SQlJeXIIMotOzubKVOm8Ne//jXPaw0aNODUqVPs27ePuXPn8uijj3L58uUCj1WY/AL/HNuHDh1aaIqtpyjznpGIjAEeAvrYr2nmS1U/BD4E26rdxT2PWbHbMIqne/fuJCcns3//frZv357nl3VhWrdujbe3NwcPHiQhIQE/Pz/q1q0LwJAhQ9i+fTuPPfaYdZnrrrvuYuzYsXl6Es6WLl3KxYsX8fPzA2zx3dHR0VaO0X333ce4ceMYN24c7dq1sxqqotxxxx3079+fLVu2WHERmZmZ/Otf/2Lv3r3Wflu3bqVHjx6FHuvKlSscPHiQMHvG/dmzZxk0aBBr1qyhY8eOVq+vQ4cO+Pv788MPP3D69GleffVVAD766CM6dix8/dL9+/eTmZlJhw4d8rwWHR1dYZYgKtOekYj0B17AFsaUWpbnNgyjcCLC8OHDGTNmDAMGDChy/CMhIYHMTNsQ8MmTJzl27Bi+vr40btyYHTt2kJqaiqryzTff0Lp1awBrTEpVWbVqFe3atSvw+FFRUcTExJCYmEhiYiJ79+61GsiYmBgyMjIAWwNw4cIFGjZ07WqIqrJ9+3b8/f2tbevXr6dVq1Y5xqFiYmKIiIgo9Fg1a9bk/PnzVhlDQ0OthiglJYWsrCwA4uPjOX78OE2bNmXw4MFWz62ohsjxfcivV3Ts2DEuXryYZ6ahpyrNqd1RwHdASxFJEpHxwN+Bu4H/2MOY3i+t8xuGUXyRkZHs37/fpYC5b7/9lqCgIIKDgxk8eDD/93//R506dejSpQtDhw6lffv2BAQEkJ2dbU29HjVqFAEBAQQEBHD+/HlefvnlfI+dmJjIqVOnCA0Ntbb5+flRo0YNdu7cyddff027du0ICgriwQcfZPbs2fzmN78ptLzz5s0jODiYdu3akZmZyR//+OvVk/wug23atClHLPi0adPw8fEhNTUVHx8fZs6cWej5tmzZYoXvDR06lPfff5/atWvnu6+vry/PPfccixYtwsfHh8OHD1uv5Z5U4RAVFcXIkSPzjCN5qgoZrmcYhnErkpKSeOKJJ/jqq6/cXZRb4gjXc3c5XGFWYDAMw8jFx8fH4xsiT2PyjAzDcKuFCxfyzjvv5NjWvXv3CjMwb7jGXKYzDMOooMxlOsMwDMMoBtMYGYZhGG5nGiPDMAzD7UxjZBiGYbidaYwMwzAMtzONkWEYhuF2HjG1W0RSgJPFfFsd4HwpFKc8ul3qervUE0xdK6qyrmsTVa1bhue7aR7RGN0MEdnjKfPrb9XtUtfbpZ5g6lpR3U51LS5zmc4wDMNwO9MYGYZhGG5XkRujD91dgDJ0u9T1dqknmLpWVLdTXYulwo4ZGYZhGJ6jIveMDMMwDA/h8Y2RiPQXkWMickJEXszn9btEZKn99Z0i4lv2pbx1LtSzl4h8LyKZIjLUHWUsKS7U9TkROSwicSLyjYg0cUc5S4ILdX1KRA7Yk5G/FZE27ihnSSiqrk77DRURFRGPnXXmwuf6uIik2D/XWBGZ4I5yliuq6rFfgBfwI9AUuBPYD7TJtc8fgfftj0cCS91d7lKqpy8QCCwGhrq7zKVc195ANfvjP3jiZ1qMutZwejwIiHF3uUurrvb97ga2ADuAju4udyl+ro8Df3d3WcvTl6f3jDoDJ1Q1XlXTgWjgd7n2+R3wif3xcqCPeF5ofJH1VNVEVY0Dst1RwBLkSl03qmqq/ekOwKeMy1hSXKnrZaen3oCnDvK68n8V4M/A20BaWRauhLlaV8OJpzdGDYHTTs+T7Nvy3UdVM4FLwL1lUrqS40o9K4ri1nU84Kn50C7VVUQmisiP2H5JTy6jspW0IusqIiFAI1VdW5YFKwWu/gw/Yr/UvFxEGpVN0covT2+M8uvh5P7L0ZV9yruKUAdXuVxXEXkM6AjMLtUSlR6X6qqq76mqP/AC8HKpl6p0FFpXEakEzAP+VGYlKj2ufK5fAL6qGgis59erN7ctT2+MkgDnvyh8gDMF7SMilYGawM9lUrqS40o9KwqX6ioifYHpwCBVvVFGZStpxf1co4GHS7VEpaeout4NtAM2iUgiEAqs8dBJDEV+rqp6wenn9p9AhzIqW7nl6Y3RbqC5iPiJyJ3YJiisybXPGmCM/fFQYIPaRxA9iCv1rCiKrKv9cs4H2Bqic24oY0lxpa7NnZ7+FjhehuUrSYXWVVUvqWodVfVVVV9sY4GDVHWPe4p7S1z5XBs4PR0EHCnD8pVLld1dgFuhqpki8jSwDtsMlgWqekhEXgP2qOoa4GPgUxE5ga1HNNJ9Jb45rtRTRDoBK4F7gIEi8qqqtnVjsW+Ki5/pbKA68Ll9LsopVR3ktkLfJBfr+rS9F5gBXOTXP6w8iot1rRBcrOtkERkEZGL7vfS42wpcTpgVGAzDMAy38/TLdIZhGEYFYBojwzAMw+1MY2QYhmG4nWmMDMMwDLczjZFhGIbhdqYxMjyOiPiKyMEyPuemom7ALK1yiUiYiHRzer7I01dmN4zcTGNkVFgi4uXuMpSQMKBbUTsZhiczjZHhqSqLyCdOC01WAxCRRBGZISLfAsNE5AkR2S0i+0VkhdN+i0TkXRHZLiLxzj0NEZlmzxDaLyJvOp1zmIjsEpEfRKRnYYUTES8RmW0/d5yIPGnfHmbvZS0XkaMi8pljFXkRGWDf9q29bGvFlr/1FDDFnnvjOG+v/MpuGJ7KNEaGp2oJfGhfaPIyttwqhzRV7aGq0cC/VLWTqgZhW3JlvNN+DYAewEPAmwAiEoFt/bcu9ve87bR/ZVXtDDwLvFJE+cYDl1S1E9AJeEJE/OyvhdiP0QZb5k13EamCbYmjCFXtAdQFWzQI8D4wT1WDVXVrQWU3DE9mGiPDU51W1W32x0uw/WJ2WOr0uJ2IbBWRA8AowHmJpFWqmq2qh4H69m19gYWOvCRVdV5U91/2f/diCzMsTDgwWkRigZ3YYksc68ztUtUkVc0GYu3HagXEq2qCfZ+oIo6fX9kNw2N59Np0xm0t9zpWzs+vOT1eBDysqvtF5HFs4y8Ozqt9i9O/Ba2R5dg/i6L/7wgwSVXX5dgoEpbrvI5jFTfwMb+yG4bHMj0jw1M1FpGu9seRwLcF7Hc3kCwid2DrGRXla2Cc09hS7Zss3zrgD/bzIiItRMS7kP2PAk3tY0QAI5xeu4KtHoZRYZnGyPBUR4AxIhIH1Ab+UcB+/4vtMtl/sP3CL5SqxmBb7n+P/RLb8zdZvo+Aw8D39uneH1BIb0pVr2Mb94qxT774L7ZUYrAFsQ3ONYHBMCoUs2q3YZQTIlJdVa/aZ9e9BxxX1XnuLpdhlAXTMzKM8uMJe2/sELZE4g/cXB7DKDOmZ2QYhmG4nekZGYZhGG5nGiPDMAzD7UxjZBiGYbidaYwMwzAMtzONkWEYhuF2pjEyDMMw3O7/A4KDYDDMuJSFAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<Figure size 432x288 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "Phylo.draw(tree)"
   ]
  }
 ],
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
